<main class="html-publication" lang="en" role="main">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          
<div class="gem-c-organisation-logo brand--department-of-health">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color" href="/government/organisations/medicines-and-healthcare-products-regulatory-agency">
      <span class="gem-c-organisation-logo__name">Medicines &amp; Healthcare products<br>Regulatory Agency</span>
</a>
</div>
        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  

<div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
      <span class="govuk-caption-xl gem-c-title__context">
    Guidance
  </span>


  <h1 class="gem-c-title__text govuk-heading-xl">
    Guidance for manufacturers: diagnostic assurance with SARS-CoV-2 variants in circulation
  </h1>
</div>
  <p class="publication-header__last-changed">Updated 14 February 2022</p>

  </header>



<div class="govuk-grid-row sidebar-with-body" data-module="sticky-element-container">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav aria-label="Contents" class="gem-c-contents-list" data-module="gem-track-click" role="navigation">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 1" data-track-category="contentsClicked" data-track-label="#expectations-for-manufacturers" data-track-options="{&quot;dimension29&quot;:&quot;Expectations for manufacturers&quot;}" href="#expectations-for-manufacturers">Expectations for manufacturers</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 2" data-track-category="contentsClicked" data-track-label="#expectations-for-manufacturers-of-molecular-assays" data-track-options="{&quot;dimension29&quot;:&quot;Expectations for manufacturers of molecular assays&quot;}" href="#expectations-for-manufacturers-of-molecular-assays">Expectations for manufacturers of molecular assays</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 3" data-track-category="contentsClicked" data-track-label="#expectations-for-manufacturers-of-antigen-assays-including-antigen-lateral-flow-tests" data-track-options="{&quot;dimension29&quot;:&quot;Expectations for manufacturers of antigen assays (including antigen lateral flow tests)&quot;}" href="#expectations-for-manufacturers-of-antigen-assays-including-antigen-lateral-flow-tests">Expectations for manufacturers of antigen assays (including antigen lateral flow tests)</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 4" data-track-category="contentsClicked" data-track-label="#ivd-in-silico-recommendations" data-track-options="{&quot;dimension29&quot;:&quot;IVD in silico Recommendations&quot;}" href="#ivd-in-silico-recommendations">IVD in silico Recommendations</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 5" data-track-category="contentsClicked" data-track-label="#introduction" data-track-options="{&quot;dimension29&quot;:&quot;\n1.  Introduction&quot;}" href="#introduction"><span class="gem-c-contents-list__number">1. </span><span class="gem-c-contents-list__numbered-text">Introduction</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 6" data-track-category="contentsClicked" data-track-label="#frequency-recommendations-for-in-silico-analysis" data-track-options="{&quot;dimension29&quot;:&quot;\n2.  Frequency recommendations for in silico analysis&quot;}" href="#frequency-recommendations-for-in-silico-analysis"><span class="gem-c-contents-list__number">2. </span><span class="gem-c-contents-list__numbered-text">Frequency recommendations for in silico analysis</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 7" data-track-category="contentsClicked" data-track-label="#database-considerations" data-track-options="{&quot;dimension29&quot;:&quot;\n3.  Database considerations&quot;}" href="#database-considerations"><span class="gem-c-contents-list__number">3. </span><span class="gem-c-contents-list__numbered-text">Database considerations</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 8" data-track-category="contentsClicked" data-track-label="#bioinformatics-recommendations" data-track-options="{&quot;dimension29&quot;:&quot;\n4.  Bioinformatics recommendations&quot;}" href="#bioinformatics-recommendations"><span class="gem-c-contents-list__number">4. </span><span class="gem-c-contents-list__numbered-text">Bioinformatics recommendations</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 9" data-track-category="contentsClicked" data-track-label="#assessment-of-impact-on-molecular-assay" data-track-options="{&quot;dimension29&quot;:&quot;\n5.  Assessment of impact on molecular assay&quot;}" href="#assessment-of-impact-on-molecular-assay"><span class="gem-c-contents-list__number">5. </span><span class="gem-c-contents-list__numbered-text">Assessment of impact on molecular assay</span></a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img class="print-meta-data-licence" src="/assets/government-frontend/open-government-licence-min-e98134ae65ff5fe99a524d8c383350b3998f91c298dc24b36126e65f1656c578.png">
</p>
<p>
  Â© Crown copyright 2022
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/how-tests-and-testing-kits-for-coronavirus-covid-19-work/guidance-for-manufacturers-diagnostic-assurance-with-sars-cov-2-variants-in-circulation
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
    
    
      <div class="govspeak">
<p>Since December 2020, the MHRA have been made aware of increasing concerns relating to the performance of diagnostic tests for SARS-CoV-2 virus variants, with Public Health England (PHE) producing the <a class="govuk-link" href="https://www.gov.uk/government/collections/new-sars-cov-2-variant">first report</a> of new SARS-CoV-2 variants of concern (VOCs) on 21.12.2020.</p>

<p>Diagnostic assays for detection of SARS-CoV-2 are crucial in the management of the pandemic. However, where diagnostic assays directly target the genetic sequence, different strains of the virus might impact assay performance. The nature and severity of any impact will depend on the assay but can, in a worst-case scenario, lead to a false negative result.</p>

<p>This issue can be mitigated by in silico sequence monitoring. The MHRA has been monitoring the issue and collaborating with external partners.</p>

<h3>Variants of Concern (VOC) assurance working group</h3>

<p>A VOC assurance working group has been established to provide an assurance framework for the performance of SARS-COV-2 In-vitro diagnostic (IVD) assays. The VOC assurance working group includes MHRA as well as specialist members from the UKHSA, NHS England, NHS Improvement, and partners from all 4 devolved administrations.</p>

<p>The VOC assurance working group monitor the performance of assays through:</p>

<ul>
  <li>in silico monitoring (based on computer modelling)</li>
  <li>laboratory test performance quality monitoring</li>
  <li>in vitro testing (where indicated by the in silico and quality monitoring processes)</li>
</ul>

<p>The MHRA regularly engages with suppliers and manufacturers of test products in the UK to review their post-market assurance processes for the most recently published variants in circulation in the UK.</p>

<h3>Omicron variant update</h3>

<p>The MHRA have reminded manufacturers of their obligations with respect to monitoring and validation of their assays when new variants are identified. This is a previous commitment and requirement that MHRA has requested from manufacturers with COVID-19 IVDs in the UK market. With the concerns associated with the Omicron (B.1.1.529) variant, manufacturers/suppliers have been asked to notify MHRA immediately of their evaluation of the performance of their devices. As needed, the MHRA will be requesting additional information/action from specific manufacturers if their devices are suspected or at risk of performance issues based on available information.</p>

<p>This is the course of action taken for any emerging variants of concern.</p>

<h2>Expectations for manufacturers</h2>

<p>Due to the diversity of genomic sequences of SARS-CoV-2, a proactive approach is required in both diagnostic assay design and surveillance to monitor pathogen evolution in real time. This will allow assays that may be affected by variants to be identified and redesigned to avoid the potential effect of false negative results on the management of SARS-CoV-2 infected individuals.</p>

<p>Manufacturers are responsible for the safety and performance of their assays whilst they are available on the UK market.</p>

<p>To help fulfil this responsibility, MHRA have reminded manufacturers to have a Post Market Surveillance plan (PMSP) in place to continuously monitor, investigate and assess the influence of newly emerging variants of SARS-CoV-2 on assay performance.</p>

<p>In line with the <a class="govuk-link" href="https://www.legislation.gov.uk/uksi/2002/618/contents/made" rel="external">UK Medical Devices Regulations 2002</a>, the MHRA considers reports relating to variants to be serious public health threats, therefore significant safety issues (for example, decrease in performance) should be reported within 48h.</p>

<p>The use of available SARS-CoV-2 sequencing data to monitor assay performance will provide the clinical community and those tasked with managing the COVID-19 pandemic with the foresight to avoid diagnostic assay failure in the face of viral genome evolution.</p>

<h2>Expectations for manufacturers of molecular assays:</h2>

<ol>
  <li>Manufacturers should inform the MHRA of the outcome of the initial risk assessment of any impact on each primer/probe/other oligonucleotide target sequence and on the performance of the assay in light of the newly identified variants, as identified in the Technical Briefing Documents found under <a class="govuk-link" href="https://www.gov.uk/government/publications/investigation-of-novel-sars-cov-2-variant-variant-of-concern-20201201#history">Investigation of novel SARS-CoV-2 Variants of Concern</a> on the Public Health England website.</li>
  <li>A monthly update report should be submitted to the MHRA. Information to include on the report:</li>
</ol>

<p>a. Information on whether the sensitivity of any gene targets in the diagnostic assay is affected by any of the variant</p>

<p>b. Information of either: all sequence identifiers (for example, COG-UK ID or accession) of sequences used in the analysis, or sufficient metadata (for example, lineage and epi week/date range) and sequence selection criteria to ensure that the data used can be independently identified if required</p>

<ol>
  <li>If the performance of an assay is directly and negatively impacted by new virus variant(s), a Field Safety Notice (FSN) should be issued immediately to alert customers. Further guidance on effective FSNs can be found <a class="govuk-link" href="https://www.gov.uk/guidance/effective-field-safety-notices-fsns-guidance-for-manufacturers-of-medical-devices">here</a>.</li>
</ol>

<p>Ongoing essential post market surveillance actions, relevant to all GISAID published SARS-CoV-2 variants:</p>

<ol>
  <li>A Post Market Surveillance plan (PMSP) should be in place to continuously monitor, investigate and assess the influence of newly emerging variants of SARS-CoV-2 on assay performance. The PMSP can include:</li>
</ol>

<p>a. Fortnightly in silico checks of assay targets against GISAID sequence databases <a class="govuk-link" href="https://www.gisaid.org" rel="external">https://www.gisaid.org</a> [*Please note that high profile potential issues should be immediately investigated]</p>

<p>b. Scientific literature and post market intelligence gathering</p>

<p>c. Outcomes of EQA schemes when available</p>

<p>d. Use of reference materials when available</p>

<p>e. Reporting potential safety issues of any new clinically significant variant SARS-CoV-2 strain on the performance of your assay to MHRA</p>

<h2>Expectations for manufacturers of antigen assays (including antigen lateral flow tests):</h2>

<ol>
  <li>Manufacturers should monitor the emergence of variants using the available information on the GISAID website (https://www.gisaid.org/), reviewing variants where there are &gt;100 sequences for the variant lineage with change(s) in the target antigen(s) of their assays. This should be conducted on a fortnightly basis unless they are notified by the MHRA that a new variant of concern has arisen, which requires immediate attention.</li>
  <li>As part of routine monitoring, if variant strains arise revealing amino acid changes in target antigen(s) that may compromise the assay performance, this should be communicated to the MHRA within 2 days as a priority.</li>
  <li>A monthly update report should be submitted to the MHRA. Evidence should be based on in silico modelling of target antigen structure with amino acid changes compared to antibody binding sites and backed up with wet lab testing against one or more of the following:</li>
</ol>

<p>a. recombinant protein containing amino acid changes observed in variant virus strains</p>

<p>b. against characterised clinical samples</p>

<p>c. characterised viral variant isolates</p>

<p>It is required that both favourable and unfavourable data should be reported to the MHRA as assurance of either positive or negative performance.</p>

<p>If the performance of your assay is directly impacted by these new virus variant(s), a Field Safety Notice (FSNs) should be issued immediately to alert customers. Further guidance on effective FSNs can be found <a class="govuk-link" href="https://www.gov.uk/guidance/effective-field-safety-notices-fsns-guidance-for-manufacturers-of-medical-devices">here</a>.</p>

<p>In line with this, we have developed specific guidance for manufacturers of molecular and/or antigen assays (including lateral flow tests):</p>

<h2>IVD in silico Recommendations</h2>

<h2>
<span class="number">1. </span> Introduction</h2>

<p>All in vitro diagnostic (IVD) solutions that detect pathogens directly (such as antigen or molecular tests targeting proteins or RNA respectively) shall monitor the impact of significant variants (defined as an organism that has one or more mutations that differentiate it from the reference sequence) during routine test evaluation and ensure that the performance specified by the manufacturer during initial validation is maintained.</p>

<p>Ideally this should be done empirically with real clinical samples, however this may not always be possible due to availability of samples containing the variants of concern (VOC). Alternatively, cultured organism or synthetic molecules may also be utilised to explore potential impact of VOC on performance.  IVD assay manufacturers shall take into consideration all relevant VOCs at the time of test development and validation and the potential impact of future genetic variants.</p>

<p>As molecular diagnostic assays directly target the genetic sequence any changes have the potential to disrupt their performance. This problem can be mitigated by in silico sequence assessment. However, for this, two pieces of information are required:</p>

<ul>
  <li>The sequence of the genetic variants in question.</li>
  <li>The sequence of the genetic region targeted by the molecular assays being used (typically small DNA sequences known as primers or probes).</li>
</ul>

<p>While antigen-based assays do not directly target the genetic sequence, non-synonymous mutations within the associated genomic region can have adverse and unpredictable effects on assay performance and therefore active and ongoing in silico monitoring is recommended.</p>

<p>These recommendations are intended to guide manufacturers of authorised IVD solutions provided within the United Kingdom (UK), however they are also provided to increase awareness of in silico monitoring of for developers of IVD solutions and may be subject to re-evaluation and revisions as appropriate.</p>

<h2>
<span class="number">2. </span> Frequency recommendations for in silico analysis</h2>
<p>The frequency with which in silico analysis shall be performed is of great importance but will depend on the frequency that mutations occur in the genomic region targeted by the IVD assay. Initial test validation must take into account all sequence variation in the target region and post marketing in silico surveillance be tailored to the specific target.</p>

<p>Ongoing scientific literature review shall be performed at a frequency appropriate to the pathogen in question in order to identify any new sequence variants or periods of rapid expansion of pathogens that would likely impact the rate of mutations of the assay target. The MHRA may be able to offer guidance on specific frequency requirements, but broad categories are offered as guidance below.</p>

<ul>
  <li>Rapidly Evolving Targets
For targets with rapid rates of mutations or expansion within populations, such as pandemic SARS-CoV-2, monthly in silico analysis shall be performed. Increased frequency or individual analyses may be required upon the release of new information relating to specific VOCs, variants under investigation (VUI) or strains as directed by Public Health England (for example, upon receipt of information from âSARS-CoV-2 variants of concern and variants under investigation in Englandâ Technical briefings). Ideally the use of automated analytical processes and database monitoring can ensure that the process is run in near real-time as novel sequences become available.</li>
  <li>Targets with Moderate Evolution
For moderately stable target pathogen genetic regions (such as established bacterial and viral pathogens where mutation rates are known) the frequency of analysis may be reduced to biannually or annually.</li>
  <li>Highly Conserved Targets
Highly conserved or stable target pathogen genetic regions may require infrequent monitoring (&gt;1-year frequency) or no regular monitoring with analysis only performed upon receipt of new information provided by literature or other external sources.</li>
</ul>

<h2>
<span class="number">3. </span> Database considerations</h2>
<p>Considerations for the nucleotide database against which query sequences are compared include the quality of sequence, diversity and whether the data is representative of the UK during the time-period the assay will be in use. Sequence quality is of particular importance and care should be taken in assigning high quality sequences where available to the database, so as to reduce the number of aligned sequences containing unknown or ambiguous bases.</p>

<p>Ideally this database should contain all high-quality nucleotide records available for the target sequence, however this may not be possible in all situations. In these cases, particular care must be taken to ensure that the data is representative of the overall diversity.</p>

<p>The source of sequences used in the creation of a nucleotide database and either sequence identifiers of all sequences used (such as Accession codes or COG-UK IDs) or sufficient metadata (for example, lineage and epi week/date range) and sequence selection criteria to ensure that the data used can be independently identified if required, shall be recorded and made available to MHRA at request.</p>

<h2>
<span class="number">4. </span> Bioinformatics recommendations</h2>
<p>The analysis of sequences shall be performed by a trained bioinformatician or person with experience in the handling of large databases and the tools used should be both peer-reviewed and version controlled.</p>

<p>The version information and references of all tools and software shall be recorded for each analysis to allow for audit, and the use of differing tools or versions between analyses is discouraged unless adequate validation and quality assurance has been performed. For this purpose it is recommended that bioinformatics pipelines or tools used are containerised using services such as <a class="govuk-link" href="https://www.docker.com/" rel="external">Docker</a> or <a class="govuk-link" href="https://sylabs.io/" rel="external">Singularity</a>.
Manufacturers or third parties involved in bioinformatics analysis should ensure that code written for the purpose of in silico analysis follows best coding practices, including that the code is readable and easily maintained.</p>

<p>For regular monitoring of target sequences, it is recommended that short sequences are extracted from complete records that correspond to regions associated with the target nucleotide sequence in order to reduce the chances of non-specific alignment and allow the region of interest to be more accurately quality controlled. Any sequence containing unknown or ambiguous bases within the target region should be removed from the database to prevent misidentification of mutations.</p>

<p>From these extracted sequences multiple sequence alignments (MSA) or local alignments should be performed to identify mutations (SNPs/SNVs, deletions or insertions) within target sequences, using peer-reviewed tools such as Clustal Omega offered by EMBL-EBI (available as a command line or online tool from: <a class="govuk-link" href="https://www.ebi.ac.uk/" rel="external">https://www.ebi.ac.uk/</a>) or nucleotide BLASTÂ® offered by NIH (available as a command line or online: <a class="govuk-link" href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" rel="external">https://blast.ncbi.nlm.nih.gov/Blast.cgi</a>).</p>

<p>In the case of Clustal Omega the assay sequence can be added to a multi-fasta database and a MSA performed, with the identification of mutations performed on the returned data.</p>

<p>However, if nucleotide BLASTÂ® is performed care should be taken in both running the analysis and interpreting the results, making sure that the assay sequence is inputted as the âQueryâ and with both identity and coverage taken into account in the results.</p>

<p>The frequency and location of all mutations within all assay targets shall be noted and percentage frequency based on target sequences analysed should be calculated for regular surveillance purposes (for example, a single SNP/SNV should be recorded as: C21026T, 4.5% frequency).</p>

<p>For molecular assays further in silico analysis can be performed including the calculation of melting temperature (Tm) with the assay primers or probes and the assay target sequence. It is recommended that all conditions of the assay are included in the calculation (such as ion concentration and primer excess) in order to best reflect the real-world conditions of the assay.</p>

<p>In silico analysis should be used as a method of prioritising further investigations based on the predicted impact of sequence variation on assay performance. It is strongly recommended that further in vitro or âwetâ testing is performed in order to validate these predictions.</p>

<h2>
<span class="number">5. </span> Assessment of impact on molecular assay</h2>
<p>For molecular assays the impact of any mutations identified through in silico analysis which result in mismatches within the target region that are likely to impact the performance of the IVD assay should be assessed through in vitro or âwetâ testing of synthetic constructs or an isolate containing the known mutation.</p>

<p>The impact on antigen based IVD assays is not as clear as for molecular tests and therefore identification of any non-synonymous mutations in the genetic sequence should be evaluated through wet testing.</p>

<h4>Useful links</h4>
<p>Urgent enquiries related to variants should be reported to <a class="govuk-link" href="mailto:covid19.variants@mhra.gov.uk">covid19.variants@mhra.gov.uk</a>.</p>

<p>Private providers of SARS-CoV-2 antigen and nucleic acid tests should contact the MHRA at <a class="govuk-link" href="mailto:device.registrations@mhra.gov.uk">device.registrations@mhra.gov.uk</a> to ensure products are registered and monitored.</p>

</div>


</div>
</div>
  </div>

  <div class="sticky-element" data-sticky-element="">
    <a class="govuk-link app-c-back-to-top dont-print" href="#contents">
    <svg class="app-c-back-to-top__icon" focusable="false" height="17" viewBox="0 0 13 17" width="13" xmlns="http://www.w3.org/2000/svg">
      <path d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z" fill="currentColor"></path>
    </svg>
    Contents
</a>

    <div class="sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>