<main role="main" class="html-publication" lang="en">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          <div class="gem-c-organisation-logo brand--department-of-health">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color" href="/government/organisations/medicines-and-healthcare-products-regulatory-agency">
      <span class="gem-c-organisation-logo__name">Medicines &amp; Healthcare products<br>Regulatory Agency</span>
</a>
</div>

        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  <div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
    <span class="govuk-caption-xl gem-c-title__context">
      Guidance
    </span>
  <h1 class="gem-c-title__text govuk-heading-xl">
    Target Product Profile: Laboratory-Based SARS-CoV-2 Viral Detection tests 
  </h1>
</div>
  <p class="publication-header__last-changed">Updated 25 May 2021</p>

  </header>




<div data-module="sticky-element-container" class="govuk-grid-row sidebar-with-body">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav aria-label="Contents" data-module="gem-track-click" role="navigation" class="gem-c-contents-list  gem-c-contents-list--no-underline ">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link " data-track-action="content_item 1" data-track-options="{&quot;dimension29&quot;:&quot;The purpose of a Target Product Profile (TPP)&quot;}" data-track-category="contentsClicked" data-track-label="#the-purpose-of-a-target-product-profile-tpp" href="#the-purpose-of-a-target-product-profile-tpp">The purpose of a Target Product Profile (TPP)</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link " data-track-options="{&quot;dimension29&quot;:&quot;TPPs for detection of SARS-CoV-2 in order to diagnose COVID-19&quot;}" data-track-category="contentsClicked" data-track-action="content_item 2" data-track-label="#tpps-for-detection-of-sars-cov-2-in-order-to-diagnose-covid-19" href="#tpps-for-detection-of-sars-cov-2-in-order-to-diagnose-covid-19">TPPs for detection of SARS-CoV-2 in order to diagnose COVID-19</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 3" data-track-category="contentsClicked" data-track-label="#clinical-performance-requirements" data-track-options="{&quot;dimension29&quot;:&quot;Clinical performance requirements&quot;}" class="gem-c-contents-list__link govuk-link " href="#clinical-performance-requirements">Clinical performance requirements</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-label="#future-developments" data-track-category="contentsClicked" data-track-action="content_item 4" class="gem-c-contents-list__link govuk-link " data-track-options="{&quot;dimension29&quot;:&quot;Future developments&quot;}" href="#future-developments">Future developments</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-options="{&quot;dimension29&quot;:&quot;Key to Table&quot;}" class="gem-c-contents-list__link govuk-link " data-track-action="content_item 5" data-track-category="contentsClicked" data-track-label="#key-to-table" href="#key-to-table">Key to Table</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 6" data-track-label="#target-product-profile-covid-19---laboratory-based-sars-cov-2-viral-detection-testing" data-track-options="{&quot;dimension29&quot;:&quot;Target Product Profile: COVID-19 - Laboratory-based SARS-CoV-2 Viral detection testing&quot;}" data-track-category="contentsClicked" class="gem-c-contents-list__link govuk-link " href="#target-product-profile-covid-19---laboratory-based-sars-cov-2-viral-detection-testing">Target Product Profile: COVID-19 - Laboratory-based SARS-CoV-2 Viral detection testing</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link " data-track-action="content_item 7" data-track-options="{&quot;dimension29&quot;:&quot;Operational characteristics&quot;}" data-track-category="contentsClicked" data-track-label="#operational-characteristics" href="#operational-characteristics">Operational characteristics</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 8" data-track-category="contentsClicked" href="#other" data-track-label="#other" data-track-options="{&quot;dimension29&quot;:&quot;Other&quot;}" class="gem-c-contents-list__link govuk-link ">Other</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-options="{&quot;dimension29&quot;:&quot;Annex 1: Assay validation&quot;}" class="gem-c-contents-list__link govuk-link " data-track-category="contentsClicked" data-track-label="#annex-1-assay-validation" href="#annex-1-assay-validation" data-track-action="content_item 9">Annex 1: Assay validation</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-category="contentsClicked" data-track-label="#annex-2-diagnostic-accuracy-considerations" data-track-options="{&quot;dimension29&quot;:&quot;Annex 2: Diagnostic accuracy considerations&quot;}" href="#annex-2-diagnostic-accuracy-considerations" data-track-action="content_item 10" class="gem-c-contents-list__link govuk-link ">Annex 2: Diagnostic accuracy considerations</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link " href="#annex-3-glossary" data-track-category="contentsClicked" data-track-options="{&quot;dimension29&quot;:&quot;Annex 3: Glossary&quot;}" data-track-action="content_item 11" data-track-label="#annex-3-glossary">Annex 3: Glossary</a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button data-module="print-link" class="govuk-link govuk-body-s gem-c-print-link__button">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img src="/assets/government-frontend/open-government-licence-min-e98134ae65ff5fe99a524d8c383350b3998f91c298dc24b36126e65f1656c578.png" class="print-meta-data-licence">
</p>
<p>
  © Crown copyright 2021
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/how-tests-and-testing-kits-for-coronavirus-covid-19-work/target-product-profile-laboratory-based-sars-cov-2-viral-detection-tests
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
    
    
      <div class="govspeak">
<p>Version control
Version
1.0</p>

<p>Date issued
October 2020</p>

<p>Description
Initial document</p>

<h2>The purpose of a Target Product Profile (<abbr title="target product profile">TPP</abbr>)</h2>

<p>Target product profiles (<abbr title="target product profile">TPP</abbr>) outline the ‘profile’ or characteristics of a target product that is aimed at a particular disease or diseases. TPPs state intended use, target populations and other attributes of products, including safety and performance-related characteristics. They help guide industry development towards desired characteristics.</p>

<p>A <abbr title="target product profile">TPP</abbr> provides a common foundation for the development of tests and contains sufficient detail to allow device developers and key stakeholders to understand the characteristics a test must have to be successful for the particular intended use.</p>

<p>Included is a description of (1) the preferred and (2) the minimally acceptable profiles based on the intended use, setting of use, and intended user, with respect to the performance and operational characteristics expected of the target products. As new scientific evidence is generated, this <abbr title="target product profile">TPP</abbr> may require further review and revision.</p>

<h2>TPPs for detection of SARS-CoV-2 in order to diagnose COVID-19</h2>

<p>These product profiles have been developed to assist manufacturers to design and deliver tests that might be useful in support of the UK COVID-19 testing strategy. How closely a product matches the profile may inform procurement and regulatory decision making. Any deviation from the target specification must be fully justified. Production lead time may also factor into decision making.</p>

<p>This <abbr title="target product profile">TPP</abbr> is for laboratory based SARS-CoV-2 Viral detection tests.</p>

<p>It should be noted that a different <abbr title="target product profile">TPP</abbr> could apply for different scenarios. As such, the contents of the <abbr title="target product profile">TPP</abbr> in this document are restricted. This <abbr title="target product profile">TPP</abbr> contains profiles based on our best information, but the science and use requirements are rapidly evolving. Manufacturers should ensure they are working to the most recent <abbr title="target product profile">TPP</abbr> and the most recent science.</p>

<h2>Clinical performance requirements</h2>
<p>This is a specification of the clinically acceptable performance requirements for SARS-CoV-2 viral detection tests. It sets out the clinical requirements based on the consensus of what is ‘minimally acceptable’ in the opinion of UK healthcare professionals. A test kit with specifications other than this may not be suited to support the UK testing strategy.</p>

<p>The intended use of the assays that match this profile (or one that does not yet meet the specifications but looks promising) is to aid in the management of individuals with a current SARS-CoV-2 infection by detection of SARS-CoV-2 nucleic acid in human samples. Assays that don’t yet meet the specifications but look promising may require further technical and clinical validation.</p>

<p>The acceptable criteria for clinical sensitivity and specificity are an initial estimate of minimally acceptable performance based on current expert opinion in a potentially limited number of use cases. Please see <a href="https://www.finddx.org/covid-19/dx-use-cases/" rel="external" class="govuk-link">examples of potential use cases of such devices from the Foundation for Innovative New Diagnostics (<abbr title="the Foundation for Innovative New Diagnostics">FIND</abbr>)</a> for more information.</p>

<p>To ensure ongoing public safety and value for money, procurement and deployment of tests should consider the specific clinical decision the test is being used to make and the prevalence of SARS-CoV-2 within the intended test population, as well as the potential consequences of false positives and false negatives. The acceptable clinical sensitivity and specificity may need to be higher for some uses of the tests. For example, in populations with a low prevalence of COVID-19 a very high specificity may be needed to increase assurance that a positive test result is a true positive. The impact of changing sensitivity, specificity and prevalence on the numbers of false positives and negatives can be seen in Annex 2 below.</p>

<p>These specification criteria are based on similar TPPs published by the World Health Organisation (WHO), Program for Appropriate Technology in Health (PATH), and <abbr title="the Foundation for Innovative New Diagnostics">FIND</abbr> for in-vitro diagnostics (<abbr title="in-vitro diagnostic">IVDs</abbr>) for other diseases.</p>

<h2>Future developments</h2>
<p>These profiles are subject to review and change as we gain a greater knowledge of SARS-CoV-2, the disease it causes (COVID-19) and our needs for an effective response. They may need to be updated at short notice.</p>

<p>As our knowledge and understanding of the disease improves and the UK clinical needs change, so may the <abbr title="target product profile">TPP</abbr>. A test that meets this version of the <abbr title="target product profile">TPP</abbr> may not meet future versions. Manufacturers should ensure that they are working to the most recent <abbr title="target product profile">TPP</abbr>.</p>

<p>It is expected that as more well characterised samples become available, then test performance can be established on a higher number and more diverse samples.</p>

<h2>Key to Table</h2>
<p>Acceptable: Defines the minimum acceptable feature.</p>

<p>Desired: Highly desirable features of considerable benefit. As time is of the essence, if omitting one of these features significantly accelerates development and production it can be considered.</p>

<h2>Target Product Profile: COVID-19 - Laboratory-based SARS-CoV-2 Viral detection testing</h2>

<h3>Scope</h3>

<h4>Intended use</h4>

<h5>Desired</h5>
<p>Multiplex -  Determining current infection by detecting SARS-CoV-2 virus in samples from people of all ages at any point during active infection and differentiating from other respiratory infections</p>

<h5>Acceptable</h5>
<p>Determining current SARS-CoV-2 infection by detection of SARS-CoV-2 virus in samples from people of all ages during the active phase of infection.</p>

<h5>Comment</h5>
<p>Active infection may be asymptomatic or symptomatic</p>

<h4>Target population</h4>

<h5>Desired</h5>
<p>People with or without clinical signs associated with SARS-CoV-2 infection.</p>

<h5>Acceptable</h5>
<p>People with clinical signs associated with SARS-CoV-2 infection</p>

<h4>Target user</h4>

<h5>Desired</h5>
<p>Trained healthcare / public health professionals (i.e. one of the 10 health and social care professional bodies that are overseen by the <a rel="external" href="https://www.professionalstandards.org.uk/" class="govuk-link">professional standards authority</a> ) and suitably trained and assessed as competent lab technician or scientist.</p>

<h5>Acceptable</h5>
<p>Trained healthcare / public health professionals (i.e. one of the 10 health and social care professional bodies that are overseen by the <a rel="external" href="https://www.professionalstandards.org.uk/" class="govuk-link">professional standards authority</a> ) and suitably trained and assessed as competent lab technician or scientist.</p>

<h5>Comment</h5>
<p>A target user will perform the assay, interpret and communicate the results.</p>

<p>Full training appropriate to the intended user is required.</p>

<h4>Target use Setting</h4>

<h5>Desired</h5>
<p>Healthcare and Medical Laboratories</p>

<h5>Acceptable</h5>
<p>Healthcare and Medical Laboratories</p>

<h5>Comment</h5>
<p>These exclude Point of Care (<abbr title="Point of Care">POC</abbr>) testing environments which will utilise test types that are appropriate for that setting.</p>

<h3>Test design characteristics</h3>

<h4>Test format</h4>

<h5>Desired</h5>
<p>A standardised kit that contains all materials required for the laboratory procedure that includes controls, reagents and Instruction for Use (<abbr title="Instruction for Use">IFU</abbr>).</p>

<p>All accessories needed to perform the assay and sample processing included, with the exception of routine laboratory consumables such as pipettes, pipette tips, disinfectants, screw cap eppendorfs, heat blocks, fluorescent readers etc.</p>

<h5>Acceptable</h5>
<p>A standardised kit that contains all materials required for the laboratory procedure that includes controls, reagents and Instruction for Use (<abbr title="Instruction for Use">IFU</abbr>).</p>

<p>All accessories needed to perform the assay and sample processing included, with the exception of routine laboratory consumables such as pipettes, pipette tips, disinfectants, screw cap eppendorfs, heat blocks, fluorescent readers etc.</p>

<h5>Comment</h5>
<p>May apply to, for example:</p>

<ul>
  <li>Open and Closed high throughput platforms</li>
  <li>Microarray testing (for broad range pathogen testing)</li>
</ul>

<p>All accessories need to be validated for use in combination with the test as part of the CE marking.</p>

<p>Specifications for general reagents that can be used with open or closed systems must be clearly defined</p>

<h4>SARS-CoV-2 Target</h4>

<h5>Desired</h5>
<p>Dual (or more) SARS-CoV-2 <abbr title="Ribonucleic acid">RNA</abbr></p>

<h5>Acceptable</h5>
<p>Single SARS-CoV-2 <abbr title="Ribonucleic acid">RNA</abbr></p>

<h4>Analyte</h4>

<h5>Desired</h5>
<p>Multiplex panel for a range of infectious respiratory viruses</p>

<p>Detection and differentiation of SARS-CoV-2, influenza A virus, and/or influenza B virus, RSV A &amp; B</p>

<h5>Acceptable</h5>
<p>SARS CoV 2 only</p>

<h5>Comment</h5>
<p>Multiplex systems must be able to clearly distinguish between targets included in the panel.</p>

<h4>Sample type</h4>

<h5>Desired</h5>
<p>Oral Fluid</p>

<h5>Acceptable</h5>
<p>Nasopharyngeal or oropharyngeal swabs, lower respiratory tract aspirates, bronchoalveolar lavage, nasopharyngeal wash/aspirate or nasal aspirate</p>

<h5>Comment</h5>
<p>Methods not using invasive swabs are desirable due to the individuals discomfort, pre-analytical errors. Consideration should be given to compatibility of sample with collection media/buffers and assay performance. Sample types must be validated as part of the performance assessment.</p>

<h4>Compatible sample collection</h4>

<h5>Desired</h5>
<p>Test is validated for use with saliva and/or non-propriety collection media</p>

<h5>Acceptable</h5>
<p>Test is validated for use with a variety of non-propriety sample collection media e.g. viral transport medium, inactivation medium, dry swabs, and saline in addtion to propriety media</p>

<h5>Comment</h5>
<p>Supply chain constraints</p>

<h4>Result output</h4>

<h5>Desired</h5>
<p>Semi quantitative</p>

<h5>Acceptable</h5>
<p>Qualitative</p>

<h5>Comment</h5>
<p>Reference <abbr title="Ribonucleic acid">RNA</abbr>/DNA materials are now available from NIBSC1 for <abbr title="Reverse transcription">RT</abbr>-<abbr title="Quantitative polymerase chain reaction">qPCR</abbr>. Some reference materials may perform poorly in <abbr title="Reverse transcription">RT</abbr>-<abbr title="Loop-mediated isothermal amplification">LAMP</abbr> assays because of intact fragment length availability. This may be revised when an International Standard becomes available.</p>

<h4>Size</h4>

<h5>Desired</h5>
<p>Standard mains power supply with UPS and the capability for battery power.</p>

<h5>Acceptable</h5>
<p>Standard mains power supply</p>

<h4>Internal controls</h4>

<h5>Desired</h5>
<p>Whole process positive controls, negative controls, internal and external controls are required to confirm validity end-end processing and clearly identify inhibitory or cellular deficient results as invalid.</p>

<h5>Acceptable</h5>
<p>Should include positive controls and negative controls with option to include internal controls/external controls in a single or multiplex format.</p>

<h5>Comment</h5>
<p>Invalid results may be due to sampling technique or presence of biological inhibitors.</p>

<h4>Technical failure rate</h4>

<h5>Desired</h5>
<p>Less than 0.2%</p>

<h5>Acceptable</h5>
<p>Less than 1%</p>

<h5>Comment</h5>
<p>In use failures resulting from mechanical, controls, calibration or other factors which may not be regarded as  reportable as an adverse event under the IVD regulations.</p>

<p>Does not include failure due to sample collection or technical issues outside of scope of test (e.g. house-keeping genes).</p>

<h4>Ease of use and result interpretation</h4>

<h5>Desired</h5>
<p>Suitable for target user groups (i.e. trained healthcare professionals)</p>

<h5>Acceptable</h5>
<p>Suitable for target user groups (i.e. trained healthcare professionals)</p>

<h4>Need for calibration</h4>

<h5>Desired</h5>
<p>No calibration required</p>

<h5>Acceptable</h5>
<p>Remote or auto-calibration</p>

<h4>Identification capability</h4>

<h5>Desired</h5>
<p>Unique barcode or equivalent for integration into electronic systems</p>

<h5>Acceptable</h5>
<p>Labelling of the device with the subjects identification must be feasible</p>

<h3>Performance characteristics</h3>

<h4>Clinical (diagnostic) sensitivity (or Positive Percent Agreement)</h4>

<h5>Desired</h5>
<p>For <abbr title="Ribonucleic acid">RNA</abbr> extraction based assays a greater than 99% (with 95% two-sided confidence interval entirely above 97%).</p>

<h5>Acceptable</h5>
<p>For <abbr title="Ribonucleic acid">RNA</abbr> extraction based assays a greater than 95% (with 95% two-sided confidence interval entirely above 90%).</p>

<h5>Comment</h5>
<p>At least 150 positive clinical samples with viral loads that span the upper and lower limit of detection for the comparator assay</p>

<p>The samples should cover a clinically meaningful range of viral loads and the percentages of each concentration (low, medium and high) used should match the concentrations anticipated in the test population.</p>

<h4>Clinical (diagnostic) specificity (or Negative Percent Agreement)</h4>

<h5>Desired</h5>
<p>For <abbr title="Ribonucleic acid">RNA</abbr> extraction based assays and those assays which detect SARS-CoV-2 directly from clinical samples a greater than 99% (with 95% two-sided confidence interval entirely above 97%).</p>

<p>National Institute of Biological Standards and Control (NIBSC) <a rel="external" href="https://www.nibsc.org/science_and_research/virology/centre_for_aids_reagents/covid-19_reagents.aspx" class="govuk-link">is developing a number of reference materials</a> including a candidate International Standard for SARS CoV 2 nucleic acid.</p>

<h5>Acceptable</h5>
<p>For <abbr title="Ribonucleic acid">RNA</abbr> extraction based assays and those assays which detect SARS-CoV-2 directly from clinical samples a greater than 95% (with 95% two-sided confidence interval entirely above 90%).</p>

<h5>Comment</h5>
<p>At least 250 negative clinical samples.</p>

<h4>Comparison method</h4>

<h5>Desired</h5>
<p><abbr title="Reverse transcription">RT</abbr>-<abbr title="Quantitative polymerase chain reaction">qPCR</abbr> or <abbr title="Digital polymerase chain reaction">dPCR</abbr> reference methods.</p>

<h5>Acceptable</h5>
<p>Reference method against which the Negative/Positive Percent Agreement is calculated</p>

<h5>Comment</h5>
<p>The comparison should consider the effect of treatment or storage on virus material prior to use and the impact this may have had on the ability of that assay to detect inactivated virus/degraded <abbr title="Ribonucleic acid">RNA</abbr>. Sample must be processed in accordance with the instructions for use.</p>

<h4>Analytical specificity</h4>

<h5>Desired</h5>
<p>No clinically relevant cross reactivity or interference</p>

<h5>Acceptable</h5>
<p>No clinically relevant cross reactivity or interference</p>

<p>Minimal interference caused by common interferents at clinically relevant concentrations (dependant on sample type and analyte)</p>

<h5>Comment</h5>
<p>See annex 1 for list</p>

<p>Demonstrated specificity in the presence of other respiratory pathogens, Flu A/B, RSV, hMPV etc.</p>

<h4>Analytical sensitivity (Limit of Detection (<abbr title="Limit of Detection">LOD</abbr>)</h4>

<h5>Desired</h5>
<p>≤100 SARS-CoV-2 copies/mL</p>

<h5>Acceptable</h5>
<p>≤ 1000 SARS-CoV-2 copies/mL</p>

<h5>Comment</h5>
<p>Positive samples for which the quantity value and measurement uncertainty have been assigned (i.e. by <abbr title="Digital polymerase chain reaction">dPCR</abbr>) should be used to characterize the true positive detection rate.</p>

<p>Where a different unit of measurement is used (e.g. copies/swab, ng/ml) equivalence must be demonstrated and copies/ml equivalent stated in the validation data.</p>

<p>International Standards should be used when available to report <abbr title="Limit of Detection">LOD</abbr></p>

<p>Treatment of virus material on ability of the technology to detect sample needs to be considered particularly in the case of <abbr title="Reverse transcription">RT</abbr>-<abbr title="Loop-mediated isothermal amplification">LAMP</abbr> assays.</p>

<h4>Clinical utility</h4>

<h5>Desired</h5>
<p>Evidence that using the test improves system and individual outcomes (for example, time to diagnosis, subject’s experience, use of pre-cautionary COVID-19 isolation facilities).</p>

<h5>Acceptable</h5>

<h5>Comment</h5>
<p>Refer to <a class="govuk-link" rel="external" href="https://www.nice.org.uk/Media/Default/About/what-we-do/covid-19/Diagnostic-tests-for-COVID-19-evidence-standards-framework.pdf">NICE evidence standards</a> for further information.</p>

<h4>Turnaround time</h4>

<h5>Desired</h5>
<p>Less than 90 minutes from sample to result.</p>

<h5>Acceptable</h5>
<p>Less than five hours from sample to result</p>

<h5>Comment</h5>
<p>The time is from receipt of sample in test laboratory.</p>

<h4>Throughput</h4>

<h5>Desired</h5>
<p>More than 200 tests in unit per 4 hours</p>

<h5>Acceptable</h5>
<p>More than 50 tests per unit in 4 hours</p>

<h4>Hands-on time</h4>

<h5>Desired</h5>
<p>Less than five minutes per sample</p>

<h5>Acceptable</h5>
<p>Less than 20 minutes per sample</p>

<h4>Sample processing and handling</h4>

<h5>Desired</h5>
<p>Standardised sample-processing steps, using standard laboratory equipment (centrifuge, vortex, pipette etc).</p>

<h5>Acceptable</h5>
<p>Standardised sample-processing steps, using standard laboratory equipment (centrifuge, vortex, pipette etc).</p>

<h4>Biosafety</h4>

<h5>Desired</h5>
<p>Buffers provided with kit are proven to inactivate SARS-CoV-2 and any other respiratory viruses in scope of assay enabling entire process to be carried out outside of CL3 or CL2+</p>

<h5>Acceptable</h5>
<p>Requires inactivation pre-step at CL3 or CL2+. Remaining steps can be preformed at CL2.</p>

<h5>Comment</h5>
<p>Systems specifying need for lower biosafety environments must demonstrate sample inactivation and virus containment.</p>

<p>Instructions for use must confirm biosafety requirements for sample handling and inactivation evidence</p>

<p>Appropriate consideration should be given for biosafety in the environment which the test should be carried out</p>

<h4>Risk in use</h4>

<h5>Desired</h5>
<p>Risks have been managed according to ISO 14971</p>

<h5>Acceptable</h5>
<p>Risks have been managed according to ISO 14971</p>

<h2>Operational characteristics</h2>

<h4>Test kit storage and stability conditions</h4>

<h5>Desired</h5>
<p>No cold chain (15 to 30° C)</p>

<h5>Acceptable</h5>
<p>Storage of kit and reagents at -20° C  and above for at least 12 months. Stable for 12 hours once removed from cold storage.</p>

<h5>Comment</h5>
<p>Packaging to be as compact as possible to facilitate storage. Must be made clear if reagents can withstand freeze and thaw and will not be detrimental to the assay</p>

<h4>Assay end point stability (time window during which signal remains valid)</h4>

<h5>Desired</h5>
<p>Up to 1 hour</p>

<h5>Acceptable</h5>
<p>Up to 30 minutes</p>

<h4>Operating conditions</h4>

<h5>Desired</h5>
<p>15 to 30° C</p>

<h5>Acceptable</h5>
<p>15 to 30° C</p>

<h4>Connectivity</h4>

<h5>Desired</h5>
<p>Wireless connectivity into NHS <abbr title="Laboratory Information Management System">LIMS</abbr> systems</p>

<h5>Acceptable</h5>
<p>Cable connectivity into NHS <abbr title="Laboratory Information Management System">LIMS</abbr> system</p>

<h5>Comment</h5>
<p>Results may need to be transferred by digital media e.g. CSV files</p>

<h4>Presentation of results</h4>

<h5>Desired</h5>
<p>Results do not require post run analysis (automatically called). Ability to access raw data e.g. <abbr title="Reverse transcription">RT</abbr>-<abbr title="Quantitative polymerase chain reaction">qPCR</abbr>/<abbr title="Reverse transcription">RT</abbr>-<abbr title="Loop-mediated isothermal amplification">LAMP</abbr> fluorescence traces</p>

<h5>Acceptable</h5>
<p>Easy to capture for interpretation and able to record public health data</p>

<h5>Comment</h5>
<p>Integrity of data must be maintained</p>

<h4>Reproducibility</h4>

<h5>Desired</h5>
<p>More than 95% between repeats at LoD</p>

<p>More than 99% at higher concentrations</p>

<h5>Acceptable</h5>
<p>More than 95% between repeats at LoD</p>

<p>More than 99% at higher concentrations</p>

<h5>Comment</h5>
<p>Manufacturers should consider ISO 20395:2019 and ISO 5725-1 when evaluating reproducibility.</p>

<h4>Volume of sample</h4>

<h5>Desired</h5>
<p>Depends on sample type, but no more than 0.5mL</p>

<h5>Acceptable</h5>
<p>Depends on sample type, but no more than 0.5mL</p>

<h4>Disposal requirements</h4>

<h5>Desired</h5>
<p>No additional disposal requirements beyond normal laboratory practice. Any special containment or disposal requirements need to be clearly specified (e.g. contamination control for <abbr title="Reverse transcription">RT</abbr>-<abbr title="Loop-mediated isothermal amplification">LAMP</abbr> post amplification products).</p>

<h5>Acceptable</h5>
<p>No additional disposal requirements beyond normal laboratory practice. Any special containment or disposal requirements need to be clearly specified (e.g. contamination control for <abbr title="Reverse transcription">RT</abbr>-<abbr title="Loop-mediated isothermal amplification">LAMP</abbr> post amplification products).</p>

<h4>Training needs (Time dedicated to training session for end users)</h4>

<h5>Desired</h5>
<p>Less than half day training needed for laboratory staff</p>

<h5>Acceptable</h5>
<p>Less than one day of training needed</p>

<h2>Other</h2>

<h4>Immediate supply volumes (Tests per week, within 4 weeks)</h4>

<h5>Desired</h5>
<p>10,000 tests per day</p>

<h5>Acceptable</h5>
<p>5000 tests per day</p>

<h4>Label and Instructions for Use</h4>

<h5>Desired</h5>
<p>Conforms to IVD Directive and relevant harmonised standards</p>

<h5>Acceptable</h5>
<p>Conforms to IVD Directive and relevant harmonised standards</p>

<h4>Regulatory status</h4>

<h5>Desired</h5>
<p>CE marked</p>

<h5>Acceptable</h5>
<p>Exempt according to Article 9 para 12 or para 13 of IVD Directive</p>

<h4>Maintenance</h4>

<h5>Desired</h5>
<p>Preventive maintenance should not be needed until after 2 years or 100,000 samples. An alert should be included to indicate when maintenance is needed.</p>

<h5>Acceptable</h5>
<p>Preventive maintenance should not be needed until after 1 year or 10,000 samples; an alert should be included to indicate when maintenance is needed.</p>

<h5>Comment</h5>
<p>Assuming the equipment is used at capacity 24 hours a day, seven days a week</p>

<h4>Design and manufacturing environment</h4>

<h5>Desired</h5>
<p>ISO 13485:2016</p>

<h5>Acceptable</h5>
<p>ISO 13485:2016</p>

<h2>Annex 1: Assay validation</h2>

<h3>Establishing Performance Characteristics</h3>

<p>It is recommended that the following aspects are considered when designing and validating the assay.</p>

<ul>
  <li>Reference material should be used to establish performance, including standard validation panels, quality control materials and proficiency testing materials</li>
  <li>Some technologies may require dilution series of untreated whole virus then comparing with <abbr title="Digital polymerase chain reaction">dPCR</abbr>. Treated material may artificially lower sensitivity of a <abbr title="Loop-mediated isothermal amplification">LAMP</abbr> assay</li>
  <li>When establishing analytical specificity, the following should be considered:
    <ul>
      <li>Prepandemic samples,</li>
      <li>Other coronavirus, SARS-CoV-1,</li>
      <li>hCoV 229E, OC43, HKU1, NL63 epitopes /genome</li>
      <li>Adenovirus (e.g. C1 Ad. 71)</li>
      <li>Human Metapneumovirus (hMPV)</li>
      <li>Parainfluenza virus 1-4</li>
      <li>Influenza A &amp; B</li>
      <li>Enterovirus (e.g. EV68)</li>
      <li>Respiratory syncytial virus</li>
      <li>Rhinovirus</li>
      <li>Middle East Respiratory Syndrome (MERS)</li>
      <li>Chlamydia pneumoniae</li>
      <li>Haemophilus influenzae</li>
      <li>Legionella pneumophila</li>
      <li>Mycobacterium tuberculosis</li>
      <li>Streptococcus pneumoniae</li>
      <li>Streptococcus pyogenes</li>
      <li>Bordetella pertussis</li>
      <li>Mycoplasma pneumoniae</li>
      <li>Pneumocystis jirovecii (PJP)</li>
      <li>Epstein Barr Virus</li>
    </ul>
  </li>
  <li>Potential interferents may originate from the following endogenous and exogenous sources and may be more relevant to ligand-binding based antigen tests than conventional <abbr title="Polymerase chain reaction">PCR</abbr> based assays. Manufacturers should declare if any other endogenous/ exogenous substances will impact the assay.
    <ul>
      <li>Antibacterial, systemic</li>
      <li>Antibiotic, nasal ointment</li>
      <li>Anti-viral drugs</li>
      <li>Antibodies developed against protein expression system used to generate recombinant antigens</li>
      <li>Bilirubin</li>
      <li>Biotin</li>
      <li>Blood (human)</li>
      <li>Haemoglobin</li>
      <li>Human Anti-mouse Antibody (HAMA)</li>
      <li>Medications most often prescribed in the population for which the test is ordered</li>
      <li>Mucin: bovine submaxillary gland, type I-S</li>
      <li>Nasal sprays or drops</li>
      <li>Nasal corticosteroids</li>
      <li>Nasal gel</li>
      <li>Protein</li>
      <li>Rheumatoid Factor</li>
      <li>Throat lozenges, oral anaesthetic and analgesic</li>
      <li>Triglycerides</li>
    </ul>
  </li>
</ul>

<p>In addition the assay must deliver across the clinical range of haematocrit values</p>

<h3>Post-market performance considerations for probes and primers</h3>

<p>Manufacturers should consider monitoring for potential genetic change to ensure probes and primers are not adversely affected by such events. In silico analysis may aid ongoing monitoring.</p>

<h2>Annex 2: Diagnostic accuracy considerations</h2>

<p>When considering procurement and deployment of devices for any clinical and public health use-case, it is recommended to consider the maximum number of false positives and false negatives that would be acceptable for the new test based on the possible consequences of these misdiagnoses.</p>

<p>The table below presents the numbers of false positives and negatives in a cohort of fixed size (10,000) with varying prevalence of COVID-19 (NPV/PPV rounded to nearest whole number).</p>

<p>Therefore, for a test with a sensitivity of 80% and specificity of 95%:</p>

<table>
  <thead>
    <tr>
      <th scope="col">COVID-19 prevalence</th>
      <th scope="col">False Positives</th>
      <th scope="col">Positive predictive value (proportion of people with positive results who have COVID-19)</th>
      <th scope="col">False Negatives</th>
      <th scope="col">Negative predictive value (proportion of people with negative results that don’t have COVID-19)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1%</td>
      <td>495</td>
      <td>14%</td>
      <td>20</td>
      <td>100%</td>
    </tr>
    <tr>
      <td>5%</td>
      <td>475</td>
      <td>46%</td>
      <td>100</td>
      <td>99%</td>
    </tr>
    <tr>
      <td>10%</td>
      <td>450</td>
      <td>64%</td>
      <td>200</td>
      <td>98%</td>
    </tr>
    <tr>
      <td>25%</td>
      <td>250</td>
      <td>94%</td>
      <td>1000</td>
      <td>83%</td>
    </tr>
  </tbody>
</table>

<p>It should also be noted that sensitivity and specificity values estimated in a particular population (i.e. intensive care patients) may not be generalisable to other populations (i.e. general practice) with a different prevalence of COVID-19, if these populations are made up of people with less or more severe COVID-19. For example, accuracy estimates generated in a population of people with early symptoms of COVID-19 may be higher, due to viral load, than a test would achieve in a population of people with no symptoms of the condition.</p>

<h3>Prevalence 1%</h3>
<p>Numbers per 10,000 tested</p>

<table>
  <thead>
    <tr>
      <td></td>
      <td style="text-align: right"></td>
      <th scope="col">Sensitivity</th>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th scope="col">Specificity</th>
      <th scope="col" style="text-align: right">Test result</th>
      <th scope="col">98%</th>
      <th scope="col">95%</th>
      <th scope="col">90%</th>
      <th scope="col">85%</th>
      <th scope="col">80%</th>
      <th scope="col">75%</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>98%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>198 <br><br> 2</td>
      <td>198 <br><br> 5</td>
      <td>198 <br><br> 10</td>
      <td>198 <br><br> 15</td>
      <td>198 <br><br> 20</td>
      <td>198 <br><br> 25</td>
    </tr>
    <tr>
      <td>95%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>495 <br><br> 2</td>
      <td>495 <br><br> 5</td>
      <td>495 <br><br> 10</td>
      <td>495 <br><br> 15</td>
      <td>495 <br><br> 20</td>
      <td>495 <br><br> 25</td>
    </tr>
    <tr>
      <td>90%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>990 <br><br> 2</td>
      <td>990 <br><br> 5</td>
      <td>990 <br><br> 10</td>
      <td>990 <br><br> 15</td>
      <td>990 <br><br> 20</td>
      <td>990  <br><br> 25</td>
    </tr>
    <tr>
      <td>85%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>1,485 <br><br> 2</td>
      <td>1,485 <br><br> 5</td>
      <td>1,485 <br><br> 10</td>
      <td>1,485 <br><br> 15</td>
      <td>1,485 <br><br> 20</td>
      <td>1,485 <br><br> 25</td>
    </tr>
    <tr>
      <td>80%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>1,980 <br><br> 2</td>
      <td>1,980 <br><br> 5</td>
      <td>1,980 <br><br> 10</td>
      <td>1,980 <br><br> 15</td>
      <td>1,980 <br><br> 20</td>
      <td>1,980 <br><br> 25</td>
    </tr>
    <tr>
      <td>75%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>2,475 <br><br> 2</td>
      <td>2,475 <br><br> 5</td>
      <td>2,475 <br><br> 10</td>
      <td>2,475 <br><br> 15</td>
      <td>2,475 <br><br> 20</td>
      <td>2,475 <br><br> 25</td>
    </tr>
  </tbody>
</table>

<h3>Prevalence 5%</h3>
<p>Numbers per 10,000 tested</p>

<table>
  <thead>
    <tr>
      <td></td>
      <td style="text-align: right"></td>
      <th scope="col">Sensitivity</th>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th scope="col">Specificity</th>
      <th scope="col" style="text-align: right">Test result</th>
      <th scope="col">98%</th>
      <th scope="col">95%</th>
      <th scope="col">90%</th>
      <th scope="col">85%</th>
      <th scope="col">80%</th>
      <th scope="col">75%</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>98%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>190 <br><br> 10</td>
      <td>190 <br><br> 25</td>
      <td>190 <br><br> 50</td>
      <td>190 <br><br> 75</td>
      <td>190 <br><br> 100</td>
      <td>190 <br><br> 125</td>
    </tr>
    <tr>
      <td>95%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>475 <br><br> 10</td>
      <td>475 <br><br> 25</td>
      <td>475 <br><br> 50</td>
      <td>475 <br><br> 75</td>
      <td>475 <br><br> 100</td>
      <td>475 <br><br> 125</td>
    </tr>
    <tr>
      <td>90%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>950 <br><br> 10</td>
      <td>950 <br><br> 25</td>
      <td>950 <br><br> 50</td>
      <td>950  <br><br> 75</td>
      <td>950 <br><br> 100</td>
      <td>950 <br><br> 125</td>
    </tr>
    <tr>
      <td>85%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>1,425 <br><br> 10</td>
      <td>1,425 <br><br> 25</td>
      <td>1,425 <br><br> 50</td>
      <td>1,425 <br><br> 75</td>
      <td>1,425 <br><br> 100</td>
      <td>1,425 <br><br> 125</td>
    </tr>
    <tr>
      <td>80%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>1,900 <br><br> 10</td>
      <td>1,900 <br><br> 25</td>
      <td>1,900 <br><br> 50</td>
      <td>1,900 <br><br> 75</td>
      <td>1,900 <br><br> 100</td>
      <td>1,900 <br><br> 125</td>
    </tr>
    <tr>
      <td>75%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>2,375 <br><br> 10</td>
      <td>2,375 <br><br> 25</td>
      <td>2,375 <br><br> 50</td>
      <td>2,375 <br><br> 75</td>
      <td>2,375 <br><br> 100</td>
      <td>2,375 <br><br> 125</td>
    </tr>
  </tbody>
</table>

<h3>Prevalence 10%</h3>
<p>Numbers per 10,000 tested</p>

<table>
  <thead>
    <tr>
      <td></td>
      <td style="text-align: right"></td>
      <th scope="col">Sensitivity</th>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th scope="col">Specificity</th>
      <th style="text-align: right" scope="col">Test result</th>
      <th scope="col">98%</th>
      <th scope="col">95%</th>
      <th scope="col">90%</th>
      <th scope="col">85%</th>
      <th scope="col">80%</th>
      <th scope="col">75%</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>98%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>180 <br><br> 20</td>
      <td>180 <br><br> 50</td>
      <td>180 <br><br> 100</td>
      <td>180 <br><br> 150</td>
      <td>180 <br><br> 200</td>
      <td>180 <br><br> 250</td>
    </tr>
    <tr>
      <td>95%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>450 <br><br> 20</td>
      <td>450 <br><br> 50</td>
      <td>450 <br><br> 100</td>
      <td>450 <br><br> 150</td>
      <td>450 <br><br> 200</td>
      <td>450 <br><br> 250</td>
    </tr>
    <tr>
      <td>90%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>900 <br><br> 20</td>
      <td>900 <br><br> 50</td>
      <td>900 <br><br> 100</td>
      <td>900 <br><br> 150</td>
      <td>900 <br><br> 200</td>
      <td>900 <br><br> 250</td>
    </tr>
    <tr>
      <td>85%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>1,350 <br><br> 20</td>
      <td>1,350 <br><br> 50</td>
      <td>1,350 <br><br> 100</td>
      <td>1,350 <br><br> 150</td>
      <td>1,350 <br><br> 200</td>
      <td>1,350 <br><br> 250</td>
    </tr>
    <tr>
      <td>80%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>1,800 <br><br> 20</td>
      <td>1,800 <br><br> 50</td>
      <td>1,800 <br><br> 100</td>
      <td>1,800 <br><br> 150</td>
      <td>1,800 <br><br> 200</td>
      <td>1,800 <br><br> 250</td>
    </tr>
    <tr>
      <td>75%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>2,250 <br><br> 20</td>
      <td>2,250 <br><br> 50</td>
      <td>2,250 <br><br> 100</td>
      <td>2,250 <br><br> 150</td>
      <td>2,250 <br><br> 200</td>
      <td>2,250 <br><br> 250</td>
    </tr>
  </tbody>
</table>

<h3>Prevalence 50%</h3>
<p>Numbers per 10,000 tested</p>

<table>
  <thead>
    <tr>
      <td></td>
      <td style="text-align: right"></td>
      <th scope="col">Sensitivity</th>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th scope="col">Specificity</th>
      <th style="text-align: right" scope="col">Test result</th>
      <th scope="col">98%</th>
      <th scope="col">95%</th>
      <th scope="col">90%</th>
      <th scope="col">85%</th>
      <th scope="col">80%</th>
      <th scope="col">75%</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>98%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>100 <br><br> 100</td>
      <td>100 <br><br> 250</td>
      <td>100 <br><br> 500</td>
      <td>100 <br><br> 750</td>
      <td>100 <br><br> 1,000</td>
      <td>100 <br><br> 1,250</td>
    </tr>
    <tr>
      <td>95%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>250 <br><br> 100</td>
      <td>250 <br><br> 250</td>
      <td>250 <br><br> 500</td>
      <td>250 <br><br> 750</td>
      <td>250 <br><br> 1,000</td>
      <td>250 <br><br> 1,250</td>
    </tr>
    <tr>
      <td>90%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>500 <br><br> 100</td>
      <td>500  <br><br> 250</td>
      <td>500 <br><br> 500</td>
      <td>500 <br><br> 750</td>
      <td>500 <br><br> 1,000</td>
      <td>500 <br><br> 1,250</td>
    </tr>
    <tr>
      <td>85%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>750 <br><br> 100</td>
      <td>750 <br><br> 250</td>
      <td>750 <br><br> 500</td>
      <td>750 <br><br> 750</td>
      <td>750 <br><br> 1,000</td>
      <td>750 <br><br> 1,250</td>
    </tr>
    <tr>
      <td>80%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>1,000 <br><br> 100</td>
      <td>1,000 <br><br> 250</td>
      <td>1,000 <br><br> 500</td>
      <td>1,000 <br><br> 750</td>
      <td>1,000 <br><br> 1,000</td>
      <td>1,000 <br><br> 1,250</td>
    </tr>
    <tr>
      <td>75%</td>
      <td style="text-align: right">False +ves <br><br> False -ves</td>
      <td>1,250 <br><br> 100</td>
      <td>1,250 <br><br> 250</td>
      <td>1,250 <br><br> 500</td>
      <td>1,250 <br><br> 750</td>
      <td>1,250 <br><br> 1,000</td>
      <td>1,250 <br><br> 1,250</td>
    </tr>
  </tbody>
</table>

<h2>Annex 3: Glossary</h2>

<ul>
  <li>
<abbr title="Biological Safety Level">BSL</abbr>: Biological Safety Level</li>
  <li>
<abbr title="Containment level">CL</abbr>: Containment level</li>
  <li>
<abbr title="Digital polymerase chain reaction">dPCR</abbr>: Digital polymerase chain reaction</li>
  <li>IVD: In Vitro Diagnostic</li>
  <li>
<abbr title="Loop-mediated isothermal amplification">LAMP</abbr>: Loop-mediated isothermal amplification</li>
  <li>
<abbr title="Laboratory Information Management System">LIMS</abbr>: Laboratory Information Management System</li>
  <li>LoD: Limit of Detection</li>
  <li>
<abbr title="Nucleic acid testing">NAT</abbr>: Nucleic acid testing</li>
  <li>
<abbr title="Polymerase chain reaction">PCR</abbr>: Polymerase chain reaction</li>
  <li>
<abbr title="Personal Protective Equipment">PPE</abbr>: Personal Protective Equipment</li>
  <li>
<abbr title="Quantitative polymerase chain reaction">qPCR</abbr>: Quantitative polymerase chain reaction</li>
  <li>
<abbr title="Ribonucleic acid">RNA</abbr>: Ribonucleic acid</li>
  <li>
<abbr title="Reverse transcription">RT</abbr>: Reverse transcription</li>
</ul>

<p>analytical sensitivity: sensitivity of a measurement procedure quotient of the change in a measurement indication and the corresponding change in a value of a quantity being measured</p>

<p>analytical specificity: selectivity of a measurement procedure capability of a measuring system, using a specified measurement procedure, to provide measurement results for one or more measurands which do not depend on each other nor on any other quantity in the system undergoing measurement</p>

<p>Clinical (Diagnostic) Sensitivity: ability of an IVD examination procedure to identify the presence of a target marker associated with a particular disease or condition</p>

<p>Clinical (Diagnostic) Specificity ability of an IVD examination procedure to recognise the absence of a target marker associated with a particular disease or condition</p>

<p>The above definitions of performance characteristics taken from BS EN ISO 18113-1:2011, In vitro diagnostic medical devices — Information supplied by the manufacturer (labelling): Terms, definitions and general requirements.</p>

<p>Positive Percent Agreement: calculated in the same way as Clinical (Diagnostic) Sensitivity, but indicate that a non-reference standard was used</p>

<p>Negative Percent Agreement: calculated in the same way as Clinical (Diagnostic) Specificity, but indicate that a non-reference standard was used</p>

</div>


</div>
</div>
  </div>

  <div data-sticky-element="" class="sticky-element">
    <a class="govuk-link app-c-back-to-top dont-print" href="#contents">
    <svg class="app-c-back-to-top__icon" viewBox="0 0 13 17" height="17" xmlns="http://www.w3.org/2000/svg" width="13" focusable="false">
      <path d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z" fill="currentColor"></path>
    </svg>
    Contents
</a>

    <div class="sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>