<main lang="en" class="html-publication" role="main">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          
<div class="gem-c-organisation-logo brand--department-of-health">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color" href="/government/organisations/medicines-and-healthcare-products-regulatory-agency">
      <span class="gem-c-organisation-logo__name">Medicines &amp; Healthcare products<br>Regulatory Agency</span>
</a>
</div>
        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  

<div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
      <span class="govuk-caption-xl gem-c-title__context">
    Guidance
  </span>

  
  <h1 class="gem-c-title__text govuk-heading-xl">
    Target Product Profile: Laboratory-Based SARS-CoV-2 Viral Detection tests 
  </h1>
</div>
  <p class="publication-header__last-changed">Updated 29 December 2021</p>

  </header>



<div class="govuk-grid-row sidebar-with-body" data-module="sticky-element-container">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav aria-label="Contents" data-module="gem-track-click" class="gem-c-contents-list" role="navigation">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 1" data-track-label="#the-purpose-of-a-target-product-profile-tpp" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" href="#the-purpose-of-a-target-product-profile-tpp" data-track-category="contentsClicked" data-track-options="{&quot;dimension29&quot;:&quot;The purpose of a Target Product Profile (TPP)&quot;}">The purpose of a Target Product Profile (TPP)</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-options="{&quot;dimension29&quot;:&quot;Intended use for this TPP&quot;}" data-track-label="#intended-use-for-this-tpp" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" href="#intended-use-for-this-tpp" data-track-action="content_item 2" data-track-category="contentsClicked">Intended use for this TPP</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-category="contentsClicked" data-track-options="{&quot;dimension29&quot;:&quot;Clinical performance requirements&quot;}" href="#clinical-performance-requirements" data-track-action="content_item 3" data-track-label="#clinical-performance-requirements">Clinical performance requirements</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-label="#future-developments" href="#future-developments" data-track-action="content_item 4" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-category="contentsClicked" data-track-options="{&quot;dimension29&quot;:&quot;Future developments&quot;}">Future developments</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a href="#key-to-table" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-label="#key-to-table" data-track-options="{&quot;dimension29&quot;:&quot;Key to Table&quot;}" data-track-category="contentsClicked" data-track-action="content_item 5">Key to Table</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-category="contentsClicked" data-track-label="#annex-1-assay-validation" data-track-options="{&quot;dimension29&quot;:&quot;Annex 1: Assay validation&quot;}" data-track-action="content_item 6" href="#annex-1-assay-validation" class="gem-c-contents-list__link govuk-link govuk-link--no-underline">Annex 1: Assay validation</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-category="contentsClicked" data-track-action="content_item 7" data-track-options="{&quot;dimension29&quot;:&quot;Annex 2: Diagnostic accuracy considerations&quot;}" data-track-label="#annex-2-diagnostic-accuracy-considerations" href="#annex-2-diagnostic-accuracy-considerations">Annex 2: Diagnostic accuracy considerations</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-options="{&quot;dimension29&quot;:&quot;Annex 3 Glossary&quot;}" href="#annex-3-glossary" data-track-label="#annex-3-glossary" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 8" data-track-category="contentsClicked">Annex 3 Glossary</a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button data-module="print-link" class="govuk-link govuk-body-s gem-c-print-link__button">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img class="print-meta-data-licence" src="/assets/government-frontend/open-government-licence-min-e98134ae65ff5fe99a524d8c383350b3998f91c298dc24b36126e65f1656c578.png">
</p>
<p>
  © Crown copyright 2021
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/how-tests-and-testing-kits-for-coronavirus-covid-19-work/target-product-profile-laboratory-based-sars-cov-2-viral-detection-tests
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div data-module="govspeak" class="gem-c-govspeak govuk-govspeak ">
    
    
      <div class="govspeak">
<p>Version control</p>

<table>
  <thead>
    <tr>
      <th scope="col">Version</th>
      <th scope="col">Date issued</th>
      <th scope="col">Description</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1.0</td>
      <td>October 2020</td>
      <td>Initial document</td>
    </tr>
    <tr>
      <td>2.0</td>
      <td>10 November 2021</td>
      <td>Changes to Clinical Sensitivity and Limits of Detection to accommodate assays with additional benefits, including: multiplex, short turn around times or high throughput.</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>Increased Clinical Specificity in line with the changing state of the art and continuous quality improvement</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>Updated regulatory sections in line with the UK departure from the EU</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>Updated intended-use and included statements concerning SARS-CoV-2 variants of concern</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>Updated sections on Reference Standard and Sample types</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>Harmonised introductory sections and Annex’s with newer TPP formatting</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>Included considerations of clinical performance evaluation, clinical impact, downstream sequencing and carry over</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>Removed requirements considering supply volumes, equipment size, sample volume</td>
    </tr>
  </tbody>
</table>

<h2>The purpose of a Target Product Profile (TPP)</h2>

<p>MHRA Target Product Profiles (TPP) are guidance documents which aim to support and accelerate the development and evaluation of new medical technologies to address specific unmet clinical or public health needs of high strategic priority to the UK population.</p>

<p>TPPs summarise the key features and anticipated performance specifications of a new device in advance, to enable innovators to design and develop high quality products that are fit for purpose and meet specific health-related goals.  They are intended to be used to support product design, research and development planning and to facilitate discussions with regulators.</p>

<p>MHRA TPP’s are aspirational documents aimed at test manufacturers and are based upon the best available evidence and independent expert opinion. They do not represent UK government policy and are not regulatory requirements.  For information on the current National technical validation process and their relevant performance goals, please see <a href="https://www.gov.uk/government/publications/assessment-and-procurement-of-coronavirus-covid-19-tests/coronavirus-covid-19-serology-and-viral-detection-testing-uk-procurement-overview" class="govuk-link">National technical validation process for manufacturers of SARS-CoV-2 (COVID-19) tests - GOV.UK (www.gov.uk)</a></p>

<p>MHRA TPPs are living documents that are reviewed on a frequent basis, dependent on the specific disease area, and updated as additional evidence and information becomes available.  Manufacturers should ensure they are working to the most recent version of a TPP.</p>

<h2>Intended use for this TPP</h2>

<p>This TPP is intended to be used by manufacturers to support the development of laboratory-based in vitro diagnostic tests for detection of current SARS-CoV-2 infection in people with and without symptoms., as part of UK government national testing programmes or <a href="https://www.gov.uk/government/publications/list-of-private-providers-of-coronavirus-testing/list-of-private-providers-of-coronavirus-testing" class="govuk-link">accredited private testing services</a>.</p>

<p>The exact performance requirements will vary depending on the specific use-case, taking into consideration the following:</p>

<ol>
  <li>The target population (e.g. Children, young people, adults, older people)</li>
  <li>The setting (e.g. A&amp;E, testing centres, workplaces)</li>
  <li>The intervention decision being informed (e.g. release from isolation, contact tracing, infection control measures etc).</li>
</ol>

<p>This TPP does not consider tests for use in the following possible scenarios:</p>

<ul>
  <li>Self-Tests for use at home</li>
  <li>Point of care tests to be performed by a professional (<a href="https://www.gov.uk/government/publications/how-tests-and-testing-kits-for-coronavirus-covid-19-work/for-industry-and-manufactures-covid-19-tests-and-testing-kits" class="govuk-link">see here for full definitions</a>) in a health and/or social care setting.</li>
  <li>Prognose a patient’s likely outcome, including disease severity or survival.</li>
  <li>Predict or monitor a patient’s likely response to treatment.</li>
</ul>

<h2>Clinical performance requirements</h2>

<p>The TPP sets out the requirements based on the consensus of what is “desired” and minimally “acceptable” in the opinion of healthcare professionals and scientists given the current situation. Products meeting the “desirable” criteria will likely have a role in a greater number of use-cases than products that only meet the “acceptable” criteria.</p>

<p>The decision to use a particular test for a specific use must be informed by clinical expert opinion at the time, considering the disease prevalence, risks, benefits and downstream consequences of testing vs not-testing. Annex 2 provides tables and further discussion which may be useful in supporting decision making, by demonstrating the impact of changing sensitivity, specificity and prevalence on the numbers of false positives and negatives.</p>

<h2>Future developments</h2>

<p>These profiles are subject to review and change, as we gain a greater knowledge of the virus, the disease and our needs for an effective response. They may need to be updated at short notice.</p>

<p>As our knowledge and understanding of the disease changes and the UK clinical needs change, so will the specifications. A test that meets this version of the TPP may not meet future versions.</p>

<h2>Key to Table</h2>

<p><strong>Desired</strong>: Highly desirable features of considerable benefit. As time is of the essence, if omitting one of these features significantly accelerates development and production it can be considered.</p>

<p><strong>Acceptable</strong>: Defines the minimum acceptable feature.</p>

<h3>Target Product Profile: COVID-19 - Laboratory-based SARS-CoV-2 Viral detection testing</h3>

<h3>Scope</h3>

<h4>Key feature</h4>

<p>Intended Function(s)</p>

<h4>Desired</h4>

<p>Used in the detection of current SARS-CoV-2 infection<br><br>And<br><br>
Differentiation from recent or recovered infection.<br><br>And/or<br><br>Differentiation of SARS-CoV-2 from other respiratory infections.<br><br>And/or <br><br>Differentiation of SARS-CoV-2 Variants of Concern (VoCs)</p>

<h4>Acceptable</h4>

<p>Used in the detection of current SARS-CoV-2 infection</p>

<h4>Comments</h4>

<p>Current infection: an infection in which the causative organism has the potential, either now or in the future, to cause disease or onward transmission. An individual with a current infection may not display disease symptoms, require treatment or be infectious at the time of testing.</p>

<p>Manufacturers should indicate if the test is only intended for use in a specific population or setting and would be expected to justify the balance between risk and benefits.  Tests with “acceptable” levels of sensitivity and specificity may only have application in a limited number of use-cases and will likely require additional risk mitigation measures, such as confirmatory testing or infection control.  Refer to Annex 2.</p>

<p>Whilst multiplexing is desirable for specific use-cases, it may not be appropriate for all.</p>

<h4>Target population</h4>

<p>Adults and children with or without symptoms</p>

<h4>Comments</h4>

<p>Populations “without symptoms” represents a range of pre-symptomatic, peri-symptomatic, sub-symptomatic and truly asymptomatic phenotypes, spanning all stages of disease (early and late) and a wide range of viral loads (low to high).</p>

<p>Manufacturers should carefully consider the design features of tests intended for use in children to ensure their safe and effective operation and acceptability.</p>

<p>Manufacturers should ensure that risk-management and clinical performance is appropriate for their claimed target populations.  Evidence of clinical performance should be provided for each population claimed by the manufacturer and performed by the relevant user group.</p>

<p>As the immune status of the UK population changes over time, manufacturers should undertake studies to evaluate how the test continues to perform in vaccinated and re-infected individuals.</p>

<h4>Target user</h4>

<p>Trained healthcare / public health professionals (i.e. one of the 10 health and social care professional bodies that are overseen by the <a class="govuk-link" href="https://www.professionalstandards.org.uk/" rel="external">professional standards authority</a>) and suitably trained and assessed as competent lab technicians or scientists.</p>

<h4>Comments</h4>

<p>A target user will perform the assay, interpret and communicate the results.</p>

<p>Full training appropriate to the intended user is required.</p>

<h4>Target use Setting</h4>

<p>Healthcare and Medical Laboratories</p>

<h4>Comments</h4>

<p>These exclude Point of Care (POC) testing environments which will utilise tests types that are appropriate for that setting.</p>

<p>Includes both private and public sector laboratories.</p>

<h3>Test design characteristics</h3>

<h4>Test format</h4>

<p>A standardised kit that contains all materials required for the laboratory procedure that includes controls, reagents and Instruction for Use (IFU).</p>

<p>All equipment and accessories needed to perform the assay and sample processing included, with the exception of routine laboratory consumables such as pipettes, pipette tips, disinfectants, screw cap eppendorfs, heat blocks, fluorescent readers etc.</p>

<p>Should the technology not be compatible with routinely used sample collection devices these should also be provided at the same time as the test kit.</p>

<h4>Comments</h4>

<p>May apply to, for example:</p>

<ul>
  <li>Open and Closed high throughput platforms</li>
  <li>Microarray testing (for broad range pathogen testing)</li>
</ul>

<p>All accessories need to be validated for use in combination with the test as part of the CE, CE UKNI or UKCA marking.</p>

<p>Specifications for general reagents that can be used with open or closed systems must be clearly defined.</p>

<h4>Target Analyte (Measurand)</h4>

<h4>Desired</h4>

<p>Dual (or more) SARS-CoV-2 targets (e.g. nucleic acid(s), antigen(s) or other targets)</p>

<p>Single (or more) targets for a range of infectious respiratory viruses, including influenza A virus, and influenza B virus, RSV A &amp; B</p>

<p>Single (or more) targets for all current SARS-CoV-2 variants of concern</p>

<h4>Acceptable</h4>

<p>Single SARS-CoV-2 target (e.g. nucleic acid, antigen or other target)</p>

<h4>Comments</h4>

<p>Evidence demonstrating the association of the analyte(s) with current SARS-CoV-2 infection (scientific validity) should be provided.</p>

<p>Manufacturers should consider targeting assays to conserved regions/epitopes of SARS-CoV-2 to ensure the detection of current and future variants</p>

<p>Dual antigen targets could include multiple epitopes for the same protein.</p>

<p>Multiplex sytems must be able to clearly distinguish between targets included in the panel.</p>

<p>Assays that detect whole SARS-CoV-2 Virus may also be acceptable</p>

<p>There is a requirement on suppliers to confirm performance in detection of both current and emerging strain variants of SARS-CoV-2 as they arise and confirm this to MHRA where VOC (variants of concern) or VUI (variants under investigation) are reported in line with MHRA requirements, see Annex 1</p>

<h4>Sample type</h4>

<h4>Desired</h4>

<p>Method not requiring a swab (e.g. saliva, sputum, stool, breath sample).</p>

<p>Validation of assays for use with respiratory tract samples’ (sputum, endotracheal, bronchoalveolar lavage, nasopharyngeal aspirate), tissue samples or cerebral spinal fluid may also be desirable for some specific use cases.</p>

<h4>Acceptable</h4>

<p>Nasal and/or throat swab</p>

<h4>Comments</h4>

<p>All sample types claimed as appropriate in the instructions for use must be validated as part of the performance assessment. Study sample sizes for each sample type must be sufficiently powered.</p>

<p>Methods not using invasive swabs are desirable due to the individual’s discomfort and pre-analytical errors. They may also facilitate improved quantification of viral load.</p>

<p>Manufacturers should ensure that swabs are appropriate for the intended population (i.e. smaller swabs for children).Consideration should be given to compatibility of sample with collection media/buffers and assay performance</p>

<p>Not all sample collection buffers or methods are compatible with downstream testing requirements e.g. genotyping and sequencing.</p>

<h4>Compatible sample collection</h4>

<h4>Desired</h4>

<p>Test is validated for use with the sample types listed above which may be collected into a variety of sample collection medias e.g. viral transport medium, inactivation medium, solvents, dry swabs, and saline in addtional to media provided with the kit.</p>

<p>Sample media is suitable for downstream processing including sequencing for identification of lineage, mutations and variants of concern</p>

<h4>Acceptable</h4>

<p>Test is validated for use with  sample types above,  which may be collected into a media provided with the kit.</p>

<h4>Comments</h4>

<p>Additional processing or re-sampling may be required if the availability of quality RNA for downstream processing cannot be met by any single test method</p>

<h4>Result output</h4>

<h4>Desired</h4>

<p>Semi-Quantitative</p>

<h4>Acceptable</h4>

<p>Qualitative</p>

<h4>Comments</h4>

<p>Whilst it is desirable for analytical methods to be quantitative, the nature of  swab sample collection means that results are likely to only be semi-quantitative.</p>

<p>For semi-quantitative assays, an appropriate number of calibration points and replicates covering the range of reliable signal should be applied.</p>

<p>Qualitative results may not support all downstream applications (e.g sequencing).</p>

<h4>Power requirement</h4>

<h4>Desired</h4>

<p>Standard mains power supply with UPS and the capability for battery power.</p>

<h4>Acceptable</h4>

<p>Mains power supply</p>

<h4>Internal controls</h4>

<h4>Desired</h4>

<p>Whole process positive controls, negative controls, internal and external controls are required to confirm validity of end-end processing and clearly identify sample inadequacy results as invalid.</p>

<h4>Acceptable</h4>

<p>Should include positive controls and negative controls with option to include internal controls/external controls in a single or multiplex format.</p>

<h4>Comments</h4>

<p>Invalid results may be due to sampling technique, the presence of biological inhibitors or matrix effects.</p>

<h4>Technical failure rate</h4>

<h4>Desired</h4>

<p>Less than 0.2%</p>

<h4>Acceptable</h4>

<p>Less than 1%</p>

<h4>Comments</h4>

<p>In use failures resulting from mechanical, controls, calibration or other factors which may not be regarded as  reportable as an adverse event under the UK Medical Devices Regulations 2002 (SI 2002 No 618, as amended) (UK MDR 2002).</p>

<p>Does not include failure due to sample collection or technical issues outside of scope of test  (e.g house-keeping genes)</p>

<h4>Ease of use and result interpretation</h4>

<p>Suitable for target user groups  (i.e trained healthcare professionals)</p>

<h4>Need for calibration</h4>

<h4>Desired</h4>

<p>No calibration required</p>

<h4>Acceptable</h4>

<p>Remote or auto-calibration</p>

<h4>Identification capability</h4>

<h4>Desired</h4>

<p>Unique barcode or equivalent for integration into electronic systems</p>

<h4>Acceptable</h4>

<p>Labelling of the device with the subjects identification must be feasible</p>

<h4>Carry Over</h4>

<p>Bioinformatics approaches used to detect human-human sample contamination to monitor carryover.</p>

<h4>Comments</h4>

<p>The risk of carryover should be evaluated at each step of the assay.</p>

<h3>Performance characteristics</h3>

<h4>Clinical (diagnostic) sensitivity (or Positive Percent Agreement)</h4>

<h4>Desired</h4>

<p>Greater than 99% (with 95% two-sided confidence interval entirely above 97%).</p>

<h4>Acceptable</h4>

<p>Greater than 80% (with 95% two-sided confidence interval entirely above 70%).</p>

<h4>Comments</h4>

<p>At least 150 positive cases (per sample type). Greater sample numbers will be required to support higher performance claims.</p>

<p>Multiplex assays and assays with shorter turn-around-times or greater throughput may justify the lower 80% sensitivity requirement, in specific use cases. However, standard RNA extraction based RT-PCR assays should be achieving the Desirable performance.</p>

<h4>Clinical (diagnostic) specificity (or Negative Percent Agreement)</h4>

<h4>Desired</h4>

<p>Greater than 99% (with 95% two-sided confidence interval entirely above 97%).</p>

<h4>Acceptable</h4>

<p>Greater than 97% (with 95% two-sided confidence interval entirely above 93%).</p>

<h4>Comments</h4>

<p>A minimum of 250 COVID-19 negative cases (per sample type).</p>

<p>Testing should include all claimed specimen types and provide details of collection device and transport medium that have been validated for use with the assay.</p>

<h4>Clinical Performance Evaluation</h4>

<h4>Desired</h4>

<p>Positive and negative cases should be recruited prospectively and consecutively or randomly from the target population without prior knowledge of their disease status (e.g. single-gate design).</p>

<p>Studies directly comparing a new assay to the current state of the art test would be advantageous.</p>

<p>Test samples should be collected in the target setting at the same time as samples for the reference standard and any comparator methods.</p>

<p>In some cases the type of test or order of testing may need to be randomised.</p>

<p>Alternative study designs looking at the impact of using tests on disease spread (for tests to rule-in) or outbreaks (for tests to rule-out) should also be considered.</p>

<h4>Acceptable</h4>

<p>When prevalence is low, or in emergency use situations (e.g. a surge in cases of a new variant of Concern (VoC)), case-control designs using clinical samples with viral load distributions (determined by PCR) generalisable to the target population may be necessary.</p>

<p>Once in clinical use, manufacturers should immediately start to collect and make available in a timely way “Desirable” clinical performance evidence from post-market surveillance studies.</p>

<h4>Comments</h4>

<p>Two-gated case-control designs can introduce selection and spectrum bias and should be avoided if possible, refer to <a href="https://academic.oup.com/clinchem/article/51/8/1335/5629913" class="govuk-link" rel="external">Rutjes, 2005</a>. Claims made using such approaches (minimum criteria) are likely to overestimate the Clinical Performance.</p>

<p>Testing of all claimed specimen types should be performed with sufficient power, and details provided for collection devices and transport media that have been validated for use with the assay.</p>

<p>Practical recommendations for designing diagnostic accuracy studies in low prevalence settings can be found in <a class="govuk-link" rel="external" href="https://www.sciencedirect.com/science/article/abs/pii/S0895435618306619?via%3Dihub">Holtman, 2020</a>.</p>

<p>Further information on the design and conduct of clinical performance evaluations of SARs-CoV-2 tests can be found in <a rel="external" href="https://www.bmj.com/content/372/bmj.n568" class="govuk-link">Doust, 2021</a>.</p>

<p>Reporting of clinical performance evaluation studies should be in line with <a rel="external" href="https://academic.oup.com/clinchem/article/61/12/1446/5611598" class="govuk-link">STARD 2015</a>.  Results should include a diagram of participant flow, participant clinical and demographic characteristics including distibutions of disease severity/stage and alternative diagnoses, time intervals between index and reference tests,and a 2x2 table of results in addition to the measures of diagnostic performance with 95% confidence intervals.</p>

<p>All efforts should be made to establish the disease/infection status of a study participant. E.g. evidence of prior infection (describing test results and timing) and vaccination status (date of vaccination/s, vaccine).</p>

<h4>Clinical Reference Standard</h4>

<h4>Desired</h4>

<p>A composite clinical reference standard, against which the clinical sensitivity and specificity are calculated.</p>

<p>This could, if scientifically valid and appropriate for the defined context of use, include considerations of  immunity status, disease phase, virus characteristics, and infectivity.</p>

<h4>Acceptable</h4>

<p>A validated CE, CE UKNI or UKCA marked RNA extracted RT-PCR laboratory method in current clinical use that itself performs within the desirable analytical and clinical performance specifications of this TPP, against which the Negative/Positive Percent Agreement is calculated.</p>

<h4>Comments</h4>

<p>See the <a class="govuk-link" rel="external" href="https://www.nice.org.uk/Media/Default/About/what-we-do/covid-19/Diagnostic-tests-for-COVID-19-evidence-standards-framework.pdf">NICE evidence standards framework</a> for more information on composite clinical reference standards.</p>

<p>An example of a <a class="govuk-link" rel="external" href="https://www.cebm.net/covid-19/a-composite-reference-standard-for-covid-19-diagnostic-accuracy-studies-a-roadmap/">temporary Composite Reference Standard for COVID-19 can be found here</a>.</p>

<p>For samples with discordant results further testing could be done to try and explain the direction of discordance (for example, repeating the sample run on both tests or using a third method, if available).  But this should not influence claims of sensitivity and specificity.</p>

<h4>Analytical specificity</h4>

<h4>Desired</h4>

<p>No clinically relevant cross reactivity or interference to all organisms and agents listed in Annex 1.</p>

<p>Inclusive of all SARS-CoV-2 variants of concern. Demonstrated in silico and in vitro, where suitable reference materials are available.</p>

<h4>Acceptable</h4>

<p>No clinically relevant cross reactivity to common seasonal respiratory pathogens.</p>

<p>Minimal interference caused by common interferents at clinically relevant concentrations (dependant on sample type and analyte)</p>

<p>Inclusive of SARS-CoV-2 variants of concern. Demonstrated in silico and in vitro, where suitable reference materials are available.</p>

<h4>Comments</h4>

<p>Manufacturers should consider inclusivity, exclusivity, cross-reactions and exogenous/endogenous
interference.  See annex 1 for list.</p>

<p>In silico analyses including database searches should be performed to confirm the species specificity, avoiding the possibility of accidental cross reaction with human and bacteria.</p>

<h5>Limit of Detection (LOD)</h5>

<h4>Desired</h4>

<p>An appropriate unit of measurement for the target analyte (e.g. International Units) equivalent to a viral load of less than 100 SARS-CoV-2 RNA copies/mL of sample.</p>

<p>Standard RNA extraction based RT-PCR assays should be achieving the Desirable LoD performance.</p>

<h4>Acceptable</h4>

<p>An appropriate unit of measurement for the target analyte (e.g. International Units) equivalent to a viral load of less than 10,000 SARS-CoV-2 RNA copies/mL of sample.</p>

<p>Multiplex assays and assays with shorter turn-around-times or greater throughput may justify the acceptable LoD performance.</p>

<h4>Comments</h4>

<p>The LoD is the lowest concentration of analyte that can be consistently detected in ≥95% of samples tested under routine laboratory conditions and in the appropriate sample matrix. This concentration must yield an assay value that can be reproducibly distinguished from values obtained with samples that do not contain the analyte. Refer to appropriate standards (e.g. CLSI EP17) in the design of studies.  Ideally multiple batches/lots of kits/reagents should be used when establishing the LoD.</p>

<p>Where an appropriate International Standard,  reference material or reference measurement procedure is available for the analyte(s) this should be used.  <a rel="external" class="govuk-link" href="https://www.nibsc.org/science_and_research/idd/cfar/covid-19_reagents.aspx">For examples see here</a>.</p>

<p>If there are no comparative reference materials or measurement procedures available, evidence should be provided to demonstrate the choice of strategy for determining LOD is appropriate.</p>

<p>To demonstrate equivalence of the analyte(s) with viral load in copies/mL, the quantity value and measurement uncertainty of the clinical samples used should be assigned using an appropriate reference method (e.g. dPCR).  Commercially available quality control materials may not be value assigned with sufficeint accuracy to enable LoD evaluation.</p>

<p>The evaluation of LoD for some analytes using samples characterised in terms of RNA copies/mL may not be optimal and different criteria may need to be considered, if accompanied by sufficient evidence of scientific validity.</p>

<h4>Clinical impact</h4>

<h4>Desired</h4>

<p>Evidence that the test improves system and/or an individual’s outcomes (for example, time to diagnosis, subjects experience, use of pre-cautionary COVID-19 isolation facilities). 
Evidence that the test provides good value.</p>

<h4>Comments</h4>

<p>Refer to <a class="govuk-link" rel="external" href="https://www.nice.org.uk/Media/Default/About/what-we-do/covid-19/Diagnostic-tests-for-COVID-19-evidence-standards-framework.pdf">NICE evidence standards</a> for further information.</p>

<p>The purchase price/cost of a test alone may not be a good indicator of its valueNICE use the incremental cost effectiveness ratio (ICER) as their preferred measure of value.  Whilst NICE do not have a fixed threshold for cost-effectiveness, technologies exceeding £30,000/QALY have a higher probability of rejection and need to identify a strong case with regards to the certainty of evidence and innovative nature of the technology.</p>

<h4>Turnaround time</h4>

<p>Less than 90 minutes from sample to result.</p>

<h4>Acceptable</h4>

<p>Less than five hours from sample to result.</p>

<h4>Comments</h4>

<p>The time is from receipt of sample in test laboratory.</p>

<h5>Throughput</h5>

<h4>Desired</h4>

<p>More than 200 tests per machine/module every 4 hours.</p>

<h4>Acceptable</h4>

<p>More than 300 tests per machine/module every 24 hours.</p>

<h3>Test procedure characteristics</h3>

<h5>Hands-on time</h5>

<h4>Desired</h4>

<p>Less than five minutes per sample.</p>

<h4>Acceptable</h4>

<p>Less than 20 minutes per sample.</p>

<h4>Sample processing and handling</h4>

<p>Standardised sample-processing steps, using medical laboratory equipment (centrifuge, vortex, pipette etc).</p>

<h4>Biosafety</h4>

<h4>Desired</h4>

<p>Buffers or other components provided with kit or sample collection devices are proven to inactivate SARS-CoV-2 and any other respiratory viruses in scope of assay enabling entire process to be carried at CL2.</p>

<h4>Acceptable</h4>

<p>Requires inactivation pre-step at CL3 or CL2+. Remaining steps can be preformed at CL2.</p>

<h5>Comments</h5>

<p>Systems specifying need for lower biosafety environments must demonstrate sample inactivation and virus containment.
Instructions for use must confirm biosafety requirements for sample handling and inactivation evidence</p>

<p>Appropriate consideration should be given for biosafety in the environment which the test should be carried out.</p>

<h4>Risk in use</h4>

<p>Risks have been managed according to ISO 14971.</p>

<h3>Operational characteristics</h3>

<h5>Test kit storage and stability conditions</h5>

<h4>Desired</h4>

<p>No cold chain (15 to 30º C).</p>

<h4>Acceptable</h4>

<p>Storage of kit and reagents at 
-20ºC  and above for at least 12 months.  Stable for 3 hours once removed from cold storage.</p>

<h4>Comments</h4>

<p>Packaging to be as compact as possible to facilate storage. Must be made clear if reagents can withstand freeze and thaw and will not be detrimental to the assay.</p>

<h4>Assay end point stability (time window during which signal remains valid)</h4>

<h4>Desired</h4>

<p>Up to 1 hour.</p>

<h4>Acceptable</h4>

<p>Up to 30 minutes.</p>

<h4>Operating conditions</h4>

<p>15 to 30ºC.</p>

<h5>Connectivity</h5>

<h4>Desired</h4>

<p>Wireless and cable connectivity via LIMS systems.</p>

<h4>Acceptable</h4>

<p>Cable connectivity via LIMS system.</p>

<h4>Comments</h4>

<p>Results may need to be transferred by digital media e.g. CSV files.</p>

<h4>Presentation of results</h4>

<h4>Desired</h4>

<p>Easy to capture for interpretation and able to record public health data.</p>

<p>Results do not require post run analysis (automatically called). Ability to access raw data e.g. RT-qPCR/RT-LAMP fluorescence traces.</p>

<h4>Acceptable</h4>

<p>Easy to capture for interpretation and able to record public health data.</p>

<h4>Comments</h4>

<p>Integrity of data must be maintained.</p>

<h4>Reproducibility</h4>

<p>More than 95% between repeats at LoD.
More than 99% at higher concentrations.</p>

<h4>Comments</h4>

<p>Manufacturers should consider ISO 20395:2019 and ISO 5725-1 when evaluating reproducibility.</p>

<h4>Disposal requirements</h4>

<p>No additional disposal requirements beyond normal laboratory practice. Any special containment or disposal requirements need to be clearly specified (e.g. contamination control for RT-LAMP post amplification products).</p>

<h4>Training needs (Time dedicated to training session for end users)</h4>

<h4>Desired</h4>

<p>Less then half day training needed for for laboratory staff.</p>

<h4>Acceptable</h4>

<p>Less than one day of training needed.</p>

<h3>Other</h3>

<h4>Label and Instructions for Use</h4>

<p>Conforms to UK MDR 2002 and relevant designated standards.</p>

<h4>Regulatory status</h4>

<h4>Desired</h4>

<p>CE, CE UKNI or UKCA marked.</p>

<h4>Acceptable</h4>

<p>Exempt according to Regulation 12 of the UK MDR 2002.</p>

<h4>Comments</h4>

<p>For further information on the regulation of medical devices on the UK market, please see <a class="govuk-link" href="https://www.gov.uk/guidance/regulating-medical-devices-in-the-uk">our guidance</a>.</p>

<p>Specific <a href="https://www.gov.uk/government/publications/how-tests-and-testing-kits-for-coronavirus-covid-19-work/for-industry-and-manufactures-covid-19-tests-and-testing-kits#how-to-apply-to-mhra-for-a-derogation" class="govuk-link">guidance for manufacturers of COVID-19 tests can be found here</a>.</p>

<h4>Maintenance</h4>

<h4>Desired</h4>

<p>Preventive maintenance should not be needed until after 2 years or 100,000 samples. An alert should be included to indicate when maintenance is needed.</p>

<h4>Acceptable</h4>

<p>Preventive maintenance should not be needed until after 1 year
or
10,000 samples; an alert should be included to indicate when maintenance is needed.</p>

<h4>Comments</h4>

<p>Assuming the equipment is used at capacity 24 hours a day, seven days a week.</p>

<h4>Design and manufacturing environment</h4>

<p>ISO 13485:2016</p>

<h2>Annex 1: Assay validation</h2>

<h3>Establishing Performance Characteristics</h3>

<p>It is recommended that the following aspects are considered when designing and validating the assay. Analytical performance evaluations should test any multiplex molecular test system in its final configuration, and not in separate singleplex experiments.</p>

<ul>
  <li>Reference material should be used to establish performance, including standard validation panels, quality control materials and proficiency testing materials</li>
  <li>When establishing the performance of the test, manufacturers must consider the analytical sensitivity and specificity of the test to SARS-CoV-2 Variants Under Investigation (VUI) and of Concern (VOC) , including those listed on gov.uk by Public Health England). A full up to date list of variants can be found at <a class="govuk-link" rel="external" href="https://www.gisaid.org/">https://www.gisaid.org/</a>. Manufacturers are expected to routinely perform in silico analysis of listed VOC and VUI of gov.uk as a standard post market surveillance analysis. Where an assay is suspected or known to be affected by a listed VOC OR VUI, the manufacturer should inform MHRA within 48 hours of discovery.</li>
  <li>When establishing analytical specificity, the following should be considered:
    <ul>
      <li>Samples from patients who have received any licenced vaccine at several time points post vaccination ( e.g. &lt;5 days and 1-6 months).</li>
      <li>pre-pandemic samples,</li>
      <li>other coronavirus, SARS-CoV-1,</li>
      <li>MERS- coronavirus</li>
      <li>hCoV 229E, OC43, HKU1, NL63</li>
      <li>Adenovirus (e.g. C1 Ad. 71)</li>
      <li>Human Metapneumovirus (hMPV)</li>
      <li>Parainfluenza virus 1-4</li>
      <li>Influenza A &amp; B</li>
      <li>Enterovirus (e.g. EV68)</li>
      <li>Respiratory syncytial virus</li>
      <li>Rhinovirus</li>
      <li>Chlamydia pneumoniae</li>
      <li>Haemophilus influenzae</li>
      <li>Legionella pneumophila</li>
      <li>Mycobacterium tuberculosis</li>
      <li>Streptococcus pneumoniae</li>
      <li>Streptococcus pyogenes</li>
      <li>Bordetella pertussis</li>
      <li>Mycoplasma pneumoniae</li>
      <li>Pneumocystis jirovecii (PJP)</li>
    </ul>
  </li>
  <li>Potential interferents may originate from the following endogenous and exogenous sources and may be more relevant to ligand-binding based antigen tests than conventional PCR based assays. Manufacturers should declare if any other endogenous/ exogenous substances will impact the assay.
    <ul>
      <li>Antibacterial, systemic</li>
      <li>Antibiotic, nasal ointment</li>
      <li>Anti-viral drugs</li>
      <li>Antibodies developed against protein expression system used to generate recombinant antigens</li>
      <li>Bilirubin</li>
      <li>Biotin</li>
      <li>Blood (human)</li>
      <li>Haemoglobin</li>
      <li>Human Anti-mouse Antibody (HAMA)</li>
      <li>Medications most often prescribed in the population for which the test is ordered</li>
      <li>Mucin: bovine submaxillary gland, type I-S</li>
      <li>Nasal sprays or drops</li>
      <li>Nasal corticosteroids</li>
      <li>Nasal gel</li>
      <li>Protein</li>
      <li>Rheumatoid Factor</li>
      <li>Throat lozenges, oral anaesthetic and analgesic</li>
      <li>Triglycerides</li>
    </ul>
  </li>
</ul>

<p>In addition the assay must deliver across the clinical range of haematocrit  values.</p>

<h3>Post Market Performance considerations</h3>

<p>An increase in the number of cases with new SARS-CoV-2 variants has been observed in the United Kingdom. MHRA is aware that some laboratories have reported poor performance of some diagnostic assays that include an S-gene target. Such impact on test performance may be linked to the virus variant alpha or other variants. Mutations are not exclusive to the S-gene and action is required by manufacturers regardless of the diagnostic assay targets.</p>

<p>In line with UK MDR 2002 vigilance and field safety corrective action reporting requirements, MHRA consider reports relating to Variants of Concern (VOC) to be serious public health threats, therefore significant safety issues should be reported within 48h.</p>

<p>Actions specific to new VOCs:</p>

<ol>
  <li>Inform MHRA of the outcome of your initial risk assessment on the performance of your assay in light of identified variants of concern and your plan to mitigate against any new risks from mutations, including your timelines for addressing these.</li>
  <li>If the performance of your assay is directly impacted by new virus variant(s), a Field Safety Notice should be issued immediately to alert customers.</li>
  <li>A Post Market Surveillance plan (PMSP) should be in place to continuously monitor, investigate and assess newly emerging variants of SARS-CoV-2. The PMSP can include:</li>
</ol>

<p>a. Fortnightly  in silico checks of assay targets against GISAID sequence databases (https://www.gisaid.org) [*Please note that high profile potential issues should be immediately investigated]
  b. Scientific literature and post market intelligence gathering
  c. Outcomes of EQA schemes when available
  d. Use of reference materials when available
  e. Reporting potential safety issues of any new clinically significant variant SARS-CoV-2 strain on the performance of your assay to the MHRA</p>

<h4>Public Visibility of device assurance</h4>

<p>The MHRA intend to publish safety actions resulting from manufacturer in silico analysis and in vitro testing against variants of concern. The manufacturer should request if specific submitted information should not be made public. Public access to test device assurance will mutually benefit commercial suppliers and test device users.</p>

<h2>Annex 2: Diagnostic accuracy considerations</h2>

<p>When considering procurement and deployment of devices for any given clinical use-case, it is recommended to consider the maximum number of false positives and false negatives (Table 1) that would be acceptable for the new test based on the possible consequences of these misdiagnoses.  It is also helpful to consider the post-test probability that someone with a positive or negative result has infection (Table 2).  Programmes and testing services deploying self-tests into routine use should consider such information when determining where and when testing may be clinically/cost-effective and acceptable to end users.</p>

<p><strong>Table 1:</strong> The tables below presents the numbers of false positives and negatives in a cohort of fixed size (1,000,000) with varying prevalence of SARS-CoV-2 infection.</p>

<h3>Prevalence 20%</h3>
<p>Numbers per 10,000 tested</p>

<table>
  <thead>
    <tr>
      <td></td>
      <td style="text-align: right"></td>
      <th scope="col">Sensitivity</th>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th scope="col">Specificity</th>
      <th scope="col" style="text-align: right">Test result</th>
      <th scope="col">99%</th>
      <th scope="col">97%</th>
      <th scope="col">95%</th>
      <th scope="col">80%</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>99%</td>
      <td style="text-align: right">False +ves</td>
      <td>8000</td>
      <td>8000</td>
      <td>8000</td>
      <td>8000</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td style="text-align: right">False -ves</td>
      <td>2000</td>
      <td>6000</td>
      <td>10000</td>
      <td>40000</td>
    </tr>
    <tr>
      <td>95%</td>
      <td style="text-align: right">False +ves</td>
      <td>40000</td>
      <td>40000</td>
      <td>40000</td>
      <td>40000</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td style="text-align: right">False -ves</td>
      <td>2000</td>
      <td>6000</td>
      <td>10000</td>
      <td>40000</td>
    </tr>
  </tbody>
</table>

<h3>Prevalence 5%</h3>
<p>Numbers per 10,000 tested</p>

<table>
  <thead>
    <tr>
      <td></td>
      <td style="text-align: right"></td>
      <th scope="col">Sensitivity</th>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th scope="col">Specificity</th>
      <th style="text-align: right" scope="col">Test result</th>
      <th scope="col">99%</th>
      <th scope="col">97%</th>
      <th scope="col">95%</th>
      <th scope="col">80%</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>99%</td>
      <td style="text-align: right">False +ves</td>
      <td>9500</td>
      <td>9500</td>
      <td>9500</td>
      <td>9500</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td style="text-align: right">False -ves</td>
      <td>500</td>
      <td>1500</td>
      <td>2500</td>
      <td>10000</td>
    </tr>
    <tr>
      <td>95%</td>
      <td style="text-align: right">False +ves</td>
      <td>47500</td>
      <td>47500</td>
      <td>47500</td>
      <td>47500</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td style="text-align: right">False -ves</td>
      <td>500</td>
      <td>1500</td>
      <td>2500</td>
      <td>10000</td>
    </tr>
  </tbody>
</table>

<h3>Prevalence 1%</h3>
<p>Numbers per 10,000 tested</p>

<table>
  <thead>
    <tr>
      <td></td>
      <td style="text-align: right"></td>
      <th scope="col">Sensitivity</th>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th scope="col">Specificity</th>
      <th scope="col" style="text-align: right">Test result</th>
      <th scope="col">99%</th>
      <th scope="col">97%</th>
      <th scope="col">95%</th>
      <th scope="col">80%</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>99%</td>
      <td style="text-align: right">False +ves</td>
      <td>9900</td>
      <td>9900</td>
      <td>9900</td>
      <td>9900</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td style="text-align: right">False -ves</td>
      <td>100</td>
      <td>300</td>
      <td>500</td>
      <td>2000</td>
    </tr>
    <tr>
      <td>95%</td>
      <td style="text-align: right">False +ves</td>
      <td>49500</td>
      <td>49500</td>
      <td>49500</td>
      <td>49500</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td style="text-align: right">False -ves</td>
      <td>100</td>
      <td>300</td>
      <td>500</td>
      <td>2000</td>
    </tr>
  </tbody>
</table>

<h3>Prevalence 0.2%</h3>
<p>Numbers per 10,000 tested</p>

<table>
  <thead>
    <tr>
      <td></td>
      <td style="text-align: right"></td>
      <th scope="col">Sensitivity</th>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th scope="col">Specificity</th>
      <th style="text-align: right" scope="col">Test result</th>
      <th scope="col">99%</th>
      <th scope="col">97%</th>
      <th scope="col">95%</th>
      <th scope="col">80%</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>99%</td>
      <td style="text-align: right">False +ves</td>
      <td>9980</td>
      <td>9980</td>
      <td>9980</td>
      <td>9980</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td style="text-align: right">False -ves</td>
      <td>20</td>
      <td>60</td>
      <td>100</td>
      <td>400</td>
    </tr>
    <tr>
      <td>95%</td>
      <td style="text-align: right">False +ves</td>
      <td>49900</td>
      <td>49900</td>
      <td>49900</td>
      <td>49900</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td style="text-align: right">False -ves</td>
      <td>20</td>
      <td>60</td>
      <td>100</td>
      <td>400</td>
    </tr>
  </tbody>
</table>

<p><strong>Table 2:</strong> The following tables show the post-test probability of having an infection as the prevalence (or pre-test probability) changes. The percentage of people testing positive who are infected (Positive Predictive Value)) and the percentage of people testing negative who are infected (1-Negative Predictive Value) are shown for different prevalences from 0.2% to 20% (1 in 500 to 1 in 5). Programmes and testing services should determine their own acceptance criteria.</p>

<h4>Desirable Test</h4>

<p>Sensitivity 9%
Specificity 99%</p>

<table>
  <thead>
    <tr>
      <th scope="col">Prevalence</th>
      <th style="text-align: right" scope="col">0.2%</th>
      <th scope="col">0.5%</th>
      <th scope="col">1%</th>
      <th scope="col">2%</th>
      <th scope="col">5%</th>
      <th scope="col">10%</th>
      <th scope="col">20%</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>% +ves infected (PPV)</td>
      <td style="text-align: right">16.56%</td>
      <td>33.22%</td>
      <td>50%</td>
      <td>66.89%</td>
      <td>83.9%</td>
      <td>91.67%</td>
      <td>96.12%</td>
    </tr>
    <tr>
      <td>% -ves infected (1-NPV)</td>
      <td style="text-align: right">0.0020%</td>
      <td>0.0051%</td>
      <td>0.0101%</td>
      <td>0.0202%</td>
      <td>0.0505%</td>
      <td>0.1009%</td>
      <td>0.2016%</td>
    </tr>
  </tbody>
</table>

<h4>Acceptable Test</h4>

<p>Sensitivity 80%
Specificity 95%</p>

<table>
  <thead>
    <tr>
      <th scope="col">Prevalence</th>
      <th scope="col" style="text-align: right">0.2%</th>
      <th scope="col">0.5%</th>
      <th scope="col">1%</th>
      <th scope="col">2%</th>
      <th scope="col">5%</th>
      <th scope="col">10%</th>
      <th scope="col">20%</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>% +ves infected (PPV)</td>
      <td style="text-align: right">3.11%</td>
      <td>7.44%</td>
      <td>13.91%</td>
      <td>24.62%</td>
      <td>45.71%</td>
      <td>64%</td>
      <td>80%</td>
    </tr>
    <tr>
      <td>% -ves infected (1-NPV)</td>
      <td style="text-align: right">0.042%</td>
      <td>0.105%</td>
      <td>0.210%</td>
      <td>0.4193%</td>
      <td>1.0417%</td>
      <td>2.0619%</td>
      <td>4.0404%</td>
    </tr>
  </tbody>
</table>

<h2>Annex 3 Glossary</h2>

<ul>
  <li>BSL: Biological Safety Level</li>
  <li>CL: Containment level</li>
  <li>dPCR: Digital polymerase chain reaction</li>
  <li>IVD: in-vitro diagnostic</li>
  <li>LAMP: Loop-mediated isothermal amplification</li>
  <li>LIMS: Laboratory Information Management System</li>
  <li>LOD: Limit of Detection</li>
  <li>NAT: Nucleic acid testing</li>
  <li>PCR: Polymerase chain reaction</li>
  <li>PPE: Personal Protective Equipment</li>
  <li>qPCR: Quantitative polymerase chain reaction</li>
  <li>RNA: Ribonucleic acid</li>
  <li>RT: Reverse transcription</li>
  <li>VOC: Varients of concern</li>
</ul>

<p><strong>Analytical sensitivity:</strong> quotient of the change in an indication and the corresponding change in the value of a quantity being measured (ISO 15193)</p>

<p><strong>Analytical specificity:</strong> ability of a measurement procedure to determine solely the quantity it purports to measure (ISO 15193)</p>

<p><strong>Clinical (Diagnostic) Sensitivity:</strong> ability of an IVD examination procedure to identify the presence of a target marker(s) in individuals with a particular disease or condition</p>

<p><strong>Clinical (Diagnostic) Specificity:</strong> ability of an IVD examination procedure to recognise the absence of a target marker(s) in individuals with a particular disease or condition</p>

<p><strong>Positive Percent Agreement:</strong> the proportion of non-reference standard positive subjects in whom the new test is positive.</p>

<p><strong>Negative Percent Agreement:</strong> the proportion of non-reference standard negative subjects in whom the new test is negative</p>

<p><strong>Positive Predictive Value:</strong> the proportion of patients with positive test results who have the target condition (as determined by the reference standard)</p>

<p><strong>Negative Predictive Value:</strong> the proportion of patients with negative test results who do not have the target condition (as determined by the reference standard)</p>

</div>


</div>
</div>
  </div>

  <div class="sticky-element" data-sticky-element="">
    <a href="#contents" class="govuk-link app-c-back-to-top dont-print">
    <svg focusable="false" class="app-c-back-to-top__icon" xmlns="http://www.w3.org/2000/svg" height="17" width="13" viewBox="0 0 13 17">
      <path d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z" fill="currentColor"></path>
    </svg>
    Contents
</a>

    <div class="sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>