<main lang="en" class="html-publication" role="main">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          
<div class="gem-c-organisation-logo brand--department-of-health">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color" href="/government/organisations/department-of-health-and-social-care">
      <span class="gem-c-organisation-logo__name">Department<br>of Health &amp;<br>Social Care</span>
</a>
</div>
        </li>
        <li class="organisation-logos__logo">
          
<div class="gem-c-organisation-logo brand--department-of-health">
    <a href="/government/organisations/uk-health-security-agency" class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color">
      <span class="gem-c-organisation-logo__name">UK Health<br>Security<br>Agency</span>
</a>
</div>
        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  

<div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
      <span class="govuk-caption-xl gem-c-title__context">
    Guidance
  </span>

  
  <h1 class="gem-c-title__text govuk-heading-xl">
    COVID-19 test approval – step 2: process for desktop review
  </h1>
</div>
  <p class="publication-header__last-changed">Updated 20 January 2022</p>

  </header>



<div data-module="sticky-element-container" class="govuk-grid-row sidebar-with-body">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav role="navigation" data-module="gem-track-click" aria-label="Contents" class="gem-c-contents-list">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-category="contentsClicked" data-track-label="#overview-of-the-process" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 1" href="#overview-of-the-process" data-track-options="{&quot;dimension29&quot;:&quot;Overview of the process&quot;}">Overview of the process</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a href="#manufacturer-and-test-information" data-track-category="contentsClicked" data-track-label="#manufacturer-and-test-information" data-track-options="{&quot;dimension29&quot;:&quot;\n1.  Manufacturer and test information&quot;}" data-track-action="content_item 2" class="gem-c-contents-list__link govuk-link govuk-link--no-underline"><span class="gem-c-contents-list__number">1. </span><span class="gem-c-contents-list__numbered-text">Manufacturer and test information</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-label="#regulatory-status" data-track-category="contentsClicked" data-track-action="content_item 3" data-track-options="{&quot;dimension29&quot;:&quot;\n2.  Regulatory status&quot;}" href="#regulatory-status" class="gem-c-contents-list__link govuk-link govuk-link--no-underline"><span class="gem-c-contents-list__number">2. </span><span class="gem-c-contents-list__numbered-text">Regulatory status</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-options="{&quot;dimension29&quot;:&quot;\n3.  Intended use case&quot;}" href="#intended-use-case" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-label="#intended-use-case" data-track-category="contentsClicked" data-track-action="content_item 4"><span class="gem-c-contents-list__number">3. </span><span class="gem-c-contents-list__numbered-text">Intended use case</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a href="#test-performance" data-track-label="#test-performance" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-options="{&quot;dimension29&quot;:&quot;\n4.  Test performance&quot;}" data-track-category="contentsClicked" data-track-action="content_item 5"><span class="gem-c-contents-list__number">4. </span><span class="gem-c-contents-list__numbered-text">Test performance</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 6" data-track-options="{&quot;dimension29&quot;:&quot;\n5.  Biosafety&quot;}" data-track-label="#biosafety" data-track-category="contentsClicked" href="#biosafety" class="gem-c-contents-list__link govuk-link govuk-link--no-underline"><span class="gem-c-contents-list__number">5. </span><span class="gem-c-contents-list__numbered-text">Biosafety</span></a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img class="print-meta-data-licence" src="/assets/government-frontend/open-government-licence-min-e98134ae65ff5fe99a524d8c383350b3998f91c298dc24b36126e65f1656c578.png">
</p>
<p>
  © Crown copyright 2022
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/assessment-and-procurement-of-coronavirus-covid-19-tests/covid-19-test-approval-step-2-process-for-desktop-review
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
    
    
      <div class="govspeak">
<p>This guidance outlines how the government will undertake the desktop review of COVID-19 detection tests submitted for assessment under the Coronavirus Test Device Approvals Regulation. The Regulation requires manufacturers or distributors of molecular and antigen tests for COVID-19 to apply to the Department of Health and Social Care (<abbr title="Department of Health and Social Care">DHSC</abbr>) for approval to market their test in the UK. See <a class="govuk-link" href="https://www.gov.uk/guidance/covid-19-test-approval-how-to-apply">COVID-19 test approval: how to apply</a>.</p>

<p>Their product must meet the requirements of <a class="govuk-link" href="https://www.legislation.gov.uk/uksi/2021/910/contents/made" rel="external">The Medical Devices (Coronavirus Test Device Approvals) (Amendment) Regulations 2021</a>.</p>

<p>Molecular tests detect viral RNA or DNA and include polymerase chain reaction (<abbr title="polymerase chain reaction">PCR</abbr>), isothermal amplification and loop-mediated isothermal amplification (<abbr title="loop-mediated isothermal amplification">LAMP</abbr>) tests. Antigen tests directly detect viral proteins or components of the virus other than nucleic acids, and include those that require an analyser or reader and lateral flow tests that do not. All COVID-19 molecular and antigen tests are subject to approval, whether they are designed for use in a laboratory, at point of care, near patient or for home use (self-test).</p>

<h2>Overview of the process</h2>

<p>The desktop review is a systematic assessment of the evidence submitted by a supplier against a minimum required data set. The purpose of this step is to prevent tests that are below the expected standards from progressing to a technical validation in a laboratory.</p>

<p>The information submitted will be reviewed to check for completeness and then passed to a scientific advisor who will undertake the initial assessment. The assessment will be peer reviewed and presented to the Desktop Review Assurance Group meeting who will make a recommendation for pass or fail. All decisions will be ratified by the Regulatory Approvals Committee. The main areas of assessment are as follows:</p>

<ol>
  <li>Manufacturer and test information</li>
  <li>Regulatory status</li>
  <li>Intended use case</li>
  <li>Product performance</li>
  <li>Biosafety</li>
</ol>

<h2>
<span class="number">1. </span> Manufacturer and test information</h2>

<p>As a minimum the applicant should supply a Word or PDF document of the instructions for use (<abbr title="instructions for use">IFU</abbr>) for the submitted test. This document will be required to demonstrate the submitted test aligns with the intended use and evidence the stated performance characteristics of the submitted test. Several supporting documents in addition to the <abbr title="instructions for use">IFU</abbr> will also be required (see table below).</p>

<p>The submitted information will be assessed to ensure that the key documents for the application are present.</p>

<table>
  <thead>
    <tr>
      <th scope="col">Supplementary documents uploaded with application</th>
      <th scope="col">Assessment</th>
      <th scope="col">Response</th>
      <td></td>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Current version of <abbr title="instructions for use">IFU</abbr> (with date of publication). The   applicant must inform the <abbr title="Coronavirus Test Device Approval">CTDA</abbr> administration team at <a class="govuk-link" href="mailto:support@test-validation.test-and-trace.nhs.uk">support@test-validation.test-and-trace.nhs.uk</a>   if they make any changes to the <abbr title="instructions for use">IFU</abbr> for the submitted test whilst their   application for approval is being considered, and provide the updated <abbr title="instructions for use">IFU</abbr>   within 5 working days of publication</td>
      <td>Supplied or not supplied</td>
      <td>Automatic reject if not supplied</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>
<abbr title="instructions for use">IFU</abbr> for all comparator assays that were used to calculate   performance characteristics</td>
      <td>Supplied or not supplied</td>
      <td>Automatic reject if not supplied</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>Biosafety documents (including viral inactivation where a   claim of inactivation is stated in the <abbr title="instructions for use">IFU</abbr>)</td>
      <td>Not applicable, supplied or not supplied</td>
      <td>Automatic reject if not supplied where required</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>Evidence of performance characteristic including materials   and method used for limit of detection (<abbr title="limit of detection">LOD</abbr>) and raw cycle threshold (<abbr title="cycle threshold">CT</abbr>)   data per gene target for the comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr> results. For molecular tests that   provide quantitative results, provide the raw <abbr title="cycle threshold">CT</abbr> data (or equivalent) for the   submitted test. For antigen tests, values should be given in appropriate   units for semi-quantitative and quantitative tests, and as much data as   possible supplied for qualitative tests</td>
      <td>Supplied or not supplied</td>
      <td>Automatic reject if not supplied</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>Field safety notices (wherever issued) related to the   submitted test issued from 1 January 2020 to date of application, and any   pending notices</td>
      <td>Not applicable, supplied or not supplied</td>
      <td>SA to assess impact and resolution of issue</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>Regulatory certification, for example CE, <abbr title="in vitro diagnostic">IVD</abbr>, <abbr title="UK Conformity Assessed">UKCA</abbr>, <abbr title="Medicines and Healthcare products Regulatory Agency">MHRA</abbr>   derogation</td>
      <td>Supplied or not supplied</td>
      <td>Automatic reject if not supplied</td>
      <td>&nbsp;</td>
    </tr>
  </tbody>
</table>

<h2>
<span class="number">2. </span> Regulatory status</h2>

<h3>Regulatory approval</h3>

<table>
  <thead>
    <tr>
      <th scope="col">Desirable</th>
      <th scope="col">Minimum requirement</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>UK Conformity Assessed  (<abbr title="UK Conformity Assessed">UKCA</abbr>) or Certificate European (CE)</td>
      <td>Alternatives – health institution   exemptions or exceptional use authorisation (for example <abbr title="Medicines and Healthcare products Regulatory Agency">MHRA</abbr> derogation)</td>
    </tr>
  </tbody>
</table>

<p>From 1 July 2023, a <abbr title="UK Conformity Assessed">UKCA</abbr> mark will be required in order to place a device on the Great Britain market. Prior to this date, manufacturers can use the <abbr title="UK Conformity Assessed">UKCA</abbr> mark on a voluntary basis and the CE mark will continue to be accepted on the Great Britain market until 30 June 2023.</p>

<p>You will need to supply a copy of the certificate and any exemption reports.</p>

<p>The regulatory approval documentation will be assessed for the following:</p>

<table>
  <thead>
    <tr>
      <th scope="col">Review area</th>
      <th scope="col">Documentation</th>
      <th scope="col">Evidence</th>
      <th scope="col">Guidance notes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Regulatory status at time of application</td>
      <td>Regulatory certificate</td>
      <td>Automatic reject if not supplied</td>
      <td>The submitted test must have either CE, <abbr title="in vitro diagnostic">IVD</abbr>, <abbr title="UK Conformity Assessed">UKCA</abbr> or <abbr title="Medicines and Healthcare products Regulatory Agency">MHRA</abbr> exemption. Tests that do not meet this criterion will be automatically rejected</td>
    </tr>
  </tbody>
</table>

<h2>
<span class="number">3. </span> Intended use case</h2>

<p>The <abbr title="instructions for use">IFU</abbr> will be assessed for:</p>

<table>
  <thead>
    <tr>
      <th scope="col">Review area</th>
      <th scope="col">Documentation</th>
      <th scope="col">Evidence</th>
      <th scope="col">Guidance   notes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Intended use</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and publications<sup role="doc-noteref"><a class="govuk-link" rel="footnote" href="#fn:1">[footnote 1]</a></sup>
</td>
      <td>Automatic reject if not supplied</td>
      <td>Intended use case as described in the <abbr title="instructions for use">IFU</abbr>
</td>
    </tr>
    <tr>
      <td>Target population</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and publications</td>
      <td>Automatic reject if not supplied</td>
      <td>Please describe if the submitted test is intended for symptomatic and/or asymptomatic cases (that is people that are infected but do not display symptoms)</td>
    </tr>
    <tr>
      <td>Sample collection type and requirements</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and publications</td>
      <td>Automatic reject if not supplied</td>
      <td>Please state the specifications and product refence numbers for collection devices (including swabs, media and containers) that have been validated for use with your product</td>
    </tr>
    <tr>
      <td>Target use setting</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and publications</td>
      <td>Automatic reject if not supplied</td>
      <td>Please state if the submitted test is intended for use at point of care, near patient testing and/or laboratory based</td>
    </tr>
    <tr>
      <td>Target user</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and publications</td>
      <td>Automatic reject if not supplied</td>
      <td>Please state if the submitted test can be used by a trained healthcare professional; member of the allied health professions; trained lay-person and/or self-test</td>
    </tr>
    <tr>
      <td>Please state the time to result for the submitted test</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and publications</td>
      <td>Automatic reject if not supplied</td>
      <td>Time to result excludes delivery and receipt of samples but is inclusive of relaying of results into <abbr title="laboratory information management system">LIMS</abbr> system (where appropriate)</td>
    </tr>
  </tbody>
</table>

<h2>
<span class="number">4. </span> Test performance</h2>

<h3>Minimum evidence to support performance claim</h3>

<table>
  <thead>
    <tr>
      <th scope="col">Desirable</th>
      <th scope="col">Essential</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>150 clinical ‘positive’ samples across the full dynamic range of viral loads; 250 clinical ‘negative’ samples as determined by a comparator laboratory <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr>
</td>
      <td>100 clinical ‘positive’ samples across the full dynamic range of viral loads and 150 clinical ‘negative’ samples as determined by a comparator laboratory <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr>
</td>
    </tr>
  </tbody>
</table>

<p>The essential and desirable numbers of samples apply to each sample type, for example: nasal, nasopharyngeal, throat and saliva. Results for different sample types should not be aggregated. Approval will only be provided for sample types where sufficient data is provided for that sample type, and where that data demonstrates performance above the thresholds specified in the regulation. The clinical performance of the assessed product, for the purposes of the regulatory thresholds, must be assessed across the full dynamic range of viral loads.</p>

<p>The comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr> method must meet the diagnostic criteria from the guidance <a href="https://www.gov.uk/guidance/technologies-validation-group-using-tests-to-detect-covid-19" class="govuk-link">Technologies Validation Group: using tests to detect COVID-19</a>. This means it should be CE-marked, have a sensitivity of 97% or above with the lower 95% confidence interval at 93% or above, and a specificity of 99% or above with the lower 95% confidence interval at 97% or above.</p>

<p>Clinical diagnosis or contrived samples (for example those that have been spiked or diluted) are not acceptable comparator methods.</p>

<p>The raw data (<abbr title="cycle threshold">CT</abbr> values or equivalent) from the comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr> method are required in the supplementary evidence submitted via the portal (to evidence that the full dynamic range of clinically meaningful viral loads have been evaluated).</p>

<p>Where a test has already been validated by the Technologies Validation Group (TVG), the applicant can make a request to use the TVG data for the purposes of the <abbr title="Coronavirus Test Device Approval">CTDA</abbr> application. Please email your request for data to <a href="mailto:tvgdata@dhsc.gov.uk" class="govuk-link">tvgdata@dhsc.gov.uk</a>.</p>

<h3>Clinical performance</h3>

<p>The <abbr title="instructions for use">IFU</abbr>, relevant raw data and published supporting documents will be assessed for key requirements for evidence of clinical performance.</p>

<div class="call-to-action">
<p>The clinical performance characteristics of the submitted test. Performance characteristics must be based on the comparison of the submitted test with results obtained using an RT-qPCR method that meets diagnostic criteria from the guidance <a href="https://www.gov.uk/guidance/technologies-validation-group-using-tests-to-detect-covid-19" class="govuk-link">Technologies Validation Group: using tests to detect COVID-19</a>.</p>
</div>

<table>
  <thead>
    <tr>
      <th scope="col">Review area</th>
      <th scope="col">Documentation</th>
      <th scope="col">Response</th>
      <th scope="col">Guidance    notes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>The comparator assays name, test number and <abbr title="instructions for use">IFU</abbr> version</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and   publications</td>
      <td>Automatic reject without <abbr title="instructions for use">IFU</abbr>
</td>
      <td>The comparator assays must    meet the requirements of the ‘diagnostic criteria’ from the guidance <a class="govuk-link" href="https://www.gov.uk/guidance/technologies-validation-group-using-tests-to-detect-covid-19">Technologies   Validation Group: using tests to detect COVID-19</a>
</td>
    </tr>
    <tr>
      <td>Sample size of positive clinical samples</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and   publications</td>
      <td>Automatic reject if less than100</td>
      <td>Minimal acceptable number of samples is 100, desired   number is greater than or equal to 150</td>
    </tr>
    <tr>
      <td>Sample size for negative clinical samples</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and   publications</td>
      <td>Automatic reject is less than 150</td>
      <td>Minimal acceptable number of samples is 150, desired   number is greater than or equal to 250</td>
    </tr>
    <tr>
      <td>Viral load or <abbr title="cycle threshold">CT</abbr> values of positive samples obtained in   the comparator assays</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and   publications</td>
      <td>Automatic reject without raw data</td>
      <td>The <abbr title="cycle threshold">CT</abbr> or viral load of the positive clinical samples as   measured using the comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr> assays should span the dynamic range of   clinically meaningful viral loads. This means that positive samples should be   from people with high, high-medium, low-medium and low viral load approximate   to <abbr title="cycle threshold">CT</abbr> values less than 25, 25 to 30, 30 to 35, greater than 35; with no less   than 10% of samples in any one category and no more than 40% in the <abbr title="cycle threshold">CT</abbr> less   than 25 category) and represent the population that the test is intended to   be used in (such as symptomatic only)</td>
    </tr>
    <tr>
      <td>Please state the clinical sensitivity of the submitted   test determined by comparison with an <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr> assay and provide raw data to   evidence the statement at 10^6 to 10^5 copies per ml (approximate to <abbr title="cycle threshold">CT</abbr> less   than or equal to 25)</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and   publications</td>
      <td>Automatic reject without raw data</td>
      <td>This is the sensitivity of your test when compared to <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr>   for positive samples up to and including <abbr title="cycle threshold">CT</abbr> 25 (or equivalent copies/ml). Raw   data, at the per sample level, of the comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr> and submitted test are   required to assess the validity of claim</td>
    </tr>
    <tr>
      <td>Please state the clinical sensitivity of submitted test   determined by comparison with an <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr> assays and provide raw data to   evidence the statement across the dynamic range of the assay (approximate to <abbr title="cycle threshold">CT</abbr>   less than 25, 25 to 30, 30 to 35, greater than 35)</td>
      <td>-</td>
      <td>Automatic reject without raw data</td>
      <td>This is the sensitivity of your test when compared to the   comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr> for ALL positive samples. Raw data, at the per sample   level, of the comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr> and submitted test are required to assess   the validity of claim. These samples should cover the dynamic range of clinically   meaningful viral loads (that is should be from people with high, high-medium,   low-medium and low viral load approximate to <abbr title="cycle threshold">CT</abbr> less than 25, 25 to 30, 30 to   35, greater than 35; with no less than 10% of samples in any one category and   no more than 40% in the <abbr title="cycle threshold">CT</abbr> less than 25 category)</td>
    </tr>
    <tr>
      <td>Please state the clinical specificity of submitted test   and provide raw data to evidence the statement</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and   publications</td>
      <td>Automatic reject</td>
      <td>This is the specificity of your test when compared to   comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr> for all negative samples. Raw data, at the per sample   level, of the comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr> and submitted test are required to assess   the validity of claim</td>
    </tr>
    <tr>
      <td>What evidence do you have of the performance of submitted   test against contrived samples?</td>
      <td>Raw data and relevant published papers</td>
      <td>For information only</td>
      <td>Contrived samples could include diluted clinical samples   or negative clinical samples that have been spiked with positive control   material. This data is not a requirement but may be desirable as prevalence   changes impact on access to positive samples</td>
    </tr>
    <tr>
      <td>For molecular assays, please state the number of   SARS-CoV-2 targets in the submitted test and which genes they are associated   with</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and   publications</td>
      <td>Automatic reject if information is not provided</td>
      <td>These data are required to aid the assessment of your test</td>
    </tr>
  </tbody>
</table>

<h3>Analytical performance</h3>

<p>The <abbr title="instructions for use">IFU</abbr>, relevant raw data and published supporting documents will be assessed for key requirements for evidence of analytical performance.</p>

<div class="call-to-action">
<p>The analytical performance characteristics of the assay. Performance characteristics must be based on the comparison of the submitted test with results obtained using an RT-qPCR  method that meets diagnostic criteria from the guidance <a class="govuk-link" href="https://www.gov.uk/guidance/technologies-validation-group-using-tests-to-detect-covid-19">Technologies Validation Group: using tests to detect COVID-19</a>.</p>
</div>

<table>
  <thead>
    <tr>
      <th scope="col">Review area</th>
      <th scope="col">Documentation</th>
      <th scope="col">Response</th>
      <th scope="col">Guidance    notes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Please state the <abbr title="limit of detection">LOD</abbr> of your   submitted test preferably in copies per ml. Where <abbr title="instructions for use">IFU</abbr> states another unit, for example copies or reaction, please translate to copies/ml where possible. For some   tests other units may be applicable, that is <abbr title="tissue culture infectious dose">TCID</abbr><sub>50</sub>/ml</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and   publications</td>
      <td>Automatic rejection if do not state a limit of detection</td>
      <td>Please also include the materials and method used to determine the   limit of detection (for example run a quantified standard against the   comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr>)</td>
    </tr>
    <tr>
      <td>Please confirm in silico and where appropriate in vitro   testing exclusivity of the submitted test (for molecular tests only)</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and   publications</td>
      <td>Automatic rejection if not supplied</td>
      <td>Please provide details of in silico tests for primer and   probe sequences and any subsequent in vitro testing resulting from organisms   with identified in silico homology</td>
    </tr>
    <tr>
      <td>For molecular tests the applicant must provide evidence of   regular in silico analysis against SARS-CoV-2 <a href="https://www.gov.uk/government/publications/covid-19-variants-genomically-confirmed-case-numbers/variants-distribution-of-cases-data" class="govuk-link">variants   of concern or under investigation data</a> as published on GOV.UK. During the   period the application is being considered, the applicant must inform the   <abbr title="Coronavirus Test Device Approval">CTDA</abbr> administration team at <a href="mailto:support@test-validation.test-and-trace.nhs.uk" class="govuk-link">support@test-validation.test-and-trace.nhs.uk</a>   within 48 hours of confirmation that a variant of concern (<abbr title="variant of concern">VOC</abbr>) affects the   assay target regions by in silico analysis</td>
      <td>Please attach in silico evidence for variants of concern</td>
      <td>For information only</td>
      <td>Please provide details of in silico tests for primer and probe sequences against   variants of concern. These data are not a requirement but will aid the   assessment of your test. Please note this is a requirement from the <abbr title="Medicines and Healthcare products Regulatory Agency">MHRA</abbr> for   all COVID-19 tests</td>
    </tr>
    <tr>
      <td>For antigen-based assays, please state the protein targets   and regions</td>
      <td>Please attach protein target information</td>
      <td>Automatically reject if not provided</td>
      <td>Please provide details of your test design and target   regions, for example monoclonal, polyclonal, terminus and target protein. This   data is required to assess your test. Please note this is a requirement from   the <abbr title="Medicines and Healthcare products Regulatory Agency">MHRA</abbr> for all COVID-19 tests</td>
    </tr>
    <tr>
      <td>In vitro or in silico cross reactivity with common   respiratory targets</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and   publications</td>
      <td>Automatic rejection if not supplied</td>
      <td>Please provide raw data of in silico or in vitro studies   with clinical or contrived samples known to contain other respiratory viruses.   Examples include commercial <abbr title="external quality assessment">EQA</abbr> and control panels</td>
    </tr>
    <tr>
      <td>In vitro effect of common interferants</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and   publications</td>
      <td>Automatic rejection if not supplied</td>
      <td>Please provide raw data of any interfering substances that   have been tested in combination with the submitted test. Please see the   relevant <a class="govuk-link" href="https://www.gov.uk/government/publications/how-tests-and-testing-kits-for-coronavirus-covid-19-work"><abbr title="Medicines and Healthcare products Regulatory Agency">MHRA</abbr>   Target Product Profile</a> for the submitted test for the required checklist   of interfering substances</td>
    </tr>
  </tbody>
</table>

<h3>Technology performance thresholds for approval under <abbr title="Coronavirus Test Device Approval">CTDA</abbr>
</h3>

<p>The <abbr title="Coronavirus Test Device Approval">CTDA</abbr> approval process requires key mainstream technology groups to meet a minimum performance threshold reflecting standards identified for the technology area identified. These are set out below.</p>

<p>There is separate guidance on what level of test performance is required in different settings and for different purposes, see <a href="https://www.gov.uk/guidance/technologies-validation-group-using-tests-to-detect-covid-19" class="govuk-link">Technologies Validation Group: using tests to detect COVID-19</a>.</p>

<table>
  <thead>
    <tr>
      <th scope="col">Requirement</th>
      <th scope="col">PCR and isothermal (extracted)</th>
      <th scope="col">PCR and isothermal (direct)</th>
      <th scope="col">Antigen</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Minimum Performance category expectation*</td>
      <td>Sensitivity   with 95% 2-sided confidence interval entirely above 93%    <br>  <br>         Specificity   with 95% 2-sided confidence interval entirely above 97%</td>
      <td>Sensitivity   with 95% 2-sided confidence interval entirely above 70%   <br>  <br>          Specificity   with 95% 2-sided confidence interval entirely above 93%</td>
      <td>Sensitivity   with 95% 2-sided confidence interval entirely above 60%   <br>  <br>          Specificity   with 95% 2-sided confidence interval entirely above 93%</td>
    </tr>
  </tbody>
</table>

<p>*Setting a standard for technology types so government does not endorse tests that are substandard for their type.</p>

<p>Sensitivity in regard to a coronavirus test device means the percentage of coronavirus positive samples detected by the comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr> that are correctly identified by the test, calculated using the following equation.</p>

<figure class="image embedded"><div class="img"><img src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/129448/Sensitivity_formula.png" alt=""></div></figure>

<p>The above equation states the percentage sensitivity equals the number of true positives divided by the sum of the number of true positives and the number of false negatives, all multiplied by 100.</p>

<p>Specificity in regard to a coronavirus test device means the percentage of coronavirus negative samples classified by the comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr> that are correctly identified as negative by the test, calculated using the following equation.</p>

<figure class="image embedded"><div class="img"><img src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/129449/Specificity_formula.png" alt=""></div></figure>

<p>The above equation states the percentage specificity equals the number of true negatives divided by the sum of the number of true negatives and the number of false positives, all multiplied by 100.</p>

<p>True positive results are those that have a positive result on both the test device and the comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr>. False negative results are those that have a negative result on the test device and positive on the comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr>.</p>

<p>True negative results are those that have a negative result on both the test device and the comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr>. False positive results are those that have a positive result on the test device and a negative result on the comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr>.</p>

<table>
  <thead>
    <tr>
      <th scope="col">Lower 95% CI</th>
      <th scope="col">Positive result on comparator RT-qPCR</th>
      <th scope="col">Negative result on comparator RT-qPCR</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Positive result on submitted   test</td>
      <td>True positive</td>
      <td>False positive</td>
    </tr>
    <tr>
      <td>Negative result on submitted   test</td>
      <td>False negative</td>
      <td>True negative</td>
    </tr>
  </tbody>
</table>

<p>We will use the following formula<sup role="doc-noteref"><a rel="footnote" href="#fn:2" class="govuk-link">[footnote 2]</a></sup> for calculating 95% confidence intervals.</p>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/129450/95__CI_formula.png"></div></figure>

<p>The above equation states the lower 95% confidence interval equals A minus B, all divided by C. The upper 95% confidence interval equals A plus B, all divided by C, where:</p>

<ul>
  <li>A equals 2 r, plus 1.96 to the power of 2</li>
  <li>B equals 1.96 multiplied by the square root of D</li>
  <li>C equals n, plus 1.96 to the power of 2, all multiplied by 2</li>
  <li>D equals 1 minus p, all multiplied by 4r, plus 1.96 to the power of 2</li>
  <li>r equals the number of true positives (for sensitivity) or the number of true negatives (for specificity)</li>
  <li>n equals the number of positives on the comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr> (for sensitivity) or the number of negatives on the comparator <abbr title="quantitative reverse transcription polymerase chain reaction">RT-qPCR</abbr> (for specificity)</li>
  <li>p equals r divided by n (that is p is the sensitivity or specificity as a proportion)</li>
</ul>

<p>When calculating the percentage sensitivity and specificity and 95% confidence intervals, we will round to the nearest 0.1% for comparison with the thresholds. For example, a test with a lower 95% confidence interval that rounds to 92.9% or 93.0% would fail against a threshold of 93%, as the requirement is that the lower 95% <abbr title="confidence interval">CI</abbr> is entirely above 93%. For examples, see the table below.</p>

<table>
  <thead>
    <tr>
      <th style="text-align: right" scope="col">Lower 95% CI</th>
      <th style="text-align: right" scope="col">Rounded to the nearest 0.1%</th>
      <th scope="col" style="text-align: right">If compared to a threshold of</th>
      <th scope="col">Pass or fail</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: right">92.85%</td>
      <td style="text-align: right">92.9%</td>
      <td style="text-align: right">93%</td>
      <td>Fail</td>
    </tr>
    <tr>
      <td style="text-align: right">92.95%</td>
      <td style="text-align: right">93.0%</td>
      <td style="text-align: right">93%</td>
      <td>Fail</td>
    </tr>
    <tr>
      <td style="text-align: right">93.03%</td>
      <td style="text-align: right">93.0%</td>
      <td style="text-align: right">93%</td>
      <td>Fail</td>
    </tr>
    <tr>
      <td style="text-align: right">93.05%</td>
      <td style="text-align: right">93.1%</td>
      <td style="text-align: right">93%</td>
      <td>Pass</td>
    </tr>
    <tr>
      <td style="text-align: right">93.15%</td>
      <td style="text-align: right">93.2%</td>
      <td style="text-align: right">93%</td>
      <td>Pass</td>
    </tr>
  </tbody>
</table>

<table>
  <thead>
    <tr>
      <th scope="col">Review area</th>
      <th scope="col">Documentation</th>
      <th scope="col">Response</th>
      <th scope="col">Guidance notes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Technology   identified meets key performance requirement</td>
      <td>Supplier confirmation   or supporting documents</td>
      <td>Automatic reject   if not stated</td>
      <td>Confirmation that the   submitted test meets requirement</td>
    </tr>
    <tr>
      <td>Indicative   performance category</td>
      <td>None required</td>
      <td>-</td>
      <td>This is the result   of the assessment made by <abbr title="Department of Health and Social Care">DHSC</abbr> and will be included in the published results   on the <abbr title="Coronavirus Test Device Approval">CTDA</abbr> website</td>
    </tr>
  </tbody>
</table>

<h2>
<span class="number">5. </span> Biosafety</h2>

<p>Applicant must state the biosafety requirements of their submission.</p>

<p>Where the submitted assay includes the use of a reagent that claims to inactivate virus, the manufacturer must be able to evidence according to the <abbr title="British Standards Institution">BSI</abbr> safety standards and/or <a class="govuk-link" href="https://www.gov.uk/government/publications/position-statement-regarding-covid-19-tests-evaluated-by-phe/position-statement-on-inactivation-of-sars-cov-2-implications-for-laboratory-testing">PHE position statement on inactivation</a>.</p>

<p>The <abbr title="instructions for use">IFU</abbr> and any clearly identified supporting documents will be assessed for:</p>

<table>
  <thead>
    <tr>
      <th scope="col">Review area</th>
      <th scope="col">Documentation</th>
      <th scope="col">Response</th>
      <th scope="col">Guidance   notes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Please state the biosafety containment requirements for   your submission</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and   publications</td>
      <td>Automatic reject if not supplied</td>
      <td>The biosafety of the submitted test will impact the   use-case scenario in which the submitted test can be utilised</td>
    </tr>
    <tr>
      <td>Does the submitted test include a viral inactivation step?</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and   publications</td>
      <td>Automatic reject if not supplied</td>
      <td>Please highlight where this is claimed within the <abbr title="instructions for use">IFU</abbr>
</td>
    </tr>
    <tr>
      <td>If yes, have you provided documented evidence. Please   provide documented efficacy evidence according to <abbr title="British Standards Institution">BSI</abbr> standards</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and   publications</td>
      <td>Automatic reject if not supplied if answer above is yes</td>
      <td>Viral inactivation statements are not acceptable without   documented evidence. Equivalence to similar products are not acceptable     British Standard for quantitative suspension tests for   evaluation of virucidal activity, which requires that disinfectant products   demonstrate a titre reduction of at least 4 log (to base 10) in virus titre     <br>  <br>        Please provide data to support the viral inactivation   claim.            See <a href="https://www.gov.uk/government/publications/position-statement-regarding-covid-19-tests-evaluated-by-phe/position-statement-on-inactivation-of-sars-cov-2-implications-for-laboratory-testing" class="govuk-link">Position   statement on inactivation of SARS-Cov-2: implications for laboratory testing</a> <br>      <br>      <a class="govuk-link" href="https://www.gov.uk/government/publications/wuhan-novel-coronavirus-guidance-for-clinical-diagnostic-laboratories/wuhan-novel-coronavirus-handling-and-processing-of-laboratory-specimens">COVID-19:   safe handling and processing for samples in laboratories</a>
</td>
    </tr>
    <tr>
      <td>If your submission does not include inactivation of   clinical samples, can the submitted test be used outside of a laboratory   setting</td>
      <td>Please attach and highlight relevant reference in <abbr title="instructions for use">IFU</abbr> and   publications</td>
      <td>Automatic reject if not supplied</td>
      <td>Please provide evidence of sample preparation and   processing method, such as non-aerosol creating, need for full <abbr title="personal protective equipment">PPE</abbr> to allow scientific   advisor to assess this claim</td>
    </tr>
  </tbody>
</table>

<div role="doc-endnotes" class="footnotes">
  <ol>
    <li role="doc-endnote">
      <p>Publications of interest are scientific or medical journal articles that have been peer reviewed prior to publication. Publications that have not been peer-reviewed are acceptable but should be labelled as non-peer reviewed.&nbsp;<a role="doc-backlink" href="#fnref:1" class="govuk-link" aria-label="go to where this is referenced">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Wilson E.B. (1927) <a rel="external" class="govuk-link" href="https://www.tandfonline.com/doi/abs/10.1080/01621459.1927.10502953">Probable inference, the law of succession, and statistical inference</a>. Journal of the American Statistical Association  22: 209 to 212 <a class="govuk-link" href="https://doi.org/10.2307/2276774" rel="external">https://doi.org/10.2307/2276774</a>. As described in Altman D.G., Machin D., Bryant T.N. &amp; Gardner M.J. (2000) Statistics with Confidence, 2nd edition. BMJ Books, London.&nbsp;<a href="#fnref:2" role="doc-backlink" aria-label="go to where this is referenced" class="govuk-link">↩</a></p>
    </li>
  </ol>
</div>
</div>


</div>
</div>
  </div>

  <div class="sticky-element" data-sticky-element="">
    <a href="#contents" class="govuk-link app-c-back-to-top dont-print">
    <svg class="app-c-back-to-top__icon" width="13" viewBox="0 0 13 17" xmlns="http://www.w3.org/2000/svg" height="17" focusable="false">
      <path d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z" fill="currentColor"></path>
    </svg>
    Contents
</a>

    <div class="sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>