<main class="html-publication" lang="en" role="main">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          
<div class="gem-c-organisation-logo brand--department-of-health">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color" href="/government/organisations/uk-health-security-agency">
      <span class="gem-c-organisation-logo__name">UK Health<br>Security<br>Agency</span>
</a>
</div>
        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  

<div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
      <span class="govuk-caption-xl gem-c-title__context">
    Transparency data
  </span>


  <h1 class="gem-c-title__text govuk-heading-xl">
    EMHP wastewater monitoring of SARS-CoV-2 in England: 15 July 2020 to 30 March 2022
  </h1>
</div>
  <p class="publication-header__last-changed">Published 12 May 2022</p>

  </header>



<div class="govuk-grid-row sidebar-with-body" data-module="sticky-element-container">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav aria-label="Contents" class="gem-c-contents-list" data-module="gem-track-click" role="navigation">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 1" data-track-category="contentsClicked" data-track-label="#background-on-the-environmental-monitoring-for-health-protection-emhp-wastewater-monitoring-programme" data-track-options="{&quot;dimension29&quot;:&quot;Background on the Environmental Monitoring for Health Protection (EMHP) wastewater monitoring programme&quot;}" href="#background-on-the-environmental-monitoring-for-health-protection-emhp-wastewater-monitoring-programme">Background on the Environmental Monitoring for Health Protection (EMHP) wastewater monitoring programme</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 2" data-track-category="contentsClicked" data-track-label="#emhp-sampling-sites" data-track-options="{&quot;dimension29&quot;:&quot;\nEMHP sampling sites&quot;}" href="#emhp-sampling-sites">
EMHP sampling sites</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 3" data-track-category="contentsClicked" data-track-label="#concentration-of-sars-cov-2-rna-in-wastewater-samples-15-july-2020-to-31-may-2021" data-track-options="{&quot;dimension29&quot;:&quot;Concentration of SARS-CoV-2 RNA in wastewater samples: 15 July 2020 to 31 May 2021&quot;}" href="#concentration-of-sars-cov-2-rna-in-wastewater-samples-15-july-2020-to-31-may-2021">Concentration of SARS-CoV-2 RNA in wastewater samples: 15 July 2020 to 31 May 2021</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 4" data-track-category="contentsClicked" data-track-label="#concentration-of-sars-cov-2-rna-in-wastewater-samples-1-june-2021-to-30-march-2022" data-track-options="{&quot;dimension29&quot;:&quot;Concentration of SARS-CoV-2 RNA in wastewater samples: 1 June 2021 to 30 March 2022&quot;}" href="#concentration-of-sars-cov-2-rna-in-wastewater-samples-1-june-2021-to-30-march-2022">Concentration of SARS-CoV-2 RNA in wastewater samples: 1 June 2021 to 30 March 2022</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 5" data-track-category="contentsClicked" data-track-label="#regional-average-concentration-of-sars-cov-2-rna-in-wastewater-1-june-2021-to-30-march-2022" data-track-options="{&quot;dimension29&quot;:&quot;Regional average concentration of SARS-CoV-2 RNA in wastewater: 1 June 2021 to 30 March 2022&quot;}" href="#regional-average-concentration-of-sars-cov-2-rna-in-wastewater-1-june-2021-to-30-march-2022">Regional average concentration of SARS-CoV-2 RNA in wastewater: 1 June 2021 to 30 March 2022</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 6" data-track-category="contentsClicked" data-track-label="#detection-of-sars-cov-2-variants-and-mutations-in-wastewater-samples-1-may-2021-to-27-march-2022" data-track-options="{&quot;dimension29&quot;:&quot;Detection of SARS-CoV-2 variants and mutations in wastewater samples: 1 May 2021 to 27 March 2022&quot;}" href="#detection-of-sars-cov-2-variants-and-mutations-in-wastewater-samples-1-may-2021-to-27-march-2022">Detection of SARS-CoV-2 variants and mutations in wastewater samples: 1 May 2021 to 27 March 2022</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 7" data-track-category="contentsClicked" data-track-label="#detection-of-the-omicron-variant-in-wastewater-samples-22-november-2021-to-27-march-2022" data-track-options="{&quot;dimension29&quot;:&quot;Detection of the Omicron variant in wastewater samples: 22 November 2021 to 27 March 2022&quot;}" href="#detection-of-the-omicron-variant-in-wastewater-samples-22-november-2021-to-27-march-2022">Detection of the Omicron variant in wastewater samples: 22 November 2021 to 27 March 2022</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 8" data-track-category="contentsClicked" data-track-label="#methodology" data-track-options="{&quot;dimension29&quot;:&quot;Methodology&quot;}" href="#methodology">Methodology</a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img class="print-meta-data-licence" src="/assets/government-frontend/open-government-licence-min-93b6a51b518ff99714a1aa2a7d2162735c155ec3cb073c75fb88b2a332fa83d3.png">
</p>
<p>
  Â© Crown copyright 2022
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/monitoring-of-sars-cov-2-rna-in-england-wastewater-monthly-statistics-15-july-2020-to-30-march-2022/emhp-wastewater-monitoring-of-sars-cov-2-in-england-15-july-2020-to-30-march-2022
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
    
    
      <div class="govspeak">
<p>Data showing the concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> and variants of concern (<abbr title="variant of concern">VOC</abbr>) and variants under investigation (<abbr title="variant under investigation">VUI</abbr>) detected in wastewater by the Environmental Monitoring for Health Protection (<abbr title="Environmental Monitoring for Health Protection">EMHP</abbr>) wastewater monitoring programme.</p>

<p>The <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> SARS-CoV-2 wastewater monitoring programme was scaled down and paused at the end of March 2022. Wastewater Based Epidemiology is likely to be of significant importance in the future, both domestically and internationally, and we are exploring further opportunities for this technology.</p>

<p>These publications are therefore being discontinued. Transparency data covering the time period up to the end of March 2022 is included in this publication after which regular publications will cease. Any future ad-hoc transparency publications will be announced in advance on this page and on the Statistics at <abbr title="UK Health Security Agency">UKHSA</abbr> page.</p>

<div aria-label="Information" class="application-notice info-notice" role="note">
<p>Note: As of 1 April 2022, previous variants of concern which no longer meet the criteria for VOC label according to the UK Health Security Agency (UKHSA) variant classification system have been redesignated. New variants of note based on genomic features, vaccine escape and growth, now receive a variant number (V-date-number).</p>
</div>

<h2>Background on the Environmental Monitoring for Health Protection (<abbr title="Environmental Monitoring for Health Protection">EMHP</abbr>) wastewater monitoring programme</h2>

<p>People infected with coronavirus (COVID-19) shed the virus during daily activities such as going to the toilet and blowing their nose. The virus enters the sewer system through sinks, drains and toilets. Fragments of SARS-CoV-2 (the virus that causes COVID-19) can be detected in samples of wastewater (untreated sewage).</p>

<p>The <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> programme is led by the <abbr title="UK Health Security Agency">UKHSA</abbr> and run in partnership with the Department for Environment, Food and Rural Affairs (Defra), the Environment Agency (EA), the Centre for Environment, Fisheries and Aquaculture Science (Cefas), academia, and water companies. It provides coverage of approximately 74% of the population in England. The <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> team coordinates with programmes in the devolved administrations to provide UK-wide wastewater monitoring.</p>

<p>The programme tests sewage for fragments of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr>. Untreated influent samples are taken from approximately 302 Sewage Treatment Works in England, 3 times a week (see figures 1 to 9). <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> also sample from sewer network sites (manholes), untreated influent samples are taken from approximately 197 sewer network sites in England, 4 times a week.</p>

<p>Data from sewer network sites are included in the âDetection of the Omicron variant in wastewater samples; 22 November 2021 to 27 March 2022â section of this publication to ensure continuity with previous publications. Data from sewer network sites are not included in the rest of this publication as these sites generally fall within <abbr title="sewage treatment work">STW</abbr> catchments.</p>

<p>As the threat of variants has emerged, the programme has played a key role in the detection of mutations of the virus, including those associated with known variants of concern (<abbr title="variant of concern">VOC</abbr>), and variants under investigation (<abbr title="variant under investigation">VUI</abbr>). This is achieved through genomic sequencing of wastewater samples, to provide an indication of where <abbr title="variants of concern">VOCs</abbr> and <abbr title="variants under investigation">VUIs</abbr> may be present across England.</p>

<p>Findings are reported to national decision makers and local stakeholders on a regular basis, helping to inform national strategy and localised action. Wastewater analysis has the benefit of detecting the virus regardless of whether people have symptoms or whether they are tested. Wastewater monitoring complements other testing programmes and public health actions to help protect against the threat of new variants.</p>

<p>The data presented in this publication are generated by non-accredited research laboratories and should be considered experimental. Formal validation of wastewater monitoring as a variant surveillance system is underway. <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> advise caution when directly comparing data from Sewage Treatment Works and concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> in wastewater with clinical testing data. Wastewater detections of a variant are reported as either confirmed or possible, based on the number of signature or unique <abbr title="single nucleotide polymorphisms">SNPs</abbr>, and co-occurrence on the same amplicon.</p>

<p>This describes the level of confidence <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> has in the specificity of the detection; âconfirmedâ status is assigned when there is a high degree of confidence the variant was present in the wastewater sample, âpossibleâ is assigned where there is some evidence that the variant was present.</p>

<p>These definitions are different to the clinical case definitions and the 2 should not be compared. Lack of a positive wastewater sequencing result is not necessarily confirmation that no variant cases reside within a given geography, due to factors such as rainfall dilution impacting <abbr title="ribonucleic acid">RNA</abbr> recovery.</p>

<p>Interpretation of all data in this publication should be made with careful consideration of the points in the âUncertainty, data quality, and revisionsâ section.</p>

<h2>
<abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> sampling sites</h2>

<p><abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> sample at sewer network sites (manholes in the street which generally serve a local area) and Sewage Treatment Works (which generally serve wider areas such as a city or town). Associated with every Sewage Treatment Works and sewer network site is a catchment, the geographical area from which wastewater flows into the sampling location. <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> concentrations data from sewer network sites are not included in this publication as these sites generally fall within <abbr title="sewage treatment work">STW</abbr> catchments.</p>

<p>In addition, <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> change sampling strategy and locations of sewer network sites to meet epidemiological priorities (for example to support local public health responses). Further detail about <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> wastewater coverage can be found in this <a class="govuk-link" href="https://www.gov.uk/government/publications/wastewater-testing-coverage-data-for-23-february-2022-emhp-programme">publication</a>.</p>

<p>Figures 1 to 9 show the location of Sewage Treatment Works that <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> sample at in each region.</p>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149587/East_Midlands_May_2022_STW_Locations_Only.png"></div>
<figcaption><p>Figure 1. Map showing the location of Sewage Treatment Works sampled in the East Midlands.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in the East Midlands:</p>

<ul>
  <li>Alfreton</li>
  <li>Anwick</li>
  <li>Blaby</li>
  <li>Boston</li>
  <li>Bourne</li>
  <li>Brackley</li>
  <li>Chesterfield</li>
  <li>Coalville</li>
  <li>Corby</li>
  <li>Daventry</li>
  <li>Derby</li>
  <li>Eastwood</li>
  <li>Gainsborough</li>
  <li>Grantham</li>
  <li>Hinckley</li>
  <li>Ilkeston</li>
  <li>Ingoldmells</li>
  <li>Leicester</li>
  <li>Lincoln</li>
  <li>Long Eaton</li>
  <li>Loughborough</li>
  <li>Louth</li>
  <li>Mablethorpe</li>
  <li>Market Harborough</li>
  <li>Mansfield</li>
  <li>Melton Mowbray</li>
  <li>Northampton</li>
  <li>Nottingham</li>
  <li>Oakham</li>
  <li>Retford</li>
  <li>Spalding</li>
  <li>Stamford</li>
  <li>Wellingborough</li>
  <li>Whaley Bridge</li>
  <li>Wigston</li>
  <li>Worksop</li>
</ul>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149588/East_of_England_May_2022_STW_Locations_Only.png"></div>
<figcaption><p>Figure 2. Map showing the location of Sewage Treatment Works sampled in the East of England.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in the East of England:</p>

<ul>
  <li>Basildon</li>
  <li>Basildon (Vange)</li>
  <li>Beccles</li>
  <li>Bedford</li>
  <li>Braintree</li>
  <li>Breckland</li>
  <li>Bury St Edmunds</li>
  <li>Cambridge</li>
  <li>Chalton</li>
  <li>Chelmsford</li>
  <li>Colchester</li>
  <li>Cromer</li>
  <li>Diss</li>
  <li>Downham Market</li>
  <li>Felixstowe</li>
  <li>Great Yarmouth</li>
  <li>Harlow</li>
  <li>Haverhill</li>
  <li>Hunstanton</li>
  <li>Huntingdon</li>
  <li>Ipswich</li>
  <li>Ipswich (Chantry)</li>
  <li>Jaywick</li>
  <li>Kings Lynn</li>
  <li>Letchworth</li>
  <li>Lowestoft</li>
  <li>Luton</li>
  <li>March</li>
  <li>Needham Market</li>
  <li>Newmarket</li>
  <li>Norwich</li>
  <li>Peterborough</li>
  <li>Radlett</li>
  <li>Reepham</li>
  <li>Rochford</li>
  <li>Royston</li>
  <li>Saffron Walden</li>
  <li>Shefford</li>
  <li>Soham</li>
  <li>St Albans</li>
  <li>Stalham</li>
  <li>Stowmarket</li>
  <li>Sudbury</li>
  <li>Southend-on-Sea</li>
  <li>Thetford</li>
  <li>Tilbury</li>
  <li>Wells-next-the-Sea</li>
  <li>Wickford</li>
  <li>Wisbech</li>
  <li>Witham</li>
</ul>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149590/London_May_2022_STW_Locations_Only.png"></div>
<figcaption><p>Figure 3. Map showing the location of Sewage Treatment Works sampled in London.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in London:</p>

<ul>
  <li>Beckton</li>
  <li>Beddington</li>
  <li>Crossness</li>
  <li>Deepham</li>
  <li>Hogsmill Valley</li>
  <li>Long Reach</li>
  <li>Mogden</li>
  <li>Riverside</li>
</ul>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149591/North_East_May_2022_STW_Locations_Only.png"></div>
<figcaption><p>Figure 4. Map showing the location of Sewage Treatment Works sampled in the North East.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in the North East:</p>

<ul>
  <li>Billingham</li>
  <li>Bishop Auckland</li>
  <li>Blyth</li>
  <li>Consett</li>
  <li>Cramlington</li>
  <li>Darlington</li>
  <li>Durham (Barkers Haugh)</li>
  <li>Durham (Belmont)</li>
  <li>Hartlepool</li>
  <li>Horden</li>
  <li>Houghton-le-Spring</li>
  <li>Marske Redcar</li>
  <li>Middlesbrough</li>
  <li>Newcastle</li>
  <li>Newton Aycliffe</li>
  <li>Seaham</li>
  <li>Sunderland</li>
  <li>Washington</li>
</ul>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149592/North_West_May_2022_STW_Locations_Only.png"></div>
<figcaption><p>Figure 5. Map showing the location of Sewage Treatment Works sampled in the North West.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in the North West:</p>

<ul>
  <li>Barnoldswick</li>
  <li>Barrow-in-Furness</li>
  <li>Birkenhead</li>
  <li>Blackburn</li>
  <li>Bolton</li>
  <li>Bromborough</li>
  <li>Burnley</li>
  <li>Bury</li>
  <li>Carlisle</li>
  <li>Clitheroe</li>
  <li>Congleton</li>
  <li>Crewe</li>
  <li>Ellesmere Port</li>
  <li>Fleetwood</li>
  <li>Huyton and Prescot</li>
  <li>Hyde</li>
  <li>Hyndburn</li>
  <li>Kendal</li>
  <li>Lancaster</li>
  <li>Leigh</li>
  <li>Liverpool (Sandon)</li>
  <li>Macclesfield</li>
  <li>Maghull</li>
  <li>Northwich</li>
  <li>Penrith</li>
  <li>Preston</li>
  <li>Rochdale</li>
  <li>Rossendale</li>
  <li>Skelmersdale</li>
  <li>St. Helens</li>
  <li>Stockport</li>
  <li>Walton-Le-Dale</li>
  <li>Warrington</li>
  <li>Wigan</li>
  <li>Wirral</li>
  <li>Workington</li>
</ul>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149593/South_East_May_2022_STW_Locations_Only.png"></div>
<figcaption><p>Figure 6. Map showing the location of Sewage Treatment Works sampled in the South East.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in the South East:</p>

<ul>
  <li>Andover</li>
  <li>Alton</li>
  <li>Ashford</li>
  <li>Aylesbury</li>
  <li>Banbury</li>
  <li>Basingstoke</li>
  <li>Bexhill</li>
  <li>Bicester</li>
  <li>Bordon</li>
  <li>Buckingham</li>
  <li>Burgess Hill</li>
  <li>Camberley</li>
  <li>Canterbury</li>
  <li>Chatham</li>
  <li>Chesham</li>
  <li>Chichester</li>
  <li>Crawley</li>
  <li>Didcot</li>
  <li>Dover and Folkestone</li>
  <li>Eastleigh</li>
  <li>Fareham and Gosport</li>
  <li>Guildford</li>
  <li>Hailsham</li>
  <li>Hythe</li>
  <li>Isle of Wight</li>
  <li>Lewes</li>
  <li>Littlehampton and Bognor</li>
  <li>Lymington and New Milton</li>
  <li>Maidstone and Aylesford</li>
  <li>Milton Keynes</li>
  <li>Newbury</li>
  <li>New Forest</li>
  <li>Oxford</li>
  <li>Portsmouth and Havant</li>
  <li>Ramsgate, Sandwich and Deal</li>
  <li>Reading</li>
  <li>Reigate</li>
  <li>Scaynes Hill</li>
  <li>Sittingbourne</li>
  <li>Slough</li>
  <li>Southampton</li>
  <li>Tonbridge</li>
  <li>Tunbridge Wells</li>
  <li>Witney</li>
  <li>Woking</li>
  <li>Worthing</li>
  <li>Wycombe</li>
</ul>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149594/South_West_May_2022_STW_Locations_Only.png"></div>
<figcaption><p>Figure 7. Map showing the location of Sewage Treatment Works sampled in the South West.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in the South West:</p>

<ul>
  <li>Barnstaple</li>
  <li>Bath</li>
  <li>Bideford</li>
  <li>Blandford Forum</li>
  <li>Bodmin Sc Well</li>
  <li>Bournemouth (Central)</li>
  <li>Bridport</li>
  <li>Bristol</li>
  <li>Camborne</li>
  <li>Chard</li>
  <li>Cheltenham</li>
  <li>Chippenham</li>
  <li>Cirencester</li>
  <li>Clevedon and Nailsea</li>
  <li>Ernesettle and Saltash</li>
  <li>Exmouth</li>
  <li>Falmouth</li>
  <li>Gloucester</li>
  <li>Helston</li>
  <li>Liskeard</li>
  <li>Lydney</li>
  <li>Menagwins</li>
  <li>Minehead</li>
  <li>Newquay</li>
  <li>Newton Abbot</li>
  <li>Par</li>
  <li>Plymouth (Camels Head)</li>
  <li>Plymouth</li>
  <li>Plympton</li>
  <li>Salisbury</li>
  <li>Shaftesbury</li>
  <li>Shepton Mallet</li>
  <li>Sidmouth</li>
  <li>St Ives and Penzance</li>
  <li>Stroud</li>
  <li>Swanage</li>
  <li>Swindon</li>
  <li>Taunton</li>
  <li>Tavistock</li>
  <li>Tiverton</li>
  <li>Torquay</li>
  <li>Trowbridge</li>
  <li>Wellington</li>
  <li>Weston-super-Mare</li>
  <li>Weymouth</li>
  <li>Yeovil</li>
</ul>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149595/West_Midlands_May_2022_STW_Locations_Only.png"></div>
<figcaption><p>Figure 8. Map showing the location of Sewage Treatment Works sampled in the West Midlands.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in the West Midlands:</p>

<ul>
  <li>Barston</li>
  <li>Brancote</li>
  <li>Checkley</li>
  <li>Birmingham (Coleshill)</li>
  <li>Birmingham (Minworth)</li>
  <li>Burton on Trent</li>
  <li>Coventry</li>
  <li>Evesham</li>
  <li>Kidderminster</li>
  <li>Leek</li>
  <li>Ludlow</li>
  <li>Malvern</li>
  <li>Market Drayton</li>
  <li>Nuneaton</li>
  <li>Oswestry</li>
  <li>Rugby</li>
  <li>Spernal</li>
  <li>Stoke-on-Trent</li>
  <li>Stourbridge and Halesowen</li>
  <li>Telford</li>
  <li>Telford South</li>
  <li>Walsall</li>
  <li>Warwick</li>
  <li>Wolverhampton</li>
  <li>Worcester</li>
</ul>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149596/Yorkshire_and_The_Humber_May_2022_STW_Locations_Only.png"></div>
<figcaption><p>Figure 9. Map showing the location of Sewage Treatment Works sampled in Yorkshire and the Humber.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in Yorkshire and the Humber:</p>

<ul>
  <li>Barnsley</li>
  <li>Barton-upon-Humber</li>
  <li>Beverley</li>
  <li>Bingley</li>
  <li>Bradford</li>
  <li>Bridlington</li>
  <li>Castleford</li>
  <li>Colburn</li>
  <li>Dewsbury</li>
  <li>Doncaster (Thorne)</li>
  <li>Doncaster (Sandall)</li>
  <li>Driffield</li>
  <li>Grimsby</li>
  <li>Halifax</li>
  <li>Harrogate North</li>
  <li>Harrogate South</li>
  <li>Hemsworth and South Elmsall</li>
  <li>Huddersfield</li>
  <li>Hull</li>
  <li>Keighley</li>
  <li>Leeds</li>
  <li>Maltby</li>
  <li>Malton</li>
  <li>Mexborough and Conisbrough</li>
  <li>Northallerton</li>
  <li>Pontefract</li>
  <li>Scarborough</li>
  <li>Scunthorpe</li>
  <li>Sheffield (Blackburn Meadows)</li>
  <li>Sheffield (Woodhouse Mill)</li>
  <li>Wakefield</li>
  <li>York</li>
</ul>

<h2>Concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> in wastewater samples: 15 July 2020 to 31 May 2021</h2>

<p><abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> wastewater concentration data from 15 July 2020 to 31 May 2021 is available in the accompanying spreadsheet. Samples collected from 182 <abbr title="sewage treatment works">STWs</abbr> during this time period were analysed at the David Jones laboratory. By 31 May 2021 this had decreased to 10 <abbr title="sewage treatment works">STWs</abbr> then to 0 <abbr title="sewage treatment works">STWs</abbr> by the 31 July 2021.</p>

<p>All other <abbr title="sewage treatment work">STW</abbr> samples throughout the life of the <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> Programme have been analysed at the Environment Agency laboratory. Methods used at both labs should enable direct comparison of historical data (15 July 2020 to 31 May 2021) and later data (1 June 2021 to 30 March 2022) but this change in laboratories should be taken into account when interpreting results of any further analysis.</p>

<p>In addition, population coverage of the <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> programme gradually increased over this time period. Therefore, regional and national averages have not been calculated for this time period and graphical visualisations of this historical data are not shown in this report.</p>

<h2>Concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> in wastewater samples: 1 June 2021 to 30 March 2022</h2>

<p><abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> wastewater concentration data from 1 June 2021 to 30 March 2022 is available in the accompanying spreadsheet. The spreadsheet includes data from previous monthsâ publications, along with the latest data from March 2022. The most recent 4 weeks of data are visualised in figures 10 and 11; England concentration maps. Note that Sewage Treatment Works are omitted from figures 10 and 11 if no samples were taken during the week.</p>

<p>In the week beginning 8 March, 57 of 298 <abbr title="sewage treatment works">STWs</abbr> (19%) had a decrease in average concentration levels relative to the previous week. In the week beginning 15 March, 92 of 299 <abbr title="sewage treatment works">STWs</abbr> (31%) had lower average concentration than the preceding week. In the week beginning 22 March, 196 of 268 <abbr title="sewage treatment works">STWs</abbr> (73%) had a decrease in average concentration compared to the previous week. Interpretation should be made with careful consideration of the points in the âUncertainty, data quality and revisionsâ section below.</p>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149597/LOD_160_National_STW_Locations_Last_Two_Weeks__1_.png"></div>
<figcaption><p>Figure 10. England concentration map: map showing the location of Sewage Treatment Works sampled. Colour indicates the weekly-average SARS-CoV-2 RNA concentration (gene copies per litre) in the last week. See regional maps for the names of each Sewage Treatment Works. Note due to the pause in the EMHP Programme on 30 March 2022 there is insufficient data to calculate weekly estimates for the week commencing 29 March 2022.</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149598/LOD_160_National_STW_Locations_First_Two_Weeks__1_.png"></div>
<figcaption><p>Figure 11. England concentration map: map showing the location of Sewage Treatment Works sampled. Colour indicates the weekly-average SARS-CoV-2 RNA concentration (gene copies per litre) in the first two weeks. See regional maps for the names of each Sewage Treatment Works.</p></figcaption></figure>

<h2>Regional average concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> in wastewater: 1 June 2021 to 30 March 2022</h2>

<p>In the latest period (8 March to 30 March) concentration data showed signs of an upward trend in all regions. Concentration levels in the West Midlands, North East, North West and Yorkshire and the Humber were below the national average in February. In the month of March, despite a peak in mid-March, concentrations in these regions were below the national average by the end of March.</p>

<p>Yorkshire and the Humber showed the lowest average concentrations relative to other regions. There is high variability in concentration between individual <abbr title="sewage treatment works">STWs</abbr>, nationally. Interpretation should be made with careful consideration of the points in the âUncertainty, data quality, and revisionsâ section.</p>

<p>Figures 12 to 20 show the 7-day rolling average SARS-CoV-2 concentration detected in wastewater for each region in England, together with the national average of SARS-CoV-2 concentration detected in wastewater and individual site data. Samples where no SARS-CoV-2 was detected are assigned the value 160 (gc/l) which is the Theoretical Limit of Detection. Data are weighted by catchment population to account for varying catchment sizes. For further information, see âmethodologyâ section below.</p>

<p>Note: there was limited wastewater sampling over the Christmas period, therefore trends are not shown for the period 21 December 2021 to 28 December 2021</p>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149599/East_Midlands_May_2022.png"></div>
<figcaption><p>Figure 12. East Midlands 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149600/East_of_England_May_2022.png"></div>
<figcaption><p>Figure 13. East of England 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149601/London_May_2022.png"></div>
<figcaption><p>Figure 14. London 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149602/North_East_May_2022.png"></div>
<figcaption><p>Figure 15. North East 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149603/North_West_May_2022.png"></div>
<figcaption><p>Figure 16. North West 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149604/South_East_May_2022.png"></div>
<figcaption><p>Figure 17. South East 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149605/South_West_May_2022.png"></div>
<figcaption><p>Figure 18. South West 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149606/West_Midlands_May_2022.png"></div>
<figcaption><p>Figure 19. West Midlands 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149607/Yorkshire_and_The_Humber_May_2022.png"></div>
<figcaption><p>Figure 20. Yorkshire and the Humber 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<h2>Detection of SARS-CoV-2 variants and mutations in wastewater samples: 1 May 2021 to 27 March 2022</h2>

<h3>Detections of SARS-CoV-2 variants</h3>

<p>Applying the Wastewater Variant Genomic Case Definition (see âMethodologyâ), wastewater samples collected from <abbr title="sewage treatment work">STW</abbr> sites across England from 1 May 2021 up to 27 March 2022 were analysed for the presence of <abbr title="variant of concern">VOC</abbr> or <abbr title="variant under investigation">VUI</abbr>. <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> variant detection data showing the most common variants is available in the accompanying spreadsheet.</p>

<p>The spreadsheet includes a table showing the number of sequenced samples and whether those samples contained a Confirmed or Possible detection of Delta, Kappa, B.1.1.318, AY4.2, Omicron, Omicron-BA.1 and Omicron-BA.2. Note that this data is subject to change as methods are further developed.</p>

<p>The prevalence of a variant in a catchment area (that is, the number of individuals infected with that variant) cannot be deduced from wastewater data. Instead, a variant is categorised Confirmed, Possible or not detected in each sequenced sample. Lack of a positive sequencing result is not necessarily confirmation that no variant cases reside within a given geography, due to factors impacting <abbr title="ribonucleic acid">RNA</abbr> recovery (see âMethodologyâ).</p>

<p>Figure 21 shows the proportion of wastewater <abbr title="sewage treatment work">STW</abbr> catchment sites in which each variant was detected. Given the consideration above, this should be interpreted as a geographical metric and not a measure of prevalence. The replacement of the Delta variant with Omicron can be seen clearly in Figure 21, as can the replacement of Omicron-BA.1 with Omicron-BA.2. By 27 March, using a 7-day rolling average, Omicron-BA.1 and Omicron-BA.2 were detected in 29% and 92% of <abbr title="sewage treatment work">STW</abbr> sites, respectively.</p>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149608/variant_line_chart_2.png"></div>
<figcaption><p>Figure 21. VOC detections (STWs) in England: 7-day moving average of variant detections (confirmed and possible) as a proportion of wastewater STW catchment sites sampled and sequenced each day, 1 May 2021 to 27 March 2022. Note that any days with less than 50 sequenced catchment sites have been excluded and are considered as missing data for the purposes of this chart.</p></figcaption></figure>

<h3>Detections of SARS-CoV-2 single mutations</h3>

<p>Applying the Wastewater Variant Genomic Case Definition (see âMethodologyâ), wastewater samples collected from <abbr title="sewage treatment work">STW</abbr> sites across England from 1 May 2021 up to 27 March 2022 were analysed for the presence of 2 single mutations, K417N and E484K. Detections of a single mutations were only reported as confirmed when found in samples in which no variant with these <abbr title="single nucleotide polymorphisms">SNPs</abbr> in their definition were detected. The accompanying spreadsheet includes a table showing confirmed detections of K417N and E484K. Note that this data is subject to change as methods are further developed.</p>

<h2>Detection of the Omicron variant in wastewater samples: 22 November 2021 to 27 March 2022</h2>

<p>Applying the wastewater variant genomic case definition (see âMethodologyâ), wastewater samples collected from <abbr title="sewage treatment work">STW</abbr> and network sites across England from 22 November 2021 up to 27 March 2022 have been analysed for the presence of Omicron, Omicron-BA.1 and Omicron-BA.2. To ensure continuity with previous publications we have included an updated table showing the number of sequenced samples by region and the number of confirmed and possible Omicron detections in the accompanying spreadsheet.</p>

<p>This table includes data from both <abbr title="sewage treatment works">STWs</abbr> and network sites and is the only data in this publication to include network sites. Note that this data is subject to change as methods are further developed. The most recent 4 weeks of Omicron detections data are visualised in figures 22 to 25.</p>

<p>Over the 4 weeks from 28 February to 27 March, wastewater detections data showed a continuous decrease in the proportion of sequenced samples with detections of Omicron-BA.1. In comparison, the proportion of samples with detections of Omicron-BA.2 continued to increase. In the latest week, South East and South West had the highest proportion of samples with Omicron detections while North East had the lowest proportion of samples with Omicron detections.</p>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149609/omicron_BA1_first_two_weeks.png"></div>
<figcaption><p>Figure 22. Confirmed and possible detections of Omicron-BA.1 (VOC-21NOV-01) in wastewater samples collected in England, data from 28 February to 13 March 2022</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149610/omicron_BA1_last_two_weeks.png"></div>
<figcaption><p>Figure 23. Confirmed and possible detections of Omicron-BA.1 (VOC-21NOV-01) in wastewater samples collected in England, data from 14 March to 27 March 2022</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149611/omicron-BA.2_first_two_weeks.png"></div>
<figcaption><p>Figure 24. Confirmed and possible detections of Omicron-BA.2 (VOC-22JAN-01) in wastewater samples collected in England, data from 28 February to 13 March 2022</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149612/omicron-BA.2_last_two_weeks.png"></div>
<figcaption><p>Figure 25. Confirmed and possible detections of Omicron-BA.2 (VOC-22JAN-01) in wastewater samples collected in England, data from 14 March to 27 March 2022</p></figcaption></figure>

<h2>Methodology</h2>

<h3>About these data</h3>

<h4>SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> Concentration</h4>

<p><abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> sample wastewater from Sewage Treatment Works (<abbr title="sewage treatment works">STWs</abbr>) throughout England. Samples are analysed for the concentration of SARS-CoV-2 by quantifying the number of copies of the nucleocapsid gene (N1). Samples are transported to Environment Agency laboratory for quantification of the N1 gene using a process called Reverse Transcriptase Polymerase Chain Reaction.</p>

<p>The data reported here are the number of SARS-CoV-2 N1 gene copies per litre of wastewater (shortened to gene copies/litre throughout this document). Generally, the more people with COVID-19 in the community the more viral <abbr title="ribonucleic acid">RNA</abbr> will be shed into wastewater. Therefore, the concentration in wastewater is indicative of the prevalence of COVID-19 in the community.</p>

<p>Samples are analysed for the concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> and concentration is adjusted for flow (see âUncertainty, data quality, and revisionsâ). Typically, 4 samples per week are analysed from each <abbr title="sewage treatment work">STW</abbr>. However, occasionally fewer samples will be collected and/or analysed. This can be due to several reasons such as <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> altering sampling strategy because of testing capacity constraints, changing epidemiological priorities and current local response activities.</p>

<p>Weekly Concentration Table and Daily Concentration Table in the accompanying spreadsheet show the concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> in wastewater samples (gene copies/litre). Note that there is sensitivity associated with the analysis of wastewater; the Limit of Detection (<abbr title="Limit of Detection">LOD</abbr>) is the minimum concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> that can be reliably detected. Data tables contain the values â<abbr title="Theoretical Limit of Detection">tLOD</abbr>â where analysis of the sample did not detect presence of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr>.</p>

<p>For the purposes of averaging data to produce weekly averages in the Weekly Concentration Table and Figures 10 and 11, <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> assign a concentration of 160 gene copies/litre to all â<abbr title="Theoretical Limit of Detection">tLOD</abbr>â samples. This concentration is the Theoretical Limit of Detection and is calculated by the Environment Agency. Note that there is uncertainty in this calculation, please see the âUncertainty, data quality, and revisionsâ section for further information.</p>

<p>To calculate region averages, data from each <abbr title="sewage treatment work">STW</abbr> was weighted by catchment population. To estimate the population of <abbr title="sewage treatment works">STWs</abbr>, the catchment areas of <abbr title="sewage treatment works">STWs</abbr> are matched to lower layer super output areas (LSOA)s. Catchment areas were provided by the 9 water companies serving England.</p>

<p>See the <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> publication Wastewater testing coverage data for the Environmental Monitoring for Health Protection (<abbr title="Environmental Monitoring for Health Protection">EMHP</abbr>) programme for a detailed description of this method. Regional and national averages were calculated by taking the 7-day rolling arithmetic mean. Figures are displayed on a logarithmic scale for ease of interpretation.</p>

<h4>SARS-CoV-2 variant and single mutation detections</h4>

<p>The wastewater routine analysis is to look for the presence of pre-defined sets of single nucleotide polymorphisms (<abbr title="single nucleotide polymorphisms">SNPs</abbr>) that identify known variants of concern, variants under investigation, and signals in monitoring. It is possible to look for mutations associated with variants in the wastewater, but detection of variants can be transient and the correlation between population prevalence and wastewater variant detection has not been established for Omicron. Wastewater monitoring remains under development and is currently considered as supplementary data in variant monitoring.</p>

<p>A proportion of samples are sent to Liverpool, Nottingham and Exeter Universities Sequencing facilities each week for genomic sequencing, results are analysed by <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> for the presence of known <abbr title="variants of concern">VOCs</abbr> and <abbr title="variants under investigation">VUIs</abbr> officially recognised by <abbr title="UK Health Security Agency">UKHSA</abbr>. Wastewater detections of a variant are reported as either confirmed or possible, based on the number of signature or unique <abbr title="single nucleotide polymorphisms">SNPs</abbr>, and co-occurrence on the same amplicon.</p>

<p>This describes the level of confidence <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> has in the specificity of the detection; âconfirmedâ status is assigned when there is a high degree of confidence the variant is present, âpossibleâ is assigned where there is some evidence that the variant is present. These definitions are different to the clinical case definitions and the 2 should not be compared.  Lack of a positive sequencing result is not necessarily confirmation that no variant cases reside within a given geography, due to factors such as rainfall dilution impacting <abbr title="ribonucleic acid">RNA</abbr> recovery.</p>

<p>Results were also analysed by <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> for the presence of some single mutations. Detections of a single mutation were only reported as confirmed when found in samples in which no variant with these <abbr title="single nucleotide polymorphisms">SNPs</abbr> in their definition were detected.</p>

<p>The wastewater variant genomic case definition used for detections in this publication is detailed below. Note that these definitions are subject to change.</p>

<h3>Wastewater variant genomic case definition</h3>

<h4>B.1.1.529-BA.2â¯(<abbr title="variant of concern">VOC</abbr>-22JAN-01) Omicron-BA.2</h4>

<p>B.1.1.529-BA.2 lineage has 4 synonymous and 16 non-synonymous signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> of which 12 are unique amongst known <abbr title="variant of concern">VOC</abbr> and <abbr title="variant under investigation">VUI</abbr>. Co-occurrence of signature mutations occurs on 1 amplicon for Nimagen protocol*.</p>

<p>Confirmedâ¯ââ¯â¥â¯13â¯of 20â¯signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected,â¯â¥â¯6â¯ofâ¯12â¯unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> detectedâ¯andâ¯co-occurrence detected OR ifâ¯â¥â¯16â¯signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> andâ¯â¥â¯6â¯unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> are present, but with no co-occurrence confirmed presence can be assigned if some position of the cooccurring pairs are not covered</p>

<p>Possibleâ¯ââ¯â¥â¯8â¯ofâ¯16â¯signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> detectedâ¯andâ¯â¥â¯4 ofâ¯12â¯unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected OR ifâ¯&lt;â¯8 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> andâ¯â¥â¯4 unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> are detected, butâ¯â¥â¯6â¯<abbr title="single nucleotide polymorphisms">SNPs</abbr> are not covered possible presence can be assigned if those not covered are present across 2 dates from the same site, but in the same sequencing run.</p>

<p>Not detectedâ¯-â¯&lt;â¯7â¯ofâ¯16â¯signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected.</p>

<p>(*: amplicon_119 [22595-22837] positions 22679, 22688U,22775U and 22786U)</p>

<h4>B.1.1.529-BA.1â¯(<abbr title="variant of concern">VOC</abbr>-21NOV-01) OMICRON-BA.1 (updated 25/01/2022)</h4>

<p>B.1.1.529-BA.1 lineage has 1â¯synonymous and 16 non-synonymousâ¯signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> of which 11â¯are unique amongst known <abbr title="variant of concern">VOC</abbr> and <abbr title="variant under investigation">VUI</abbr>.â¯Co-occurrence of signature mutations occurs on 2 amplicons forâ¯Nimagenâ¯protocol*.â¯</p>

<p>Confirmedâ¯ââ¯â¥â¯11â¯of 16â¯signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected,â¯â¥â¯6â¯ofâ¯11â¯unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> detectedâ¯andâ¯co-occurrence detected OR ifâ¯â¥â¯16â¯signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> andâ¯â¥â¯6â¯unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> are present, but with no co-occurrence confirmed presence can be assigned if some position of the cooccurring pairs are not covered</p>

<p>Possibleâ¯ââ¯â¥â¯7â¯ofâ¯16â¯signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> detectedâ¯andâ¯â¥â¯3 ofâ¯11â¯unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected OR ifâ¯&lt;â¯7 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> andâ¯â¥â¯3 unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> are detected, butâ¯â¥â¯6â¯<abbr title="single nucleotide polymorphisms">SNPs</abbr> are not covered possible presence can be assigned if those not covered are present across 2 dates from the same site, but in the same sequencing run.</p>

<p>Not detectedâ¯ââ¯&lt;â¯5â¯ofâ¯16â¯signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected.</p>

<p>(*: amplicon_121 [22876-23119] positions 22898 and 23048, amplicon_139 [26476-26705] position 26530 and 26577.)</p>

<h4>B.1.1.529â¯(<abbr title="variant of concern">VOC</abbr>-21NOV-01) Omicron</h4>

<p>The wastewater routine analysis is to look for the presence of pre-defined sets of single nucleotide polymorphisms (<abbr title="single nucleotide polymorphisms">SNPs</abbr>) that identify known variants of concern, variants under investigation, and signals in monitoring. For the detection of Omicron <abbr title="variant of concern">VOC</abbr>-21NOV-01 (B.1.1.529) in wastewater, the following definition is based on the detection of a number of <abbr title="single nucleotide polymorphisms">SNPs</abbr> from the list in the official definition is noted below. No distinction is made between <abbr title="single nucleotide polymorphisms">SNPs</abbr> which can be used to define wastewater detections, unless otherwise stated.</p>

<p>Confirmed â â¥ 16 of 22 variant defining <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected, â¥ 9 of 20 unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected and co-occurrence detected on at least 7 OR if â¥ 16 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> and â¥ 9 unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> are present without co-occurrence, OR co-presence of the following <abbr title="single nucleotide polymorphisms">SNPs</abbr> on amplicon 121 with co-occurrence: 22882, 22898, 22992, 23013, 23040, 23048 and 23055</p>

<p>Possibleâ¯ââ¯â¥â¯10â¯ofâ¯22â¯variant defining <abbr title="single nucleotide polymorphisms">SNPs</abbr> detectedâ¯andâ¯â¥â¯6 ofâ¯20â¯unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected OR ifâ¯&lt;â¯10 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> andâ¯â¥â¯6 unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> are detected, butâ¯â¥â¯9â¯<abbr title="single nucleotide polymorphisms">SNPs</abbr> are not covered possible presence can be assigned if those not covered are present across 2 dates from the same site, but in the same sequencing run.</p>

<p>Not detected â &lt; 7 of 22 variant defining <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected unless co-presence of the following <abbr title="single nucleotide polymorphisms">SNPs</abbr> on amplicon 121: 22882, 22898, 22992, 23013, 23040, 23048 and 23055</p>

<h4>AY.4.2 (V-21OCT-01)</h4>

<p>AY.4.2 is Delta + 3 unique signatures among <abbr title="variant of concern">VOC</abbr>/<abbr title="variant under investigation">VUI</abbr>:  A1711V (protein nsp3), A222V (Spike protein) and Y145H (Spike protein).</p>

<p>Confirmed â â¥ 11 of 16 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected including the presence of A1711V, A222V and Y145H. If â¥ 11 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> are present and at least 5 are not covered, confirmed presence can also be assigned if at least 2 of the 3 unique SNPS are covered</p>

<p>Possible â â¥ 8 of 16 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected and presence of A222V and Y145H. If &lt; 8 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> and 2 out of 3 unique snips are detected, but â¥ 5 <abbr title="single nucleotide polymorphisms">SNPs</abbr> are not covered possible presence can be assigned if those not covered are present across 2 dates from the same site within the same sequencing run.</p>

<p>Not detected â â¤ 7 of 14 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected.</p>

<h4>B.1.617.2â¯(V-21APR-02) Delta</h4>

<p>B.1.617.2â¯has 13â¯signature non-synonymous <abbr title="single nucleotide polymorphisms">SNPs</abbr>,â¯of whichâ¯noneâ¯are unique amongst known <abbr title="variant of concern">VOC</abbr> and <abbr title="variant under investigation">VUI</abbr>.â¯Co-occurrence of signature mutations occurs on 1 amplicon for both ARTIC andâ¯Nimagenâ¯protocols,â¯respectively.â¯</p>

<p>Confirmedâ¯ââ¯â¥â¯9â¯of 13â¯signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected andâ¯co-occurrence detected on one amplicon. Ifâ¯â¥â¯9â¯signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> are present, but those co-occurring are not covered, confirmed presence can also be assigned.â¯</p>

<p>Possibleâ¯ââ¯â¥â¯5 of 13â¯signature <abbr title="single nucleotide polymorphisms">SNPs</abbr>â¯detected.â¯Ifâ¯&lt;â¯5 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> are detected, butâ¯â¥â¯5 <abbr title="single nucleotide polymorphisms">SNPs</abbr> are not covered possible presence can be assigned if those not covered are present across 2 dates from the same site, but in the same sequencing run.</p>

<p>Not detectedâ¯-â¯â¤â¯4 of 13â¯signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected.</p>

<h4>B.1.617.1 (V-21APR-01) Kappa</h4>

<p>The B.1.617 lineage has 11 signature synonymous and non-synonymous <abbr title="single nucleotide polymorphisms">SNPs</abbr>, of which 4 are unique amongst known <abbr title="variant of concern">VOC</abbr> and <abbr title="variant under investigation">VUI</abbr>. Co-occurrence of signature mutations occurs on 1 amplicon for both ARTIC and Nimagen protocols, respectively.</p>

<p>Confirmed â â¥ 8 of 11 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected, â¥ 3 of 4 unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected and co-occurrence detected on one amplicon. If â¥ 8 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> and â¥ 3 unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> are present, but those co-occurring are not covered, confirmed presence can also be assigned.</p>

<p>Possible â â¥ 5 of 11 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected and â¥ 2 of 4 unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected. If &lt; 5 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> and â¥ 2 unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> are detected, but â¥ 5 <abbr title="single nucleotide polymorphisms">SNPs</abbr> are not covered possible presence can be assigned if those not covered are present across 2 dates from the same site, but in the same sequencing run.</p>

<p>Not detected â â¤ 4 of 11 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected.</p>

<h4>B.1.1.318â¯(V-21FEB-04)</h4>

<p>The B.1.1.318 lineage has 14â¯signatureâ¯synonymousâ¯and non-synonymous <abbr title="single nucleotide polymorphisms">SNPs</abbr>,â¯of which 12â¯are unique amongst known <abbr title="variant of concern">VOC</abbr> and <abbr title="variant under investigation">VUI</abbr>. Co-occurrence of signature mutations occurs on 3â¯and 2â¯amplicons for ARTIC andâ¯Nimagenâ¯protocols,â¯respectively.â¯</p>

<p>Confirmedâ¯ââ¯â¥â¯11 of 14â¯signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected,â¯â¥â¯6â¯of 12â¯unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> detectedâ¯andâ¯co-occurrence detected on at leastâ¯oneâ¯amplicon. Ifâ¯â¥â¯11â¯signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> andâ¯â¥â¯8â¯unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> are present, but those co-occurring are not covered, confirmed presence can also be assigned.â¯</p>

<p>Possibleâ¯ââ¯â¥â¯5â¯of 14â¯signature <abbr title="single nucleotide polymorphisms">SNPs</abbr>â¯detected andâ¯â¥â¯3 of 12â¯unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected.â¯Ifâ¯&lt;â¯5 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> andâ¯â¥â¯5 unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> are detected, butâ¯â¥â¯6 <abbr title="single nucleotide polymorphisms">SNPs</abbr> are not covered possible presence can be assigned, if those not covered are present across 2 dates from the same site, but in the same sequencing run.â¯</p>

<p>Not detectedâ¯ââ¯â¤â¯4â¯of 14â¯signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected.â¯</p>

<h3>Data sources</h3>

<p>Wastewater samples are collected by water companies and transported to an Environment Agency Laboratory. Concentration data are from Environment Agency analysis of wastewater samples. Sequencing data is from Liverpool, Nottingham and Exeter Universities / Sequencing facilities. Variant detectionâs data are from <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> Genomics Analysis.</p>

<p>Figures in this publication show the location of <abbr title="sewage treatment works">STWs</abbr>. These locations were obtained from the <a class="govuk-link" href="https://uwwtd.eu/United-Kingdom/treatment-plant/ukenthtwutp000014/2016" rel="external">European Commission urban wastewater website</a>. Note that names differ between those used in this document and the European Commission urban wastewater website, sites can be linked using the â<abbr title="sewage treatment work">STW</abbr> Site Codeâ in the attached data tables.</p>

<p>The figures and analysis presented will evolve over time to ensure the most relevant information is included and the needs of stakeholders are met. These figures were initially published monthly, from June 2021 to March 2022.</p>

<p>This publication is the final regular publication, covering data up to the pausing of the <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> Programme on 30 March, and regular publications will now cease. Any future changes to these figures and analysis, or any future ad-hoc publications, will be announced in advance on the Statistics at <abbr title="UK Health Security Agency">UKHSA</abbr> page.</p>

<h3>How these figures can be used</h3>

<p>These figures and data can be used to view the concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> in wastewater, which is indicative of prevalence in the community and an indication of where <abbr title="variants of concern">VOCs</abbr> and <abbr title="variants under investigation">VUIs</abbr> may be present across England. Due to factors relating to data uncertainty, <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> recommend careful interpretation of this data alongside other data sources such as Weekly Statistics for NHS Test and Trace (England).</p>

<h3>Uncertainty, data quality, and revisions</h3>

<p>The figures here are compiled by professional analysts and have been quality assured. However, the analytical pipeline to produce this data should still be considered experimental; it is subject to an ongoing quality assurance process. Any revisions to past publications will be in line with DHSCâs revision policy and highlighted in future publications accordingly.</p>

<h4>SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> concentration</h4>

<p>From September to November 2021 the ratio of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> concentrations measured in wastewater to clinical measures of prevalence (Pillar I+II, CIS data) diverged significantly. This manifested as a monotonically decreasing ratio indicative of reduced <abbr title="ribonucleic acid">RNA</abbr> concentrations in wastewater in contrast to plateauing or rising case rates in the population. No firm conclusions as to the cause of this divergence can be drawn at this time.</p>

<p>However, from the end of November 2021, coinciding with the emergence of the Omicron variant in England, this trend in the ratio changed. While there remained a significant difference between the measured and expected <abbr title="ribonucleic acid">RNA</abbr> concentrations based on pre-divergence observations, the dynamics of this measure adhered to the trends observed in clinical estimations of cases and prevalence.</p>

<p>Given reports from a number of other countries of similar observations in their data, the so-called âOmicron effectâ was postulated to be a result of lower viral shedding in faeces in individuals infected with the Omicron variant. In the absence of empirical evidence to test this hypothesis and relying on as yet unpublished analyses ruling out other possible causes (for example, reduction in testing uptake), no firm conclusions can yet be drawn.</p>

<p><abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> advise caution when directly comparing data from Sewage Treatment Works and concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> in wastewater with clinical testing data.</p>

<p>Interpretation should be made with careful consideration of the points below:</p>

<p>The sites from which <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> sample can change on a regular basis depending on testing capacity, epidemiological priorities, and current local response activities. This can result in changes to specific Sewage Treatment Works (<abbr title="sewage treatment works">STWs</abbr>) covered. This means the <abbr title="sewage treatment work">STW</abbr> included in subsequent publications may change.</p>

<p>This publication shows <abbr title="sewage treatment works">STWs</abbr> as single points. Every <abbr title="sewage treatment work">STW</abbr> has an associated geographical catchment area (area from which wastewater flows into the sampling location). These catchments are not yet presented in these statistics publications. There is uncertainty surrounding catchment geography, as catchments vary in size and population covered, and in some cases an <abbr title="sewage treatment work">STW</abbr> may not be located within the catchment it serves (for example, if wastewater is pumped to the <abbr title="sewage treatment work">STW</abbr>). Therefore, the location of <abbr title="sewage treatment work">STW</abbr> should be used as a general guide only.</p>

<p>Wastewater is an inherently variable material and can change greatly within and between days (foodstuffs being consumed; when washing machines are switched on and many other factors), the physical infrastructure and the type and levels of industrial wastes. It is generally more useful to consider trends over medium- and long-term periods of time, rather than between individual days.</p>

<p>There are several factors that can impact the quality of the signal recorded from wastewater and introduce uncertainty:</p>

<p>1. The concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> in wastewater may be impacted by weather, for example heavy rainfall wastewater may cause dilution. <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> mitigate this by adjusting concentration to consider flow.</p>

<p>2. Whether samples are âcompositeâ or âgrabâ. Composite samples are where an autosampler (a machine which gathers wastewater at regular intervals over a set time period) gathers wastewater throughout the day. Grab samples are where a single sample is taken at one point in the day and are subject to greater variability; they are influenced by flow characteristics, time of day, and are more susceptible to outliers in the sewage. <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> aim to mitigate this by sampling during times of peak load and ensuring the sample is collected mid-stream rather than close to the edges or bottom of the pipe.</p>

<p>3. Calculating the Limit of Detection (<abbr title="Limit of Detection">LOD</abbr>; the minimum concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> that can be reliably detected in a wastewater sample) can be achieved by several methods. Since June 2021 â March 2022 we have used the Theoretical <abbr title="Limit of Detection">LOD</abbr> as calculated by the Environment Agency. Data from July 2020 â May 2021, samples were assigned Theoretical <abbr title="Limit of Detection">LOD</abbr> value of 133 gc/l (Environment Agency laboratory) or 610 gc/l (David Jones laboratory).</p>

<p>4. The Theoretical <abbr title="Limit of Detection">LOD</abbr> uses several assumptions. Laboratory data is required to inform the more realistic Practical <abbr title="Limit of Detection">LOD</abbr>. The Practical <abbr title="Limit of Detection">LOD</abbr> for this analysis is not yet reported here. It is likely that the <abbr title="Limit of Detection">LOD</abbr> used in producing these statistics will change in future publications. <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> will clearly state when this occurs.</p>

<p>5. Although the exact value of the <abbr title="Limit of Detection">LOD</abbr> is uncertain, it is unlikely to have a noticeable impact on the interpretation of this data. This is because the difference between Theoretical and Practical <abbr title="Limit of Detection">LOD</abbr> is relatively small, with most impact at low levels of the target being measured.</p>

<p>6. Samples collected from 182 <abbr title="sewage treatment works">STWs</abbr> during 15 July 2020 to 31 May 2021 were analysed at the David Jones laboratory. By 31 May 2021 this had decreased to 10 <abbr title="sewage treatment works">STWs</abbr> then to 0 <abbr title="sewage treatment works">STWs</abbr> by 31 July 2021. All other <abbr title="sewage treatment work">STW</abbr> samples throughout the life of the <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> Programme have been analysed at the Environment Agency laboratory. Methods used at both labs should enable direct comparison of historical data (15 July 2020 to 31 May 2021) and later data (1 June 2021 to 30 March 2022) but this change in laboratories should be taken into account when interpreting results of any further analysis.</p>

<p>7. Similarly the evolution of lab methods over time may also affect fair comparisons of the wastewater data across location and time.</p>

<p>8. As explained in the âAbout this dataâ section, typically 4 wastewater samples are taken at each <abbr title="sewage treatment work">STW</abbr> per week. The average of all samples within the week is presented in the Weekly Concentration Table. Uncertainty occurs where fewer samples are taken. For example, if only one sample is taken during a week, that one sample defines the week average. This means weeks with fewer samples are more susceptible to variability and outliers. Sample Count Table shows how many samples were taken at an <abbr title="sewage treatment work">STW</abbr> for each week in this release.</p>

<p>For the above reasons <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> advise caution when directly comparing data from Sewage Treatment Works and concentrations of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> in wastewater with clinical testing data. Methodology and limitations will differ across data sources and this should be taken into account.</p>

<h4>SARS-CoV-2 variant and single mutation detections</h4>

<p>The lack of a positive sequencing result is not confirmation that no variant cases reside within a given geography (given limitations of wastewater testing).</p>

<p>Environmental factors such as rainfall dilution can influence the detection of variants in wastewater. Therefore, low total detections on some days may not be reflective of low community infection levels.</p>

<p>Due to limitations of wastewater testing, <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> cannot estimate the number of cases associated with a detection. In small catchments it is possible, but unlikely, the detection is due to a single individual. In larger catchments a detection is likely caused by a higher number of cases.</p>

<p>Figure 21 shows the daily proportion of wastewater catchments with detections. Data has been smoothed by taking 7 day rolling averages. Detections include confirmed and possible detections. Samples are from <abbr title="sewage treatment works">STWs</abbr> only. The prevalence of a variant in a catchment area (that is,  the number of individuals infected with that variant) cannot be deduced from wastewater data. In a typical week there are approximately 200 sequenced samples per day (3 days per week). Any days with less than 50 sequenced samples (equivalent to 50 sequenced catchment areas since catchment areas are sampled once a day) have been excluded and are considered as missing data for the purposes of this chart.</p>

<p>Detections of a single mutation are only reported as confirmed when found in samples in which no variant with these <abbr title="single nucleotide polymorphisms">SNPs</abbr> in their definition were detected.</p>

<p>Figures 22 and 25 show the location of Omicron detections in <abbr title="sewage treatment works">STWs</abbr> and sewer networks as single points. Sewer networks tend to be relatively small geographical areas. Please note that the underlying characteristics of city sewer networks can be complex. For example, there may be varying flow from other areas into the catchment, and/or the exact catchment geography may be uncertain. With this in mind, maps are presented as a general guide to the location of a detection.</p>

<p>It is possible that a detection in a catchment is contributed to by case(s) who do not reside in the catchment; interpretation with other epidemiological indicators, such as known clinical cases, is encouraged.</p>

<p>The data presented in this publication are generated by non-accredited research laboratories and should be considered experimental. Formal validation of wastewater monitoring as a variant surveillance system is underway.</p>

<p>If you would like to give user feedback, please email <a class="govuk-link" href="mailto:TestandTrace.Statistics@ukhsa.gov.uk">TestandTrace.Statistics@ukhsa.gov.uk</a></p>

</div>


</div>
</div>
  </div>

  <div class="sticky-element" data-sticky-element="">
    <a class="govuk-link app-c-back-to-top dont-print" href="#contents">
    <svg class="app-c-back-to-top__icon" focusable="false" height="17" viewBox="0 0 13 17" width="13" xmlns="http://www.w3.org/2000/svg">
      <path d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z" fill="currentColor"></path>
    </svg>
    Contents
</a>

    <div class="sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>