<main lang="en" role="main" class="html-publication">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          
<div class="gem-c-organisation-logo brand--cabinet-office">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link brand__border-color" href="/government/organisations/scientific-advisory-group-for-emergencies">
      <span class="gem-c-organisation-logo__name">Scientific Advisory Group for Emergencies</span>
</a>
</div>
        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  

<div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
      <span class="govuk-caption-xl gem-c-title__context">
    Research and analysis
  </span>

  
  <h1 class="gem-c-title__text govuk-heading-xl">
    Imperial College London: Omicron vs Delta replication, 19 December 2021
  </h1>
</div>
  <p class="publication-header__last-changed">Published 23 December 2021</p>

  </header>



<div class="govuk-grid-row sidebar-with-body" data-module="sticky-element-container">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav aria-label="Contents" class="gem-c-contents-list" role="navigation" data-module="gem-track-click">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-category="contentsClicked" data-track-action="content_item 1" data-track-label="#the-sars-cov-2-variant-omicron-shows-enhanced-replication-in-human-primary-nasal-epithelial-cells" data-track-options="{&quot;dimension29&quot;:&quot;\n1. The SARS-CoV-2 variant, Omicron, shows enhanced replication in human primary nasal epithelial cells.&quot;}" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" href="#the-sars-cov-2-variant-omicron-shows-enhanced-replication-in-human-primary-nasal-epithelial-cells"><span class="gem-c-contents-list__number">1. </span><span class="gem-c-contents-list__numbered-text">The SARS-CoV-2 variant, Omicron, shows enhanced replication in human primary nasal epithelial cells.</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-category="contentsClicked" data-track-label="#introduction" data-track-action="content_item 2" data-track-options="{&quot;dimension29&quot;:&quot;\n2. Introduction&quot;}" href="#introduction" class="gem-c-contents-list__link govuk-link govuk-link--no-underline"><span class="gem-c-contents-list__number">2. </span><span class="gem-c-contents-list__numbered-text">Introduction</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 3" href="#results" data-track-category="contentsClicked" data-track-label="#results" data-track-options="{&quot;dimension29&quot;:&quot;\n3. Results&quot;}" class="gem-c-contents-list__link govuk-link govuk-link--no-underline"><span class="gem-c-contents-list__number">3. </span><span class="gem-c-contents-list__numbered-text">Results</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a href="#discussion" data-track-options="{&quot;dimension29&quot;:&quot;\n4. Discussion&quot;}" data-track-action="content_item 4" data-track-category="contentsClicked" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-label="#discussion"><span class="gem-c-contents-list__number">4. </span><span class="gem-c-contents-list__numbered-text">Discussion</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a href="#materials-and-methods" data-track-action="content_item 5" data-track-category="contentsClicked" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-options="{&quot;dimension29&quot;:&quot;\n5. Materials and methods&quot;}" data-track-label="#materials-and-methods"><span class="gem-c-contents-list__number">5. </span><span class="gem-c-contents-list__numbered-text">Materials and methods</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-category="contentsClicked" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" href="#acknowledgements-and-funding" data-track-action="content_item 6" data-track-label="#acknowledgements-and-funding" data-track-options="{&quot;dimension29&quot;:&quot;\n6. Acknowledgements and funding&quot;}"><span class="gem-c-contents-list__number">6. </span><span class="gem-c-contents-list__numbered-text">Acknowledgements and funding</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-label="#references" data-track-action="content_item 7" data-track-category="contentsClicked" data-track-options="{&quot;dimension29&quot;:&quot;\n7. References&quot;}" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" href="#references"><span class="gem-c-contents-list__number">7. </span><span class="gem-c-contents-list__numbered-text">References</span></a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button data-module="print-link" class="govuk-link govuk-body-s gem-c-print-link__button">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img class="print-meta-data-licence" src="/assets/government-frontend/open-government-licence-min-e98134ae65ff5fe99a524d8c383350b3998f91c298dc24b36126e65f1656c578.png">
</p>
<p>
  © Crown copyright 2021
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/imperial-college-london-omicron-vs-delta-replication-19-december-2021/imperial-college-london-omicron-vs-delta-replication-19-december-2021
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div data-module="govspeak" class="gem-c-govspeak govuk-govspeak ">
    
    
      <div class="govspeak">
<h2>
<span class="number">1. </span>The <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> variant, Omicron, shows enhanced replication in human primary nasal epithelial cells.</h2>

<p>Authors: Jonathan Brown, Jie Zhou, Thomas P. Peacock and Wendy S Barclay</p>

<ol>
  <li>Department of Infectious Disease, Imperial College London, <abbr title="United Kingdom">UK</abbr>, W2 1PG</li>
</ol>

<p>Corresponding author: Wendy Barclay</p>

<h2>
<span class="number">2. </span>Introduction</h2>

<p>The second year of the <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> pandemic was marked by the repeated emergence of <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> ‘variants’ with altered properties such as enhanced transmissibility or altered antigenicity. The fifth of these ‘variants of concern’, Omicron (Pango lineage <abbr title="a omicron substrain">BA.1</abbr>), was first detected in Southern Africa and, within a just a month of its initial detection, has rapidly spread across the world and now comprises the majority of <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> cases in many places including the <abbr title="United Kingdom">UK</abbr>. Omicron is marked by an unprecedented number of mutations in the Spike protein that result in a large antigenic distance between Omicron and the Wuhan-like Spike used in current vaccines with a consequent fall in vaccine effectiveness against Omicron and a higher frequency of reinfections. However, it is unclear if vaccine breakthrough and reinfections alone fully account for Omicron’s rapid growth and displacement of the Delta variant, or whether an inherent increase in viral fitness in the human airway also contributes to its enhanced transmissibility. Here we demonstrate an increase in replication of an isolate
of Omicron compared to Delta in cultures of primary human airway epithelia derived from nasal cells.</p>

<h2>
<span class="number">3. </span>Results</h2>

<p><strong>Omicron produces smaller plaques than Delta on cells overexpressing <abbr title="angiotensin-converting enzyme 2">ACE2</abbr> and <abbr title="transmembrane serine protease 2">TMPRSS2</abbr></strong></p>

<p>An Omicron and Delta isolate were propagated in <abbr title="kidney epithetical cell line derived from an African Green Monkey used in cell culture">Vero</abbr> cells overexpressing human <abbr title="angiotensin-converting enzyme 2">ACE2</abbr> and <abbr title="transmembrane serine protease 2">TMPRSS2</abbr> protease and then titrated by plaque assay on these cells. Both Delta and Omicron variants formed plaques, but the plaques for Omicron were smaller and less distinct (Figure 1).</p>

<figure class="image embedded"><div class="img"><img src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/139262/imperialpic1.png" alt="Two photos showing the results of the plaque assay; culture plates of Vero cells inoculated with either the Delta or Omicron variant of SARS-CoV-2 and stained with crystal violet, which stains only living cells.   "></div></figure>

<p><strong>Figure 1:</strong> Plaque phenotype of Delta and Omicron variants of <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr>. Virus isolates were propagated on <abbr title="kidney epithetical cell line derived from an African Green Monkey used in cell culture">Vero</abbr>-<abbr title="angiotensin-converting enzyme 2">ACE2</abbr>-<abbr title="transmembrane serine protease 2">TMPRSS2</abbr> (<abbr title="Vero E6 ACE2/TMPRSS2 cells">VAT</abbr>) cells and plaqued on <abbr title="Vero E6 ACE2/TMPRSS2 cells">VAT</abbr> cells for 3 days before staining with crystal violet to visualise. Omicron produces smaller plaques than Delta.</p>

<p><strong>Omicron demonstrates a replicative advantage over Delta in primary human airway epithelial cells</strong></p>

<p>The virus stocks of the Omicron and Delta variants contained similar viral <abbr title="ribonucleic acid">RNA</abbr> to infectivity ratios as measured by <abbr title="encodes the envelope protein">E gene</abbr> <abbr title="reverse transcription">RT</abbr>-<abbr title="quantitative polymerase chain reaction">qPCR</abbr> and plaque assay. To compare replication of the two variants, primary human airway epithelial cultures derived from nasal cells were inoculated at a multiplicity of 0.1 plaque forming unit (<abbr title="plaque forming unit">pfu</abbr>) per cell of each variant. <abbr title="kidney epithetical cell line derived from an African Green Monkey used in cell culture">Vero</abbr> <abbr title="a subclone of Vero 76 and it is supplied by European Collection of Authenticated Cell Cultures">E6</abbr>-<abbr title="angiotensin-converting enzyme 2">ACE2</abbr>/<abbr title="transmembrane serine protease 2">TMPRSS2</abbr> (<abbr title="Vero E6 ACE2/TMPRSS2 cells">VAT</abbr>) cells, or human lung <abbr title="lung cell line">Calu3</abbr> cells were inoculated at 0.001 <abbr title="plaque forming unit">pfu</abbr> per cell. Virus released at the apical surface of the airway cultures or into the supernatants overlaying the monocultures was collected daily for 3 days post inoculation and viral loads determined by <abbr title="reverse transcription">RT</abbr>-<abbr title="quantitative polymerase chain reaction">qPCR</abbr> for <abbr title="encodes the envelope protein">E gene</abbr> and by plaque assay for infectious virus on <abbr title="Vero E6 ACE2/TMPRSS2 cells">VAT</abbr> cells (Figure 2). In human nasal airway epithelial cells (<abbr title="human nasal airway epithelial cells">HAE</abbr>) Omicron showed a large early replication advantage, yielding viral load titres ~100-fold higher than for Delta by 24 hours post infection. By 48 and 72 hours post infection, infectious titres of Omicron were lower compared to Delta and at 72 hours the <abbr title="ribonucleic acid">RNA</abbr> collected from Omicron infected wells was also lower in <abbr title="encodes the envelope protein">E gene</abbr> copies. This coincided with observations by light microscopy at 72 hours that all cilia had stopped beating in the Omicron infected wells, suggesting cell death, whereas cilia were still observed to be beating in the Delta virus infected wells. In <abbr title="Vero E6 ACE2/TMPRSS2 cells">VAT</abbr> cells, replication of the two variants was equal despite the observation that Omicron formed smaller plaques in these cells. In <abbr title="lung cell line">Calu3</abbr> cells, which are a monoculture of immortalized human lung cells, the viral yields of Omicron were lower than for Delta.</p>

<figure class="image embedded"><div class="img"><img alt="Three bar charts showing viral load released and three bar charts showing plaques formed, between 0 and 72 hours after infection, when infected with either the Omicron or Delta variant of SARS-CoV-2. One chart per cell type" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/139263/imperialpic2.png"></div></figure>

<p><strong>Figure 2:</strong> Comparative replication kinetics of <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> Omicron and Delta variants in vitro. Human nasal airway epithelial (<abbr title="human nasal airway epithelial cells">HAE</abbr>) cells were inoculated with Delta or Omicron variants at 0.1 <abbr title="plaque forming unit">pfu</abbr> per cell, and <abbr title="kidney epithetical cell line derived from an African Green Monkey used in cell culture">Vero</abbr>-<abbr title="angiotensin-converting enzyme 2">ACE2</abbr>-<abbr title="transmembrane serine protease 2">TMPRSS2</abbr> (<abbr title="Vero E6 ACE2/TMPRSS2 cells">VAT</abbr>) and <abbr title="lung cell line">Calu3</abbr> cells were inoculated at 0.001 <abbr title="plaque forming unit">pfu</abbr> per cell. Infections were performed in triplicate wells and back titration confirmed equal inputs of the variants by <abbr title="encodes the envelope protein">E gene</abbr> copies measured by <abbr title="reverse transcription">RT</abbr>-<abbr title="quantitative polymerase chain reaction">qPCR</abbr> and infectious units measured by plaque assay. Omicron showed no replicative advantage in <abbr title="Vero E6 ACE2/TMPRSS2 cells">VAT</abbr> cells and was defective in <abbr title="lung cell line">Calu3</abbr> cells, but demonstrated a ~100-fold higher titre than Delta at 24 hours post-infection of <abbr title="human nasal airway epithelial cells">HAE</abbr> cells. Bars show individual replicate titres with the standard deviation.</p>

<p>Statistical differences measured by ANOVA on log transformed data:</p>

<ul>
  <li>for <abbr title="lung cell line">Calu3</abbr> <abbr title="encodes the envelope protein">E gene</abbr> copies per <abbr title="millilitre">ml</abbr> for 48 hours and 72 hours and for <abbr title="plaque forming unit">Pfu</abbr> per <abbr title="millilitre">ml</abbr> for 72 hours the P is less than 0.05.</li>
  <li>for <abbr title="human nasal airway epithelial cells">HAE</abbr> <abbr title="encodes the envelope protein">E gene</abbr> copies per <abbr title="millilitre">ml</abbr> for 24 hours, for <abbr title="lung cell line">Calu3</abbr> <abbr title="encodes the envelope protein">E gene</abbr> copies per <abbr title="millilitre">ml</abbr> for 0 hours and for <abbr title="plaque forming unit">Pfu</abbr> per <abbr title="millilitre">ml</abbr> for 24 hours, 48 hours and 72 hours the P is less than 0.01.</li>
  <li>and for P is less than 0.001 and for P is less than 0.0001</li>
</ul>

<p><strong>Omicron rapidly outcompetes Delta in mixed infections of primary human airway epithelial Cells</strong></p>

<p>Next we performed a direct head-to-head competition assay inoculating an equal mix of each variant onto the same wells of <abbr title="human nasal airway epithelial cells">HAE</abbr>, <abbr title="Vero E6 ACE2/TMPRSS2 cells">VAT</abbr>, or <abbr title="lung cell line">Calu3</abbr> cells. Once again virus released from apical surface or the primary cultures or into the supernatant overlaying the immortalize cells was collected daily and assessed for total virus replication by <abbr title="encodes the envelope protein">E gene</abbr> <abbr title="reverse transcription">RT</abbr>-<abbr title="quantitative polymerase chain reaction">qPCR</abbr> (dotted black lines in figure 3) and also for the <abbr title="ribonucleic acid">RNA</abbr> yield for each variant using a novel <abbr title="Polymerase chain reaction">PCR</abbr> protocol. Variant specific <abbr title="reverse transcription">RT</abbr>-<abbr title="quantitative polymerase chain reaction">qPCR</abbr> was carried out using a primer-probe set specific for the insertion in the
Omicron <abbr title="S-gene encodes the spike protein">S gene</abbr> at position 214 and a primer-probe set for the same region in the Delta <abbr title="S-gene encodes the spike protein">S gene</abbr> which lacks this insertion, allowing for the proportion of each variant in the same sample to be determined. We first validated the specific <abbr title="reverse transcription">RT</abbr>-<abbr title="quantitative polymerase chain reaction">qPCR</abbr> by creating a series of artificial mixes of <abbr title="ribonucleic acid">RNA</abbr> from each variant (left hand graph, Figure 3). During mixed infection of primary <abbr title="human nasal airway epithelial cells">HAE</abbr> cells, Omicron rapidly outcompeted Delta so that it was the only virus yielding detectible <abbr title="ribonucleic acid">RNA</abbr> products to apical wash by 24 hours post infection. Single infections may have suggested
that Delta has a replication advantage over Omicron at later timepoints (Figure 2). However, this mixed infection demonstrated that Omicron remained dominant throughout the time course (Figure 3). This supports the idea that it was Omicron-mediated cell damage which resulted in lower viral titres of Omicron than Delta at later timepoints in single infections (Figure 2). In contrast to <abbr title="human nasal airway epithelial cells">HAE</abbr> cells, both viruses in mixed infections remained in similar
quantities in the <abbr title="Vero E6 ACE2/TMPRSS2 cells">VAT</abbr> cells, and in <abbr title="lung cell line">Calu3</abbr> cells Delta was marginally fitter than Omicron (Figure 3).</p>

<figure class="image embedded"><div class="img"><img src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/139264/imperialpic3.png" alt=" Four graphs; one showing test validation and three showing viral load of Omicron and Delta at time points between 0 and 72 hours after infection when each of the three cell types are infected with both variants simultaneously"></div></figure>

<p><strong>Figure 3:</strong> Competition assay between <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> Omicron and Delta variants in vitro. A probebased <abbr title="reverse transcription">RT</abbr>-<abbr title="quantitative polymerase chain reaction">qPCR</abbr> assay was developed to measure the proportion of variants present in mixed samples. First panel – Omicron (O) and Delta (Δ) <abbr title="ribonucleic acid">RNA</abbr> were mixed at different ratios shown on the x-axis and measured for the signal produced using an Omicron-specific or a Delta-specific probe by <abbr title="reverse transcription">RT</abbr>-<abbr title="quantitative polymerase chain reaction">qPCR</abbr>. The proportion of variant-specific signal is shown with the <abbr title="standard deviation">SD</abbr> of 3 replicates. Equal mixtures of Omicron and Delta variant were then inoculated onto human airway epithelial (<abbr title="human nasal airway epithelial cells">HAE</abbr>) cells at a final multiplicity of 0.1 <abbr title="plaque forming unit">pfu</abbr> per cell and onto <abbr title="Vero E6 ACE2/TMPRSS2 cells">VAT</abbr> and <abbr title="lung cell line">Calu3</abbr> cells at 0.001 <abbr title="plaque forming unit">pfu</abbr> per cell. Daily harvests were measured for total viral load by <abbr title="encodes the envelope protein">E gene</abbr> qRT-<abbr title="Polymerase chain reaction">PCR</abbr> (black dots) and by the variant specific <abbr title="reverse transcription">RT</abbr>-<abbr title="quantitative polymerase chain reaction">qPCR</abbr> to give variant proportions. <abbr title="standard deviation">SD</abbr> of 3 replicates shown.</p>

<h2>
<span class="number">4. </span>Discussion</h2>

<p>Taken together the replication assays here show that the <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> Omicron variant has increased replicative fitness in primary cells derived from human upper respiratory tract compared to the previously dominant Delta variant. This suggests that at least part of the rapid growth of Omicron observed in <abbr title="United Kingdom">UK</abbr> in the face of a preceding Delta wave is due to high transmissibility of the virus between people resulting from efficient nasal replication. This, acting in combination with evasion of control by pre-existing neutralizing antibodies suggests that Omicron is likely to spread rapidly regardless of prior vaccine or infection status, and will require a combination of <abbr title="Non-pharmaceutical intervention">NPI</abbr> measures for effective control in the community.</p>

<h2>
<span class="number">5. </span>Materials and methods</h2>

<h3>
<span class="number">5.1 </span>Biosafety and ethics statement</h3>

<p>All laboratory work was approved by the local genetic manipulation safety committee of Imperial College London, St. Mary’s Campus (centre number GM77), and the Health and Safety Executive of the United Kingdom, under reference CBA1.77.20.1.</p>

<h3>
<span class="number">5.2 </span>Cell lines</h3>

<p>African green monkey kidney cells overexpressing human <abbr title="angiotensin-converting enzyme 2">ACE2</abbr> and <abbr title="transmembrane serine protease 2">TMPRSS2</abbr> (<abbr title="kidney epithetical cell line derived from an African Green Monkey used in cell culture">Vero</abbr> <abbr title="a subclone of Vero 76 and it is supplied by European Collection of Authenticated Cell Cultures">E6</abbr>-<abbr title="angiotensin-converting enzyme 2">ACE2</abbr>-<abbr title="transmembrane serine protease 2">TMPRSS2</abbr>; Glasgow University)<sup role="doc-noteref"><a href="#fn:1" rel="footnote" class="govuk-link">[footnote 1]</a></sup>, were maintained in complete media supplemented with 200 µg per <abbr title="millilitre">ml</abbr> hygromycin B (Gibco) and 2 <abbr title="Milligram">mg</abbr> per <abbr title="millilitre">ml</abbr> G418 (Gibco) to maintain human <abbr title="angiotensin-converting enzyme 2">ACE2</abbr> and <abbr title="transmembrane serine protease 2">TMPRSS2</abbr> expression. <abbr title="lung cell line">Calu3</abbr> airway epithelial cells were maintained in complete medium with 20% foetal calf serum. Nasal origin air–liquid interface primary human airway epithelial (<abbr title="human airway epithelium cells">HAEs</abbr>) cells were purchased from Epithelix and maintained in Mucilair cell culture medium (Epithelix). All Cells were kept at 5% <abbr title="carbon dioxide">CO2</abbr>, 37 degrees C.</p>

<h3>
<span class="number">5.3 </span>Primary virus isolates</h3>

<p>A virus isolate matching Omicron sequence was kindly donated by Gavin Screaton at Oxford University and propagated on <abbr title="kidney epithetical cell line derived from an African Green Monkey used in cell culture">Vero</abbr>-<abbr title="angiotensin-converting enzyme 2">ACE2</abbr>-<abbr title="transmembrane serine protease 2">TMPRSS2</abbr> (<abbr title="Vero E6 ACE2/TMPRSS2 cells">VAT</abbr>) cells for 3 days until cytopathic effect was observed<sup role="doc-noteref"><a rel="footnote" class="govuk-link" href="#fn:1">[footnote 1]</a></sup> . A Delta swab sample was kindly provided by Thushan de Silva at University of Sheffield and virus isolated and propagated on <abbr title="Vero E6 ACE2/TMPRSS2 cells">VAT</abbr> cells. Virus isolates used are as follows:</p>

<table>
  <thead>
    <tr>
      <th scope="col">Virus name</th>
      <th scope="col">Variant</th>
      <th scope="col">PANGO lineage</th>
      <th scope="col">GISAID Accession</th>
      <th scope="col">Spike mutations</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>hCoV- 19/England/SHEF- 10E8F3B/2021</td>
      <td>Delta</td>
      <td>B.1.617.2</td>
      <td>EPI_ISL_17 31019</td>
      <td>T19R, K77R, G142D, Δ156-157/R158G, A222V, L452R, T478K, D614G, P681R, D950N</td>
    </tr>
    <tr>
      <td>M21021166</td>
      <td>Omicron</td>
      <td>B.1.1.529 /<abbr title="a omicron substrain">BA.1</abbr>
</td>
      <td>N/A</td>
      <td>A67V, Δ69-70, T95I, G142D/Δ143-145, Δ211/L212I, ins214EPE, G339D, S371L, S373P, S375F, K417N, N440K, G446S, S477N, T478K, E484A, Q493R, G496S, Q498R, N501Y, Y505H, T547K, D614G, H655Y, N679K, P681H, N764K, A701V, D796Y, N856K, Q954H, N969K, L981F</td>
    </tr>
  </tbody>
</table>

<p><strong>Table 1 – Virus isolates derived from clinical samplesVirus</strong> growth kinetics and competition assays were performed as described previously<sup role="doc-noteref"><a href="#fn:2" class="govuk-link" rel="footnote">[footnote 2]</a></sup> . Briefly, in air-liquid interface <abbr title="human airway epithelium cells">HAEs</abbr>, before infection cells were washed with serum-free media to remove mucus and debris. Cells were infected with 200 µL of virus-containing serum-free <abbr title="Dulbecco's Modified Eagle Medium">DMEM</abbr> and incubated at 37 degrees centigrade for 1 hour. Inoculum was then removed and cells were washed twice. Time points were taken by adding 200 µL of serum-free <abbr title="Dulbecco's Modified Eagle Medium">DMEM</abbr> and incubating for 10 minutes and 37 degrees centigrade before removal and titration.</p>

<h3>
<span class="number">5.4 </span>Plaque assay</h3>

<p>Serial dilutions of virus supernatant in serum-free <abbr title="Dulbecco's Modified Eagle Medium">DMEM</abbr>, 1% NEAA and 1% P/S were performed and inoculated onto <abbr title="Vero E6 ACE2/TMPRSS2 cells">VAT</abbr> cells for 1 h at 37 °C. Inoculum was then removed and replaced with <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> overlay medium (1× MEM, 0.2% w/v BSA, 0.16% w/v <abbr title="Sodium bicarbonate">NaHCO3</abbr>, 10 mM <abbr title="a zwitterionic sulfonic acid buffering agent; one of the twenty Good's buffers">HEPES</abbr>, 2mM L-Glutamine, 1× P/S, 1% w/v Avicel). Plates were incubated for 3 days at 37 °C before overlay was removed and cells were stained for 1 hour at room temperature in crystal violet solution.</p>

<h3>
<span class="number">5.5 </span>Viral <abbr title="ribonucleic acid">RNA</abbr> extraction and <abbr title="reverse transcription">RT</abbr>-<abbr title="quantitative polymerase chain reaction">qPCR</abbr> for <abbr title="encodes the envelope protein">E gene</abbr> and Delta or Omicron mixture genotyping</h3>

<p>Virus genomes were quantified by <abbr title="encodes the envelope protein">E gene</abbr> <abbr title="reverse transcription">RT</abbr>-<abbr title="quantitative polymerase chain reaction">qPCR</abbr> as previously described<sup role="doc-noteref"><a href="#fn:4" class="govuk-link" rel="footnote">[footnote 4]</a></sup> . Viral <abbr title="ribonucleic acid">RNA</abbr> was extracted from cell culture supernatants using the <abbr title="technology combines the speed and efficiency of silica-based purification">QIAsymphony</abbr> DSP Virus or Pathogen Mini Kit on the <abbr title="technology combines the speed and efficiency of silica-based purification">QIAsymphony</abbr> instrument (Qiagen). <abbr title="reverse transcription">RT</abbr>-<abbr title="quantitative polymerase chain reaction">qPCR</abbr> was then performed using the AgPath <abbr title="reverse transcription PCR">RT-PCR</abbr> (Life Technologies) kit on a QuantStudio™ 7 Flex Real-Time <abbr title="Polymerase chain reaction">PCR</abbr> System with the primers specific for <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> <abbr title="encodes the envelope protein">E gene</abbr> <sup role="doc-noteref"><a rel="footnote" href="#fn:5" class="govuk-link">[footnote 5]</a></sup> . For absolutely quantification of <abbr title="encodes the envelope protein">E gene</abbr> <abbr title="ribonucleic acid">RNA</abbr> copies, a standard curve was generated using dilutions viral <abbr title="ribonucleic acid">RNA</abbr> of known copy number. <abbr title="encodes the envelope protein">E gene</abbr> copies per <abbr title="millilitre">ml</abbr> of original virus supernatant were then calculated using this standard curve.</p>

<p>To measure the proportions of Omicron and Delta <abbr title="ribonucleic acid">RNA</abbr> from mixed samples a modified <abbr title="reverse transcription">RT</abbr>-<abbr title="quantitative polymerase chain reaction">qPCR</abbr> protocol was developed using probes specific for the <abbr title="S-gene encodes the spike protein">S gene</abbr> of Omicron or Delta variants. A FAM probe specific to a 9 base pair insertion in the Omicron <abbr title="S-gene encodes the spike protein">S gene</abbr> at amino acid position 214, and a VIC probe specific to the same region of the Delta <abbr title="S-gene encodes the spike protein">S gene</abbr> sequence lacking this insertion were used. Reactions were run on AgPath <abbr title="reverse transcription PCR">RT-PCR</abbr> (Life Technologies) kit on a QuantStudio™ 7 Flex Real-Time <abbr title="Polymerase chain reaction">PCR</abbr> System and the relative amounts of FAM and VIC signal used to determine the proportion of Omicron and Delta in the original samples. Validation of the assay was carried out by extracting <abbr title="ribonucleic acid">RNA</abbr> from pure Omicron and Delta supernatants, normalising by <abbr title="encodes the envelope protein">E gene</abbr> copies and mixing at different ratios before measuring using the assay.</p>

<table>
  <tbody>
    <tr>
      <td><strong>Primer or Probe Sequence</strong></td>
      <td><strong>Sequence</strong></td>
    </tr>
    <tr>
      <td>Forward <abbr title="Polymerase chain reaction">PCR</abbr> primer upstream of Omicron insertion</td>
      <td>TGGACCTTGAAGGAAAACAGGG</td>
    </tr>
    <tr>
      <td>Reverse <abbr title="Polymerase chain reaction">PCR</abbr> primer downstream of Omicron insertion</td>
      <td>TGGTTCTAAAGCCGAAAAACCC</td>
    </tr>
    <tr>
      <td>FAM probe specific to Omicron Spike insertion sequence</td>
      <td>TTATAGTGCGTGAGCCAGAAGA</td>
    </tr>
    <tr>
      <td>VIC probe specific to Delta Spike sequence at site of Omicron insertion</td>
      <td>CTATTAATTTAGTGCGTGATCT</td>
    </tr>
  </tbody>
</table>

<h2>
<span class="number">6. </span>Acknowledgements and funding</h2>

<p>The authors would like to thank Dr Matthew Turnbull and Dr Suzannah Rihn of the <abbr title="Medical Research Council">MRC</abbr> University of Glasgow Centre for Virus Research (<abbr title="Centre for Virus Research">CVR</abbr>) for sharing their <abbr title="kidney epithetical cell line derived from an African Green Monkey used in cell culture">Vero</abbr> <abbr title="a subclone of Vero 76 and it is supplied by European Collection of Authenticated Cell Cultures">E6</abbr>-<abbr title="angiotensin-converting enzyme 2">ACE2</abbr>-<abbr title="transmembrane serine protease 2">TMPRSS2</abbr> cells and Gavin Screaton, Wanwisa Dejnirattisai and Alison Cowper from Oxford University for sharing the Omicron isolate.</p>

<p>This work was supported by the G2P-<abbr title="United Kingdom">UK</abbr> National Virology Consortium funded by <abbr title="UK Research and Innovation">UKRI</abbr>.</p>

<h2>
<span class="number">7. </span>References</h2>

<div role="doc-endnotes" class="footnotes">
  <ol>
    <li role="doc-endnote">
      <p>Rihn, S. J. et al. A plasmid <abbr title="deoxyribonucleic acid">DNA</abbr>-launched <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> reverse genetics system and coronavirus toolkit for <abbr title="coronavirus">COVID-19</abbr> research. PLoS Biol 19, e3001091, doi:10.1371/journal.pbio.3001091 (2021).&nbsp;<a class="govuk-link" aria-label="go to where this is referenced" href="#fnref:1" role="doc-backlink">↩</a>&nbsp;<a role="doc-backlink" href="#fnref:1:1" class="govuk-link" aria-label="go to where this is referenced 2">↩<sup>2</sup></a></p>
    </li>
    <li role="doc-endnote">
      <p>Peacock, T. P. et al. The furin cleavage site in the <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> spike protein is required for transmission in ferrets. Nat Microbiol, doi:10.1038/s41564-021-00908-w (2021).&nbsp;<a aria-label="go to where this is referenced" href="#fnref:2" class="govuk-link" role="doc-backlink">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Zhou, J. et al. Investigating <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> surface and air contamination in an acute healthcare setting during the peak of the <abbr title="coronavirus">COVID-19</abbr> pandemic in London. Clin Infect Dis, doi:10.1093/cid/ciaa905 (2020).&nbsp;<a class="govuk-link" aria-label="go to where this is referenced" href="#fnref:4" role="doc-backlink">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Corman, V. M. et al. Detection of 2019 novel coronavirus (<abbr title="2019 novel coronavirus">2019-nCoV</abbr>) by real-time <abbr title="reverse transcription PCR">RT-PCR</abbr>. Eurosurveillance 25, 2000045, doi:doi:<a class="govuk-link" href="https://doi.org/10.2807/1560-7917.ES.2020.25.3.2000045" rel="external">https://doi.org/10.2807/1560-7917.ES.2020.25.3.2000045 (2020)</a>.&nbsp;<a href="#fnref:5" class="govuk-link" role="doc-backlink" aria-label="go to where this is referenced">↩</a></p>
    </li>
  </ol>
</div>
</div>


</div>
</div>
  </div>

  <div class="sticky-element" data-sticky-element="">
    <a class="govuk-link app-c-back-to-top dont-print" href="#contents">
    <svg class="app-c-back-to-top__icon" viewBox="0 0 13 17" height="17" width="13" xmlns="http://www.w3.org/2000/svg" focusable="false">
      <path fill="currentColor" d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z"></path>
    </svg>
    Contents
</a>

    <div class="sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button data-module="print-link" class="govuk-link govuk-body-s gem-c-print-link__button">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>