<main lang="en" role="main" class="html-publication">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          <div class="gem-c-organisation-logo brand--department-of-health">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color" href="/government/organisations/department-of-health-and-social-care">
      <span class="gem-c-organisation-logo__name">Department<br>of Health &amp;<br>Social Care</span>
</a>
</div>

        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  <div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
    <span class="govuk-caption-xl gem-c-title__context">
      Research and analysis
    </span>
  <h1 class="gem-c-title__text ">
    Rapid evaluation of Oxford Nanopore Technologies’ LamPORE assay
  </h1>
</div>
  <p class="publication-header__last-changed">Published 28 January 2021</p>

  </header>




<div class="govuk-grid-row sidebar-with-body" data-module="sticky-element-container">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav role="navigation" class="gem-c-contents-list  gem-c-contents-list--no-underline " data-module="track-click" aria-label="Contents">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-label="#executive-summary" data-track-options="{&quot;dimension29&quot;:&quot;Executive summary&quot;}" data-track-category="contentsClicked" href="#executive-summary" class="gem-c-contents-list__link govuk-link " data-track-action="content_item 1">Executive summary</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-category="contentsClicked" data-track-label="#background" class="gem-c-contents-list__link govuk-link " data-track-action="content_item 2" data-track-options="{&quot;dimension29&quot;:&quot;\n1.  Background&quot;}" href="#background"><span class="gem-c-contents-list__number">1. </span><span class="gem-c-contents-list__numbered-text">Background</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-category="contentsClicked" data-track-options="{&quot;dimension29&quot;:&quot;\n2.  Technical validation&quot;}" class="gem-c-contents-list__link govuk-link " data-track-label="#technical-validation" href="#technical-validation" data-track-action="content_item 3"><span class="gem-c-contents-list__number">2. </span><span class="gem-c-contents-list__numbered-text">Technical validation</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-category="contentsClicked" class="gem-c-contents-list__link govuk-link " data-track-action="content_item 4" data-track-label="#diagnostic-sensitivity-and-specificity" data-track-options="{&quot;dimension29&quot;:&quot;\n3. \tDiagnostic sensitivity and specificity&quot;}" href="#diagnostic-sensitivity-and-specificity"><span class="gem-c-contents-list__number">3. </span><span class="gem-c-contents-list__numbered-text">Diagnostic sensitivity and specificity</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-options="{&quot;dimension29&quot;:&quot;\n4. \tConclusions&quot;}" data-track-category="contentsClicked" data-track-label="#conclusions" class="gem-c-contents-list__link govuk-link " href="#conclusions" data-track-action="content_item 5"><span class="gem-c-contents-list__number">4. </span><span class="gem-c-contents-list__numbered-text">Conclusions</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-category="contentsClicked" data-track-options="{&quot;dimension29&quot;:&quot;\n5. \tAdditional data&quot;}" class="gem-c-contents-list__link govuk-link " data-track-label="#additional-data" href="#additional-data" data-track-action="content_item 6"><span class="gem-c-contents-list__number">5. </span><span class="gem-c-contents-list__numbered-text">Additional data</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link " data-track-label="#sample-provenance" data-track-category="contentsClicked" data-track-options="{&quot;dimension29&quot;:&quot;\n6.  Sample provenance&quot;}" href="#sample-provenance" data-track-action="content_item 7"><span class="gem-c-contents-list__number">6. </span><span class="gem-c-contents-list__numbered-text">Sample provenance</span></a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button data-module="print-link" class="govuk-link gem-c-print-link__button">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img src="/assets/government-frontend/open-government-licence-min-e98134ae65ff5fe99a524d8c383350b3998f91c298dc24b36126e65f1656c578.png" class="print-meta-data-licence">
</p>
<p>
  © Crown copyright 2021
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/rapid-evaluation-of-oxford-nanopore-technologies-lampore-assay/rapid-evaluation-of-oxford-nanopore-technologies-lampore-assay
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div data-module="govspeak" class="gem-c-govspeak govuk-govspeak ">
    
    
      <div class="govspeak">
<h2>Executive summary</h2>

<p>The LamPORE assay was evaluated according to the published <a class="govuk-link" href="https://www.gov.uk/government/publications/assessment-and-procurement-of-coronavirus-covid-19-tests/coronavirus-covid-19-serology-and-viral-detection-testing-uk-procurement-overview">Technical Validation Group protocol</a>.</p>

<p>This was a collaboration across 5 NHS trusts and 2 university partners who evaluated <abbr title="loop-mediated isothermal amplification">LAMP</abbr> molecular testing technologies (including Oxford Nanopore LamPORE), in 22,941 samples, in an NHS asymptomatic staff pilot study.</p>

<p>As part of this, we also investigated the accuracy of LamPORE on 848 symptomatic patients who had undergone previous respiratory pathogen testing in a ‘real-world’ situation using RNA from samples sent to Public Health West Midlands for testing of patients with influenza-like illness (<abbr title="influenza-like illness">ILI</abbr>).</p>

<p>The LamPORE assay was assessed on saliva and swab samples from asymptomatic participants in the setting of RNA extraction when the underlying disease incidence was 1.99%.</p>

<p>The LamPORE assay on swabs in asymptomatic patients, with RNA extraction, returned a sensitivity of 99.57% (95% confidence interval (<abbr title="confidence interval">CI</abbr>) 98.46% to 99.99%) and the positive predictive value was 78.91% (95% <abbr title="confidence interval">CI</abbr> 75.84% to 81.69%). Specificity was 99.40% (95% <abbr title="confidence interval">CI</abbr> 99.28% to 99.50%) and negative predictive value was 99.99% (95% <abbr title="confidence interval">CI</abbr> 99.96% to 100.0%). For comparison, previous studies have estimated the <abbr title="positive predictive value">PPV</abbr> of RT-qPCR in low-prevalence settings to be between 20% and 74%.</p>

<p>The LamPORE assay on saliva in asymptomatic patients returned a sensitivity of 98.94% (95% <abbr title="confidence interval">CI</abbr> 96.23% to 99.87%) and the positive predictive value was 62.54% (95% <abbr title="confidence interval">CI</abbr> 58.11% to 66.77%). Specificity was 99.39% (95% <abbr title="confidence interval">CI</abbr> 99.26% to 99.49%) and negative predictive value was 99.99% (95% <abbr title="confidence interval">CI</abbr> 99.96% to 100.0%).</p>

<p>The LamPORE assay on swab samples in symptomatic patients, with RNA extraction returned a sensitivity of 100% (95% <abbr title="confidence interval">CI</abbr> 85.18% to 100%) and a specificity of 100%.</p>

<p>Participating NHS and academic laboratories developed an SOP during the study that eliminated contamination related false positives.</p>

<p>LamPORE demonstrates equivalent sensitivity and specificity to ‘gold standard’ RT-qPCR with the advantage of enhanced multiplexing (up to 768 samples per flow cell), which provides additional capacity for winter respiratory syndrome testing with significantly higher throughput than some existing technologies (96 samples per plate).</p>

<h2>
<span class="number">1. </span> Background</h2>

<p>1.1.	The importance of testing, isolation and contact tracing for control of SARS-CoV-2 transmission has been highlighted by the World Health Organisation (<abbr title="World Health Organisation">WHO</abbr>) as a critical intervention to prevent the spread of infection and ensuing morbidity and mortality from COVID-19. International efforts have largely focused on the detection of infection in symptomatic individuals as well as evolving clinical testing systems to detect those with asymptomatic or pre-symptomatic (subsequently referred to collectively as asymptomatic) infection that may still be infectious to others.</p>

<p>1.2.	LamPORE combines loop-mediated isothermal amplification (<abbr title="loop-mediated isothermal amplification">LAMP</abbr>) and nanopore sequencing to provide a highly scalable, multi-gene assay for the detection of SARS-CoV-2. <abbr title="loop-mediated isothermal amplification">LAMP</abbr> is a single-tube technique for the amplification of DNA and as a standalone method it may be considered as a low-cost, rapid alternative to RT-<abbr title="polymerase chain reaction">PCR</abbr>. Reverse transcription loop-mediated isothermal amplification (<abbr title="reverse transcription loop-mediated isothermal amplification">RT-LAMP</abbr>) combines <abbr title="loop-mediated isothermal amplification">LAMP</abbr> with a reverse transcription step to allow the detection of RNA. Target sequence is amplified at a constant temperature. Typically, 4 different primers are used to amplify 6 distinct regions on the target gene, which increases specificity. Additional pairs of ‘loop primers’ can further accelerate the reaction. The amount of amplified product produced in <abbr title="loop-mediated isothermal amplification">LAMP</abbr> is considerably higher than <abbr title="polymerase chain reaction">PCR</abbr>-based amplification, due to the use of multiple primer sets.  LamPORE builds on the <abbr title="loop-mediated isothermal amplification">LAMP</abbr> method to add sequencing as a process, for high-sensitivity, high-specificity analysis.</p>

<p>1.3.	The technology of choice for this provision to date has been reverse transcription polymerase chain reaction testing (RT-qPCR), typically provided in central laboratory facilities, with a sample receipt to result time of more than 24 hours and reliance on international supply chains for a singular testing system. Considering these challenges, alternative technology approaches are being evaluated, including other forms of nucleic acid amplification (that is, <abbr title="loop-mediated isothermal amplification">LAMP</abbr> or LamPORE) and protein-based methods (that is, lateral flow devices) for viral antigen detection, potentially providing diversification to the overall supply chain and options for an integrated testing strategy. These additional testing assets have also been considered in the context of other components of the wider end to end testing system, including the source and ease of sample collection and frequency of provision (saliva versus swab), RNA extraction requirements, and efficiency of returning results from a testing system into a clinical system for full maximal clinical utility, public health reporting and intervention requirements.</p>

<p>1.4.	<abbr title="loop-mediated isothermal amplification">LAMP</abbr> is an isothermal nucleic acid amplification technique, in contrast to the polymerase chain reaction (<abbr title="polymerase chain reaction">PCR</abbr>) technology, in which the reaction is carried out with a series of alternating temperature steps or cycles, isothermal amplification is carried out at a constant temperature, and does not require a thermal cycler.  In the case of LamPORE the <abbr title="loop-mediated isothermal amplification">LAMP</abbr> process is combined with a sequencing process for high-sensitivity, high-specificity analysis.</p>

<p>1.5.	This report outlines the technical validation undertaken in conjunction with the NHS Test and Trace Technical Validation Group and, given the need to prevent nosocomial spread and maintain NHS capacity and capability, it was conducted in collaboration with NHS trusts and university partners as a real-world evidence study.</p>

<h2>
<span class="number">2. </span> Technical validation</h2>

<p>2.1.	The Technical and Validation function was established under NHS Test and Trace, inclusive of NHS and PHE experts and working closely with the Medicines and Healthcare Products Regulatory Agency (<abbr title="Medicines and Healthcare Products Regulatory Agency">MHRA</abbr>) and research bodies. The Technical and Validation function considers manufacturers of SARS-CoV-2 (COVID-19) tests for viral detection (including <abbr title="loop-mediated isothermal amplification">LAMP</abbr> technologies) and registers their interest in the national procurement process if their test meets, or are intended to meet, the requirements of a relevant <abbr title="Medicines and Healthcare Products Regulatory Agency">MHRA</abbr> Target Product Profile (<abbr title="Target Product Profile">TPP</abbr>).  The Technical and Validation function reviews product information, undertakes technical and clinical validation, and establishes and/or works with service evaluation projects and pilots.</p>

<p>2.2.	The asymptomatic pilot study recruited 1,200 healthcare workers across 4 hospital sites (University Hospitals Birmingham NHS Foundation Trust, University Hospitals Southampton NHS Foundation Trust, Hampshire Hospitals NHS Foundation Trust and Manchester Hospitals NHS Foundation Trust). Participants underwent self-administered nasophayngeal swabbing at days 1, 7, 14 and 21 and saliva sampling on a daily basis. A total of 3,966 swab and 18,435 saliva samples were collected as part of this study.</p>

<h3>Assay description and intended purpose</h3>

<p>2.3	LamPORE test is CE marked for use on the Oxford Nanopore GridION device. Each GridION device can run between 8 and 768 samples using up to 5 individually addressable flow cells (consumable units on which the sequencing is performed). Each individual flow cell can used to test up to 96 samples, with recalibration of the flow cell allowing for multiple uses for batch sizes of less than 96 samples. LamPORE is an assay  produced by Oxford Nanopore Technologies consisting of 3 key stages:</p>

<ul>
  <li>the assay uses reverse-transcription coupled to loop-mediated isothermal amplification (<abbr title="reverse transcription loop-mediated isothermal amplification">RT-LAMP</abbr>) to amplify 3 highly conserved genes in SARS-CoV-2 positive samples (E1 gene, N2 gene, ORF1ab) and a negative control gene ACTB (human beta actin)</li>
  <li>samples are given a molecular barcode so that multiple samples can be analysed together</li>
  <li>these are then sequenced on the GridION Mk1 device, and analysed</li>
</ul>

<p>2.4	In order for <abbr title="reverse transcription loop-mediated isothermal amplification">RT-LAMP</abbr> to be combined with Nanopore sequencing technologies, several modifications to the <abbr title="reverse transcription loop-mediated isothermal amplification">RT-LAMP</abbr> protocol have been made. Each sample is uniquely identified in a 96 well plate by the addition of 2 sets of molecular barcodes. The first set is attached to the forward inner primer (FIP). The second set is attached during sequencing library preparation allowing for between 96 and 768 samples to be pooled onto each flow cell.</p>

<p>2.5	Samples undergo Nanopore sequencing for one hour, and then sequence data is aligned via a custom algorithm to the SARS-CoV-2 genome. Each sample in the pool is identified by its molecular barcodes and absolute read counts per sample for the ORF1ab, N2 and E1 genes, as well as an ACTB (human beta actin) internal control are obtained. A positive is defined as any of the target genes having greater than 50 reads per sample, indeterminate being between 30 and 50 reads and negative anything below this. Samples are called as invalid if there are less than 50 ACTB reads detected.</p>

<p>2.6	The validation was performed on nasal/oropharyngeal swabs and saliva. Nasal and oropharyngeal swabs were collected in viral transport medium (Sigma Virocult). Saliva was collected neat in universal plastic tubes. The samples were all collected in clinical settings from asymptomatic healthcare workers across 4 hospital sites (Birmingham, Southampton, Basingstoke and Manchester).</p>

<p>2.7	All the equipment required to perform the test is supplied by the manufacturer except for a plate centrifuge, thermal cycler, pipettes, magnetic stand, laboratory consumables, saliva collection containers and nasopharyngeal/oropharyngeal swabs.</p>

<h3>Performance characteristics</h3>

<h3>
<span class="number">2.8 </span>	Analytical Sensitivity of SARS CoV-2 targets</h3>

<p>The analytical sensitivity (ASe) for the LamPORE assay was evaluated using a blinded panel of guanidine isothiocynate inactivated SARS-CoV-2 virus grown on Vero E6 cells and quantified by droplet digital <abbr title="polymerase chain reaction">PCR</abbr> to 20,000 copies/ml. The CerTest ViaSure SARS-CoV-2 multiplex RT-qPCR assay was used as a direct comparator.  LamPORE successfully detected SARS-CoV-2 to a detection threshold of 20 copies/ml (Tables 1a and 1b). All assays were performed in triplicate.</p>

<h3>Table 1.a: LamPORE analytical sensitivity – limit of detection of LamPORE</h3>

<table>
  <thead>
    <tr>
      <th scope="col">Concentration     (copies/ml)</th>
      <th scope="col">Detected</th>
      <th scope="col">ORF1ab median reads</th>
      <th scope="col">E1 median reads</th>
      <th scope="col">N2 median reads</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>20,000</td>
      <td>Positive</td>
      <td>6,429</td>
      <td>808</td>
      <td>2,288</td>
    </tr>
    <tr>
      <td>2,000</td>
      <td>Positive</td>
      <td>1,385</td>
      <td>18</td>
      <td>602</td>
    </tr>
    <tr>
      <td>200</td>
      <td>Positive</td>
      <td>27</td>
      <td>4</td>
      <td>82</td>
    </tr>
    <tr>
      <td>20</td>
      <td>Positive</td>
      <td>67</td>
      <td>6</td>
      <td>979</td>
    </tr>
    <tr>
      <td>2</td>
      <td>Negative</td>
      <td>0</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td>0.2</td>
      <td>Negative</td>
      <td>16</td>
      <td>0</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

<h3>Table 1.b: LamPORE analytical sensitivity – reference standard (ViaSure qPCR)</h3>

<table>
  <thead>
    <tr>
      <th scope="col">Concentration     (copies/ml)</th>
      <th scope="col">Detected</th>
      <th scope="col">ORF1ab     median CT</th>
      <th scope="col">N1   median CT</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>20,000</td>
      <td>Positive</td>
      <td>19.4</td>
      <td>23.0</td>
    </tr>
    <tr>
      <td>2,000</td>
      <td>Positive</td>
      <td>21.8</td>
      <td>22.7</td>
    </tr>
    <tr>
      <td>200</td>
      <td>Positive</td>
      <td>30.2</td>
      <td>27.3</td>
    </tr>
    <tr>
      <td>20</td>
      <td>Positive</td>
      <td>33.3</td>
      <td>28.7</td>
    </tr>
    <tr>
      <td>2</td>
      <td>Equivocal</td>
      <td>N/A</td>
      <td>29.8</td>
    </tr>
    <tr>
      <td>0.2</td>
      <td>Negative</td>
      <td>N/A</td>
      <td>N/A</td>
    </tr>
  </tbody>
</table>

<h3>
<span class="number">2.9 </span> Precision and robustness</h3>

<p>Intra-assay repeatability and inter-assay reproducibility was carried out using quantified inactivated live virus (as previously quantified by droplet digital <abbr title="polymerase chain reaction">PCR</abbr>). A series of 5 replicates of the same sample undertaken on the same day is shown in Table 2. For intra-assay precision the Standard Deviation (SD) was 50 reads with a coefficient of variation (<abbr title="coefficient of variation">CV</abbr>) of +/- 2.3% (a satisfactory assay has a <abbr title="coefficient of variation">CV</abbr> less than 10%) (Table 2).  As LamPORE is designed as a qualitative assay rather than quantitative, the applicability of precision metrics is uncertain in this context.</p>

<h3>Table 2: intra-assay precision</h3>

<table>
  <thead>
    <tr>
      <th scope="col">Sample</th>
      <th scope="col">Inhibition   control value (Unmapped reads)</th>
      <th scope="col">ORF1ab</th>
      <th scope="col">E1   gene</th>
      <th scope="col">N2   gene</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Replicate   1</td>
      <td>658</td>
      <td>2,207</td>
      <td>970</td>
      <td>11</td>
    </tr>
    <tr>
      <td>Replicate   2</td>
      <td>788</td>
      <td>2,176</td>
      <td>112</td>
      <td>306</td>
    </tr>
    <tr>
      <td>Replicate   3</td>
      <td>1,261</td>
      <td>2,286</td>
      <td>637</td>
      <td>782</td>
    </tr>
    <tr>
      <td>Replicate   4</td>
      <td>3,126</td>
      <td>2,161</td>
      <td>159</td>
      <td>312</td>
    </tr>
    <tr>
      <td>Replicate   5</td>
      <td>565</td>
      <td>2,237</td>
      <td>11</td>
      <td>121</td>
    </tr>
  </tbody>
</table>

<p>For inter-assay precision the same sample was analysed on multiple days and is seen in Table 3. The SD was 178 reads with a <abbr title="coefficient of variation">CV</abbr> of +/- 7.8%. Satisfactory assay performance has a <abbr title="coefficient of variation">CV</abbr> less than 15%.</p>

<h3>Table 3: inter-assay precision</h3>

<table>
  <thead>
    <tr>
      <th scope="col">Sample</th>
      <th scope="col">Inhibition   control value (unmapped reads)</th>
      <th scope="col">ORF1ab</th>
      <th scope="col">E1   gene</th>
      <th scope="col">N2   gene</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Sample   1 (day 1)</td>
      <td>796</td>
      <td>2,269</td>
      <td>821</td>
      <td>207</td>
    </tr>
    <tr>
      <td>Sample   1 (day 2)</td>
      <td>1,248</td>
      <td>2,359</td>
      <td>93</td>
      <td>93</td>
    </tr>
    <tr>
      <td>Sample   1 (day 3)</td>
      <td>564</td>
      <td>2,007</td>
      <td>124</td>
      <td>219</td>
    </tr>
    <tr>
      <td>Sample   1 (day 4)</td>
      <td>3,747</td>
      <td>2,292</td>
      <td>1,541</td>
      <td>510</td>
    </tr>
    <tr>
      <td>Sample   1 (day 5)</td>
      <td>1,380</td>
      <td>2,495</td>
      <td>1,687</td>
      <td>1,547</td>
    </tr>
  </tbody>
</table>

<p>Quantified SARS-CoV-2 virus (via droplet digital <abbr title="polymerase chain reaction">PCR</abbr>) was used to input 20,000 copies/ml of extracted SARS-CoV-2 RNA into the LamPORE assay for 24 replicates in order to understand reproducibility. Standard deviation was 128 reads with a <abbr title="coefficient of variation">CV</abbr> of +/- 3.9% (Table 4).</p>

<h3>Table 4: reproducibility assessment for LamPORE</h3>

<table>
  <thead>
    <tr>
      <th scope="col">Replicate</th>
      <th scope="col">Concentration</th>
      <th scope="col">ORF1ab</th>
      <th scope="col">E1</th>
      <th scope="col">N2</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>20,000   copies/ml</td>
      <td>3,334</td>
      <td>1,384</td>
      <td>1,342</td>
    </tr>
    <tr>
      <td>2</td>
      <td>20,000   copies/ml</td>
      <td>3,084</td>
      <td>445</td>
      <td>0</td>
    </tr>
    <tr>
      <td>3</td>
      <td>20,000   copies/ml</td>
      <td>3,500</td>
      <td>20</td>
      <td>177</td>
    </tr>
    <tr>
      <td>4</td>
      <td>20,000   copies/ml</td>
      <td>3,174</td>
      <td>11</td>
      <td>60</td>
    </tr>
    <tr>
      <td>5</td>
      <td>20,000   copies/ml</td>
      <td>3,367</td>
      <td>60</td>
      <td>400</td>
    </tr>
    <tr>
      <td>6</td>
      <td>20,000   copies/ml</td>
      <td>3,013</td>
      <td>1</td>
      <td>0</td>
    </tr>
    <tr>
      <td>7</td>
      <td>20,000   copies/ml</td>
      <td>3,337</td>
      <td>22</td>
      <td>352</td>
    </tr>
    <tr>
      <td>8</td>
      <td>20,000   copies/ml</td>
      <td>3,375</td>
      <td>14</td>
      <td>151</td>
    </tr>
    <tr>
      <td>9</td>
      <td>20,000   copies/ml</td>
      <td>3,183</td>
      <td>11</td>
      <td>87</td>
    </tr>
    <tr>
      <td>10</td>
      <td>20,000   copies/ml</td>
      <td>3,173</td>
      <td>1,045</td>
      <td>22</td>
    </tr>
    <tr>
      <td>11</td>
      <td>20,000   copies/ml</td>
      <td>3,190</td>
      <td>128</td>
      <td>0</td>
    </tr>
    <tr>
      <td>12</td>
      <td>20,000   copies/ml</td>
      <td>3,344</td>
      <td>1,293</td>
      <td>1,837</td>
    </tr>
    <tr>
      <td>13</td>
      <td>20,000   copies/ml</td>
      <td>3,191</td>
      <td>416</td>
      <td>664</td>
    </tr>
    <tr>
      <td>14</td>
      <td>20,000   copies/ml</td>
      <td>3,233</td>
      <td>821</td>
      <td>1,823</td>
    </tr>
    <tr>
      <td>15</td>
      <td>20,000   copies/ml</td>
      <td>3,256</td>
      <td>1,792</td>
      <td>1,867</td>
    </tr>
    <tr>
      <td>16</td>
      <td>20,000   copies/ml</td>
      <td>3,237</td>
      <td>1,779</td>
      <td>1,640</td>
    </tr>
    <tr>
      <td>17</td>
      <td>20,000   copies/ml</td>
      <td>3,017</td>
      <td>377</td>
      <td>142</td>
    </tr>
    <tr>
      <td>18</td>
      <td>20,000   copies/ml</td>
      <td>3,144</td>
      <td>1,517</td>
      <td>311</td>
    </tr>
    <tr>
      <td>19</td>
      <td>20,000   copies/ml</td>
      <td>3,171</td>
      <td>230</td>
      <td>365</td>
    </tr>
    <tr>
      <td>20</td>
      <td>20,000   copies/ml</td>
      <td>3,241</td>
      <td>574</td>
      <td>2,774</td>
    </tr>
    <tr>
      <td>21</td>
      <td>20,000   copies/ml</td>
      <td>3,386</td>
      <td>60</td>
      <td>1,518</td>
    </tr>
    <tr>
      <td>22</td>
      <td>20,000   copies/ml</td>
      <td>3,493</td>
      <td>1,902</td>
      <td>16</td>
    </tr>
    <tr>
      <td>23</td>
      <td>20,000   copies/ml</td>
      <td>3,211</td>
      <td>82</td>
      <td>476</td>
    </tr>
    <tr>
      <td>24</td>
      <td>20,000   copies/ml</td>
      <td>3,206</td>
      <td>1,042</td>
      <td>1,201</td>
    </tr>
  </tbody>
</table>

<h3>
<span class="number">2.11 </span>	Analytical specificity (interferences and cross-reactions)</h3>

<p>Analytical specificity (ASp) was determined using the NATtrol™ Respiratory Verification Panel 2 (ZeptoMetrix Corporation, New York, United States) containing pathogens causing indistinguishable clinical signs to COVID-19 (n=22). No cross reactivity was observed in LamPORE to any respiratory pathogen.</p>

<h3>Table 5: specificity assessment for LamPORE</h3>

<table>
  <thead>
    <tr>
      <th scope="col">Target</th>
      <th scope="col">Result</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Parainfluenza2</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>Negative control</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>Influenza B Florida 02 06</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>Coronavirus NL63</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>B.parapertussis A747</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>Parainfluenza4</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>Influenza A H1 a/newcal/20/99</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>Parainfluenza 3</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>Coronavirus HKU-1</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>B.parapertussis A639</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>Metapneumovirus 9 - peru6-2003</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>Rhinovirus type 1a</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>Adenovirus 31</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>Parainfluenza 1</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>Adenovirus 1</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>M.pneumoniae M129</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>Coronavirus 229E</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>RSV-A2</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>Influenza A H1N1pdm</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>Coronavirus OC43</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>Influenza AH3</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>C.pneumioniae   CWL-029</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>Adenovirus3</td>
      <td>Negative</td>
    </tr>
    <tr>
      <td>NATtrolMERS-CoV stock</td>
      <td>Negative</td>
    </tr>
  </tbody>
</table>

<h2>
<span class="number">3. </span>	Diagnostic sensitivity and specificity</h2>

<h3>Clinical validation with confirmed positives and negatives</h3>

<p>3.1	Samples included in the validation were appropriate to the intended use case scenarios, including, low medium and high viral load samples as part of the asymptomatic pilot and the symptomatic samples. This dynamic range permits a rigorous evaluation of diagnostic sensitivity (Table 6).</p>

<h3>Table 6: range of viral loads for validation samples</h3>

<table>
  <thead>
    <tr>
      <th scope="col">Gene</th>
      <th scope="col">Mean   (CT)</th>
      <th scope="col">Min   (CT)</th>
      <th scope="col">Max   (CT)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>ORF1ab</td>
      <td>17.1</td>
      <td>16.2</td>
      <td>37.2</td>
    </tr>
    <tr>
      <td>N1</td>
      <td>14.3</td>
      <td>11.0</td>
      <td>37.2</td>
    </tr>
    <tr>
      <td>IC</td>
      <td>19.8</td>
      <td>22.8</td>
      <td>25.5</td>
    </tr>
  </tbody>
</table>

<p>3.2 Diagnostic sensitivity: confirmed clinical samples from all asymptomatic and asymptomatic participants (positive RT-qPCR result) were compared. The <abbr title="cycle threshold">CT</abbr> (cycle threshold) values or equivalent for both the assessed and comparator assays were included in the submitted validation data:</p>

<ul>
  <li>34 positive prospective asymptomatic swabs analysed by LamPORE</li>
  <li>299 positive prospective asymptomatic saliva samples analysed by LamPORE</li>
  <li>116 positive retrospective respiratory swab samples analysed by LamPORE</li>
</ul>

<p>3.3 Diagnostic specificity: confirmed clinical samples from patients (negative RT-qPCR result) were used. The <abbr title="cycle threshold">CT</abbr> values or equivalent for both the assessed and comparator assays were included in the submitted validation data:</p>

<ul>
  <li>3,932 negative prospective asymptomatic swabs analysed by LamPORE</li>
  <li>18,136 negative prospective asymptomatic  saliva samples analysed by LamPORE</li>
  <li>752 negative retrospective respiratory samples analysed by LamPORE</li>
</ul>

<p>3.4	The diagnostic sensitivity (<abbr title="diagnostic sensitivity">DSe</abbr>) and specificity (<abbr title="diagnostic specificity">DSp</abbr>) (diagnostic precision) derived from the above samples are as follows:</p>

<h4>3.4.1 All samples</h4>

<ul>
  <li>For LamPORE, diagnostic sensitivity (<abbr title="diagnostic sensitivity">DSe</abbr>) was calculated as 99.39% (95% <abbr title="confidence interval">CI</abbr> 97.81% to 99.93%) and positive predictive value was 72.61% (95% <abbr title="confidence interval">CI</abbr> 68.96% to 75.97%). Underlying disease incidence was 1.41% (Table 7)</li>
  <li>For LamPORE, diagnostic specificity (<abbr title="diagnostic specificity">DSp</abbr>) was calculated as 99.46% (95% <abbr title="confidence interval">CI</abbr> 99.36% to 99.55%) and negative predictive value was 99.99% (95% <abbr title="confidence interval">CI</abbr> 99.97% to 100.0%)</li>
</ul>

<h3>Table 7: diagnostic precision on all samples</h3>

<table>
  <thead>
    <tr>
      <th scope="col">—</th>
      <th style="text-align: right" scope="col">Comparator assay result:  positive</th>
      <th style="text-align: right" scope="col">Comparator assay result:  negative</th>
      <th style="text-align: right" scope="col">Total</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th scope="row">Assessed assay: <br> positive</th>
      <td style="text-align: right">326</td>
      <td style="text-align: right">123</td>
      <td style="text-align: right">449</td>
    </tr>
    <tr>
      <th scope="row">Assessed assay: <br> negative</th>
      <td style="text-align: right">2</td>
      <td style="text-align: right">22,818</td>
      <td style="text-align: right">22,820</td>
    </tr>
    <tr>
      <th scope="row">Total</th>
      <td style="text-align: right">328</td>
      <td style="text-align: right">22,941</td>
      <td style="text-align: right">23,269</td>
    </tr>
  </tbody>
</table>

<h4>3.4.2 Swab (asymptomatic) samples</h4>

<ul>
  <li>For LamPORE, diagnostic sensitivity (<abbr title="diagnostic sensitivity">DSe</abbr>) was calculated as 100% (95% <abbr title="confidence interval">CI</abbr> 85.18% to 100%) and positive predictive value was 67.65% (95% <abbr title="confidence interval">CI</abbr> 53.68% to 79.05%). Underlying disease prevalence was 0.58% (Table 8).</li>
  <li>For LamPORE, diagnostic specificity (<abbr title="diagnostic specificity">DSp</abbr>) was calculated as 99.72% (95% <abbr title="confidence interval">CI</abbr> 99.50% to 99.86%) and negative predictive value was 100%.</li>
</ul>

<h3>Table 8: diagnostic precision on swab samples</h3>

<table>
  <thead>
    <tr>
      <th scope="col">—</th>
      <th scope="col" style="text-align: right">Comparator assay result:  positive</th>
      <th scope="col" style="text-align: right">Comparator assay result:  negative</th>
      <th style="text-align: right" scope="col">Total</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th scope="row">Assessed assay: <br> positive</th>
      <td style="text-align: right">23</td>
      <td style="text-align: right">11</td>
      <td style="text-align: right">34</td>
    </tr>
    <tr>
      <th scope="row">Assessed assay: <br> negative</th>
      <td style="text-align: right">0</td>
      <td style="text-align: right">3,932</td>
      <td style="text-align: right">3,932</td>
    </tr>
    <tr>
      <th scope="row">Total</th>
      <td style="text-align: right">23</td>
      <td style="text-align: right">3,943</td>
      <td style="text-align: right">3,966</td>
    </tr>
  </tbody>
</table>

<h4>3.4.3 Saliva (asymptomatic) samples</h4>

<ul>
  <li>For LamPORE, diagnostic sensitivity (<abbr title="diagnostic sensitivity">DSe</abbr>) was calculated as 98.94% (95% <abbr title="confidence interval">CI</abbr> 96.23% to 99.87%) and positive predictive value was 62.54% (95% <abbr title="confidence interval">CI</abbr> 58.11% to 66.77%). Underlying disease prevalence was 1.03% (Table 9).</li>
  <li>For LamPORE, diagnostic specificity (<abbr title="diagnostic specificity">DSp</abbr>) was calculated as 99.39% (95% <abbr title="confidence interval">CI</abbr> 99.26% to 99.49%) and negative predictive value was 99.99% (95% <abbr title="confidence interval">CI</abbr> 99.96% to 100%).</li>
</ul>

<h3>Table 9: diagnostic precision on saliva samples</h3>

<table>
  <thead>
    <tr>
      <th scope="col">—</th>
      <th style="text-align: right" scope="col">Comparator assay result:  positive</th>
      <th scope="col" style="text-align: right">Comparator assay result:  negative</th>
      <th style="text-align: right" scope="col">Total</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th scope="row">Assessed assay: <br> positive</th>
      <td style="text-align: right">187</td>
      <td style="text-align: right">112</td>
      <td style="text-align: right">289</td>
    </tr>
    <tr>
      <th scope="row">Assessed assay: <br> negative</th>
      <td style="text-align: right">2</td>
      <td style="text-align: right">18,134</td>
      <td style="text-align: right">18,136</td>
    </tr>
    <tr>
      <th scope="row">Total</th>
      <td style="text-align: right">189</td>
      <td style="text-align: right">18,246</td>
      <td style="text-align: right">18,435</td>
    </tr>
  </tbody>
</table>

<h4>3.4.4 Retrospective (symptomatic) samples</h4>

<ul>
  <li>The symptomatic sample set was collected from patients undergoing diagnostic testing in the laboratories of Public Health West Midlands (n=848) who underwent swabbing for an influenza like illness (<abbr title="influenza-like illness">ILI</abbr>) in the period March 2020 to June 2020 or in the Milton Keynes Lighthouse Laboratory where both swabs and saliva was taken (n=20)</li>
  <li>For LamPORE, diagnostic sensitivity (<abbr title="diagnostic sensitivity">DSe</abbr>) was calculated as 100% (95% <abbr title="confidence interval">CI</abbr> 96.48% to 100%) and positive predictive value was 100%. Underlying disease prevalence was 12% (Table 10)</li>
  <li>For LamPORE, diagnostic specificity (<abbr title="diagnostic specificity">DSp</abbr>) was calculated as 100% and negative predictive value was 100% (95% <abbr title="confidence interval">CI</abbr> 99.51% to 100%)</li>
  <li>A significant increase in accuracy was observed in these samples as they were analysed at the end of project where the maximum experience with LamPORE had been gained</li>
</ul>

<h3>Table 10: diagnostic precision on retrospective samples</h3>

<table>
  <thead>
    <tr>
      <th scope="col">—</th>
      <th style="text-align: right" scope="col">Comparator assay result:  positive</th>
      <th scope="col" style="text-align: right">Comparator assay result:  negative</th>
      <th style="text-align: right" scope="col">Total</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th scope="row">Assessed assay: <br> positive</th>
      <td style="text-align: right">116</td>
      <td style="text-align: right">0</td>
      <td style="text-align: right">116</td>
    </tr>
    <tr>
      <th scope="row">Assessed assay: <br> negative</th>
      <td style="text-align: right">0</td>
      <td style="text-align: right">752</td>
      <td style="text-align: right">752</td>
    </tr>
    <tr>
      <th scope="row">Total</th>
      <td style="text-align: right">116</td>
      <td style="text-align: right">752</td>
      <td style="text-align: right">868</td>
    </tr>
  </tbody>
</table>

<h2>
<span class="number">4. </span>	Conclusions</h2>

<p>4.1	The technical performance of the LamPORE assay reveals a sensitivity of 99.57% (95% <abbr title="confidence interval">CI</abbr> 98.46% to 99.99%) and specificity of 99.40% (95% <abbr title="confidence interval">CI</abbr> 99.28% to 99.50%). This provides a comparable analysis to current standard of care RT-qPCR tests on RNA extracted swabs (in asymptomatic settings). This supports the introduction of this alterative nucleic acid amplification technology into the testing repertoire as a diagnostic tool.</p>

<p>4.2	When the technology is evaluated in the setting of saliva samples, using RNA extraction the sensitivity is 98.94% (95% <abbr title="confidence interval">CI</abbr> 96.23% to 99.87) and specificity 99.39% (95% <abbr title="confidence interval">CI</abbr> 99.26% to 99.49%) across all samples tested. The performance of LamPORE  demonstrated excellent specificity and sufficient sensitivity to be used as part of the infection control strategy for SARS-CoV-2.</p>

<p>4.3	In a low-prevalence setting, previous studies (Skitrall et al., CMI 2020) have shown that the positive predictive value (<abbr title="positive predictive value">PPV</abbr>) of RT-qPCR is between 20.1% and 73.8%. In this context, the <abbr title="positive predictive value">PPV</abbr> of LamPORE is line with what has been observed in studies using RT-qPCR as the low prevalence means estimates of <abbr title="positive predictive value">PPV</abbr> can be changed significantly by even very low numbers of false positives.</p>

<p>4.4	In an attempt to understand the accuracy of LamPORE in a high prevalence setting, using retrospective samples from Public Health England <abbr title="influenza-like illness">ILI</abbr> plates, the sensitivity was shown to be 100% with a specificity of 100%. This section of the project was carried out at the end when the testing laboratory had the maximum experience with LamPORE and the high accuracy was probably as a consequence of the experience that the participating laboratories had gained in the technique.</p>

<p>4.5	Early in the study we noted issues with contamination causing spurious false positives in saliva samples, which we found were due to <abbr title="loop-mediated isothermal amplification">LAMP</abbr> contamination and the viscosity of saliva making automated liquid handling challenging. By modifying the provided SOP from Oxford Nanopore, we found these contamination issues were largely remedied. The SOP was modified by adding additional steps to decontaminate plasticware and work surfaces of <abbr title="loop-mediated isothermal amplification">LAMP</abbr> products before each step and the use of separate laboratory areas for each step of the process. The pipetting speed of automated saliva transfer via liquid handling robots was also changed to compensate for saliva viscosity. The technique is vulnerable to contamination and so very careful attention must be paid to the SOP.</p>

<p>4.6	LamPORE is designed to be a qualitative rather than quantitative technique, due to the non-linear amplification properties of <abbr title="loop-mediated isothermal amplification">LAMP</abbr>. Therefore, the reads per sample output of LamPORE must not be used to estimate viral load as it is not suitable for this.</p>

<p>4.7	Operation of LamPORE (and any <abbr title="loop-mediated isothermal amplification">LAMP</abbr> technique) requires a dedicated lab set up according to a standardised SOP in order to prevent problems with contamination. Additionally, data exchange and integration between the GridION instrument and specific Laboratory Information Management Systems (LIMS) is required.</p>

<p>4.8	The key advantage of LamPORE, as part of a testing regime for both saliva and swabs, is its capability to offer high throughput testing of SARS-CoV-2 for both asymptomatic and symptomatic testing. The recommended use case is as a mass testing tool for large asymptomatic populations, either as a static or mobile deployment. It is also suitable as a direct replacement for current NHS  testing methodologies.</p>

<p>4.9	The aim of an asymptomatic healthcare worker testing service is to detect unsuspected COVID-19 and thereafter to reduce the risk of SARS-CoV-2 transmission to other workers, to patients and to workers’ families.  The vulnerability of patients is a key issue here, and such testing offers the opportunity to support infection control measures that are already in place.  An early detection system could also support staff workforce resilience and so maintain NHS capacity.</p>

<p>4.10	Within a use case scenario of regular, frequent, testing of asymptomatic individuals within a population (for example, healthcare worker staff group)  for SARS-CoV-2, the ability to rapidly and accurately identify individuals with high levels of infective virus to facilitate their prompt isolation will be an important contribution to reducing transmission of infection. The LamPORE assay in the RNA mode has demonstrated sufficiency accuracy  in this use case to be utilised in populations of healthcare workers with the aim of curtailing transmission between staff and to patients within healthcare settings.  LamPORE could form part of additional capacity for the national Lighthouse laboratories due to its scale up abilities.</p>

<p>4.11	LamPORE has comparable sensitivity and specificity to the current gold standard assay but with a considerably higher multiplexing capacity, with the ability to incorporate other gene targets such as Influenza and Respiratory Syncytial Virus (RSV) which can form part of a winter respiratory illness screening programme.</p>

<p>4.12	Further development of the LamPORE technology should consider applicability to raw, inactivated samples,  challenges of scaling across differently sized organisations, the adoption of a new technology in clinical laboratories and systems that are unaccustomed to this nucleic acid amplification approach and the information technology integration that is needed to support a quality assured clinical service.</p>

<h2>
<span class="number">5. </span>	Additional data</h2>

<h3>Local verification reports</h3>

<p>5.1	The LamPORE assay has been locally verified in 4 sites, with the activities of each site listed below.</p>

<table>
  <thead>
    <tr>
      <th scope="col">Site</th>
      <th scope="col">RT-LAMP evaluations</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Institute of Cancer &amp; Genomic Science   University of Birmingham   <br> (samples from University Hospitals   Birmingham NHS Foundation Trust)</td>
      <td>- Optimisation        of LamPORE assay and generation of new standard operating procedure        (SOP)    <br> -  Limit        of detection and analytical sensitivity (ASe) and specificity (ASp)  <br>  -  RNA        LamPORE as a screening tool in asymptomatic patients using swabs <br>   -  RNA        LamPORE as a screening tool in asymptomatic patients using saliva  <br>  -  RNA        LamPORE in a high prevalence symptomatic cohort</td>
    </tr>
    <tr>
      <td>Hampshire Hospital Foundation Trust <br>  (Basingstoke and North Hampshire Hospitals site)</td>
      <td>- RNA        LamPORE as a screening tool in asymptomatic patients using swabs   <br>  -  RNA        LamPORE as a screening tool in asymptomatic patients using saliva</td>
    </tr>
    <tr>
      <td>University Hospital Southampton (RNA   extraction) / University of Southampton (Testing)</td>
      <td>-      RNA        LamPORE as a screening tool in asymptomatic patients using saliva</td>
    </tr>
    <tr>
      <td>Manchester   University Foundation Trust</td>
      <td>-    RNA        LamPORE as a screening tool in asymptomatic patients using swabs   <br>  -  RNA        LamPORE as a screening tool in asymptomatic patients using saliva</td>
    </tr>
  </tbody>
</table>

<h2>
<span class="number">6. </span> Sample provenance</h2>

<p>6.1 Data tables showing sample provenance by site and by methodology are available on request from <a class="govuk-link" href="mailto:paul.chambers@dhsc.gov.uk">paul.chambers@dhsc.gov.uk</a>.</p>

</div>


</div>
</div>
  </div>

  <div class="govuk-sticky-element" data-sticky-element="">
    <a href="#contents" class="govuk-link app-c-back-to-top dont-print">
    <svg viewBox="0 0 13 17" width="13" class="app-c-back-to-top__icon" xmlns="http://www.w3.org/2000/svg" height="17">
      <path fill="currentColor" d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z"></path>
    </svg>
    Contents
</a>

    <div class="govuk-sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button data-module="print-link" class="govuk-link gem-c-print-link__button">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>