<main class="html-publication" lang="en" role="main">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          
<div class="gem-c-organisation-logo brand--department-of-health">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color" href="/government/organisations/uk-health-security-agency">
      <span class="gem-c-organisation-logo__name">UK Health<br>Security<br>Agency</span>
</a>
</div>
        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  

<div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
      <span class="govuk-caption-xl gem-c-title__context">
    Open consultation
  </span>


  <h1 class="gem-c-title__text govuk-heading-xl">
    UK Standards for Microbiology Investigations: identification of Shiga toxin-producing Escherichia coli (STEC) including Escherichia coli O157 	
  </h1>
</div>
  <p class="publication-header__last-changed">Published 20 September 2022</p>

  </header>






<div class="sidebar-with-body">
  <div class="govuk-grid-row">
      <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
          <nav aria-label="Contents" class="gem-c-contents-list" data-module="gem-track-click" role="navigation">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 1" data-track-category="contentsClicked" data-track-label="#acknowledgments" data-track-options="{&quot;dimension29&quot;:&quot;Acknowledgments&quot;}" href="#acknowledgments">Acknowledgments</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 2" data-track-category="contentsClicked" data-track-label="#amendment-table" data-track-options="{&quot;dimension29&quot;:&quot;Amendment table&quot;}" href="#amendment-table">Amendment table</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 3" data-track-category="contentsClicked" data-track-label="#general-information" data-track-options="{&quot;dimension29&quot;:&quot;\n1.  General information&quot;}" href="#general-information"><span class="gem-c-contents-list__number">1. </span><span class="gem-c-contents-list__numbered-text">General information</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 4" data-track-category="contentsClicked" data-track-label="#scientific-information" data-track-options="{&quot;dimension29&quot;:&quot;\n2.  Scientific information&quot;}" href="#scientific-information"><span class="gem-c-contents-list__number">2. </span><span class="gem-c-contents-list__numbered-text">Scientific information</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 5" data-track-category="contentsClicked" data-track-label="#scope-of-document" data-track-options="{&quot;dimension29&quot;:&quot;\n3.  Scope of document&quot;}" href="#scope-of-document"><span class="gem-c-contents-list__number">3. </span><span class="gem-c-contents-list__numbered-text">Scope of document</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 6" data-track-category="contentsClicked" data-track-label="#introduction" data-track-options="{&quot;dimension29&quot;:&quot;\n4.  Introduction&quot;}" href="#introduction"><span class="gem-c-contents-list__number">4. </span><span class="gem-c-contents-list__numbered-text">Introduction</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 7" data-track-category="contentsClicked" data-track-label="#technical-information-and-limitations" data-track-options="{&quot;dimension29&quot;:&quot;\n5.  Technical information and limitations&quot;}" href="#technical-information-and-limitations"><span class="gem-c-contents-list__number">5. </span><span class="gem-c-contents-list__numbered-text">Technical information and limitations</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 8" data-track-category="contentsClicked" data-track-label="#safety-considerations" data-track-options="{&quot;dimension29&quot;:&quot;\n6.  Safety considerations&quot;}" href="#safety-considerations"><span class="gem-c-contents-list__number">6. </span><span class="gem-c-contents-list__numbered-text">Safety considerations</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 9" data-track-category="contentsClicked" data-track-label="#target-organisms" data-track-options="{&quot;dimension29&quot;:&quot;\n7.  Target organisms&quot;}" href="#target-organisms"><span class="gem-c-contents-list__number">7. </span><span class="gem-c-contents-list__numbered-text">Target organisms</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 10" data-track-category="contentsClicked" data-track-label="#identification" data-track-options="{&quot;dimension29&quot;:&quot;\n8.  Identification&quot;}" href="#identification"><span class="gem-c-contents-list__number">8. </span><span class="gem-c-contents-list__numbered-text">Identification</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 11" data-track-category="contentsClicked" data-track-label="#reporting" data-track-options="{&quot;dimension29&quot;:&quot;\n9.  Reporting&quot;}" href="#reporting"><span class="gem-c-contents-list__number">9. </span><span class="gem-c-contents-list__numbered-text">Reporting</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 12" data-track-category="contentsClicked" data-track-label="#referral-to-reference-laboratories" data-track-options="{&quot;dimension29&quot;:&quot;\n10.  Referral to reference laboratories&quot;}" href="#referral-to-reference-laboratories"><span class="gem-c-contents-list__number">10. </span><span class="gem-c-contents-list__numbered-text">Referral to reference laboratories</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 13" data-track-category="contentsClicked" data-track-label="#algorithm-identification-of-shiga-toxin-producing-escherichia-coli-including-e-coli-o157" data-track-options="{&quot;dimension29&quot;:&quot;Algorithm: Identification of Shiga toxin-producing Escherichia coli including E. coli O157&quot;}" href="#algorithm-identification-of-shiga-toxin-producing-escherichia-coli-including-e-coli-o157">Algorithm: Identification of Shiga toxin-producing Escherichia coli including E. coli O157</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 14" data-track-category="contentsClicked" data-track-label="#references" data-track-options="{&quot;dimension29&quot;:&quot;References&quot;}" href="#references">References</a>

        </li>
    </ol>
</nav>

        
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-ga4="{&quot;event_name&quot;:&quot;print_page&quot;,&quot;type&quot;:&quot;Print this page&quot;,&quot;index&quot;:1,&quot;index_total&quot;:1,&quot;section&quot;:&quot;Contents&quot;}" data-module="ga4-event-tracker print-link">Print this page</button>
</div>
      </div>

    <div class="print-wrapper">
      <div class="print-meta-data">
        <p>
  <img class="print-meta-data-licence" src="/assets/government-frontend/open-government-licence-min-93b6a51b518ff99714a1aa2a7d2162735c155ec3cb073c75fb88b2a332fa83d3.png">
</p>
<p>
  © Crown copyright 2022
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/consultations/uk-smi-id-22-identification-of-shiga-toxin-producing-escherichia-coli-stec-including-escherichia-coli-o157/uk-standards-for-microbiology-investigations-identification-of-shiga-toxin-producing-escherichia-coli-stec-including-escherichia-coli-o157
</p>


      </div>
    </div>

    <div class="main-content-container">
      <div class="gem-c-govspeak-html-publication">
  
<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
    
    
        <div class="govspeak">
<h2>Acknowledgments</h2>

<p>UK Standards for Microbiology Investigations (<abbr title="UK Standards for Microbiology Investigations">UK SMIs</abbr>) are developed under the auspices of UKHSA working in partnership with the partner organisations whose logos are displayed below and listed on <a class="govuk-link" href="https://www.gov.uk/uk-standards-for-microbiology-investigations-smi-quality-and-consistency-in-clinical-laboratories">the UK SMI website</a>. <abbr title="UK Standards for Microbiology Investigations">UK SMIs</abbr> are developed, reviewed and revised by various working groups which are overseen by a steering committee (see <a class="govuk-link" href="https://www.gov.uk/government/groups/standards-for-microbiology-investigations-steering-committee">the Steering Committee page on GOV.UK</a>).</p>

<p>The contributions of many individuals in clinical, specialist and reference laboratories who have provided information and comments during the development of this document are acknowledged. We are grateful to the medical editors for editing the medical content.</p>

<p><abbr title="UK Standards for Microbiology Investigations">UK SMIs</abbr> are produced in association with:</p>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/161423/SMI_logos.png"></div></figure>

<ul>
  <li>The Association for Clinical Biochemistry Microbiology Group</li>
  <li>British Infection Association (BIA)</li>
  <li>British Society for Antimicrobial Chemotherapy (BSAC)</li>
  <li>British Society for Medical Mycology</li>
  <li>British Society For Microbial Technology (BSMT)</li>
  <li>British Society for Parasitology</li>
  <li>Welsh Microbiological Association (WMA)</li>
  <li>Healthcare Infection Society</li>
  <li>Public Health Agency</li>
  <li>Institute of Biomedical Science (IBMS)</li>
  <li>Microbiology Society</li>
  <li>National Services Scotland</li>
  <li>Northern Ireland Microbiology Forum and Audit Group</li>
  <li>Public Health Scotland</li>
  <li>Public Health Wales</li>
  <li>The Royal College of Pathologists</li>
  <li>Society of Anaerobic Microbiology (SAM)</li>
  <li>Society for Applied Microbiology (SfAM)</li>
  <li>Scottish Microbiology and Virology Network</li>
  <li>UK Clinical Mycology Network (UKCMN)</li>
  <li>The UK Clinical Virology Network</li>
</ul>

<h2>Amendment table</h2>

<p>Each UK SMI document has an individual record of amendments. The amendments are listed on this page. The amendment history is available from <a class="govuk-link" href="mailto:standards@ukhsa.gov.uk">standards@ukhsa.gov.uk</a>.</p>

<p>Any alterations to this document should be controlled in accordance with the local document control process.</p>

<table>
  <tbody>
    <tr>
      <td>Amendment number/date</td>
      <td>x/dd.mm.yy</td>
    </tr>
    <tr>
      <td>Issue number discarded</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>Insert issue number</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>Anticipated next review date*</td>
      <td>dd.mm.yy</td>
    </tr>
    <tr>
      <td>Section(s) involved</td>
      <td>Amendment</td>
    </tr>
    <tr>
      <td>&nbsp;</td>
      <td>Whole document has been placed into a new template</td>
    </tr>
    <tr>
      <td>Title</td>
      <td>Changed from ‘Identification of vero toxin-producing Escherichia coli including Escherichia coli O157’ to ‘Identification of Shiga toxin-producing Escherichia coli including Escherichia coli O157’</td>
    </tr>
    <tr>
      <td>Introduction</td>
      <td>Additional information added about other E.coli and more cases added.</td>
    </tr>
  </tbody>
</table>

<p>*Reviews can be extended up to 5 years where appropriate</p>

<h2>
<span class="number">1. </span> General information</h2>

<p><a class="govuk-link" href="https://www.gov.uk/guidance/uk-standards-for-microbiology-investigations-smi-quality-and-consistency-in-clinical-laboratories">View general information</a> related to <abbr title="UK Standards for Microbiology Investigations">UK SMIs</abbr>.</p>

<h2>
<span class="number">2. </span> Scientific information</h2>

<p><a class="govuk-link" href="https://www.gov.uk/guidance/uk-standards-for-microbiology-investigations-uk-smi-scientific-information">View scientific information</a> related to <abbr title="UK Standards for Microbiology Investigations">UK SMIs</abbr>.</p>

<h2>
<span class="number">3. </span> Scope of document</h2>

<p>This UK Standards for Microbiology Investigations (UK SMI) document describes the identification of presumptive Shiga toxin-producing Escherichia coli O157 (<abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157) isolated from faeces. These strains are associated with a wide spectrum of disease including haemolytic uraemic syndrome (<abbr title="haemolytic uraemic syndrome">HUS</abbr>).</p>

<p>This UK SMI includes both biochemical tests and automated methods for the identification of microorganisms. Some biochemical tests may not be done routinely in laboratory except in cases where confirmation by an alternative technique is required or automated methods are not available.</p>

<p><abbr title="UK Standards for Microbiology Investigations">UK SMIs</abbr> should be used in conjunction with other relevant <abbr title="UK Standards for Microbiology Investigations">UK SMIs</abbr>.</p>

<h2>
<span class="number">4. </span> Introduction</h2>

<p>Shiga toxin-producing E. coli (<abbr title="Shiga toxin-producing E. coli">STEC</abbr>), also known as verocytotoxin producing E. coli (<abbr title="verocytotoxin producing E. coli">VTEC</abbr>), are a group of zoonotic, foodborne pathogenic E. coli characterised by the presence of the Shiga toxin gene (<abbr title="Shiga toxin gene">stx</abbr>). <abbr title="Shiga toxin-producing E. coli">STEC</abbr> can produce stx1 (4 subtypes 1a–1d) and stx2 (7 subtypes stx2a–2g). The presence of stx2 (specifically stx2a) is more likely to cause severe disease (<a class="govuk-link" href="#ref1">1</a>), (<a class="govuk-link" href="#ref2">2</a>).</p>

<p><abbr title="Shiga toxin-producing E. coli">STEC</abbr> can cause gastrointestinal symptoms (including diarrhoea which is often blood-stained), abdominal pain, nausea and vomiting. The incubation period is usually between 2 to 4 days (<a class="govuk-link" href="#ref3">3</a>) but can be from 1 to 10 days. Symptoms usually last for 4 to 7 days but can be longer. Following the initial infection, a subset of patients develop haemolytic uraemia syndrome (<abbr title="haemolytic uraemic syndrome">HUS</abbr>), a severe systemic condition that affects the kidneys. This can sometimes cause neurological and/or cardiac complications, which can be fatal (<a class="govuk-link" href="#ref2">2</a>), (<a class="govuk-link" href="#ref4">4</a>).</p>

<p>There are 4 other clinically important pathotypes of E. coli that cause gastrointestinal symptoms:</p>

<ul>
  <li>Enterotoxigenic E. coli (<abbr title="Enterotoxigenic E. coli">ETEC</abbr>) are important causes of diarrhoea in both humans and domestic animals – infections are seen in individuals returning from regions of endemicity and accounts for 20 to 40% of traveler’s diarrhea (<a class="govuk-link" href="#ref5">5</a>)</li>
  <li>Enteropathogenic&nbsp;E.&nbsp;coli&nbsp;(<abbr title="Enteropathogenic&nbsp;E.&nbsp;coli">EPEC</abbr>) is associated with persistent diarrhoea in young children – close contact such as in day care facilities and poor hygiene increases the risk of transmission (<a class="govuk-link" href="#ref5">5</a>)</li>
  <li>Enteroaggregative&nbsp;E.&nbsp;coli&nbsp;(<abbr title="Enteroaggregative&nbsp;E.&nbsp;coli">EAEC</abbr>) infection is greatest in children living in endemicity</li>
  <li>Enteroinvasive&nbsp;E.&nbsp;coli&nbsp;(<abbr title="Enteroinvasive&nbsp;E.&nbsp;coli">EIEC</abbr>) are very similar to Shigella – they are capable of invading and multiplying in the intestinal epithelial cells of the distal large bowel in humans</li>
</ul>

<p>Ruminants are the main animal reservoir for <abbr title="Shiga toxin-producing E. coli">STEC</abbr>. In the UK, a high proportion of cattle and sheep are colonised with <abbr title="Shiga toxin-producing E. coli">STEC</abbr>. Small mammals, birds and domestic pets can act as transient carriers. Transmission to humans can occur via direct contact with animals or their environment, and by consumption of contaminated food or water. Outbreaks of <abbr title="Shiga toxin-producing E. coli">STEC</abbr> infection are common and exposure risks include activities in rural settings, visiting petting farms, consumption of unpasteurised dairy products, undercooked meat and ready-to-eat produce (<a class="govuk-link" href="#ref2">2</a>), (<a class="govuk-link" href="#ref4">4</a>).</p>

<p><abbr title="Shiga toxin-producing E. coli">STEC</abbr> has a low infectious dose and can spread by person-to-person contact, especially within households and schools. Once symptoms resolve, some people shed <abbr title="Shiga toxin-producing E. coli">STEC</abbr> in their faeces for weeks following infection. To mitigate the risk of transmission, children aged 5 years old and under, and all those in risk groups (for example food handlers, carers of the elderly, immunocompromised and those with underlying health conditions) are excluded from work, school and childcare settings until they are microbiologically clear (<a class="govuk-link" href="#ref3">3</a>).</p>

<p><abbr title="Shiga toxin-producing E. coli">STEC</abbr> was identified as a cause of <abbr title="haemolytic uraemic syndrome">HUS</abbr> in the 1980s and early outbreaks of <abbr title="Shiga toxin-producing E. coli">STEC</abbr>-<abbr title="haemolytic uraemic syndrome">HUS</abbr> in the UK were linked to infection with <abbr title="Shiga toxin-producing E. coli">STEC</abbr> serotype O157:H7. Consequently, laboratory protocols focused on the detection and identification of this specific serotype in faecal specimens (<a class="govuk-link" href="#ref6">6</a>), (<a class="govuk-link" href="#ref7">7</a>). Unlike the majority of E. coli, <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157:H7 do not ferment sorbitol and this characteristic is used to identify <abbr title="Shiga toxin-producing E. coli">STEC</abbr> on selective media.</p>

<p><abbr title="Shiga toxin-producing E. coli">STEC</abbr>&nbsp;infections can present as sporadic cases or as large national and multinational outbreaks. A national outbreak of <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 phage type 8 (<abbr title="phage type 8">PT8</abbr>) was reported in 2005, involving stx1 + 2, associated with handling raw beef or consuming beef products at home (<a class="govuk-link" href="#ref8">8</a>).</p>

<p>In 2010 <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 <abbr title="phage type 8">PT8</abbr> phage type reported. One case was linked to handling of raw leeks and potatoes. It was reported 252 cases of <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 <abbr title="phage type 8">PT8</abbr> stx1 + 2 infection occurred in England, Scotland and Wales. The Investigation identified that transmission was linked to the consumption or handling of leeks and potatoes (<a class="govuk-link" href="#ref8">8</a>).</p>

<p>The first outbreak of E. coli O157 phage type 32 was identified in Scotland in 2015. There were 12 cases reported which were linked to the consumption of commercially produced venison products (<a class="govuk-link" href="#ref9">9</a>).</p>

<p>In the UK between August to September 2013 an outbreak of <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 phage type 2, stx2 was reported from the consumption of watercress. The investigation suggested the possible routes of contamination included agricultural run-off into the watercress, contamination of water or growing materials used in watercress production or contaminated watercress seeds (<a class="govuk-link" href="#ref10">10</a>).</p>

<p>In 2019 a total of 539 confirmed cases of <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 were reported in England and Wales. Children aged 1 to 4 years had the highest incidence of infection. Nearly one-third of confirmed <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 cases in England were hospitalised and 3% were reported to have developed <abbr title="haemolytic uraemic syndrome">HUS</abbr> (<a class="govuk-link" href="#ref2">2</a>), (<a class="govuk-link" href="#ref11">11</a>).</p>

<p>As laboratories are implementing gastrointestinal diagnostics using polymerase chain reaction (<abbr title="polymerase chain reaction">PCR</abbr>), there has been an increase in the detection of non-O157 <abbr title="Shiga toxin-producing E. coli">STEC</abbr>. In 2019, 768 culture-positive non-O157 <abbr title="Shiga toxin-producing E. coli">STEC</abbr> cases (655 in England, 113 in Wales) were reported compared to 89 cases reported between 2009 and 2013 (<a class="govuk-link" href="#ref2">2</a>).</p>

<p>In England, the 5 most common non-O157 serogroups isolated in 2019 were O26 (109/655, 17%) followed by O146 (98/655, 15%), O91 (81/655, 12%), O128ab (37/655, 6%) and O103 (28/655, 4%) (<a class="govuk-link" href="#ref2">2</a>).</p>

<p>Accurate and rapid diagnosis of <abbr title="Shiga toxin-producing E. coli">STEC</abbr> infections is important for the appropriate management of infected patients and for implementation of proper public health intervention.</p>

<h2>
<span class="number">5. </span> Technical information and limitations</h2>

<h3>Commercial identification systems</h3>

<p>Laboratories should follow manufacturer’s instructions when using these kits. It is essential that all commercial kits have evidence of adequate validation demonstrating they are fit for purpose. It is also essential that appropriate on-going quality assurance procedures should be in place.</p>

<p>Some commercial biochemical tests may give a doubtful or a low percentage profile for E. coli O157 because the fermentation of sorbitol is heavily weighted for the identification of E. coli strains, and care must be taken with the interpretation of the profile (<a class="govuk-link" href="#ref12">12</a>). However, all presumptive E. coli O157 from human and non-human sources should be referred to the appropriate specialist laboratories for confirmation.</p>

<h3>Agglutination test</h3>

<p>Escherichia hermannii is sorbitol negative, cross-reacts and agglutinates in E. coli serotype O157 antiserum, and thus it could be mistaken for E. coli O157. The use of the commercial latex screen in conjunction with a combination of cultures on sorbitol MacConkey agar (<abbr title="sorbitol MacConkey agar">SMAC</abbr>) or cefixime-tellurite sorbitol (<abbr title="cefixime-tellurite sorbitol">CTSMAC</abbr>) agar should prove to be useful for rapid detection of E. coli serotype O157 (<a class="govuk-link" href="#ref13">13</a>).</p>

<h3>Culture methods</h3>

<p>Culture methods can be time consuming and limit sensitivity for <abbr title="Shiga toxin-producing E. coli">STEC</abbr> detection. Other approaches should be considered where possible. Molecular methods have a high sensitivity and specificity for all <abbr title="Shiga toxin-producing E. coli">STEC</abbr> serotypes (<a class="govuk-link" href="#ref14">2</a>).</p>

<h3>Matrix-assisted laser desorption/ionisation time of flight (<abbr title="Matrix-assisted laser desorption/ionisation time of flight">MALDI-TOF MS</abbr>) mass spectrometry</h3>

<p>One of the limitations is the current inability of <abbr title="Matrix-assisted laser desorption/ionisation time of flight">MALDI-TOF MS</abbr> to reliably distinguish pathogenic from non-pathogenic E. coli isolates. In addition, numerous reports have also described the difficulty encountered when trying to differentiate E. coli from Shigella species. In addition, further research is needed to adapt <abbr title="Matrix-assisted laser desorption/ionisation time of flight">MALDI-TOF MS</abbr> technology to be suitable for strain-specific identifications of E. coli isolates (<a class="govuk-link" href="#ref15">15</a>).</p>

<h2>
<span class="number">6. </span> Safety considerations</h2>

<p>The section covers specific safety considerations (<a class="govuk-link" href="#ref16">16 to 37</a>) related to this UK SMI, and should be read in conjunction with the general <a class="govuk-link" href="https://www.gov.uk/guidance/uk-standards-for-microbiology-investigations-uk-smi-scientific-information#safety-considerations">safety considerations on GOV.UK</a>.</p>

<p><abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 is a Hazard Group 3 organism.</p>

<p>All work with suspected isolates of <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 must be performed under Containment level 3 conditions.</p>

<p><abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 is highly infectious and as few as 10 viable organisms are required for an infective dose (<a class="govuk-link" href="#ref30">30</a>). <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 may cause severe illness that is sometimes fatal. Laboratory acquired infections have been reported (<a class="govuk-link" href="#ref39">39 to 42</a>). However, there is no knowledge of any E. coli O157 vaccine currently commercially available in the UK or indeed licensed for use. Refer to <a class="govuk-link" href="https://www.gov.uk/government/publications/immunisation-against-infectious-disease-the-green-book-front-cover-and-contents-page">immunisation against infectious disease: the green book</a> for further guidance.</p>

<p>Laboratory procedures that give rise to infectious aerosols must be conducted in a microbiological safety cabinet.</p>

<p>The above guidance should be supplemented with local COSHH and risk assessments.</p>

<p>Compliance with postal and transport regulations is essential.</p>

<h2>
<span class="number">7. </span> Target organisms</h2>

<p>E. coli O157:H7 and E. coli O157: <abbr title="(non-motile)">NM</abbr> (non-motile) also known as E. coli O157:H negative and other non-O157 serogroups O26, O45, O103, 0111, O121 and O145.</p>

<p>Other E. coli serotypes like O145 can act like O157:H7 if they acquire the ability to produce Shiga toxin. These other E. coli serotypes can also cause outbreaks of bloody diarrhoea with haemorrhagic colitis that can become complicated by <abbr title="haemolytic uraemic syndrome">HUS</abbr>.</p>

<p>This procedure may result in the identification of isolates of presumptive E. coli O157 that do not produce Shiga toxin and some organisms that give equivocal results.</p>

<h2>
<span class="number">8. </span> Identification</h2>

<h3>
<span class="number">8.1 </span> Microscopic appearance</h3>

<p>Gram stain (<a class="govuk-link" href="https://www.gov.uk/government/collections/standards-for-microbiology-investigations-smi#test-procedures">TP 39 – Staining procedures</a>).</p>

<p>Gram negative rods.</p>

<h3>
<span class="number">8.2 </span> Primary isolation media</h3>

<p><abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157:H7 do not ferment sorbitol and this characteristic is used to identify presumptive isolates of <abbr title="Shiga toxin-producing E. coli">STEC</abbr> on selective media <abbr title="cefixime-tellurite sorbitol">CTSMAC</abbr>. Non-sorbitol fermenting colonies are then agglutinated with antisera to the O157 antigen, and biochemical tests. Positive colonies are submitted to the Reference laboratory for confirmation and typing.</p>

<p>Some commercial biochemical tests may give a doubtful or a low percentage profile for E. coli O157 because the fermentation of sorbitol is heavily weighted for the identification of E. coli strains, and care must be taken with the interpretation of the profile (<a class="govuk-link" href="#ref12">12</a>).</p>

<p>Some <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 strains have been found to ferment sorbitol and to be β-glucuronidase positive (<a class="govuk-link" href="#ref12">12</a>).</p>

<p>Primary isolation media includes:</p>

<ul>
  <li>
<abbr title="cefixime-tellurite sorbitol">CTSMAC</abbr> agar incubated in air at 35 to 37°C for 16 to 24 hour – <abbr title="cefixime-tellurite sorbitol">CTSMAC</abbr> agar is used since classical sorbitol non-fermenting <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 are relatively resistant to potassium tellurite compared with other E. coli (there is a risk of isolation failure on <abbr title="sorbitol MacConkey agar">SMAC</abbr> agar lacking cefixime and tellurite)</li>
  <li>chromogenic selective agar can be used for non-O157 serogroups O26, O45, O103, 0111, O121 and O145 (<a class="govuk-link" href="#ref43">43</a>)</li>
  <li>TBX agar is used to detect all E.coli strains (<a class="govuk-link" href="#ref44">44</a>)</li>
  <li>enrichment culture in modified tryptone soya broth (MTSB) may be required in cases of outbreaks</li>
</ul>

<p>Where the clinical evidence is suggestive of <abbr title="Shiga toxin-producing E. coli">STEC</abbr> infection (particularly in children under 15 years and adults over 65 years) and no presumptive sorbitol non-fermenting E. coli O157 colonies are observed on <abbr title="sorbitol MacConkey agar">SMAC</abbr> or <abbr title="cefixime-tellurite sorbitol">CTSMAC</abbr> agar, clinical laboratories should:</p>

<ul>
  <li>test sorbitol fermenting colonies for agglutination with E. coli O157 antiserum</li>
  <li>confirm the identification of agglutination positive O157 colonies as E. coli</li>
  <li>send presumptive isolate(s) to the Reference Laboratory for confirmation</li>
</ul>

<p>Culture methods to detect <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 by its inability to ferment sorbitol on selective media (MacConkey agar). However, non-O157 <abbr title="Shiga toxin-producing E. coli">STEC</abbr> ferment sorbitol and there is no culture method to differentiate non-O157 <abbr title="Shiga toxin-producing E. coli">STEC</abbr> from non-pathogenic E.coli in frontline laboratories (<a class="govuk-link" href="#ref45">45</a>).</p>

<p>Chromogenic identification plates are commercially available and have been evaluated for certain clinical samples. The use of chromogenic agar may be of value in the isolation and confirmation of E. coli (<abbr title="Enteropathogenic&nbsp;E.&nbsp;coli">EPEC</abbr>, EHEC, <abbr title="Shiga toxin-producing E. coli">STEC</abbr>).</p>

<p>Confirmation for the <abbr title="Shiga toxin gene">stx</abbr> gene is required for colonies identified as potential <abbr title="Shiga toxin-producing E. coli">STEC</abbr>.</p>

<h3>
<span class="number">8.3 </span> Colonial appearance</h3>

<p>On <abbr title="cefixime-tellurite sorbitol">CTSMAC</abbr> agar, typical <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 colonies are smooth, colourless or slightly greyish in appearance which may appear with an orange-coloured halo and are 2 to 3mm in diameter. Some rare variant strains of <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 ferment sorbitol and may grow poorly on <abbr title="cefixime-tellurite sorbitol">CTSMAC</abbr>/<abbr title="sorbitol MacConkey agar">SMAC</abbr>.</p>

<p>Note: Although <abbr title="cefixime-tellurite sorbitol">CTSMAC</abbr> offers a degree of selection for presumptive <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157, growth of other organisms may be observed. Mixed growth from faecal specimens may contain other sorbitol non fermenters. See table below.</p>

<table>
  <thead>
    <tr>
      <th scope="col">Organism</th>
      <th scope="col">Colour and size of colonies on CTSMAC</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Shigella flexneri</td>
      <td>Pink colonies. 0.5-1mm diameter</td>
    </tr>
    <tr>
      <td>Salmonella Typhimurium</td>
      <td>Pale pink pinpoint colonies</td>
    </tr>
    <tr>
      <td>E. coli (non-O157)</td>
      <td>Generally sorbitol fermenters. Pink colonies. Pinpoint to 0.25mm diameter</td>
    </tr>
  </tbody>
</table>

<p>TBX agar-colonies are blue-green colour indicating the presence of β-glucuronidase activity.</p>

<h3>
<span class="number">8.4 </span> Test procedures</h3>

<h4>8.4.1 Oxidase test</h4>

<p>(<a class="govuk-link" href="https://www.gov.uk/government/collections/standards-for-microbiology-investigations-smi#test-procedures">TP 26 – Oxidase test</a>)</p>

<p>E. coli O157:H7 is oxidase negative. Screening with oxidase test may be helpful and should be done on a media containing non-fermentable carbohydrates.</p>

<h4>8.4.2 Agglutination test</h4>

<p>Use <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 antiserum (latex or other commercial reagent). It is important to perform the appropriate control for autoagglutination.</p>

<p><abbr title="Shiga toxin-producing E. coli">STEC</abbr> O26 is the most common non-O157 <abbr title="Shiga toxin-producing E. coli">STEC</abbr> in the UK and suspect colonies on selective agar can be tested with antisera to the O26 antigen, to improve specificity.</p>

<h4>8.4.3 Biochemical tests</h4>

<p>Commercial identification systems:</p>

<p>Laboratories should follow manufacturer’s instructions and rapid tests and kits should be validated and be shown to be fit for purpose prior to use.</p>

<p>Subculture to lactose containing media:</p>

<p>This may be the purity plate from the commercial identification kit. Chromogenic identification plates are available and may be valuable as an alternative for confirmation of identification of E. coli.</p>

<p><abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 is almost always lactose positive but rare isolates have been found to be lactose non-fermenters.</p>

<h4>8.4.4 Matrix-assisted laser desorption/ionisation time of flight (MALDI-TOF) mass spectrometry</h4>

<p><abbr title="Matrix-assisted laser desorption/ionisation time of flight">MALDI-TOF MS</abbr> can be used to analyse the protein composition of a bacterial cell, that has emerged as a new technology for species identification. This has been shown to be a rapid and powerful tool because of its reproducibility, speed and sensitivity of analysis. The advantage of MALDI-TOF as compared with other identification methods is that the results of the analysis are available within a few hours rather than several days. The speed and the simplicity of sample preparation and result acquisition associated with minimal consumable costs make this method well suited for routine and high-throughput use (<a class="govuk-link" href="#ref46">46</a>).</p>

<h4>8.4.5 Molecular testing</h4>

<p>Polymerase chain reaction (<abbr title="polymerase chain reaction">PCR</abbr>) is usually considered to be a good method for bacterial detection as it is simple, rapid, sensitive and specific. The basis for <abbr title="polymerase chain reaction">PCR</abbr> diagnostic applications in microbiology is the detection of infectious agents and the differentiation of non-pathogenic from pathogenic strains by virtue of specific genes.</p>

<p>The implementation of <abbr title="polymerase chain reaction">PCR</abbr> assays targeting common enteric pathogens has detected an increasing number of non <abbr title="Shiga toxin-producing E. coli">STEC</abbr> serotypes in the UK (<a class="govuk-link" href="#ref47">47</a>). Real time <abbr title="polymerase chain reaction">PCR</abbr> has excellent sensitivity and specificity. Multiplex assays are able to detect and differentiate between Stx1 and Stx2 and other pathogens (<a class="govuk-link" href="#ref14">14</a>).</p>

<p>Commercial real-time <abbr title="polymerase chain reaction">PCR</abbr> systems have been developed which selects for Shiga toxin genes, accelerates more sensitive detection compared with traditional culture-based methods (<a class="govuk-link" href="#ref43">43</a>) and validated for simultaneous detection of bacterial enteric pathogens either directly from faeces without any pre-enrichment or following overnight pre-enrichment (<a class="govuk-link" href="#ref48">48 to 50</a>).</p>

<p>Following a positive <abbr title="polymerase chain reaction">PCR</abbr> result indicating the presence of <abbr title="Shiga toxin-producing E. coli">STEC</abbr>, the faecal specimen should be cultured on <abbr title="cefixime-tellurite sorbitol">CTSMAC</abbr> and/or chromogenic agar selecting for <abbr title="Shiga toxin-producing E. coli">STEC</abbr>.</p>

<h3>
<span class="number">8.5 </span> Further identification</h3>

<h4>8.5.1 Rapid molecular methods</h4>

<p>Molecular methods have had an enormous impact on the taxonomy of Escherichia. Analysis of gene sequences has increased understanding of the phylogenetic relationships of Escherichia and related organisms; and has resulted in the recognition of numerous new species and strains. Molecular techniques have made identification of species more rapid and precise than is possible with for phenotypic techniques.</p>

<h4>8.5.2 Whole genome sequencing (<abbr title="Whole genome sequencing">WGS</abbr>)</h4>

<p>This is also known as ‘full genome sequencing, complete genome sequencing, or entire genome sequencing’. It is a laboratory process that determines the complete DNA sequence of an organism’s genome at a single time. There are several high-throughput techniques that are available and used to sequence an entire genome such as pyrosequencing, nanopore technology, IIIumina sequencing, Ion Torrent sequencing, and so on. This sequencing method holds great promise for rapid, accurate, and comprehensive identification of bacterial transmission pathways in hospital and community settings, with concomitant reductions in infections, morbidity, and costs.</p>

<p><abbr title="Whole genome sequencing">WGS</abbr> has been successfully used for public health surveillance and outbreak detection by performing sequencing of bacterial genomes at low cost, which has already been demonstrated for several gastrointestinal pathogens (<a class="govuk-link" href="#ref51">51</a>).</p>

<p>This method streamlines the detection and typing workflow by replacing multiple tests required for current investigations. <abbr title="Whole genome sequencing">WGS</abbr> provides better accuracy in comparison to existing methods such as pulse field gel electrophoresis (<abbr title="pulse field gel electrophoresis">PFGE</abbr>) and multiple locus variable analysis (<abbr title="multiple locus variable analysis">MLVA</abbr>) (<a class="govuk-link" href="#ref14">14</a>).</p>

<p><abbr title="Whole genome sequencing">WGS</abbr> demonstrates unparalleled sensitivity and accuracy in identifying linked cases coupled with phylogenetic clustering of how strains are related over time and space. Its ability to accurately define sporadic cases over time enables better characterisation of the population at risk and to assess the relative importance of exposures leading to sporadic infections, which may differ from those leading to outbreaks (<a class="govuk-link" href="#ref52">52</a>).</p>

<h3>
<span class="number">8.6 </span> Storage and referral</h3>

<p>All isolates of presumptive (locally confirmed) E. coli O157 (sorbitol non fermenters or sorbitol fermenting) should be saved on nutrient agar slopes. Cultures should be referred promptly to the Reference Laboratory for confirmation of identification, biochemical confirmation, detection of <abbr title="Shiga toxin gene">stx</abbr> genes, serotyping and phage typing.</p>

<p>All identification tests should ideally be performed from non-selective agar to consider the variations that may occur with biochemical tests such as sorbitol fermentation.</p>

<p>Faecal samples from cases with appropriate clinical symptoms from whom <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 has not been isolated should be submitted to the Reference Laboratory for detection of <abbr title="Shiga toxin-producing E. coli">STEC</abbr> strains belonging to serogroups other than O157 by culture and DNA-based methods.</p>

<h2>
<span class="number">9. </span> Reporting</h2>

<h3>
<span class="number">9.1 </span> Infection specialist</h3>

<p>Inform the medical microbiologist of presumptive or confirmed E. coli O157 strains.</p>

<p>According to local protocol, the medical microbiologist should also be informed if the request bears relevant information which suggests infection with E. coli O157 for example:</p>

<ul>
  <li>enterocolitis (especially if complicated by severe dehydration, anaemia, haemolytic-uraemic syndrome, neurological dysfunction and/or confusional states)</li>
  <li>recent travel, farming (or visits to farms)</li>
  <li>veterinary or laboratory work</li>
  <li>food poisoning</li>
  <li>investigation of outbreak situations</li>
</ul>

<p>Follow local protocols for reporting to clinician.</p>

<h3>
<span class="number">9.2 </span> Preliminary identification</h3>

<p>Presumptive identification of E. coli O157 is based on appropriate growth characteristics, biochemical tests, colonial appearance and agglutination with O157 antiserum or commercial antigen kits.</p>

<h3>
<span class="number">9.3 </span> Confirmation of identification</h3>

<p>For confirmation and identification please see <a class="govuk-link" href="https://www.gov.uk/guidance/specialist-and-reference-microbiology-laboratory-tests-and-services">Specialist and reference microbiology: laboratory tests and services page on GOV.UK</a> for reference laboratory user manuals and request forms.</p>

<h3>
<span class="number">9.4 </span> Health Protection Team (<abbr title="Health Protection Team">HPT</abbr>)</h3>

<p>Refer to local agreements in devolved administrations.</p>

<h3>
<span class="number">9.5 </span> UK Health Security Agency</h3>

<p>Refer to current guidelines on Second Generation Surveillance System&nbsp;(<abbr title="Second Generation Surveillance System">SGSS</abbr>) reporting (<a class="govuk-link" href="#ref32">32</a>).</p>

<p><abbr title="Shiga toxin-producing E. coli">STEC</abbr> is notifiable under the Public Health (Control of Diseases) Act 1984 and the Health Protection (Notification) Regulations 2010. In England, local diagnostic laboratories report presumptive cases of <abbr title="Shiga toxin-producing E. coli">STEC</abbr> to the UK Health Security Agency Health Protection Teams (UKHSA HPTs) and then refer samples to the reference laboratory for confirmation and further testing (<a class="govuk-link" href="#ref11">11</a>).</p>

<h3>
<span class="number">9.6 </span> Infection prevention and control team</h3>

<p>Inform the infection prevention and control team of presumptive and confirmed isolates of E. coli O157 and other <abbr title="Shiga toxin-producing E. coli">STEC</abbr> causing a clinical picture characteristic of infection with E. coli O157.</p>

<h2>
<span class="number">10. </span> Referral to reference laboratories</h2>

<p>For information on the tests offered, turnaround times, transport procedure and the other requirements of the reference laboratory <a class="govuk-link" href="https://www.gov.uk/guidance/specialist-and-reference-microbiology-laboratory-tests-and-services">see user manuals and request forms</a>.</p>

<p>Contact appropriate reference laboratory for information on the tests available, turnaround times, transport procedure and any other requirements for sample submission:</p>

<ul>
  <li><a class="govuk-link" href="https://www.gov.uk/guidance/specialist-and-reference-microbiology-laboratory-tests-and-services">England</a></li>
  <li><a class="govuk-link" href="https://phw.nhs.wales/" rel="external">Wales</a></li>
  <li><a class="govuk-link" href="http://www.hps.scot.nhs.uk/reflab/index.aspx" rel="external">Scotland</a></li>
  <li><a class="govuk-link" href="http://www.publichealth.hscni.net/directorate-public-health/health-protection" rel="external">Northern Ireland</a></li>
</ul>

<p>Note: In case of sending away to laboratories for processing, ensure that specimen is placed in appropriate package and transported accordingly.</p>

<h2>Algorithm: Identification of Shiga toxin-producing Escherichia coli including E. coli O157</h2>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/162078/ID_22_Flowchart_in_SVG.svg"></div></figure>

<p>The flowchart is for guidance only.</p>

<p>Note: Refer to clinical details – in cases and particularly clusters of cases where isolation or identification fails, but the symptoms are consistent with <abbr title="verocytotoxin producing E. coli">VTEC</abbr> infection, it is recommended that you:</p>

<ul>
  <li>send a faecal sample to the reference laboratory</li>
  <li>send a serum sample to the reference laboratory for the testing for the presence of antibodies to E. coli O157 lipopolysaccharide</li>
</ul>

<h3>ID 22: text description of algorithm ‘Identification&nbsp;of Shiga toxin-producing Escherichia coli (<abbr title="Shiga toxin-producing E. coli">STEC</abbr>) including E.coli O157’</h3>

<p>Identify suspect colonies from primary plate. Subculture onto <abbr title="cefixime-tellurite sorbitol">CTSMAC</abbr> agar and incubate in air at 35 to 37°C for 16 to 24 hours. Further identification can be carried out by using the agglutination method or by using molecular or other methods.</p>

<h4>Results for Non Sorbitol fermenting colonies:</h4>

<p>Non sorbitol fermenting colonies present as colourless colonies on <abbr title="cefixime-tellurite sorbitol">CTSMAC</abbr> agar. Perform agglutination with <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 antiserum reagent. A positive result indicates E. coli. Further identification is recommended using commercial identification systems or other biochemical methods.&nbsp;Save the isolate on a slope and report presumptive (locally confirmed) E. coli O157. Send to the Reference laboratory to confirm identification, <abbr title="Shiga toxin gene">stx</abbr> gene, type and phage type.</p>

<p>If the agglutination result is negative then other non-O157 <abbr title="Shiga toxin-producing E. coli">STEC</abbr> should be considered.</p>

<h4>Results for Sorbitol fermenting colonies:</h4>

<p>Sorbitol fermenting colonies are pink coloured colonies on the agar plate. Some rare strains of <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 ferment sorbitol. Clinical details of the patient should be considered. If the patient presents with bloody diarrhoea or <abbr title="haemolytic uraemic syndrome">HUS</abbr> then perform agglutination with <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 antiserum reagent. If the result is negative then other non-O157 <abbr title="Shiga toxin-producing E. coli">STEC</abbr> should be considered. A positive result indicates E. coli. Further identification is recommended using commercial identification systems or other biochemical methods.&nbsp;Save the isolate on a slope and report presumptive (locally confirmed) E. coli O157. Send to the Reference laboratory to confirm identification, <abbr title="Shiga toxin gene">stx</abbr> gene, type and phage type.</p>

<p>If the patient presents with diarrhoea, only then the sample can be discarded.</p>

<h2>References</h2>

<p>An explanation of the reference assessment used is available in the <a class="govuk-link" href="https://www.gov.uk/guidance/uk-standards-for-microbiology-investigations-uk-smi-scientific-information#references">scientific information section on the UK SMI website</a>.</p>

<p>1. Mulchandani R, Brehmer C, Butt S, Vishram B, Harrison M, Marchant E and others. ‘Outbreak of Shiga toxin-producing Escherichia coli O157 linked with consumption of a fast-food product containing imported cucumbers, United Kingdom, August 2020’ International Journal of Infectious Diseases 2021: volume 110 supplement 1, S62 to s8</p>

<p>2. ‘Whole-Genome Sequencing for National Surveillance of Shiga Toxin–Producing Escherichia coli O15’, GOV.UK 2020</p>

<p>3. ‘Shiga toxin-producing Escherichia coli O157 (<abbr title="Shiga toxin-producing E. coli">STEC</abbr>-O157). An information leaflet for cases vol 25 August 2022’ Public Health England 2018</p>

<p>4. Centers for Disease Control and Prevention. Escherichia coli 2022 ed.vol 7 March 2022</p>

<p>5. Buchan B.W FML, Humphries RM, Dekker J, Ledeboer NA. Escherichia, Shigella, and Salmonella Vol 12th edition Washington DC. ASM press 2019: pages 688 to 723</p>

<p>6. Joseph A, Cointe A, Mariani Kurkdjian P, Rafat C, Hertig A. ‘Shiga Toxin-Associated Hemolytic Uremic Syndrome: A Narrative Review’ Toxins (Basel) 2020: volume 12</p>

<p>7. Jenkins C, Byrne L, Vishram B, Sawyer C, Balasegaram S, Ahyow L and others. ‘Shiga toxin-producing Escherichia coli haemolytic uraemic syndrome (<abbr title="Shiga toxin-producing E. coli">STEC</abbr>-<abbr title="haemolytic uraemic syndrome">HUS</abbr>): diagnosis, surveillance and public-health management in England’ Journal of Medical Microbiology 2020: volume 69, pages 1,034 to 1,036</p>

<p>8. Launders N, Locking ME, Hanson M, Willshaw G, Charlett A, Salmon R and others. ‘A large Great Britain-wide outbreak of <abbr title="Shiga toxin-producing E. coli">STEC</abbr> O157 phage type 8 linked to handling of raw leeks and potatoes’ Epidemiology and Infection 2016: volume 144, pages 171 to 181</p>

<p>9. Smith-Palmer A, Hawkins G, Browning L, Allison L, Hanson M, Bruce R and others. ‘Outbreak of Escherichia coli O157 Phage Type 32 linked to the consumption of venison products’ Epidemiology and Infection 2018: volume 146, pages 1,922 to 1,927</p>

<p>10. Launders N BL, Adams N, Glen K, Jenkins C, Tubin-Delic D, Locking M, Williams C, Morgan D. <a class="govuk-link" href="https://doi.org/10.2807/1560-7917.ES2013.18.44.20624" rel="external">Outbreak of Shiga toxin-producing E. coli O157 associated with consumption of watercress, United Kingdom, August to September 2013</a> Eurosurveillance 2013: volume 18</p>

<p>11. UK Health Security Agency. Shiga toxin producing Escherichia coli (<abbr title="Shiga toxin-producing E. coli">STEC</abbr>) data 2019</p>

<p>12. Easton L. ‘Escherichia coli O157: occurrence, transmission and laboratory detection’ British Journal of Biomedical Science 1997: volume 54, pages 57 to 64</p>

<p>13. March SB, Ratnam S. ‘Latex agglutination test for detection of Escherichia coli serotype O157’ Journal of Clinical Microbiology 1989: volume 27, pages 1,675 to 1,677</p>

<p>14. Parsons BD, Zelyas N, Berenger BM, Chui L. ‘Detection, Characterization, and Typing of Shiga Toxin-Producing Escherichia coli’ Frontiers in Microbiology 2016: volume 7, page 478</p>

<p>15. Clark AE, Kaleta EJ, Arora A, Wolk DM. ‘Matrix-assisted laser desorption ionization-time of flight mass spectrometry: a fundamental shift in the routine practice of clinical microbiology’ Clinical Microbiology Reviews 2013:volume 26, pages 547 to 603</p>

<p>16. Advisory Committee on Dangerous Pathogens. The Approved List of Biological Agents. Health and Safety Executive 2021: pages 1 to 39. ++</p>

<p>17. British Standards Institution (BSI). ‘BS EN12469 – Biotechnology – performance criteria for microbiological safety cabinets’ 2000. <sup>A, V </sup></p>

<p>18. British Standards Institution (BSI). ‘BS 5726:2005 – Microbiological safety cabinets. Information to be supplied by the purchaser and to the vendor and to the installer, and siting and use of cabinets. Recommendations and guidance’ 2005: pages 1 to 14. <sup>A, V</sup></p>

<p>19. Centers for Disease Control and Prevention. ‘Guidelines for Safe Work Practices in Human and Animal Medical Diagnostic Laboratories’ Morbidity and Mortality Weekly Report Surveillance Summaries 2012: volume 61, pages 1 to 102. <sup>B, IV</sup></p>

<p>20. Department for Transport, Maritime and Coastguard Agency, HSENI, Civil Aviation Authority. ‘Transport of infectious substances UN2814, UN2900 and UN3373 Guidance note number 17/2012 (revision 7)’ 2013. ++</p>

<p>21. Department of Health. ‘Health Protection Legislation (England) Guidance’ 2010: pages 1 to 112. ++</p>

<p>22. Gizzie N, Adukwu E. ‘Evaluation of Liquid-Based Swab Transport Systems against the New Approved CLSI M40-A2 Standard’ Journal of Clinical Microbiology 2016: volume 54, pages 1,152 to 1,156</p>

<p>23. Health and Safety Executive. <a class="govuk-link" href="https://www.hse.gov.uk/simple-health-safety/risk/index.htm" rel="external">Managing risks and risk assessment at work</a> (viewed on 28 July 2021). ++</p>

<p>24. Health and Safety Executive. ‘Safe use of pneumatic air tube transport systems for pathology specimens’ 2009. <sup>A, V</sup></p>

<p>25. Health and Safety Executive. Control of Substances Hazardous to Health Regulations. The Control of Substances Hazardous to Health Regulations 2002 (as amended). Approved Code of Practice and guidance L5 (sixth edition). HSE Books 2013. ++</p>

<p>26. Health and Safety Executive. ‘Risk assessment: A brief guide to controlling risks in the workplace’ HSE 2014. ++</p>

<p>27. Health and Safety Executive, Advisory Committee on Dangerous Pathogens. ‘Management and operation of microbiological containment laboratories’ HSE 2019. ++</p>

<p>28. Health Services Advisory Committee. ‘Safe Working and the Prevention of Infection in Clinical Laboratories and Similar Facilities’ HSE Books 2003. <sup>A, V</sup></p>

<p>29. Home Office. Public Health Act (Northern Ireland) 1967 Chapter 36. ++</p>

<p>30. Home Office. Anti-terrorism, Crime and Security Act 2001. ++</p>

<p>31. Official Journal of the European Communities. Directive 98/79/EC of the European Parliament and of the Council of 27 October 1998 on in vitro diagnostic medical devices 1998: pages 1 to 37. <sup>A, V</sup></p>

<p>32. Public Health England. ‘Laboratory reporting to Public Health England: a guide for diagnostic laboratories’ PHE 2020: pages 1 to 31. ++</p>

<p>33. Scottish Government. Public Health (Scotland) Act 2008. ++</p>

<p>34. The Royal College of Pathologists. ‘The retention and storage of pathological records and specimens’ (5th edition) 2015: pages 1 to 59. ++</p>

<p>35. The Welsh Assembly Government. Health Protection Legislation (Wales) Guidance 2010. ++</p>

<p>36. Tyrrell KL, Citron DM, Leoncio ES, Goldstein EJ. ‘Comparison of the Copan eSwab System with an Agar Swab Transport System for Maintenance of Fastidious Anaerobic Bacterium Viability’ Journal of Clinical Microbiology 2016: volume 54, pages 1,364 to 1,367</p>

<p>37. World Health Organization. ‘Guidance on regulations for the transport of infectious substances 2019 to 2020’ WHO 2019. ++</p>

<p>38. Advisory Committee on Dangerous Pathogens. ‘Infections at work: Controlling the risks’ Her Majesty’s Stationery Office 2003. <sup>A, V</sup></p>

<p>39. Singh K. ‘Laboratory-acquired infections’ Clinical Infectious Diseases 2009: volume 49, pages 142 to 147</p>

<p>40. Baron EJ, Miller JM. ‘Bacterial and fungal infections among diagnostic laboratory workers: evaluating the risks’ 3 2008: volume 60, pages 241 to 246</p>

<p>41. Spina N, Zansky S, Dumas N, Kondracki S. ‘Four laboratory-associated cases of infection with Escherichia coli O157:H7’ Journal of Clinical Microbiology 2005: volume 43, pages 2,938 to 2,939</p>

<p>42. Burnens AP, Zbinden R, Kaempf L, Heinzer I, Nicolet J. ‘A case of laboratory acquired infection with Escherichia coli O157:H7’ Zentralblatt für Bakteriologie 1993: volume 279, pages 512 to 517</p>

<p>43. Rice T, Quinn N, Sleator RD, Lucey B. ‘Changing Diagnostic Methods and Increased Detection of Verotoxigenic Escherichia coli, Ireland’ Emerging Infectious Diseases 2016: volume 22, pages 1,656 to 1,657</p>

<p>44. Verhaegen B, De Reu K, Heyndrickx M, De Zutter L. ‘Comparison of Six Chromogenic Agar Media for the Isolation of a Broad Variety of Non-O157 Shigatoxin-Producing Escherichia coli (<abbr title="Shiga toxin-producing E. coli">STEC</abbr>) Serogroups’ International Journal of Environmental Research and Public Health 2015: volume 12, pages 6,965 to 6,978</p>

<p>45. ‘Research and analysis: Shiga toxin-producing Escherichia coli (<abbr title="Shiga toxin-producing E. coli">STEC</abbr>) data 2018’ ed.vol 25 January 2022. GOV.UK 2020</p>

<p>46. Barbuddhe SB, Maier T, Schwarz G, Kostrzewa M, Hof H, Domann E and others. ‘Rapid identification and typing of listeria species by matrix-assisted laser desorption ionization-time of flight mass spectrometry’ Applied and Environmental Microbiology 2008: volume 74, pages 5,402 to 5,407</p>

<p>47. Gentle A, Day MR, Hopkins KL, Godbole G, Jenkins C. ‘Antimicrobial resistance in Shiga toxin-producing Escherichia coli other than serotype O157 : H7 in England, 2014 to 2016’ Journal of Medical Microbiology 2020: volume 69, pages 379 to 386</p>

<p>48. Koziel M, Kiely R, Blake L, O’Callaghan I, Corcoran GD, Lucey B and others. ‘Improved detection of bacterial pathogens in patients presenting with gastroenteritis by use of the EntericBio real-time Gastro Panel I assay’ Journal of Clinical Microbiology 2013: volume 51, pages 2,679 to 2,685</p>

<p>49. Koziel M, Corcoran D, O’Callaghan I, Sleator RD, Lucey B. ‘Validation of the EntericBio Panel II(R) multiplex polymerase chain reaction system for detection of Campylobacter spp., Salmonella spp., Shigella spp., and verotoxigenic E. coli for use in a clinical diagnostic setting’ Diagnostic Microbiology and Infectious Disease 2013: volume 75, pages 46 to 49</p>

<p>50. O’Leary J, Corcoran D, Lucey B. ‘Comparison of the EntericBio multiplex <abbr title="polymerase chain reaction">PCR</abbr> system with routine culture for detection of bacterial enteric pathogens’ Journal of Clinical Microbiology 2009: volume 47, pages 3,449 to 3,453</p>

<p>51. Butcher H, Elson R, Chattaway MA, Featherstone CA, Willis C, Jorgensen F and others. ‘Whole genome sequencing improved case ascertainment in an outbreak of Shiga toxin-producing Escherichia coli O157 associated with raw drinking milk’ Epidemiology and Infection 2016: volume 144, pages 2,812 to 2,823</p>

<p>52. Dallman TJ, Byrne L, Ashton PM, Cowley LA, Perry NT, Adak G and others. ‘Whole-genome sequencing for national surveillance of Shiga toxin-producing Escherichia coli O157’ Clinical Infectious Diseases 2015: volume 61, pages 305 to 312</p>

</div>


</div>
</div>
    </div>
  </div>

  <div class="govuk-grid-row">
    <a class="govuk-link app-c-back-to-top dont-print" href="#contents">
    <svg class="app-c-back-to-top__icon" focusable="false" height="17" viewBox="0 0 13 17" width="13" xmlns="http://www.w3.org/2000/svg">
      <path d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z" fill="currentColor"></path>
    </svg>
    Back to top
</a>

  </div>
</div>

    </main>