<main class="html-publication" lang="en" role="main">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          
<div class="gem-c-organisation-logo brand--department-of-health">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color" href="/government/organisations/uk-health-security-agency">
      <span class="gem-c-organisation-logo__name">UK Health<br>Security<br>Agency</span>
</a>
</div>
        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  

<div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
      <span class="govuk-caption-xl gem-c-title__context">
    Guidance
  </span>


  <h1 class="gem-c-title__text govuk-heading-xl">
    SARS-CoV-2 variants of public health interest: 20 May 2022
  </h1>
</div>
  <p class="publication-header__last-changed">Updated 24 June 2022</p>

  </header>






<div class="govuk-grid-row sidebar-with-body" data-module="sticky-element-container">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav aria-label="Contents" class="gem-c-contents-list" data-module="gem-track-click" role="navigation">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 1" data-track-category="contentsClicked" data-track-label="#horizon-scan" data-track-options="{&quot;dimension29&quot;:&quot;Horizon scan&quot;}" href="#horizon-scan">Horizon scan</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 2" data-track-category="contentsClicked" data-track-label="#sars-cov-2-variants-of-public-health-interest-variants-detected-in-the-uk" data-track-options="{&quot;dimension29&quot;:&quot;SARS-CoV-2 variants of public health interest: variants detected in the UK&quot;}" href="#sars-cov-2-variants-of-public-health-interest-variants-detected-in-the-uk">SARS-CoV-2 variants of public health interest: variants detected in the UK</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 3" data-track-category="contentsClicked" data-track-label="#sars-cov-2-variants-of-public-health-interest-variants-present-in-gisaid-but-not-detected-in-the-uk" data-track-options="{&quot;dimension29&quot;:&quot;SARS-CoV-2 variants of public health interest: variants present in GISAID but not detected in the UK&quot;}" href="#sars-cov-2-variants-of-public-health-interest-variants-present-in-gisaid-but-not-detected-in-the-uk">SARS-CoV-2 variants of public health interest: variants present in GISAID but not detected in the UK</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 4" data-track-category="contentsClicked" data-track-label="#sars-cov-2-variants-of-public-health-interest-variants-which-have-become-provisionally-extinct" data-track-options="{&quot;dimension29&quot;:&quot;SARS-CoV-2 variants of public health interest: variants which have become provisionally extinct&quot;}" href="#sars-cov-2-variants-of-public-health-interest-variants-which-have-become-provisionally-extinct">SARS-CoV-2 variants of public health interest: variants which have become provisionally extinct</a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img class="print-meta-data-licence" src="/assets/government-frontend/open-government-licence-min-93b6a51b518ff99714a1aa2a7d2162735c155ec3cb073c75fb88b2a332fa83d3.png">
</p>
<p>
  Â© Crown copyright 2022
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/sars-cov-2-variants-of-public-health-interest/sars-cov-2-variants-of-public-health-interest-20-may-2022
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
    
    
      <div class="govspeak">
<p>Position as of 16 May 2022.</p>

<h2>Horizon scan</h2>

<table>
  <tbody>
    <tr>
      <td>International epidemiology scan</td>
      <td>16 May 2022</td>
    </tr>
    <tr>
      <td>Lineage and signals growth rates scan</td>
      <td>16 May 2022</td>
    </tr>
    <tr>
      <td>Phylogenetic cluster detection</td>
      <td>16 May 2022</td>
    </tr>
    <tr>
      <td>International mutation scan</td>
      <td>16 May 2022</td>
    </tr>
    <tr>
      <td>Domestic mutation scan</td>
      <td>16 May 2022</td>
    </tr>
    <tr>
      <td>Signals investigated from 4 May to 16 May 2022</td>
      <td>5</td>
    </tr>
    <tr>
      <td>New genomes 4 May to 16 May 2022</td>
      <td>22,963</td>
    </tr>
  </tbody>
</table>

<p>Note: signals of individual mutations and new clusters are excluded until defined variant is identified.</p>

<h2>SARS-CoV-2 variants of public health interest: variants detected in the UK</h2>

<h3>UKHSA-designated variants of concern</h3>

<table>
  <tbody>
    <tr>
      <td>Omicron (VOC-21NOV-01/B.1.1.529/BA.1 and all sub-lineages, unless separately designated by UKHSA)</td>
    </tr>
    <tr>
      <td>Omicron (VOC-22JAN-01/B.1.1.529/BA.2 and all sub-lineages, unless separately designated by UKHSA)</td>
    </tr>
    <tr>
      <td>Omicron (B.1.1.529) sub-lineage BA.5 (VOC-22APR-04)</td>
    </tr>
    <tr>
      <td>Omicron (B.1.1.529) sub-lineage BA.4 (VOC-22APR-03)</td>
    </tr>
  </tbody>
</table>

<h3>UKHSA-designated variants</h3>

<table>
  <tbody>
    <tr>
      <td>XE recombinant (V-22APR-02/BA.1 x BA.2)</td>
    </tr>
    <tr>
      <td>Delta (V-21APR-02 and all sub-lineages, unless named elsewhere on the grid)</td>
    </tr>
    <tr>
      <td>Alpha (V-20DEC-01/B.1.1.7)</td>
    </tr>
    <tr>
      <td>V-21OCT-01 (AY.4.2)</td>
    </tr>
  </tbody>
</table>

<h3>Signals currently under monitoring and investigation</h3>

<table>
  <tbody>
    <tr>
      <td>BA.3</td>
    </tr>
    <tr>
      <td>Delta and Omicron recombinant lineages (UK)</td>
    </tr>
    <tr>
      <td>XF recombinant</td>
    </tr>
    <tr>
      <td>BA.1/BA.2 recombinant (with unique mutation C3583T)</td>
    </tr>
    <tr>
      <td>BA.2.12.1</td>
    </tr>
  </tbody>
</table>

<h2>SARS-CoV-2 variants of public health interest: variants present in GISAID but not detected in the UK</h2>

<h3>International variants of concern</h3>

<table>
  <tbody>
    <tr>
      <td>Beta (V-20DEC-02, B.1.351)</td>
    </tr>
  </tbody>
</table>

<h3>International variants</h3>

<table>
  <tbody>
    <tr>
      <td>XD Recombinant (V-22APR-01, Delta x BA.1)</td>
    </tr>
    <tr>
      <td>B.1.525 (previously VUI-21FEB-03)</td>
    </tr>
  </tbody>
</table>

<h3>International signals under monitoring</h3>

<table>
  <tbody>
    <tr>
      <td>AY.119.2/BA.1.1 Recombinant</td>
    </tr>
  </tbody>
</table>

<h2>SARS-CoV-2 variants of public health interest: variants which have become provisionally extinct</h2>

<h3>Variants provisionally extinct (not detected for 12 weeks in the UK or in GISAID)</h3>

<table>
  <tbody>
    <tr>
      <td>P.1 (Gamma) (previously VOC-21JAN-02)</td>
    </tr>
    <tr>
      <td>A.23.1 with E484K (previously VUI-21FEB-01)</td>
    </tr>
    <tr>
      <td>Alpha with E484K (previously VOC-21FEB-02)</td>
    </tr>
    <tr>
      <td>B.1.324.1 with E484K (previously VUI-21MAR-01)</td>
    </tr>
    <tr>
      <td>AV.1 (previously VUI-21MAY-01)</td>
    </tr>
    <tr>
      <td>P.3 (previously VUI-21MAR-02)</td>
    </tr>
    <tr>
      <td>B.1.318 (previously VUI-21FEB-04)</td>
    </tr>
    <tr>
      <td>P.2 (previously VUI-21JAN-01)</td>
    </tr>
    <tr>
      <td>B.1.617.1 (previously VUI-21APR-01)</td>
    </tr>
    <tr>
      <td>A.27</td>
    </tr>
    <tr>
      <td>C.36.3</td>
    </tr>
    <tr>
      <td>Iota (B.1.526)</td>
    </tr>
    <tr>
      <td>Mu (V-21JUL-01, B.1.621)</td>
    </tr>
    <tr>
      <td>V-21APR-03 (B.1.617.3)</td>
    </tr>
  </tbody>
</table>

<h3>Signals provisionally extinct (not detected for 12 weeks in the UK or in GISAID)</h3>

<table>
  <tbody>
    <tr>
      <td>A.30</td>
    </tr>
    <tr>
      <td>B.1.633</td>
    </tr>
    <tr>
      <td>AT.1</td>
    </tr>
    <tr>
      <td>P.5</td>
    </tr>
    <tr>
      <td>R.1</td>
    </tr>
    <tr>
      <td>B.1.620</td>
    </tr>
    <tr>
      <td>B.1.1.7 + B.1.617.2 (XC Recombinant)</td>
    </tr>
    <tr>
      <td>B.1.214.2</td>
    </tr>
    <tr>
      <td>B.1.619</td>
    </tr>
    <tr>
      <td>B.1.640</td>
    </tr>
  </tbody>
</table>

<h3>Signals removed after review</h3>

<p>Variants under investigation and signals in monitoring may be removed from the grid if they show consistently low growth rates.</p>

<table>
  <tbody>
    <tr>
      <td>AY.33</td>
    </tr>
    <tr>
      <td>AY.34</td>
    </tr>
    <tr>
      <td>AY.43</td>
    </tr>
    <tr>
      <td>B.1.629</td>
    </tr>
    <tr>
      <td>B.1.630</td>
    </tr>
    <tr>
      <td>B.1.628 recombinant of (B.1.631 and B.1.634)</td>
    </tr>
    <tr>
      <td>C.1.2</td>
    </tr>
    <tr>
      <td>C.37 (Lambda) (previously VUI-21JUN-01)</td>
    </tr>
    <tr>
      <td>C.37 (2nd Gen)</td>
    </tr>
    <tr>
      <td>Epsilon (B.1.427/B.1.429)</td>
    </tr>
    <tr>
      <td>P.1.8</td>
    </tr>
    <tr>
      <td>Variant associated with North America: USA 214insQAS Samples</td>
    </tr>
    <tr>
      <td>Delta + E484K Phylogenetic Cluster</td>
    </tr>
  </tbody>
</table>
</div>


</div>
</div>
  </div>

  <div class="sticky-element" data-sticky-element="">
    <a class="govuk-link app-c-back-to-top dont-print" href="#contents">
    <svg class="app-c-back-to-top__icon" focusable="false" height="17" viewBox="0 0 13 17" width="13" xmlns="http://www.w3.org/2000/svg">
      <path d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z" fill="currentColor"></path>
    </svg>
    Contents
</a>

    <div class="sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>