<main role="main" lang="en" class="html-publication">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          
<div class="gem-c-organisation-logo brand--cabinet-office">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link brand__border-color" href="/government/organisations/scientific-advisory-group-for-emergencies">
      <span class="gem-c-organisation-logo__name">Scientific Advisory Group for Emergencies</span>
</a>
</div>
        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  

<div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
      <span class="govuk-caption-xl gem-c-title__context">
    Research and analysis
  </span>

  
  <h1 class="gem-c-title__text govuk-heading-xl">
    SPI-M-O: Consensus Statement on COVID-19, 7 December 2021
  </h1>
</div>
  <p class="publication-header__last-changed">Published 8 December 2021</p>

  </header>



<div class="govuk-grid-row sidebar-with-body" data-module="sticky-element-container">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav role="navigation" class="gem-c-contents-list" aria-label="Contents" data-module="gem-track-click">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-options="{&quot;dimension29&quot;:&quot;\n1. Summary&quot;}" href="#summary" data-track-action="content_item 1" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-label="#summary" data-track-category="contentsClicked"><span class="gem-c-contents-list__number">1. </span><span class="gem-c-contents-list__numbered-text">Summary</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-options="{&quot;dimension29&quot;:&quot;\n2. Omicron variant&quot;}" href="#omicron-variant" data-track-category="contentsClicked" data-track-action="content_item 2" data-track-label="#omicron-variant"><span class="gem-c-contents-list__number">2. </span><span class="gem-c-contents-list__numbered-text">Omicron variant</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-options="{&quot;dimension29&quot;:&quot;\n3. State of the nation in the UK\n&quot;}" data-track-category="contentsClicked" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" href="#state-of-the-nation-in-the-uk" data-track-action="content_item 3" data-track-label="#state-of-the-nation-in-the-uk"><span class="gem-c-contents-list__number">3. </span><span class="gem-c-contents-list__numbered-text">State of the nation in the UK</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-label="#initial-results" data-track-options="{&quot;dimension29&quot;:&quot;\n4. Initial results&quot;}" href="#initial-results" data-track-category="contentsClicked" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 4"><span class="gem-c-contents-list__number">4. </span><span class="gem-c-contents-list__numbered-text">Initial results</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 5" data-track-options="{&quot;dimension29&quot;:&quot;\n5. Annex A: PHIA framework of language for discussing probabilities&quot;}" href="#annex-a-phia-framework-of-language-for-discussing-probabilities" data-track-label="#annex-a-phia-framework-of-language-for-discussing-probabilities" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-category="contentsClicked"><span class="gem-c-contents-list__number">5. </span><span class="gem-c-contents-list__numbered-text">Annex A: PHIA framework of language for discussing probabilities</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" href="#annex-b-update-from-spi-m-o-7-december-2021" data-track-action="content_item 6" data-track-category="contentsClicked" data-track-label="#annex-b-update-from-spi-m-o-7-december-2021" data-track-options="{&quot;dimension29&quot;:&quot;\n6. Annex B: Update from SPI-M-O, 7 December 2021&quot;}"><span class="gem-c-contents-list__number">6. </span><span class="gem-c-contents-list__numbered-text">Annex B: Update from SPI-M-O, 7 December 2021</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-label="#references" data-track-action="content_item 7" data-track-options="{&quot;dimension29&quot;:&quot;\n7. References&quot;}" data-track-category="contentsClicked" href="#references"><span class="gem-c-contents-list__number">7. </span><span class="gem-c-contents-list__numbered-text">References</span></a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button data-module="print-link" class="govuk-link govuk-body-s gem-c-print-link__button">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img src="/assets/government-frontend/open-government-licence-min-e98134ae65ff5fe99a524d8c383350b3998f91c298dc24b36126e65f1656c578.png" class="print-meta-data-licence">
</p>
<p>
  © Crown copyright 2021
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/spi-m-o-consensus-statement-on-covid-19-7-december-2021/spi-m-o-consensus-statement-on-covid-19-7-december-2021
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div data-module="govspeak" class="gem-c-govspeak govuk-govspeak ">
    
    
      <div class="govspeak">
<p><strong>Date: 1 December 2021, update on 7 December 2021 at Annex B</strong></p>

<p>All probability statements are in line with the framework given in Annex A.</p>

<h2>
<span class="number">1. </span>Summary</h2>

<p>1. The emergence of the B.1.1.529 <sup role="doc-noteref"><a class="govuk-link" rel="footnote" href="#fn:1">[footnote 1]</a></sup> (henceforth referred to as omicron) variant of <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> in southern Africa is deeply concerning. Initial indications of its genetic variation compared to previous variants of this virus suggest it has potential for significant immune escape, although its full properties are unknown. While the evidence from South Africa is still emerging and subject to a range of caveats, it is highly unlikely that the pattern of growth observed there would hold true if omicron did not have either a significant transmission advantage, significant immune escape, or both, or some other fitness advantage over the predominant variant (delta).</p>

<p>2. The first cases of omicron were identified in the <abbr title="United Kingdom">UK</abbr> on 27 November 2021, so it is very early to see any signals in the various data streams available. <abbr title="spike gene">S-gene</abbr> negative cases will likely act as a leading indicator. In the early stages of any omicron transmission in the <abbr title="United Kingdom">UK</abbr>, it will be very difficult to tell the difference between imported cases (or their immediate contacts) and sustained community transmission.</p>

<p>3. The burden of disease for any wave of omicron infection will depend on the size of the pool of people who are susceptible to infection and the severity of those infections. If omicron exhibits substantial immune escape, the pool of susceptible people in the <abbr title="United Kingdom">UK</abbr> could be large, and thus transmission could increase substantially. Any significant wave of infection, almost irrespective of immune escape, will spill over into hospitalisations. If initial estimates of transmission advantage and immune escape from South Africa are applicable to the <abbr title="United Kingdom">UK</abbr> population, there is the potential for a peak of infections much larger than that experienced in January 2021.</p>

<p>4. If omicron in the <abbr title="United Kingdom">UK</abbr> combines increased transmissibility and immune escape, irrespective of severity, it is highly likely that very stringent measures would be required to control growth and keep <abbr title="reproduction number">R</abbr> below 1. Delaying any wave of infections in such a scenario would allow more time for vaccines and therapeutics to be modified to combat omicron.</p>

<h2>
<span class="number">2. </span>Omicron variant</h2>

<p>5. The emergence of omicron in South Africa shows several concerning features. Transmission appears to be rapid, with a doubling time of approximately three days for cases in Gauteng province, with evidence of spread across the region. An observed excess in the number of reinfections <sup role="doc-noteref"><a class="govuk-link" rel="footnote" href="#fn:2">[footnote 2]</a></sup> with <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> in a highly immune population (likely to be over 70 per cent from a mixture of natural infection and vaccination) suggests that there is highly likely to be some form of escape from immunity. This also aligns with the theoretical impact of the many mutations across the genomic sequence of this variant compared to others.</p>

<p>6. Data are still emerging from South Africa and, as yet, the full range of biological parameters are not well understood. Despite this, there are multiple indications from local analyses that omicron likely has a significant transmission advantage over delta at the moment, even in the presence of complete immune escape. If escape is lower, the transmission advantage would need to be much higher to account for observed data in South Africa. Initial indications from <abbr title="spike gene">S-gene</abbr> target failure [<abbr title="s-gene target failure">SGTF</abbr>] (a proxy measure for omicron) assays in South Africa suggest that cases are increasingly <abbr title="spike gene">S-gene</abbr> negative.</p>

<p>7. It is possible that initial cases of omicron may not be representative – they will be subject to a number of biases, such as testing behaviour, potential oversampling in highly affected areas, and so forth. It is not yet clear whether and or how omicron escapes from natural immunity and or vaccine-induced immunity and which of these is the likely main driver of its apparent fitness advantage. Until there are a significant number of cases in the <abbr title="United Kingdom">UK</abbr>, analysis from South Africa and other countries will provide leading indications.</p>

<p>8. South Africa is not the <abbr title="United Kingdom">UK</abbr>, both in epidemiological terms (<abbr title="coronavirus">COVID-19</abbr> epidemic timing and variant composition to date have been different, vaccination types and programmes have been different, as well as other factors) but also socially. To extrapolate the South African experience to the <abbr title="United Kingdom">UK</abbr>, it requires omicron to outcompete delta “in the <abbr title="United Kingdom">UK</abbr>”; this will not be the same as this happening in South Africa.</p>

<p>9. Analyses considered from South Africa currently assume omicron has the same generation time as delta, however this is unknown. Severity estimates, for example how many people are hospitalised after infection (the infection hospitalisation rate – <abbr title="infection hospitalisation rate">IHR</abbr>), for omicron are also unknown. Consequently, an accurate forecast of the outcome if omicron establishes in the <abbr title="United Kingdom">UK</abbr> is not currently possible.</p>

<p>10. While the evidence from South Africa is still emerging and the analysis available to date is subject to various caveats, patterns in growth observed there strongly indicate that omicron has either a significant transmission advantage, significant immune escape, or both, or some other fitness advantage over the predominant variant (delta). Despite the uncertainty, the constellation of what is known of the epidemiology of omicron to date present a compelling case for omicron to cause a wave of infection in the <abbr title="United Kingdom">UK</abbr>.</p>

<p>11. The emergence of omicron has been identified much earlier in its trajectory than for other variants, such as alpha and delta. This means it is possible to respond more quickly but also entails greater uncertainty.</p>

<h2>
<span class="number">3. </span>State of the nation in the <abbr title="United Kingdom">UK</abbr>
</h2>

<p>12. The first cases of omicron were identified in the <abbr title="United Kingdom">UK</abbr> on 27 November 2021. As of 1 December, it is very early to see any signal in the various data streams available. Regional approaches are not <em>yet</em> observing marked increases in <abbr title="spike gene">S-gene</abbr> negative cases. Scotland, however, is currently seeing very fast doubling times in the number of <abbr title="spike gene">S-gene</abbr> negative cases, albeit growing from a low base. In the early part of any omicron transmission, it will be very difficult to tell the difference between imported cases (or their immediate contacts) and sustained chains of community transmission.</p>

<p>13. There is potential for this <abbr title="spike gene">S-gene</abbr> signal to be confounded in the wider context – while the majority of cases in the <abbr title="United Kingdom">UK</abbr> now are <abbr title="spike gene">S-gene</abbr> positive, there is constant background noise of <abbr title="spike gene">S-gene</abbr> negative cases. If there are general increases in transmission, the increase in these <abbr title="spike gene">S-gene</abbr> negative non-omicron cases could mask initial leading indicators of omicron growth. In recent months, there have been small waves of non-omicron <abbr title="spike gene">S-gene</abbr> negative cases within the <abbr title="United Kingdom">UK</abbr> – if one of these happens at the same time as omicron emerges here, it will be difficult to disentangle. The inherent lags between infections, cases, testing, sequencing, and any subsequent need for healthcare means it is not possible to know how omicron is spreading <strong>now</strong>. It is currently difficult to tell whether growth in the <abbr title="United Kingdom">UK</abbr>’s <abbr title="spike gene">S-gene</abbr> negative cases is due to thorough and rapid contact tracing and testing after potential superspreading or autochthonous growth.</p>

<p>14. The burden of disease for any wave of omicron infection will depend on the size of the pool of people susceptible to infection, which in turn depends on the numbers of people vaccinated, boosted, or recovered from natural infection and the level of protection each of these affords. Initial modelling from three academic groups suggest that the proportion of completely susceptible individuals in England varies by age group, with very high (over 90 per cent) protection (either from natural infection or vaccine-acquired immunity) in 40-year olds and over. Two of these studies use relatively simple modelling techniques that do not include waning of protection.</p>

<p>15. This low level of susceptibility in the population is likely to be one of the reasons why the <abbr title="United Kingdom">UK</abbr> has had relatively stable levels of cases, hospitalisations, and deaths in recent months. Depending on omicron’s immune escape mechanisms, it is possible that the pool of people susceptible to infection with omicron is much larger than those who could catch delta. If this is true, then transmission will increase substantially.</p>

<h2>
<span class="number">4. </span>Initial results</h2>

<p>16. Initial preliminary modelling has been conducted by two academic groups to consider the potential implications of a variant with immune escape (from both vaccine-induced and natural immunity) and transmission advantage.</p>

<p>17. Any wave of significant infection, almost irrespective of immune escape, will spill over into hospitalisations, and ultimately deaths. If initial estimates of transmission advantage and immune escape from South Africa are applicable to the <abbr title="United Kingdom">UK</abbr> population, there is the potential for a very substantial peak of infections much larger than occurred during the winter wave of January 2021. Even if severity of omicron were half that of delta, the sheer number of infections could lead to significantly more pressures on health and care settings; currently there is no strong evidence that omicron infections are either more or less severe than delta infections.</p>

<p>18. If omicron’s immune escape reduces vaccine effectiveness against hospitalisation from, say, 96 per cent to 92 per cent, that would effectively <em>double</em> the number of vaccinated individuals who are <strong>not</strong> protected from hospitalisation.</p>

<p>19. Due to the significant uncertainty and need for further data to emerge, <abbr title="Scientific Pandemic Influenza Group on Modelling, Operational sub-group">SPI-M-O</abbr> has discussed potential narrative scenarios, which the committee believe cover the likely parameter space (Table 1). <abbr title="Scientific Pandemic Influenza Group on Modelling, Operational sub-group">SPI-M-O</abbr> considers scenarios A, B or C from Table 1 an almost certain future for the <abbr title="United Kingdom">UK</abbr>. In contrast, scenarios D, E and F are each highly unlikely. There remains, however, considerable uncertainty in the current situation and likely or possible futures. As more information and evidence on omicron emerges, both globally and in the <abbr title="United Kingdom">UK</abbr>, some of these scenarios will become more and more unlikely, and even untenable. <abbr title="Scientific Pandemic Influenza Group on Modelling, Operational sub-group">SPI-M-O</abbr> will continue to consider these as more data emerge.</p>

<p>20. If any of scenarios A, B, or C turns out to be the case, it is highly likely that very stringent measures would be required to control growth and keep <abbr title="reproduction number">R</abbr> below 1. The scale of infections in any of these scenarios could be substantial and lead to unsustainable pressure on health and care settings; reducing the total number of infections will be easier than reducing the <abbr title="infection hospitalisation rate">IHR</abbr>. Delaying any wave of infections in such a scenario would allow more time for vaccinations and therapeutics to be modified to combat omicron.</p>

<h2>
<span class="number">5. </span>Annex A: PHIA framework of language for discussing probabilities</h2>

<figure class="image embedded"><div class="img"><img alt="Scale showing 0% remote chance, between 0% and 25% highly unlikely, above 25% unlikely, above this and up to 50% realistic possibility, 50 per cent to 75 per cent likely or probably, above 75 per cent highly likely and before 100 per cent almost certain." src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/137754/S1325_annex.png"></div></figure>

<p><strong>Table 1</strong>: Narrative scenarios for potential omicron epidemic trajectories.</p>

<table>
  <thead>
    <tr>
      <th scope="col">Omicron scenario</th>
      <th scope="col">Description</th>
      <th scope="col">What makes this unlikely?</th>
      <th scope="col">Arguments for and against</th>
      <th scope="col">Consequence if this is how omicron behaves in the UK</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
<strong>A</strong>: Immune escape from pre-existing immunity acquired through infection or vaccination. Increased or marginally increased transmission compared to delta. Unchanged infection hospitalisation ratio (<abbr title="infection hospitalisation rate">IHR</abbr>).</td>
      <td>Increase in effective <abbr title="reproduction number">R</abbr> sustained through time because the population of susceptible individuals is larger if there is immune escape. No change in the ratio of hospitalisations to infections or cases, but more people go to hospital because more are infected.</td>
      <td>This is currently one of the most likely scenarios.</td>
      <td>
<strong>For:</strong> consistent with emerging evidence from South Africa. <strong>Against:</strong> boosters and “immune landscape” in <abbr title="United Kingdom">UK</abbr> might hold omicron to a relatively small, relatively short outbreak – would rely on boosters being highly protective.</td>
      <td>Omicron replaces delta due to an increase in effective <abbr title="reproduction number">R</abbr> that persists through several epidemic doublings because the susceptible population is larger. This situation will develop over time and the scale of the problem will become apparent with the speed of its emergence – if omicron’s advantages are only slight then its dominance as a fraction of infections will unfold slowly and the excess number of infections will be small. Genetic information (which is substantially lagged) and <abbr title="spike gene">S-gene</abbr> target failures will provide some early measures over the next two to three weeks.</td>
    </tr>
    <tr>
      <td>
<strong>B</strong>: Immune escape from pre-existing immunity acquired through infection or vaccination. Increased or marginally increased transmission compared to delta. Smaller <abbr title="infection hospitalisation rate">IHR</abbr>.</td>
      <td>Increase in effective <abbr title="reproduction number">R</abbr> sustained through time because the population of susceptible individuals is larger if there is immune escape. Reduced ratio of hospitalisations to infections or cases. It is unlikely that <abbr title="infection hospitalisation rate">IHR</abbr> will be reduced sufficiently to compensate for increased transmission.</td>
      <td>This is currently one of the most likely scenarios.</td>
      <td>
<strong>For:</strong> consistent with emerging evidence from South Africa. <strong>Against:</strong>boosters and “immune landscape” in <abbr title="United Kingdom">UK</abbr> might hold omicron to a relatively small, relatively short outbreak – would rely on boosters being highly protective. No data on severity currently available.</td>
      <td>As scenario A. Even a marked decrease in <abbr title="infection hospitalisation rate">IHR</abbr> is unlikely to offset the impact of a larger susceptible pool; it would be easy to double the number of infections and unlikely to halve the <abbr title="infection hospitalisation rate">IHR</abbr>.</td>
    </tr>
    <tr>
      <td>
<strong>C</strong>: Immune escape from pre-existing immunity acquired through infection or vaccination. Increased or marginally increased transmission compared to delta. Larger <abbr title="infection hospitalisation rate">IHR</abbr>.</td>
      <td>Increase in effective <abbr title="reproduction number">R</abbr> sustained through time because the population of susceptible individuals is larger if there is immune escape. Combination of more infections <strong>and</strong> a higher fraction of infections or cases being hospitalised compound multiplicatively.</td>
      <td>This is currently one of the most likely scenarios.</td>
      <td>
<strong>For:</strong> South Africa data are consistent with an increase in transmission and vaccine escape. No data on severity currently available. Increased pathogenicity is possible as other variants that have increased transmissibility (for example, alpha and delta) have been associated with increased severity indicators.</td>
      <td>As scenario A with added impact of more infections compounded with increased severity.</td>
    </tr>
    <tr>
      <td>
<strong>D</strong>: Marked transmission advantage. No immune escape.</td>
      <td>Uptick in effective <abbr title="reproduction number">R</abbr>, but not long-sustained growth because herd immunity threshold shifts from, say, approximately 86 per cent to approximately 93 per cent (as hypothetical example).</td>
      <td>This scenario is inconsistent with South African data on raised reinfection rates and viral sequences.</td>
      <td>
<strong>Against:</strong> Transmission advantage without immune escape would need to be exceptionally high to fit the data available from South Africa to date. Epidemiology in South Africa implies raised rates of reinfection. Viral sequence data shows many mutations in known sites of immune recognition.</td>
      <td>Omicron replaces delta with a short-lived increase in effective <abbr title="reproduction number">R</abbr>. As there is no large increase in the size of the susceptible population, the advantages of having a small susceptible pool in the <abbr title="United Kingdom">UK</abbr> are preserved.</td>
    </tr>
    <tr>
      <td>
<strong>E</strong>: Like delta sub-lineage AY 4.2 in England.</td>
      <td>Omicron outcompetes delta but with no indication of elevated repeat infections nor any change in <abbr title="infection hospitalisation rate">IHR</abbr> nor a large increase in transmission rate.</td>
      <td>This scenario is inconsistent with South African data on raised reinfection rates and viral sequences.</td>
      <td>
<strong>Against:</strong> Genomic analysis of omicron suggests that it will evade immunity to some extent and epidemiology in South Africa implies raised rates of reinfection. This scenario would also require that the large number of mutations in the omicron genome have no impact on immune escape nor vaccine escape nor intrinsic transmissibility in <abbr title="United Kingdom">UK</abbr> context of high recent transmission and extensive booster rollout. <strong>For:</strong>This might happen <em>if</em> the South African epidemiology (implying higher reinfection probability than delta) did not apply in the <abbr title="United Kingdom">UK</abbr>. This is conceivable given recent and ongoing high transmission of delta and if extremely high levels of boosted immunity were acquired before omicron took hold. This is highly unlikely.</td>
      <td>Omicron replaces delta, but no marked change in the overall hospitalisation trajectory. There may be changes in trajectories of infections and cases (as with recent epidemics in schools), but these do not spill over into admissions. This would emerge gradually over the coming month. It would be seen alongside evidence of sustained transmission but confirmation of a small increase in infections with no consequent increases in hospitalisations would only be apparent after three to four weeks due to lags in disease progression and data delays. The impact of alpha and delta variants on hospitalisations became clearer after about four weeks of established transmission.</td>
    </tr>
    <tr>
      <td>
<strong>F</strong>: Like beta in England.</td>
      <td>Small incursions that are self-limiting or universally controlled through interventions.</td>
      <td>Requires delta to outcompete omicron in the <abbr title="United Kingdom">UK</abbr>.</td>
      <td>
<strong>Against:</strong> Omicron is growing whilst delta is shrinking in multiple regions in South Africa. This could conceivably happen if these are just stochastic flares from super-spreading events. This feels unlikely with omicron growing fast in multiple locations. <strong>For:</strong> <abbr title="United Kingdom">UK</abbr> is epidemiologically different to South Africa in terms of epidemic to date (and its legacy of natural immunity) and vaccination. Omicron has to outcompete delta in <abbr title="United Kingdom">UK</abbr> context which it might not, given social and genetic differences as well. This scenario would also require all omicron’s mutations to have no impact on immune escape nor vaccine escape nor intrinsic transmissibility in a <abbr title="United Kingdom">UK</abbr> context.</td>
      <td>Small self-limiting flares of omicron do little to interrupt the delta epidemic. There will be some indication of this in two weeks (mid-December); and there will be further certainty in very early January (subject to any disruption of data streams over the festive period).</td>
    </tr>
  </tbody>
</table>

<h2>
<span class="number">6. </span>Annex B: Update from <abbr title="Scientific Pandemic Influenza Group on Modelling, Operational sub-group">SPI-M-O</abbr>, 7 December 2021</h2>

<p>21. Preliminary modelling from three <abbr title="Scientific Pandemic Influenza Group on Modelling, Operational sub-group">SPI-M-O</abbr> groups suggest a central projection for omicron growth rate of 0.1 to 0.4 per day – this is consistent with current <abbr title="spike gene">S-gene</abbr> target failure (<abbr title="s-gene target failure">SGTF</abbr>) data with a rapid doubling time of between two and three days. As a result of this and currently increasing levels of omicron cases, it is highly likely there are currently thousands of new omicron infections per day in the <abbr title="United Kingdom">UK</abbr>. It is also highly likely that omicron will account for the majority of new <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> infections within a few weeks.</p>

<p>22. These current growth rates depend on <abbr title="s-gene target failure">SGTF</abbr> data. These may be showing some biases as a result of targeting of suspected omicron cases to laboratories that can conduct such assays, but even higher growth rates cannot be ruled out.</p>

<p>23. Assuming that omicron has the same severity or pathogenicity as delta, all modelled scenarios from the three groups so far have qualitative similarities across sensitivities analysed, irrespective of immune escape and transmissibility assumptions made and the interplay between these. Even assuming the lower end of this observed growth rate, many scenarios see hospitalisations close to or going beyond previous peak levels, if there is no reduction in transmission. The situations where the peak of the epidemic is below 1,000 to 2,000 omicron hospital admissions per day (without intervention) require low immune escape and very high protection from boosters to be consistent with observed growth.</p>

<p>24. The rapid rate of increase currently observed suggests that omicron-based hospital admissions alone could be of the order of 1,000 per day by the end of 2021.</p>

<p>25. There remains considerable uncertainty. The information about the amount of transmission and immune escape compared to delta will be significantly improved over the next two weeks, although disentangling the difference between these two may take longer. The uncertainty over severity will take longer (four to six weeks, subject to any disruption to data streams over the festive period).</p>

<p>26. Other uncertainties also remain, such as how omicron infections will move through and affect different age groups, how differently omicron may evade natural and vaccine-acquired immunity and how this may hold for booster vaccinations, and the relative scale of reduction in vaccine effectiveness between infection and severe disease.</p>

<p>27. Given the inherent delays in disease progression and data production, growth in admissions and deaths would continue for several weeks even after severe curtailment of transmission. It is probable that, once hospitalisations begin to increase at a rate similar to that of cases, four doublings could already be in the system before interventions that slow infections are reflected in slower hospitalisations.</p>

<p>28. Given the known data issues and potential for different test-seeking behaviours around the festive period, it is possible the scale of cases and hospitalisations over this period will be very difficult to track at a critical time for the epidemic.</p>

<p>29. Slowing transmission immediately will reduce the number of people who are infected with omicron before they are fully vaccinated or boosted. It will also slow the rate at which omicron overtakes delta and give more time to understand omicron’s severity.</p>

<p>30. The relative severity of omicron compared to delta remains unknown. Changes in the level of hospitalisations will broadly scale linearly with changes in severity, assuming no change in hospital capacity as admissions increase, and a marked decrease in severity is unlikely to offset the impact of a larger susceptible pool and or increased transmissibility; it would be easier to halve the number of infections than the infection hospitalisation rate (<abbr title="infection hospitalisation rate">IHR</abbr>).</p>

<p>31. <abbr title="Scientific Pandemic Influenza Group on Modelling, Operational sub-group">SPI-M-O</abbr> will discuss these modelling results and their implications further, as well as any other updates, at their next meeting, 8 December 2021.</p>

<h2>
<span class="number">7. </span>References</h2>

<div class="footnotes" role="doc-endnotes">
  <ol>
    <li role="doc-endnote">
      <p>The World Health Organisation recently recommended <a href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/" class="govuk-link" rel="external">using letters of the Greek alphabet when referring to <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> variants</a>. Current variants of concern labelling stands as B.1.1.7 as alpha, B.1.351 as beta, P.1 as gamma, B.1.617.2 as delta, and B.1.1.529 as omicron.&nbsp;<a class="govuk-link" href="#fnref:1" role="doc-backlink" aria-label="go to where this is referenced">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Increased risk of <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> reinfection associated with emergence of the Omicron variant in South Africa; <a class="govuk-link" href="https://www.medrxiv.org/content/10.1101/2021.11.11.21266068v2" rel="external">Pulliam et al</a>.&nbsp;<a href="#fnref:2" class="govuk-link" aria-label="go to where this is referenced" role="doc-backlink">↩</a></p>
    </li>
  </ol>
</div>
</div>


</div>
</div>
  </div>

  <div class="sticky-element" data-sticky-element="">
    <a class="govuk-link app-c-back-to-top dont-print" href="#contents">
    <svg class="app-c-back-to-top__icon" width="13" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 13 17" height="17" focusable="false">
      <path fill="currentColor" d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z"></path>
    </svg>
    Contents
</a>

    <div class="sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button data-module="print-link" class="govuk-link govuk-body-s gem-c-print-link__button">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>