<main class="html-publication" lang="en" role="main">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          
<div class="gem-c-organisation-logo brand--department-of-health">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color" href="/government/organisations/uk-health-security-agency">
      <span class="gem-c-organisation-logo__name">UK Health<br>Security<br>Agency</span>
</a>
</div>
        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  

<div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
      <span class="govuk-caption-xl gem-c-title__context">
    Guidance
  </span>


  <h1 class="gem-c-title__text govuk-heading-xl">
    Mycobacterium tuberculosis whole-genome sequencing and cluster investigation handbook
  </h1>
</div>
  <p class="publication-header__last-changed">Updated 5 September 2022</p>

  </header>






<div class="sidebar-with-body">
  <div class="govuk-grid-row">
      <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
          <nav aria-label="Contents" class="gem-c-contents-list" data-module="gem-track-click" role="navigation">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 1" data-track-category="contentsClicked" data-track-label="#acknowledgments" data-track-options="{&quot;dimension29&quot;:&quot;Acknowledgments&quot;}" href="#acknowledgments">Acknowledgments</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 2" data-track-category="contentsClicked" data-track-label="#authors" data-track-options="{&quot;dimension29&quot;:&quot;Authors&quot;}" href="#authors">Authors</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 3" data-track-category="contentsClicked" data-track-label="#version-control" data-track-options="{&quot;dimension29&quot;:&quot;Version control&quot;}" href="#version-control">Version control</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 4" data-track-category="contentsClicked" data-track-label="#introduction" data-track-options="{&quot;dimension29&quot;:&quot;1 Introduction &quot;}" href="#introduction"><span class="gem-c-contents-list__number">1 </span><span class="gem-c-contents-list__numbered-text">Introduction</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 5" data-track-category="contentsClicked" data-track-label="#using-wgs-for-clustering-information" data-track-options="{&quot;dimension29&quot;:&quot;\n2 Using  WGS for clustering information&quot;}" href="#using-wgs-for-clustering-information"><span class="gem-c-contents-list__number">2 </span><span class="gem-c-contents-list__numbered-text">Using WGS for clustering information</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 6" data-track-category="contentsClicked" data-track-label="#naming-wgs-clusters" data-track-options="{&quot;dimension29&quot;:&quot;\n3 Naming  WGS clusters&quot;}" href="#naming-wgs-clusters"><span class="gem-c-contents-list__number">3 </span><span class="gem-c-contents-list__numbered-text">Naming WGS clusters</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 7" data-track-category="contentsClicked" data-track-label="#wgs-cluster-annotated-examples" data-track-options="{&quot;dimension29&quot;:&quot;\n4 WGS 4 WGS cluster annotated examples&quot;}" href="#wgs-cluster-annotated-examples"><span class="gem-c-contents-list__number">4 </span><span class="gem-c-contents-list__numbered-text">WGS 4 WGS cluster annotated examples</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 8" data-track-category="contentsClicked" data-track-label="#references" data-track-options="{&quot;dimension29&quot;:&quot;References&quot;}" href="#references">References</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 9" data-track-category="contentsClicked" data-track-label="#appendix-1" data-track-options="{&quot;dimension29&quot;:&quot;Appendix 1&quot;}" href="#appendix-1">Appendix 1</a>

        </li>
    </ol>
</nav>

        
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-ga4="{&quot;event_name&quot;:&quot;print_page&quot;,&quot;type&quot;:&quot;Print this page&quot;,&quot;index&quot;:1,&quot;index_total&quot;:1,&quot;section&quot;:&quot;Contents&quot;}" data-module="ga4-event-tracker print-link">Print this page</button>
</div>
      </div>

    <div class="print-wrapper">
      <div class="print-meta-data">
        <p>
  <img class="print-meta-data-licence" src="/assets/government-frontend/open-government-licence-min-93b6a51b518ff99714a1aa2a7d2162735c155ec3cb073c75fb88b2a332fa83d3.png">
</p>
<p>
  © Crown copyright 2022
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/tb-strain-typing-and-cluster-investigation-handbook/mycobacterium-tuberculosis-whole-genome-sequencing-and-cluster-investigation-handbook
</p>


      </div>
    </div>

    <div class="main-content-container">
      <div class="gem-c-govspeak-html-publication">
  
<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
    
    
        <div class="govspeak">
<h2>Acknowledgments</h2>

<p>We gratefully acknowledge all those in the National Mycobacterial Reference laboratories and <abbr title="UK Health Security Agency">UKHSA</abbr> bioinformaticians and software engineers who undertake whole-genome sequence sample preparation and processing and maintenance of the computational systems respectively. We thank all the NHS <abbr title="tuberculosis">TB</abbr> teams and laboratories for isolation and submission of <abbr title="tuberculosis">TB</abbr> cultures. We also thank all who contribute information on <abbr title="tuberculosis">TB</abbr> notifications in England, including physicians, nurses, microbiologists, scientists, and administrative staff. Special thanks are extended to the health protection teams, for their essential collaboration in the ongoing investigation of <abbr title="tuberculosis">TB</abbr> transmission.</p>

<h2>Authors</h2>

<p><abbr title="UK Health Security Agency">UKHSA</abbr> National <abbr title="tuberculosis">TB</abbr> Unit: Dr Dona Foster (DF), Dr Suzanne Coles (SC), Dr David Wyllie, Dr Esther Robinson, Dr Lauren Ahyow, Dr Sharon Cox, Dr Martin Deadicoat, Tracey L. Langham, Surinder Tamne, Prof Grace Smith <abbr title="UK Health Security Agency">UKHSA</abbr></p>

<p>Field Services: Dr Charles Beck (CB), Andy Burkitt, Neil Macdonald</p>

<h2>Version control</h2>

<table>
  <thead>
    <tr>
      <th scope="col">Version</th>
      <th scope="col">Date</th>
      <th scope="col">Authors</th>
      <th scope="col">Description</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1.0</td>
      <td>30 June 2021</td>
      <td>DF</td>
      <td>First Draft</td>
    </tr>
    <tr>
      <td>1.1</td>
      <td>19 July 2021</td>
      <td>DF/ SC</td>
      <td>Revision</td>
    </tr>
    <tr>
      <td>1.2</td>
      <td>26 October 2021</td>
      <td>DF</td>
      <td>Addition of cluster diagrams</td>
    </tr>
    <tr>
      <td>1.3</td>
      <td>12 November 2021</td>
      <td>DF/SC</td>
      <td>Revision</td>
    </tr>
    <tr>
      <td>1.4</td>
      <td>23 November 2021</td>
      <td>All authors</td>
      <td>Revision</td>
    </tr>
    <tr>
      <td>1.5</td>
      <td>19 April 2021</td>
      <td>DF/SC/CB</td>
      <td>Revision</td>
    </tr>
  </tbody>
</table>

<h2><span class="number">1 Introduction </span></h2>

<h3>
<span class="number">1.1 </span> Purpose of the handbook</h3>

<p>This handbook supports the use of tuberculosis (<abbr title="tuberculosis">TB</abbr>) whole-genome sequencing (<abbr title="whole-genome sequencing">WGS</abbr>) data for <abbr title="tuberculosis">TB</abbr> diagnosis and investigation of <abbr title="tuberculosis">TB</abbr> clusters. It provides guidance for UK Health Security Agency (<abbr title="UK Health Security Agency">UKHSA</abbr>) staff and public health teams following the introduction of <abbr title="whole-genome sequencing">WGS</abbr> into national diagnostic service as a tool for <abbr title="tuberculosis">TB</abbr> cluster investigation. <abbr title="whole-genome sequencing">WGS</abbr> is now used routinely to identify clusters of <abbr title="tuberculosis">TB</abbr> and potential transmission chains to reduce the incidence of <abbr title="tuberculosis">TB</abbr> in the UK.</p>

<p>This handbook gives an overview of the end-to-end processes of <abbr title="whole-genome sequencing">WGS</abbr> for the investigation and management of <abbr title="tuberculosis">TB</abbr> cases and shares good practice.</p>

<p>The handbook will remain an evolving document subject to modification to reflect current knowledge. It is primarily intended for regional health protection teams to support the management of <abbr title="tuberculosis">TB</abbr> cases and incidents.</p>

<p>Note: <abbr title="whole-genome sequencing">WGS</abbr> is only available on culture positive <abbr title="tuberculosis">TB</abbr> cases, not latent <abbr title="tuberculosis">TB</abbr> infections (<abbr title="latent TB infections">LTBI</abbr>) diagnosed by Interferon-Gamma Release Assay (<abbr title="interferon-gamma release assay">IGRA</abbr>) or cases diagnosed clinically or those with negative cultures. Cases may be linked epidemiologically without <abbr title="whole-genome sequencing">WGS</abbr> where clinical samples do not exist and will require follow-up as a cluster.</p>

<h3>
<span class="number">1.2 </span> Background</h3>

<h4>1.2.1 Whole-genome sequencing (<abbr title="whole-genome sequencing">WGS</abbr>)</h4>

<p>A genome contains all the genetic information of an organism as a nucleotide sequence that includes code for specific genes and some noncoding regions. <abbr title="whole-genome sequencing">WGS</abbr> is a process whereby all, or most of, the sequence of genomic material (Deoxyribonucleic acid (<abbr title="Deoxyribonucleic acid">DNA</abbr>)) from bacterial organisms, such as Mycobacterium species, is determined. <abbr title="whole-genome sequencing">WGS</abbr> is also used for other types of cells such as human cells or viruses but these applications are not relevant to this handbook.</p>

<p><abbr title="whole-genome sequencing">WGS</abbr> involves 3 steps which are:</p>

<ol>
  <li>Laboratory-based methods to extract and prepare <abbr title="Deoxyribonucleic acid">DNA</abbr> for sequencing.</li>
  <li>Instruments to sequence the <abbr title="Deoxyribonucleic acid">DNA</abbr> bases</li>
  <li>Computational analysis to understand the sequenced bases and relate them to public health and/or clinical needs.</li>
</ol>

<p>A simple overview of the end-to-end process that occurs is shown in Figure 1.</p>

<h4>Figure 1: Simple overview of process for samples undergoing whole genome sampling</h4>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/155788/Figure_1.png"></div></figure>

<p>Box 1 sample collection and laboratory processing; Box 2 the sequencing process and Box 3 the analysis stage. All stages together are referred to as an end-to-end service.</p>

<h4>1.2.2 <abbr title="tuberculosis">TB</abbr> microbiology</h4>

<p>Mycobacterium tuberculosis (<abbr title="Mycobacterium tuberculosis">MTB</abbr>) (along with other Non-Tuberculosis Mycobacterium -NTMs) is a slow-growing organism, taking approximately 2 to 6 weeks to grow under laboratory culture conditions after the sample has been taken. Traditional microbiological methods are dependent on bacterial culture, resulting in substantial time-lag between diagnostic confirmation of the presence of <abbr title="tuberculosis">TB</abbr> and further tests such as genomic sequencing and antimicrobial susceptibility. Rapid molecular tests (such as Cepheid <abbr title="Polymerase chain reaction">PCR</abbr> based tests), which can detect Mycobacterium tuberculosis <abbr title="Deoxyribonucleic acid">DNA</abbr> in sputum, can flag patients with <abbr title="tuberculosis">TB</abbr> on the same day their sample is taken.</p>

<p>Time to culture for clinical samples is a proxy measure for infectivity <a class="govuk-link" href="#ref2">(1)</a>. Patients with large numbers of <abbr title="Mycobacterium tuberculosis">MTB</abbr> organisms in their sputum, such as those with chronic untreated infection, may grow (culture) enough mycobacteria for further testing in less than a week. Samples with lower numbers of <abbr title="Mycobacterium tuberculosis">MTB</abbr> organisms may require much longer incubation to grow (culture) <abbr title="Mycobacterium tuberculosis">MTB</abbr>. There is usually a delay, sometimes weeks, between knowing that the patient has <abbr title="tuberculosis">TB</abbr> based on <abbr title="Polymerase chain reaction">PCR</abbr> tests on sputum and being able to establish whether a cultured isolate is related to other isolates within an existing or new cluster.</p>

<p>Once mycobacteria have grown and been isolated in culture, the <abbr title="Deoxyribonucleic acid">DNA</abbr> is extracted for <abbr title="whole-genome sequencing">WGS</abbr> processing. In approximately 90% of cases, the <abbr title="whole-genome sequencing">WGS</abbr> result provides speciation (the type of tuberculosis <abbr title="Mycobacterium tuberculosis">MTB</abbr> or <abbr title="Non tuberculosis mycobacterium">NTM</abbr>). If the species is <abbr title="tuberculosis">TB</abbr>, <abbr title="whole-genome sequencing">WGS</abbr> provides antimicrobial susceptibility and determines how closely related an isolated strain is to other isolated strains within the laboratory collection. The time taken to perform the <abbr title="whole-genome sequencing">WGS</abbr> and provide the results is typically about 7 days after receipt of the isolate within the reference laboratory. The 7-day time period to complete <abbr title="whole-genome sequencing">WGS</abbr> follows the initial period a sample has taken to grow, which can take several weeks. In the other 10%, low numbers of mycobacteria organisms prevent effective sequencing, and the reference laboratory must re-grow the mycobacteria, which may cause several more weeks’ delay or may not be possible (for example, lack of sample material). Further details of processes of whole genome sequencing for mycobacterial species can be found in a series of videos available on YouTube:</p>

<ul>
  <li><a class="govuk-link" href="https://www.youtube.com/watch?v=1mByG5rZw1I" rel="external">What is whole-genome sequencing?</a></li>
  <li><a class="govuk-link" href="https://www.youtube.com/watch?v=emp4V70Ol5c" rel="external">How does whole- genome sequencing identify mycobacteria?</a></li>
  <li><a class="govuk-link" href="https://www.youtube.com/watch?v=tvqNTCZdPCc" rel="external">How does whole-genome sequencing predict drug susceptibility?</a></li>
</ul>

<h4>1.2.3 <abbr title="whole-genome sequencing">WGS</abbr> and the <abbr title="UK Health Security Agency">UKHSA</abbr> <abbr title="tuberculosis">TB</abbr> Action Plan</h4>

<p>The joint <abbr title="UK Health Security Agency">UKHSA</abbr>- NHS England <abbr title="tuberculosis">TB</abbr> Action Plan for England, 2021 to 2026, was launched in July 2021 (<a class="govuk-link" href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998158/TB_Action_Plan_2021_to_2026.pdf" rel="external"><abbr title="tuberculosis">TB</abbr> Action Plan for England, 2021 to 2026 (publishing.service.gov.uk)</a>). The aim of the <abbr title="tuberculosis">TB</abbr> Action plan is to improve the prevention, detection and control of <abbr title="tuberculosis">TB</abbr> in England and support a year-on-year reduction in <abbr title="tuberculosis">TB</abbr> incidence and in-UK <abbr title="tuberculosis">TB</abbr> transmission to enable the UK to meet its commitment to the World Health Organization (<abbr title="World Health Organization">WHO</abbr>) elimination targets by 2035.</p>

<p>There are a set of objectives within the <abbr title="tuberculosis">TB</abbr> Action Plan around <abbr title="whole-genome sequencing">WGS</abbr> - specifically, that 100% culture positive <abbr title="tuberculosis">TB</abbr> cases have <abbr title="whole-genome sequencing">WGS</abbr> relatedness data reviewed as part of decisions around contact tracing or public health intervention. There is also an ambition to support the development of direct <abbr title="whole-genome sequencing">WGS</abbr> (that is, <abbr title="whole-genome sequencing">WGS</abbr> performed directly from a sample without the need for culture) from <abbr title="tuberculosis">TB</abbr> specimens in the NHS.</p>

<p>The <abbr title="tuberculosis">TB</abbr> Action Plan includes actions linked to the outcomes of the Collaborative <abbr title="tuberculosis">TB</abbr> Strategy for England, 2015 to 2020, including increasing the use and availability of <abbr title="whole-genome sequencing">WGS</abbr> (<a class="govuk-link" href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/403231/Collaborative_TB_Strategy_for_England_2015_2020_.pdf" rel="external">Collaborative Tuberculosis Strategy for England 2015 to 2020</a> ).</p>

<p>The use of <abbr title="whole-genome sequencing">WGS</abbr> is in line with the ambition of the development of <abbr title="UK Health Security Agency">UKHSA</abbr> as outlined in the 2021 remit letter <a class="govuk-link" href="https://www.gov.uk/government/publications/ukhsa-priorities-in-2021-to-2022/letter-from-lord-bethell-to-dr-jenny-harries-ukhsa-chief-executive">Letter from Lord Bethell to Dr Jenny Harries, <abbr title="UK Health Security Agency">UKHSA</abbr> chief executive</a></p>

<p>“expand genome sequencing and analysis capacity and capability to establish a world-leading pathogen genomics system to detect and provide local, regional and national surveillance of infectious threats with strategies to provide this capability within the regional and national health and public health framework – working with devolved administrations towards a UK all pathogens genomics strategy and contributing to development of a stronger global disease surveillance system.”</p>

<h4>1.2.4 Implementation of <abbr title="whole-genome sequencing">WGS</abbr> into <abbr title="UK Health Security Agency">UKHSA</abbr>
</h4>

<p>Public Health England (<abbr title="Public Health England">PHE</abbr>) and its successor organisation <abbr title="UK Health Security Agency">UKHSA</abbr>, remains a world leader in providing a national and supra-national <abbr title="tuberculosis">TB</abbr> service. Within this service, <abbr title="whole-genome sequencing">WGS</abbr> is used as the primary diagnostic tool to simultaneously:</p>

<ul>
  <li>detect and confirm M. tuberculosis within a culture from a clinical sample</li>
  <li>identify potential laboratory processing contamination events ensuring treatment is not given to patients that do not require it</li>
  <li>predict drug resistance patterns ensuring appropriate treatment</li>
  <li>provide a high resolution of <abbr title="tuberculosis">TB</abbr> strain relatedness to trace infection sources and potential transmission networks</li>
</ul>

<p>The current programme for use of <abbr title="whole-genome sequencing">WGS</abbr> for <abbr title="tuberculosis">TB</abbr> diagnostics in England was initially led by <abbr title="Public Health England">PHE</abbr> and the Modernising Medical Microbiology group at the University of Oxford. The aim was to create a pathway from the laboratory through computational analysis to clinical reports. The work included cost-analysis to determine the financial benefit of <abbr title="whole-genome sequencing">WGS</abbr> <a class="govuk-link" href="#ref2">(2 to 7)</a>.</p>

<p>In December 2016 the full service was implemented in the National Mycobacterial Reference Service- North and Central (<abbr title="National mycobacterium reference service">NMRS</abbr>-N&amp;C) in Birmingham with the service extended to the <abbr title="National mycobacterium reference service">NMRS</abbr>-South in London in January 2018. From 2019, Public Health Wales (PHW) provided <abbr title="whole-genome sequencing">WGS</abbr> for some areas of the  Southwest England as well as Wales. Currently, around 1400 mycobacterial samples per month are processed in these laboratories. The sample and results pathway are shown in Figure 2. This service provides a comprehensive national <abbr title="tuberculosis">TB</abbr> surveillance picture using <abbr title="whole-genome sequencing">WGS</abbr> data on culture positive samples to inform clusters.</p>

<h4>Figure 2: Overview of sample journey through local and national laboratories and reporting of results</h4>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/155789/Figure_2.png"></div></figure>

<p><abbr title="Complete Pathogen Analytical Software Solution">COMPASS</abbr> (COMplete Pathogen Analytical Software Solution) is the bioinformatic pipeline where sequences are processed. FOREST is an <abbr title="UK Health Security Agency">UKHSA</abbr> program that links patient information with the sequence data.</p>

<h4>1.2.5 Speciation and resistance</h4>

<p><abbr title="whole-genome sequencing">WGS</abbr> identifies all and reports all species in the Mycobacterium tuberculosis Complex (MTC or MTBC). The major human disease-causing strains in this group are M. tuberculosis and M. africanum. Other species within the group include species associated with animals where zoonotic infection may have occurred (for example, M. bovis, M. microti, M. caprae and M. canettii). M bovis infection associated with the Bacillus Calmette-Guerin (<abbr title="Bacillus Calmette-Guerin">BCG</abbr>) strain may be identified but is not notifiable.</p>

<p>The use of <abbr title="whole-genome sequencing">WGS</abbr> for speciation and antimicrobial susceptibility for mycobacterium other than MTC including all NTMs is outside the scope of this document.</p>

<h2>
<span class="number">2 Using </span> <abbr title="whole-genome sequencing">WGS</abbr> for clustering information</h2>

<h3>
<span class="number">2.1 </span> Introduction to using <abbr title="whole-genome sequencing">WGS</abbr> for clustering information</h3>

<p>As implemented in <abbr title="UK Health Security Agency">UKHSA</abbr> <abbr title="tuberculosis">TB</abbr> <abbr title="whole-genome sequencing">WGS</abbr>, clustering is a process which yields a group of strains with closely related genomes; epidemiological data is not used during cluster formation. The process offers strong guarantees that all pairs of samples whose genomes are suggestive of recent case to case transmission will lie within the same cluster. However, it does not offer guarantees that all members of the cluster transmitted to each other recently, or even at all. This important to appreciate and is discussed more later.</p>

<p>Clustering based on molecular data requires an understanding of both how similar or different one <abbr title="tuberculosis">TB</abbr> strain is from another, and whether such differences should result in isolates being placed in a cluster compared to previous methods.</p>

<p><abbr title="whole-genome sequencing">WGS</abbr> for <abbr title="tuberculosis">TB</abbr> improves sensitivity of clusters when compared to 24 loci Mycobacterial Interspersed Repetitive Unit – Variable Number Tandem Repeats (<abbr title="Mycobacterial Interspersed Repetitive Unit - Variable Number Tandem Repeats">MIRU-VNTR</abbr>) typing. The discriminatory power of <abbr title="whole-genome sequencing">WGS</abbr> enables outbreaks to be resolved with greater speed and confidence. The first use of <abbr title="whole-genome sequencing">WGS</abbr> to describe <abbr title="tuberculosis">TB</abbr> diversity <a class="govuk-link" href="#ref8">(8)</a> demonstrated limitations in <abbr title="Deoxyribonucleic acid">DNA</abbr> fingerprinting in the detection of diversity between <abbr title="tuberculosis">TB</abbr> strains. Following this, studies in the Netherlands and Canada showed that application of <abbr title="whole-genome sequencing">WGS</abbr> could resolve large outbreaks to distinct sub-clusters assisting epidemiological resolution <a class="govuk-link" href="#ref9">(9 to 11)</a>. This evidence suggested that <abbr title="whole-genome sequencing">WGS</abbr> application to <abbr title="tuberculosis">TB</abbr> cluster investigation could assist in public health understanding and resolution of <abbr title="tuberculosis">TB</abbr> transmission. A <abbr title="Public Health England">PHE</abbr>-led study indicated that <abbr title="whole-genome sequencing">WGS</abbr> was about 80 times more granular than the legacy typing method then in use, <abbr title="Mycobacterial Interspersed Repetitive Unit - Variable Number Tandem Repeats">MIRU-VNTR</abbr> <a class="govuk-link" href="#ref12">(12)</a>. Because of this increased granularity, a 2015 national review recommended that <abbr title="whole-genome sequencing">WGS</abbr> replace routine <abbr title="Mycobacterial Interspersed Repetitive Unit - Variable Number Tandem Repeats">MIRU-VNTR</abbr> typing as a public health tool for <abbr title="tuberculosis">TB</abbr> cluster management and investigation.</p>

<p><abbr title="whole-genome sequencing">WGS</abbr> uses variations in the genetic material (nucleotide bases of the <abbr title="Deoxyribonucleic acid">DNA</abbr>) of the unknown isolate compared to a reference strain (<abbr title="UK Health Security Agency">UKHSA</abbr> use H37rV) to produce a series of single polynucleotide polymorphisms (<abbr title="Single polymorphism nucleotide">SNPs</abbr>), that is the differences between the genomes. <abbr title="whole-genome sequencing">WGS</abbr> uses approximately 88% to 92% of the genome to identify the number of <abbr title="Single polymorphism nucleotide">SNPs</abbr> and their position on the genome. Some areas of the bacterial genome are difficult to align against the reference strain due to the size of the fragments of <abbr title="Deoxyribonucleic acid">DNA</abbr> generated by the sequencing method currently used. Further areas may be masked (that is not used) because they contain areas of low-quality mapping against the reference genome. The <abbr title="Single polymorphism nucleotide">SNP</abbr> threshold is the minimum number of changes in the sequenced genomes that places two people in the same transmission cluster, illustrated in Figure 3. The position of <abbr title="Single polymorphism nucleotide">SNPs</abbr> within the genome are used to create a phylogenetic tree showing how isolates may be related.</p>

<p>Note: a free training course on phylogenetic trees can be found at <a class="govuk-link" href="https://www.ebi.ac.uk/training/online/courses/introduction-to-phylogenetics/" rel="external">
Phylogenetics: an introduction</a>.</p>

<h4>Figure 3: A schematic representation of the relatedness between three mycobacterial isolates (A, B and C) (Image created by M Reynolds)</h4>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/155790/Figure_3.png"></div></figure>

<p><abbr title="whole-genome sequencing">WGS</abbr> results, when combined with epidemiological data, can identify patients that may be involved in the same chain of <abbr title="tuberculosis">TB</abbr> transmission. This can assist with strategies for control measures and help in the understanding of <abbr title="tuberculosis">TB</abbr> epidemiology in different settings.</p>

<p>The main objectives of <abbr title="tuberculosis">TB</abbr> <abbr title="whole-genome sequencing">WGS</abbr> cluster investigation are:</p>

<p>– at patient level –</p>

<ul>
  <li>to identify issues such as laboratory contamination that may lead to unnecessary investigation and treatment</li>
  <li>to distinguish between re-infection from reactivation in patients with history of <abbr title="tuberculosis">TB</abbr>
</li>
</ul>

<p>– at regional and local public health level –</p>

<ul>
  <li>to provide typing results to prospectively identify local, regional, or national incidents of suspected transmission, enabling proactive, locally led response to <abbr title="tuberculosis">TB</abbr> control and public health protection</li>
  <li>to determine whether patients are NOT part of a cluster for public health team resource management, such as targeting contact tracing</li>
</ul>

<p>– at <abbr title="UK Health Security Agency">UKHSA</abbr> <abbr title="tuberculosis">TB</abbr> Unit national level –</p>

<ul>
  <li>to understand the national and local epidemiology of <abbr title="tuberculosis">TB</abbr>, including identification of risk groups and social networks for recent transmission</li>
  <li>to understand the molecular epidemiology of <abbr title="tuberculosis">TB</abbr>, including diversity of circulating strains, geographical distribution, and lineages</li>
  <li>to monitor <abbr title="tuberculosis">TB</abbr> control by analysing trends in estimates of recent transmission</li>
  <li>to meet international obligations for molecular surveillance, Europe-wide and globally</li>
</ul>

<h3>
<span class="number">2.2 </span> <abbr title="tuberculosis">TB</abbr> <abbr title="whole-genome sequencing">WGS</abbr> and cluster investigation</h3>

<p><abbr title="tuberculosis">TB</abbr> <abbr title="whole-genome sequencing">WGS</abbr> cluster investigations use the <abbr title="Single polymorphism nucleotide">SNPs</abbr> between isolates to estimate whether they are closely related, that is indicative of likely recent transmission.</p>

<p>Isolates from <abbr title="tuberculosis">TB</abbr> cases can differ significantly in the number of <abbr title="Single polymorphism nucleotide">SNPs</abbr> identified between them (0 to &gt;1000 SNPS). Direct or recent transmission is associated with small numbers of nucleotide differences between separate samples. Initial investigation into strains from the UK indicated that <abbr title="Single polymorphism nucleotide">SNP</abbr> differences of less than 5 may flag recent transmission while <abbr title="Single polymorphism nucleotide">SNP</abbr> differences over 12 do not represent a recent transmission network <a class="govuk-link" href="#ref6">(6)</a>, <a class="govuk-link" href="#ref11">(11)</a>.</p>

<p>While this is a useful guide, <abbr title="Single polymorphism nucleotide">SNP</abbr> differences between isolates depend upon several factors including: time lapse between isolates, acquisition of drug resistance and specific phylogenetic lineage <a class="govuk-link" href="#ref13">(13 to 16)</a>. <abbr title="Single polymorphism nucleotide">SNP</abbr> differences may also indicate mixed infection or laboratory contamination <a class="govuk-link" href="#ref17">(17)</a>. The molecular clock of the tuberculosis genome, estimated as 0.5 changes in the genomic sequence per year <a class="govuk-link" href="#ref11">(11)</a>, can also vary in different settings <a class="govuk-link" href="#ref18">(18)</a>.</p>

<p>This means that while 12 SNPS can be defined as the upper threshold of genomic relatedness between epidemiologically related hosts <a class="govuk-link" href="#ref19">(19)</a>, lower limits must be assessed on an individual situation or outbreak basis. Please note that currently the tool used to display, access, and utilise <abbr title="tuberculosis">TB</abbr> relatedness results is called Forest. Forest uses a 12 <abbr title="Single polymorphism nucleotide">SNP</abbr> distance as a cut-off for clustering isolates.</p>

<h4>2.2.1 Definitions</h4>

<p>Epidemiological cluster</p>

<p>An epidemiological cluster is defined as “2 or more persons with known epidemiological links”. An epidemiological link is defined as person linked by time and place.</p>

<p>The number of <abbr title="Single polymorphism nucleotide">SNPs</abbr> identified from <abbr title="whole-genome sequencing">WGS</abbr> in related strains provides additional evidence as to whether a recent transmission event is likely to have occurred. Cases found to be greater than 12 <abbr title="Single polymorphism nucleotide">SNPs</abbr> apart from each other will generally not be considered to belong to the same chain of transmission.</p>

<p>Cluster by <abbr title="whole-genome sequencing">WGS</abbr> <abbr title="Single polymorphism nucleotide">SNP</abbr> differences</p>

<p>A <abbr title="whole-genome sequencing">WGS</abbr> cluster is defined as “2 or more persons with a diagnosis of active <abbr title="tuberculosis">TB</abbr> that have less than 12 <abbr title="Single polymorphism nucleotide">SNPs</abbr> difference”. Note that 1 person with multiple isolates will form a genomic cluster which will not require further follow up.</p>

<p>Epidemiological information is required to determine the likelihood of 2 isolates belonging to a cluster and representing recent transmission. The assessment of likely recent transmission of isolates clustered by <abbr title="whole-genome sequencing">WGS</abbr> will depend on the amount of time between samples, geographical location, whether the isolates are multidrug resistant and the estimated changes (molecular clock) associated with the isolates. The epidemiological investigation will allow understanding of the potential for two cases to have had contact.</p>

<p>The following is a guide – expert guidance is available from the <abbr title="UK Health Security Agency">UKHSA</abbr> <abbr title="tuberculosis">TB</abbr> Unit <a class="govuk-link" href="mailto:TBSection@phe.gov.uk">TBSection@phe.gov.uk</a></p>

<p>For this handbook, a cluster is defined as cases with <abbr title="tuberculosis">TB</abbr> isolates that exhibit 2 or more confirmed cases of <abbr title="tuberculosis">TB</abbr> with 0 to 12 <abbr title="Single polymorphism nucleotide">SNP</abbr> differences between their genomic sequence with supporting epidemiological evidence.</p>

<p>Epidemiologically confirmed <abbr title="whole-genome sequencing">WGS</abbr> cluster</p>

<p>This is defined as “a cluster of 2 or more persons with <abbr title="tuberculosis">TB</abbr> whose sequence data have 12 or less <abbr title="Single polymorphism nucleotide">SNP</abbr> differences between them and have known epidemiological links between at least two other members of the cluster”.</p>

<h4>2.2.2 Epidemiological links</h4>

<p>Epidemiological links are based on information collected from 2 or more cases with active <abbr title="tuberculosis">TB</abbr> who are associated in time and place identified during interview, contact tracing or cluster investigations. Epidemiological links between <abbr title="tuberculosis">TB</abbr> cases can be characterised into three broad categories: known or definite epidemiological links, possible epidemiological links, and no identified epidemiological links.</p>

<p>Known epidemiological link</p>

<p><abbr title="tuberculosis">TB</abbr> patients have a known epidemiological link if the following one of the following conditions are met:</p>

<ul>
  <li>one case volunteered the name of the other as a contact while either was potentially infectious (that is had pulmonary <abbr title="tuberculosis">TB</abbr>, irrespective of sputum smear status) and this can be validated</li>
  <li>information confirms that the cases were in the same setting, such as hospital, school, workplace, sports place, or prison, at the same time when either of them could have been potentially infectious – this applies to pulmonary <abbr title="tuberculosis">TB</abbr> cases irrespective of sputum smear status</li>
  <li>other information linking the cases is identified through follow up by <abbr title="Health Protection Teams">HPTs</abbr> and Field Service</li>
</ul>

<p>Possible epidemiological link</p>

<p><abbr title="tuberculosis">TB</abbr> cases have a possible epidemiological link if any of the following conditions are met:</p>

<ul>
  <li>2 cases spent time in the same setting around the same time, but the timing that they were present, or the timing of the infectious period was not definite enough to meet the criteria for a definite epidemiological link</li>
  <li>2 cases lived in the same setting around the same time and this can be validated</li>
  <li>2 cases worked in, or were in the same setting, around the same time and shared social, occupational or behaviour characteristic social risk factors or similar backgrounds that increase the chances of transmission</li>
  <li>one case volunteered the name of the other as a contact but both patients have non-pulmonary <abbr title="tuberculosis">TB</abbr>
</li>
  <li>2 cases were in the same setting at the same time but both patients have non-pulmonary <abbr title="tuberculosis">TB</abbr>
</li>
</ul>

<p>No identified epidemiological link</p>

<p>Two cases should be classified as having no identified epidemiological link if they do not meet the criteria above. However, be aware it is possible that within this, epidemiological links may exist that have not been ascertained or identified.</p>

<p>For this handbook, the term ‘cluster’ is subsequently used to refer to <abbr title="whole-genome sequencing">WGS</abbr> cluster.</p>

<p><abbr title="tuberculosis">TB</abbr> cases may be linked epidemiologically without <abbr title="whole-genome sequencing">WGS</abbr> (for example where clinical samples do not exist) and will require follow-up as a cluster.</p>

<h4>2.2.3 <abbr title="tuberculosis">TB</abbr> <abbr title="whole-genome sequencing">WGS</abbr> cluster information to inform public health action</h4>

<p>It is important to note that:</p>

<ul>
  <li>it is possible that <abbr title="whole-genome sequencing">WGS</abbr> gives low <abbr title="Single polymorphism nucleotide">SNP</abbr> differences between 2 isolates, indicating a cluster, but epidemiological links cannot be demonstrated – for example, a person unknown to healthcare or who has not presented in the UK may link known cases</li>
  <li>an existing isolate can change status from ‘not-clustered’ to ‘clustered’ if further information becomes available – such as the addition of a new isolate from another patient with <abbr title="whole-genome sequencing">WGS</abbr> data that exhibits <abbr title="Single polymorphism nucleotide">SNP</abbr> differences under 12 from the existing isolate</li>
  <li>a person who is epidemiologically linked to a cluster may not be linked by <abbr title="whole-genome sequencing">WGS</abbr> <abbr title="Single polymorphism nucleotide">SNP</abbr> difference – for example, they may have a re-occurrence or re-activation coincidently at the same time, or they may be part of a different cluster identified by <abbr title="whole-genome sequencing">WGS</abbr> that shares epidemiological attributes</li>
  <li>
<abbr title="whole-genome sequencing">WGS</abbr> relies on having a culture positive sample so culture negative <abbr title="tuberculosis">TB</abbr> cases will not be represented – note that persons identified with <abbr title="latent TB infections">LTBI</abbr> will not be present in trees nor will clinically treated cases of active <abbr title="tuberculosis">TB</abbr> (such as children treated as part of family cluster)</li>
  <li>
<abbr title="whole-genome sequencing">WGS</abbr> cannot prove transmission, or its direction – clustering and low <abbr title="Single polymorphism nucleotide">SNP</abbr> difference, alongside a defined epidemiological link is indicative of transmission</li>
</ul>

<h2>
<span class="number">3 Naming </span> <abbr title="whole-genome sequencing">WGS</abbr> clusters</h2>

<p>Clusters are automatically assigned a name in the Forest tool. The naming convention, adopted by <abbr title="UK Health Security Agency">UKHSA</abbr> (previously <abbr title="Public Health England">PHE</abbr>) in May 2017, is 2 letters (assigned sequentially starting with AA) followed by 3 numbers (also assigned sequentially starting with 001). For example, once AA999 is reached it moves to AB001. Lineage is added at the end of the name separated by a dash and corresponding to:</p>

<p>1 = Indo-Oceanic</p>

<p>2 = East Asian</p>

<p>3 = East-African Indian</p>

<p>4 = Euro-American</p>

<p>5 and 6 = M. africanum</p>

<p>BOV = M. bovis</p>

<p>Examples of cluster names include AA043-3 and AC344-BOV.</p>

<p>In Forest, multiple isolates from an individual will be assigned the same cluster name (providing they are within 12 <abbr title="Single polymorphism nucleotide">SNPs</abbr>) and this name will be retained should additionally samples from a different individual (2 or more people within 12 <abbr title="Single polymorphism nucleotide">SNPs</abbr>) occur.</p>

<p>Clusters may merge if an individual arises who is within 12 <abbr title="Single polymorphism nucleotide">SNPs</abbr> of 2 or more established clusters. The earliest cluster name will be retained for this merged cluster.</p>

<h3>
<span class="number">3.1 </span> Cluster management process</h3>

<p>As <abbr title="UK Health Security Agency">UKHSA</abbr>’s Regional Health Protection Teams are unable to access cluster data directly at this time, cluster information is disseminated via the Field Service Cluster Investigators to the Health Protection Teams (<abbr title="Health Protection Teams">HPTs</abbr>) and <abbr title="tuberculosis">TB</abbr> services/case managers. New additions to clusters are reviewed daily by the Field Service Cluster Investigators. Information is then shared directly with local <abbr title="Health Protection Teams">HPTs</abbr> and <abbr title="tuberculosis">TB</abbr> nurses.</p>

<p>Information concerning multi-drug resistant isolates is shared directly with the local clinical teams by the <abbr title="UK Health Security Agency">UKHSA</abbr> <abbr title="tuberculosis">TB</abbr> Unit or National Mycobacterial Reference Service (<abbr title="National mycobacterium reference service">NMRS</abbr>). <abbr title="Health Protection Teams">HPTs</abbr> can contact the National <abbr title="tuberculosis">TB</abbr> Surveillance Unit directly via email at <a class="govuk-link" href="mailto:TBSection@phe.gov.uk">TBSection@phe.gov.uk</a> if any concerns arise regarding patients and/or clusters.</p>

<p>To support regional <abbr title="Health Protection Teams">HPTs</abbr> in interpreting and acting on cluster data, information review happens at <abbr title="UK Health Security Agency">UKHSA</abbr> <abbr title="tuberculosis">TB</abbr> Unit and <abbr title="National mycobacterium reference service">NMRS</abbr> national level on some clusters. An initial cluster review is undertaken by a senior scientist from the <abbr title="UK Health Security Agency">UKHSA</abbr> <abbr title="tuberculosis">TB</abbr> Unit and two Field Service cluster investigators. Each cluster is assessed on the number of new cases, <abbr title="Single polymorphism nucleotide">SNP</abbr> differences between cases and known epidemiological information. This will determine whether further review is necessary (summarising and recording the rationale).</p>

<p>Rationale for cases not recommended for further review at <abbr title="Air Traffic control National cluster meeting">ATC</abbr> meeting are:</p>

<ul>
  <li>cluster information and epidemiological links/ transmission settings already known, for example a case is identified as part of a household grouping or cases identified as part of incident screening – the cluster may not require further review by <abbr title="Air Traffic control National cluster meeting">ATC</abbr> as the links have been established; information will be shared with <abbr title="Health Protection Teams">HPTs</abbr> as the new case may trigger further screening</li>
  <li>large <abbr title="Single polymorphism nucleotide">SNP</abbr> differences indicating cases are unlikely to reflect recent case to case transmission</li>
  <li>likely importation of strain and no indication of in-UK transmission due to date and time of symptoms and samples</li>
</ul>

<p>Reasons for cases recommended for further review are:</p>

<ul>
  <li>rapid growth in a cluster based on the number of new additions within 12 months</li>
  <li>indication of drug resistance as multi-drug resistance cases are likely to have treatment challenges</li>
  <li>clusters with cases with geographical dispersal (need for co-ordination of response)</li>
  <li>cases with social risk factors where contact tracing and control may be challenging</li>
  <li>cases linked to educational, healthcare and congregate e.g. prison or workplace settings</li>
  <li>cases are children (&lt; 16 years) without an obvious source or index case in a household</li>
  <li>dynamic risk assessment meets criteria for escalation of response from routine to national incident (or has potential)</li>
</ul>

<p>If a cluster requires further review, it is flagged for discussion by a multi-disciplinary team known as the ‘Air Traffic Control’ (<abbr title="Air Traffic control National cluster meeting">ATC</abbr>) group. The <abbr title="Air Traffic control National cluster meeting">ATC</abbr> group meets monthly and is chaired by the head of the <abbr title="UK Health Security Agency">UKHSA</abbr> <abbr title="tuberculosis">TB</abbr> unit and has representation from <abbr title="National mycobacterium reference service">NMRS</abbr>, <abbr title="tuberculosis">TB</abbr> Unit, <abbr title="Health Protection Teams">HPTs</abbr> and Field Service. The aim of the <abbr title="Air Traffic control National cluster meeting">ATC</abbr> is to support local cluster investigators and <abbr title="Health Protection Teams">HPTs</abbr> to identify ‘high risk’ clusters of concern, provided expertise for the interpretation of <abbr title="whole-genome sequencing">WGS</abbr> relatedness data, as well as expert advice relating to the public health investigation and response.</p>

<p>The <abbr title="Air Traffic control National cluster meeting">ATC</abbr> review process will assess how new cases relate to others in the cluster in terms of:</p>

<ul>
  <li>whether <abbr title="Single polymorphism nucleotide">SNP</abbr> differences are suggestive of possible transmission</li>
  <li>geographical distribution – plausibility of transmission</li>
  <li>temporal relationship – plausibility of recent transmission or reactivation</li>
  <li>demographic profile – similarity to other cases</li>
  <li>already known epidemiological links between cases in the cluster</li>
  <li>already known contextual transmission settings identified / investigated such as common risk factors, behaviours, or activities</li>
</ul>

<p>Once cases have been discussed at the <abbr title="Air Traffic control National cluster meeting">ATC</abbr> meeting, information is relayed back to the <abbr title="Health Protection Teams">HPT</abbr> and <abbr title="tuberculosis">TB</abbr> service where the case resides for local investigation follow up and public health management. Regional <abbr title="Health Protection Teams">HPTs</abbr> and Field Service teams will have their own arrangements for cluster review process to review the public health investigation and follow-up.</p>

<h3>
<span class="number">3.2 </span> How to use cluster information</h3>

<h4>Health Protection Teams</h4>

<p>Regional <abbr title="Health Protection Teams">HPTs</abbr> will have arrangements for reviewing and managing <abbr title="whole-genome sequencing">WGS</abbr> clusters. Information in relation to new cases or expanding clusters is shared with the <abbr title="Health Protection Teams">HPT</abbr> by the Field Service Cluster Investigator. Work is then undertaken alongside <abbr title="tuberculosis">TB</abbr> services to establish epidemiological links between the cases.</p>

<p><abbr title="tuberculosis">TB</abbr> services/case managers are also informed by the Field Service Cluster Investigator when their patients are found to be part of a <abbr title="whole-genome sequencing">WGS</abbr> cluster and further information can be requested. Once the local <abbr title="Health Protection Teams">HPTs</abbr> are informed that a cluster exists, the following factors may influence local cluster public health actions and risk assessment:</p>

<ul>
  <li>the cluster includes a child (defined as under 16 years), especially if under 2 years old</li>
  <li>the infectiousness of case, such as time to culture, X-ray cavities, length of illness, indicates risk of transmission</li>
  <li>the cluster includes cases with social risk factors which makes both treatment and identification of contacts more challenging</li>
  <li>the cluster includes a health care or social care professional, because of the possibility of nosocomial transmission</li>
  <li>the cluster includes a drug-resistant or multidrug -resistant strain</li>
  <li>the <abbr title="Single polymorphism nucleotide">SNP</abbr> distance and time frame between isolates</li>
  <li>a cluster of M. bovis strains (due to concern over zoonotic transmission)</li>
  <li>the cluster includes a case with known HIV infection</li>
</ul>

<h3>
<span class="number">3.3 </span> Roles and responsibilities</h3>

<h4>Identification of <abbr title="whole-genome sequencing">WGS</abbr> clusters</h4>

<p>As detailed above clusters identified by <abbr title="whole-genome sequencing">WGS</abbr> sequencing are initially reviewed on an individual basis by a senior scientist from the <abbr title="UK Health Security Agency">UKHSA</abbr> <abbr title="tuberculosis">TB</abbr> Unit and 2 Field Service cluster investigators.</p>

<p>All information is cascaded to the appropriate <abbr title="Health Protection Teams">HPT</abbr> after review and following discussion at the <abbr title="Air Traffic control National cluster meeting">ATC</abbr> meeting if the cluster is discussed.</p>

<p>A cluster involving 2 or more persons with epidemiologically linked risks, such as household contact, generally require no further public health action following <abbr title="whole-genome sequencing">WGS</abbr> cluster identification with a small number of <abbr title="Single polymorphism nucleotide">SNPs</abbr>. Clusters with <abbr title="whole-genome sequencing">WGS</abbr> results that involve 2 or more persons with small <abbr title="Single polymorphism nucleotide">SNP</abbr> numbers with unknown/possible epidemiological links or multi-drug resistance should trigger further investigation by <abbr title="Health Protection Teams">HPTs</abbr>.</p>

<p>The <abbr title="UK Health Security Agency">UKHSA</abbr> <abbr title="tuberculosis">TB</abbr> Unit can provide additional support to cluster investigation:</p>

<ul>
  <li>as national focal point for organisation and coordinating information relating to geographically dispersed and long-standing historical clusters</li>
  <li>as a source of expertise and skills of different professionals to assess, support plan and manage the public health response to complex clusters</li>
</ul>

<p><abbr title="UK Health Security Agency">UKHSA</abbr> <abbr title="tuberculosis">TB</abbr> Unit and <abbr title="National mycobacterium reference service">NMRS</abbr> has a role in seeking assurance that public health follow up has been undertaken by <abbr title="Health Protection Teams">HPTs</abbr> in line with the <abbr title="UK Health Security Agency">UKHSA</abbr> and NHSE <abbr title="tuberculosis">TB</abbr> Action Plan. <abbr title="Health Protection Teams">HPTs</abbr> will be asked for feedback and outcome of investigations of case and clusters of <abbr title="tuberculosis">TB</abbr> linked by <abbr title="whole-genome sequencing">WGS</abbr>.</p>

<h4>Investigation of <abbr title="whole-genome sequencing">WGS</abbr> clusters</h4>

<p>The investigation and response to cases linked by <abbr title="whole-genome sequencing">WGS</abbr> is led by the <abbr title="Health Protection Teams">HPT</abbr> in liaison with local <abbr title="tuberculosis">TB</abbr> services where the case usually resides geographically.</p>

<p>Where cases are located across multiple <abbr title="Health Protection Teams">HPTs</abbr>, a lead <abbr title="Health Protection Teams">HPT</abbr> may be agreed with input from the <abbr title="UK Health Security Agency">UKHSA</abbr> <abbr title="tuberculosis">TB</abbr> Unit.</p>

<p>In some clusters epidemiological links between cases will already have been established through follow up of the index case and routine contact tracing activities.</p>

<p>Where this has not been established, actions that follow the identification of a cluster may include:</p>

<ul>
  <li>requesting enhanced information from cases through questionnaires to ascertain social networks</li>
  <li>further analysis of cases in the cluster to review risk social risk factors and potential links</li>
  <li>review of potential settings for public health action, contact tracing/screening and awareness raising</li>
</ul>

<p>Management of local outbreaks of <abbr title="tuberculosis">TB</abbr> (for example in a school, hospital, residential care, prison, or community) including those identified by <abbr title="whole-genome sequencing">WGS</abbr> requires a multi-agency response.</p>

<p>An Outbreak Control Team (<abbr title="Outbreak control team">OCT</abbr>), should be convened by the local Health Protection Team as agreed through their regional process. Tiggers for an <abbr title="Outbreak control team">OCT</abbr> will depend on risk assessment of the case. Guidance for outbreaks can be found at <a class="govuk-link" href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/343723/12_8_2014_CD_Outbreak_Guidance_REandCT_2__2_.pdf" rel="external">Communicable Disease Outbreak Management: operational guidance</a>.</p>

<p>For <abbr title="tuberculosis">TB</abbr> clusters linked by <abbr title="whole-genome sequencing">WGS</abbr> that are considered to be an outbreak or where wider actions are considered, the following persons should also be included as members of the <abbr title="Outbreak control team">OCT</abbr>:</p>

<ul>
  <li>Consultant in Communicable Disease Control/ Consultant in Health Protection (<abbr title="Consultant in Communicable Disease Control/">CCDC</abbr> / <abbr title="Community Health Partnerships">CHP</abbr>) responsible for the coordination of the <abbr title="Outbreak control team">OCT</abbr>
</li>
  <li>NHS <abbr title="tuberculosis">TB</abbr> Clinical Team (<abbr title="tuberculosis">TB</abbr> nurses, Physician responsible for the care of the case to establish infectious status of index case)</li>
  <li>microbiologist and/or infectious disease consultant from the treating NHS Trust</li>
  <li>local authority representative (Director of Public Health or nominated deputy)</li>
  <li>commissioners and/or providers (that is Clinical Commissioning Group representative, NHS England)</li>
  <li>depending upon the setting, representatives as appropriate from the implicated institutions for example hospital <abbr title="Infection Prevention and Control">IPC</abbr> or school nursing service, prison service may be appropriate</li>
</ul>

<p>As needed or directed by the lead <abbr title="Consultant in Communicable Disease Control/">CCDC</abbr>/ <abbr title="Community Health Partnerships">CHP</abbr>:</p>

<ul>
  <li>
<abbr title="UK Health Security Agency">UKHSA</abbr> <abbr title="tuberculosis">TB</abbr> Unit – <abbr title="tuberculosis">TB</abbr> service senior scientist / consultant from NRMS</li>
  <li>
<abbr title="UK Health Security Agency">UKHSA</abbr> <abbr title="tuberculosis">TB</abbr> Unit – Clinical Team member Senior Nurse</li>
  <li>Field Service Cluster Investigator</li>
  <li>
<abbr title="UK Health Security Agency">UKHSA</abbr> Travel and Migrant Health Section senior scientist/consultant epidemiologist (for national outbreaks associated with overseas travel only)</li>
  <li>
<abbr title="UK Health Security Agency">UKHSA</abbr> Health and Justice Team/consultant epidemiologist (for national outbreaks associated with secure settings)</li>
</ul>

<p>Advice on outbreak investigation and management is available from the National <abbr title="tuberculosis">TB</abbr> Unit. A physician and <abbr title="tuberculosis">TB</abbr> specialist nurse are on duty every day and can be contacted at <a class="govuk-link" href="mailto:TBSection@phe.gov.uk">TBSection@phe.gov.uk</a></p>

<h2>
<span class="number">4 WGS </span>4 <abbr title="whole-genome sequencing">WGS</abbr> cluster annotated examples</h2>

<h3>
<span class="number">4.1 </span> A cluster with low <abbr title="Single polymorphism nucleotide">SNP</abbr> difference</h3>

<p>Figure 4 shows a cluster with 11 people with active culture confirmed <abbr title="tuberculosis">TB</abbr> that are 0 or 1 <abbr title="Single polymorphism nucleotide">SNP</abbr> apart alongside a timeline of symptom onset and diagnosis date (identifiers have been removed). The timeline and the blobogram (a pictorial representation of a phylogenetical tree) with demographic and geo-spacial information can be used to determine possible transmission within a community. The low <abbr title="Single polymorphism nucleotide">SNP</abbr> difference and short timeline in this example suggest that transmission within patients in this cluster has occurred, but this can be only confirmed by epidemiological data. Note that cluster information cannot determine the direction of transmission at this time.</p>

<h4>Panel 1</h4>
<p>A phylogenetic tree generated using a rooted maximum likelihood (<abbr title="Maximum Likelihood phylogenetic tree">ML</abbr>) algorithm. There are 15 samples that include repeat samples from 2 patients and a sample from a laboratory outside the UK. The tree shows 0 <abbr title="Single polymorphism nucleotide">SNPs</abbr> between 9 patients (P) with 2 patients that are one <abbr title="Single polymorphism nucleotide">SNP</abbr> apart (P2 and P11). The isolates that are one <abbr title="Single polymorphism nucleotide">SNP</abbr> apart have different changes in the sequence compared to the main group. They have a 2 <abbr title="Single polymorphism nucleotide">SNP</abbr> distance between each other. From panel 3 (the timeline), we can see that these samples differ in time, P2 diagnosed in 2020 and P11 diagnosed in 2021.</p>

<h4>Panel 2</h4>
<p>This panel shows a blobogram which is a different way to show the <abbr title="Maximum Likelihood phylogenetic tree">ML</abbr> tree. This diagram only shows samples from the UK (number=14) A blobogram takes all isolates that are 0 <abbr title="Single polymorphism nucleotide">SNPs</abbr> apart and puts them in a blue circle. Here, all samples from the patients are shown in the central blue circle. The 2 patients that have <abbr title="Single polymorphism nucleotide">SNP</abbr> differences are shown connected to the central blob. The <abbr title="Single polymorphism nucleotide">SNP</abbr> difference is indicated (blue arrows). From this, we can see more clearly that the 2 patients that are one <abbr title="Single polymorphism nucleotide">SNP</abbr> apart from the central blob are 2 <abbr title="Single polymorphism nucleotide">SNPs</abbr> away from each other. The yellow diamond in the central circle indicates the putative oldest sample. A putative sample is one that is inferred to be the oldest based on the genetic sequence of other samples. In some clusters the putative sample is a known sample but sometimes an actual sequence is not available. When it is not present, the algorithm used to create the tree will assign an assumed putative sample.</p>

<h4>Panel 3</h4>
<p>This panel shows the epidemiological timeline of the 11 patients in cluster with dates of symptom onset (a red triangle) and dates of diagnosis (a yellow circle). It also indicates whether patients have pulmonary disease (pink bar) or not (black bar) or site of disease is not known (grey). This information is taken from <abbr title="tuberculosis">TB</abbr> surveillance system records.</p>

<h4>Figure 4: Annotated cluster showing <abbr title="Maximum Likelihood phylogenetic tree">ML</abbr> tree (panel 1), blobogram (panel 2) and timeline (panel 3)</h4>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/155791/Figure_4.png"></div></figure>

<h3>
<span class="number">4.2 </span> A cluster with variable <abbr title="Single polymorphism nucleotide">SNP</abbr> differences</h3>

<p>Figure 5 shows a cluster with 10 people with active culture confirmed <abbr title="tuberculosis">TB</abbr> that are between 1 and 6 <abbr title="Single polymorphism nucleotide">SNPs</abbr> from nearest neighbour (one shown with 7 <abbr title="Single polymorphism nucleotide">SNPs</abbr> is a non-UK sample) alongside a timeline of symptom onset and diagnosis date (identifiers have been removed). The timeline and the blobogram with demographic and geospatial information can be used to determine possible transmission within a community.</p>

<p>The <abbr title="Single polymorphism nucleotide">SNP</abbr> difference and timelines in this example suggest that transmission within patients in this cluster is occurring, but this can be only confirmed by epidemiological data.</p>

<p>The larger <abbr title="Single polymorphism nucleotide">SNP</abbr> distances and longer timeframes may make it more difficult to ascertain epidemiological links in this cluster. The timeframe with one or two cases presenting per year suggest ongoing transmission with potentially unidentified cases.</p>

<p>Note that cluster information cannot determine the direction of transmission at this time but the dates of symptom onset may provide evidence of transmission.</p>

<h4>Panel 1</h4>
<p>The maximum likelihood tree of the samples with <abbr title="Single polymorphism nucleotide">SNP</abbr> distances. There are more than 10 samples for the 10 patients (for example, patient 7 has 6 samples that are 0 <abbr title="Single polymorphism nucleotide">SNPs</abbr> apart). Three samples are from non-UK patients, only one is shown on the blobogram.</p>

<h4>Panel 2</h4>
<p>The <abbr title="Maximum Likelihood phylogenetic tree">ML</abbr> tree shown as a blogogram, annotated with patient number, year of sample and <abbr title="Single polymorphism nucleotide">SNP</abbr> distance (shown in red). A blobogram takes all isolates that are 0 <abbr title="Single polymorphism nucleotide">SNPs</abbr> apart and puts them in a blue circle – in this cluster only one of the blue circles has 2 patients, the other 2 blue circles show samples from the same patient that are 0 <abbr title="Single polymorphism nucleotide">SNPs</abbr> apart. The yellow diamond in the central circle indicates the putative oldest sample. The shape of the cluster is elongated and linked by the putative unknown sample and the latest sample from 2021. The addition of the latest person that connects the other cases may indicate a transmission link that was previously unknown.</p>

<h4>Panel 3</h4>
<p>This panel shows the epidemiological timeline of the 10 patients in cluster with dates of symptom onset (a red triangle) and dates of diagnosis (a yellow circle). It also indicates whether patients have pulmonary disease (pink) or not (black) or if site of disease is not known (grey).</p>

<h4>Figure 5: Annotated cluster showing <abbr title="Maximum Likelihood phylogenetic tree">ML</abbr> tree (panel 1), blobogram (panel 2) and timeline (panel 3)</h4>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/155792/Figure_5.png"></div></figure>

<h3>
<span class="number">4.2 </span> Cluster with variable <abbr title="Single polymorphism nucleotide">SNP</abbr> differences</h3>

<p>Figure 6a and 6b shows a cluster with 20 people with active culture confirmed <abbr title="tuberculosis">TB</abbr> that are between 0 or 11 <abbr title="Single polymorphism nucleotide">SNPs</abbr> to the nearest neighbour with a timeline of symptom onset and diagnosis date (identifiers have been removed). The timeline and the blobogram with demographic and geospatial information can be used to determine possible transmission within a community. This cluster contains people who are non-UK born with recent entry dates. The higher <abbr title="Single polymorphism nucleotide">SNP</abbr> difference between patients, alongside this epidemiological information suggests that the clustering may reflect the diversity within the country of origin. However, there are 5 people with low <abbr title="Single polymorphism nucleotide">SNP</abbr> differences (indicated by the red circle) that suggest more recent transmission occurring in the UK, but this can be only confirmed by epidemiological data. Note that <abbr title="whole-genome sequencing">WGS</abbr> information alone cannot determine the direction of transmission at this time.</p>

<h4>Panel 1</h4>
<p>The maximum likelihood tree of the samples with <abbr title="Single polymorphism nucleotide">SNP</abbr> distances. There are more than 20 samples for the 20 patients (for example, patient 16 has 6 samples that are 0 <abbr title="Single polymorphism nucleotide">SNPs</abbr> apart). Several samples are from non-UK patients. The red circle indicates possible recent transmission.</p>

<h4>Panel 2</h4>
<p>The <abbr title="Maximum Likelihood phylogenetic tree">ML</abbr> tree shown as a blogogram, annotated with patient number, year of sample and <abbr title="Single polymorphism nucleotide">SNP</abbr> distance (shown in red). A blobogram takes all isolates that are 0 <abbr title="Single polymorphism nucleotide">SNPs</abbr> apart and puts them in a blue circle – in this cluster one of the blue circles has 3 patients, the other blue circles show samples from the same patient that are 0 <abbr title="Single polymorphism nucleotide">SNPs</abbr> apart. The yellow diamond in the central circle indicates the putative oldest sample. The larger <abbr title="Single polymorphism nucleotide">SNP</abbr> distances between many samples in the cluster are shown in red.</p>

<h4>Panel 3</h4>
<p>This panel shows the epidemiological timeline of the 20 patients in cluster with dates of symptoms (a red triangle) and dates of diagnosis (a yellow circle). It also indicates whether patients have pulmonary disease (pink) or not (black) or if site of disease is not known (grey).</p>

<h4>Figure 6a (top) and 6b (bottom): Annotated cluster showing <abbr title="Maximum Likelihood phylogenetic tree">ML</abbr> tree (panel 1), blobogram (panel 2) and timeline (panel 3)</h4>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/155793/Figure_6.png"></div></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/155794/Figure_7.png"></div></figure>

<h2>References</h2>

<p><strong>1</strong> O’Shea MK and others. ‘Time-to-detection in culture predicts risk of Mycobacterium tuberculosis transmission: a cohort study’ Clinical Infectious Diseases 2014: volume 59(2), pages 177-85</p>
<p><strong>2</strong> Pankhurst LJ and others. ‘Rapid, comprehensive, and affordable mycobacterial diagnosis with whole-genome sequencing: a prospective study’ Lancet Respiratory Medicine 2016: volume 4(1), pages 49-58</p>
<p><strong>3</strong> Votintseva AA and others. ‘Same-Day Diagnostic and Surveillance Data for Tuberculosis via Whole-Genome Sequencing of Direct Respiratory Samples’ Journal of Clinical Microbiology 2017: volume 55(5), pages 1285-1298</p>
<p><strong>4</strong> Votintseva AA and others. ‘Mycobacterial <abbr title="Deoxyribonucleic acid">DNA</abbr> extraction for whole-genome sequencing from early positive liquid (MGIT) cultures’ Journal of Clinical Microbiology 2015: volume 53(4), pages 1137-43</p>
<p><strong>5</strong> Quan TP and others. ‘Evaluation of Whole-Genome Sequencing for Mycobacterial Species Identification and Drug Susceptibility Testing in a Clinical Setting: a Large-Scale Prospective Assessment of Performance against Line Probe Assays and Phenotyping’ Journal of Clinical Microbiology 2018: volume 56(2)</p>
<p><strong>6</strong> Walker TM and others. ‘Assessment of Mycobacterium tuberculosis transmission in Oxfordshire, UK, 2007-12, with whole pathogen genome sequences: an observational study’ Lancet Respiratory Medicine 2014: volume 2(4), pages 285-292</p>
<p><strong>7</strong> Walker TM and others. ‘Whole genome sequencing for M/XDR tuberculosis surveillance and for resistance testing’ Clinical Microbiology and Infection 2017: volume 23(3), pages 161-166</p>
<p><strong>8</strong> Niemann S and others. ‘Genomic diversity among drug sensitive and multidrug resistant isolates of Mycobacterium tuberculosis with identical <abbr title="Deoxyribonucleic acid">DNA</abbr> fingerprints’ PLoS One 2009: volume 4(10), page e7407</p>
<p><strong>9</strong> Schurch AC and others. ‘High-resolution typing by integration of genome sequencing data in a large tuberculosis cluster’ Journal of Clinical Microbiology 2010: volume 48(9), pages 3403-6</p>
<p><strong>10</strong> Gardy JL and others. ‘Whole-genome sequencing and social-network analysis of a tuberculosis outbreak’ New England Journal of Medicine 2011: volume 364(8), pages 730-9</p>
<p><strong>11</strong> Walker TM and others. ‘Whole-genome sequencing to delineate Mycobacterium tuberculosis outbreaks: a retrospective observational study’ Lancet Infectious Diseases 2013: volume 13(2), pages 137-46</p>
<p><strong>12</strong> Wyllie DH and others. ‘A Quantitative Evaluation of <abbr title="Mycobacterial Interspersed Repetitive Unit - Variable Number Tandem Repeats">MIRU-VNTR</abbr> Typing Against Whole-Genome Sequencing for Identifying Mycobacterium tuberculosis Transmission: A Prospective Observational Cohort Study’ EBioMedicine 2018: volume 34, pages 122-130</p>
<p><strong>13</strong> Nikolayevskyy V and others. ‘Role and value of whole genome sequencing in studying tuberculosis transmission’ Clinical Microbiology and Infection 2019: volume 25(11), pages 1377-1382</p>
<p><strong>14</strong> Roetzer A and others. ‘Whole genome sequencing versus traditional genotyping for investigation of a Mycobacterium tuberculosis outbreak: a longitudinal molecular epidemiological study’ PLoS Medicine 2013: 10(2), page e1001387</p>
<p><strong>15</strong> Jajou R and others. ‘Epidemiological links between tuberculosis cases identified twice as efficiently by whole genome sequencing than conventional molecular typing: A population-based study’ PLoS One 2018: volume 13(4), pages e0195413</p>
<p><strong>16</strong> Kato-Maeda M and others. ‘Use of whole genome sequencing to determine the microevolution of Mycobacterium tuberculosis during an outbreak’ PLoS One 2013: volume 8(3), page e58235</p>
<p><strong>17</strong> Wyllie DH and others. ‘Identifying Mixed Mycobacterium tuberculosis Infection and Laboratory Cross-Contamination during Mycobacterial Sequencing Programs’ Journal of Clinical Microbiology 2018: volume 56(11)</p>
<p><strong>18</strong> Stimson J and others. ‘Beyond the <abbr title="Single polymorphism nucleotide">SNP</abbr> Threshold: Identifying Outbreak Clusters Using Inferred Transmissions’ Molecular Biology and Evolution 2019: volume 36(3), pages 587-603</p>
<p><strong>19</strong> Hunt M and others. ‘Antibiotic resistance prediction for Mycobacterium tuberculosis from genome sequence data with Mykrobe’ Wellcome Open Research 2019: volume 4, page 191</p>

<h2>Appendix 1</h2>

<h3>Figure A.1: Flow diagram of <abbr title="tuberculosis">TB</abbr> <abbr title="whole-genome sequencing">WGS</abbr> cluster data inputs, process for action and information outputs</h3>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/155795/Figure_8.png"></div></figure>

<p>Note: this process is subject to local variation, the figure outlines a suggested process.</p>

<h3>Figure A.1. Flow diagram of <abbr title="tuberculosis">TB</abbr> <abbr title="whole-genome sequencing">WGS</abbr> cluster data inputs, process for action and information outputs – text version</h3>

<h4>Inputs 1</h4>

<ul>
  <li>
<abbr title="Cluster investigator">CI</abbr> (Cluster Investigator) cluster database</li>
  <li>FOREST data of <abbr title="whole-genome sequencing">WGS</abbr> cluster cases</li>
  <li>
<abbr title="Strain Typing Module">STM</abbr> (Strain Typing Module) data of historical <abbr title="Mycobacterial Interspersed Repetitive Unit - Variable Number Tandem Repeats">MIRU-VNTR</abbr> linked cases</li>
  <li>
<abbr title="Enhanced tuberculosis surveillance">ETS</abbr> data of epi linked cases</li>
</ul>

<p>Then:</p>

<h4>Process</h4>

<ul>
  <li>
<abbr title="Cluster investigator">CIs</abbr> review new cases and clusters to make suggestions on action</li>
</ul>

<p>Then:</p>

<p>Path 1</p>

<h4>Outputs</h4>

<ul>
  <li>Cluster line lists and cluster review summary</li>
</ul>

<p>Or:</p>

<p>Path 2</p>

<h4>Process</h4>

<ul>
  <li>
<abbr title="Air Traffic control National cluster meeting">ATC</abbr> review selected priority clusters to make suggestions on action</li>
</ul>

<p>Then:</p>

<p>Path 1</p>

<h4>Outputs</h4>

<ul>
  <li>Cluster line lists and cluster review summary</li>
</ul>

<p>Or:</p>

<p>Path 2</p>

<h4>Process</h4>

<ul>
  <li>
<abbr title="Cluster investigator">CIs</abbr> provides cluster data and suggestions to <abbr title="Health Protection Teams">HPT</abbr> <abbr title="tuberculosis">TB</abbr> leads and <abbr title="Field Service Epidemiologist">FES</abbr> (Field Service Epidemiologist) (for independent- or meeting-based review)</li>
  <li>
<abbr title="Health Protection Teams">HPT</abbr> <abbr title="tuberculosis">TB</abbr> leads decide whether cluster warrants further information collected (by <abbr title="tuberculosis">TB</abbr> services) and/or cluster investigation</li>
</ul>

<p>Then:</p>

<p>Path 1</p>

<h4>Outputs</h4>

<ul>
  <li>
<abbr title="Health Protection Zone">HPZone</abbr> (Health Protection Zone) record created/updated</li>
  <li>Log of action and outcomes</li>
</ul>

<p>Or:</p>

<p>Path 2</p>

<h4>Process</h4>

<ul>
  <li>Local cluster investigation led by <abbr title="Health Protection Teams">HPT</abbr> with or without <abbr title="Outbreak control team">OCT</abbr> to agree and deliver public health action as required</li>
</ul>

<h4>Inputs 2</h4>

<ul>
  <li>Cluster line list</li>
  <li>Cluster summary report</li>
  <li>
<abbr title="tuberculosis">TB</abbr> social context data</li>
</ul>

<p>Then:</p>

<h4>Process</h4>

<ul>
  <li>
<abbr title="Cluster investigator">CIs</abbr> provides cluster data and suggestions to <abbr title="Health Protection Teams">HPT</abbr> <abbr title="tuberculosis">TB</abbr> leads and <abbr title="Field Service Epidemiologist">FES</abbr> (for independent or meeting based review)</li>
  <li>
<abbr title="Health Protection Teams">HPT</abbr> <abbr title="tuberculosis">TB</abbr> leads decide whether cluster warrants further information collected (by <abbr title="tuberculosis">TB</abbr> services) and/or cluster investigation</li>
</ul>

<p>Then:</p>

<p>Path 1</p>

<h4>Outputs</h4>

<ul>
  <li>
<abbr title="Health Protection Zone">HPZone</abbr> record created/updated</li>
  <li>Log of action and outcomes</li>
</ul>

<p>Or:</p>

<p>Path 2</p>

<h4>Process</h4>

<p>Local cluster investigation led by <abbr title="Health Protection Teams">HPT</abbr> with or without <abbr title="Outbreak control team">OCT</abbr> to agree and deliver public health action as required</p>

</div>


</div>
</div>
    </div>
  </div>

  <div class="govuk-grid-row">
    <a class="govuk-link app-c-back-to-top dont-print" href="#contents">
    <svg class="app-c-back-to-top__icon" focusable="false" height="17" viewBox="0 0 13 17" width="13" xmlns="http://www.w3.org/2000/svg">
      <path d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z" fill="currentColor"></path>
    </svg>
    Back to top
</a>

  </div>
</div>

    </main>