<main class="html-publication" lang="en" role="main">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          
<div class="gem-c-organisation-logo brand--department-of-health">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color" href="/government/organisations/uk-health-security-agency">
      <span class="gem-c-organisation-logo__name">UK Health<br>Security<br>Agency</span>
</a>
</div>
        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  

<div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
      <span class="govuk-caption-xl gem-c-title__context">
    Guidance
  </span>


  <h1 class="gem-c-title__text govuk-heading-xl">
    Technical validation protocol for SAMBA II SARS-CoV-2/RSV Test
  </h1>
</div>
  <p class="publication-header__last-changed">Published 31 March 2022</p>

  </header>



<div class="govuk-grid-row sidebar-with-body" data-module="sticky-element-container">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav aria-label="Contents" class="gem-c-contents-list" data-module="gem-track-click" role="navigation">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 1" data-track-category="contentsClicked" data-track-label="#assay-description-and-intended-purpose" data-track-options="{&quot;dimension29&quot;:&quot;\n1.  Assay description and intended purpose&quot;}" href="#assay-description-and-intended-purpose"><span class="gem-c-contents-list__number">1. </span><span class="gem-c-contents-list__numbered-text">Assay description and intended purpose</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 2" data-track-category="contentsClicked" data-track-label="#type-of-sample-to-be-used-in-validation" data-track-options="{&quot;dimension29&quot;:&quot;\n2.  Type of sample to be used in validation&quot;}" href="#type-of-sample-to-be-used-in-validation"><span class="gem-c-contents-list__number">2. </span><span class="gem-c-contents-list__numbered-text">Type of sample to be used in validation</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 3" data-track-category="contentsClicked" data-track-label="#equipment-and-reagents" data-track-options="{&quot;dimension29&quot;:&quot;\n3.  Equipment and reagents&quot;}" href="#equipment-and-reagents"><span class="gem-c-contents-list__number">3. </span><span class="gem-c-contents-list__numbered-text">Equipment and reagents</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 4" data-track-category="contentsClicked" data-track-label="#performance-characteristics" data-track-options="{&quot;dimension29&quot;:&quot;\n4.  Performance characteristics&quot;}" href="#performance-characteristics"><span class="gem-c-contents-list__number">4. </span><span class="gem-c-contents-list__numbered-text">Performance characteristics</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 5" data-track-category="contentsClicked" data-track-label="#precision-and-robustness" data-track-options="{&quot;dimension29&quot;:&quot;\n5.  Precision and robustness&quot;}" href="#precision-and-robustness"><span class="gem-c-contents-list__number">5. </span><span class="gem-c-contents-list__numbered-text">Precision and robustness</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 6" data-track-category="contentsClicked" data-track-label="#analytical-specificity-interferences-and-cross-reactions" data-track-options="{&quot;dimension29&quot;:&quot;\n6.  Analytical specificity (Interferences and cross-reactions)&quot;}" href="#analytical-specificity-interferences-and-cross-reactions"><span class="gem-c-contents-list__number">6. </span><span class="gem-c-contents-list__numbered-text">Analytical specificity (Interferences and cross-reactions)</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 7" data-track-category="contentsClicked" data-track-label="#diagnostic-sensitivity-and-specificity-clinical-validation-with-confirmed-positives-and-negatives" data-track-options="{&quot;dimension29&quot;:&quot;\n7.  Diagnostic sensitivity and specificity (clinical validation with confirmed positives and negatives)&quot;}" href="#diagnostic-sensitivity-and-specificity-clinical-validation-with-confirmed-positives-and-negatives"><span class="gem-c-contents-list__number">7. </span><span class="gem-c-contents-list__numbered-text">Diagnostic sensitivity and specificity (clinical validation with confirmed positives and negatives)</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 8" data-track-category="contentsClicked" data-track-label="#summary" data-track-options="{&quot;dimension29&quot;:&quot;Summary&quot;}" href="#summary">Summary</a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img class="print-meta-data-licence" src="/assets/government-frontend/open-government-licence-min-93b6a51b518ff99714a1aa2a7d2162735c155ec3cb073c75fb88b2a332fa83d3.png">
</p>
<p>
  © Crown copyright 2022
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/samba-ii-sars-cov-2rsv-test-technical-validation-protocol/technical-validation-protocol-for-samba-ii-sars-cov-2rsv-test
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
    
    
      <div class="govspeak">
<h2>
<span class="number">1. </span> Assay description and intended purpose</h2>

<h4>1.1</h4>

<p>The SAMBA II SARS-CoV-2/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Test is a nucleic-acid based amplification test for the qualitative detection of nucleic acids from SARS-CoV-2 and/or Respiratory Syncytial Virus (<abbr title="Respiratory Syncytial Virus">RSV</abbr>) in human respiratory specimens (combined nasal and throat/nasal/nasopharyngeal swabs).</p>

<p>SAMBA II SARS-CoV-2/<abbr title="Respiratory Syncytial Virus">RSV</abbr> test kit (List No. 8800-12):</p>

<ul>
  <li>8800A, 12 each, SCoV/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Cartridge 1</li>
  <li>8800B and 8800D, 12 each, SCoV/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Cartridge 2 and SCoV/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Cartridge 4</li>
  <li>8800C, 12 each, SCoV/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Cartridge 3</li>
</ul>

<p>Other materials required and provided:</p>

<ul>
  <li>8800E SCoV/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Buffer, 2ml</li>
</ul>

<p>Each tube contains 2ml of SAMBA Lysis Buffer which includes a detergent that inactivates the virus.</p>

<ul>
  <li>C12-0002 Fixed volume pipet, 300 μl (+ Pipet tips, filtered) OR</li>
  <li>C20-0007 Transfer pipets, 300 μl</li>
  <li>C19-0015-12 Sample Collection Tube, red cap</li>
  <li>C19-0093 Sample Card, SAMBA II SARS-CoV-2/<abbr title="Respiratory Syncytial Virus">RSV</abbr>
</li>
</ul>

<p>Other materials required:</p>

<ul>
  <li>Sterile flocked, rayon or Dacron swabs with plastic shaft for sample collection</li>
</ul>

<p>Equipment required but ordered separately:</p>

<ul>
  <li>19-0006-AM SAMBA II Assay Module</li>
  <li>19-0006-TM SAMBA II Tablet Module</li>
  <li>SAMBA II printer</li>
</ul>

<h4>1.2</h4>

<p>The SAMBA II SARS-CoV-2/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Test is a nucleic-acid based amplification test for the qualitative detection of nucleic acids from SARS-CoV-2 and/or Respiratory Syncytial Virus (<abbr title="Respiratory Syncytial Virus">RSV</abbr>) in human respiratory specimens (combined nasal and throat/nasal/nasopharyngeal swabs). The assay is intended for use by professionals and trained operators on the SAMBA II instrument system in clinical and point-of-care settings.</p>

<h4>1.3</h4>

<p>The SAMBA II SARS-CoV-2/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Test is a fully automated assay run on the CE-marked SAMBA II instrument system consisting of the SAMBA II Assay Module, and the SAMBA II Tablet Module control unit. The assay is based on the chemistry used for the SAMBA II SARS-CoV-2 test with the addition of <abbr title="Respiratory Syncytial Virus">RSV</abbr>. The SAMBA II SARS-CoV-2/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Test detects 2 gene targets for SARS-CoV2 (ORF 1ab and nucleocapsid protein – N) and one target for <abbr title="Respiratory Syncytial Virus">RSV</abbr> (non structural protein 1 - NSP1). The test requires the use of 4 cartridges that contain reagents for extraction of viral nucleic acid from the specimen, amplification of the nucleic acid target and the detection of the amplification products.</p>

<p>The sample buffer is inactivating and the instrument is intended for use in clinical and point of care settings.</p>

<h2>
<span class="number">2. </span> Type of sample to be used in validation</h2>

<p>The performance characteristics of the SAMBA II SARS-CoV-2/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Test were determined using various reference materials including prepared panels and clinical specimens:</p>

<ul>
  <li>heat inactivated 2019 Novel Coronavirus, Isolate USA-WA1/2020 (VR-1986HK, ATCC), lot 70035039 (3.75x108 copies/ml)</li>
  <li>human respiratory syncytial virus A, strain Long (VR-26PQ, ATCC), lot 70024412 (1.6x107 TCID50/ml or 1.7 x 1010 copies/ml)</li>
  <li>human respiratory syncytial virus B, strain B/WV/1461/85 (VR-1400, ATCC), lot 70013461 (2.8 x 105 TCID50/ml)</li>
  <li>World Health Organization First International Standard for SARS CoV-2 RNA, NIBSC Code: 20/146</li>
  <li>clinical specimens (only nasal specimens were used for clinical evaluation for data presented in the IFU)</li>
  <li>microorganisms from the American Type Culture Collection and BEI Resources</li>
</ul>

<p>This report only reviews the performance of the SARS CoV2 components of the combined assay.</p>

<h3>For analytical sensitivity and linearity</h3>
<p>The limit of detection (<abbr title="limit of detection">LoD</abbr>) of the SAMBA II SARS-CoV-2/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Test was determined for SARS-CoV-2 using 2 lots of reagents. Serial dilutions of inactivated SARS-CoV-2 virus (ATCC VR-1986HK) were prepared in pooled negative combined nose and throat swab samples in SAMBA SCoV/<abbr title="Respiratory Syncytial Virus">RSV</abbr> buffer. The initial <abbr title="limit of detection">LoD</abbr> was determined by testing target concentrations of 1000, 500, 350, 300 250 copies/ml.</p>

<p>Each panel member was initially tested in at least 2 replicates.</p>

<p>The final <abbr title="limit of detection">LoD</abbr> was confirmed as 250 copies/ml (100% in a total of 20 replicates). The results are summarised in Table 1.</p>

<h3>For analytical specificity and cross-reactivity</h3>
<p>For SARS CoV-2 target in silico analysis for possible cross-reactions with all the organisms in Table 2 was conducted by carrying out a BLASTn search for of each of the SAMBA primers and probes against NCBI databases and retrieving all sequences with homologies &gt; 80%. Only one probe (N region) had greater than 80% homology (81%) to one of the high priority organisms (Pneumocystis jirovecii [PJP]). This marginal homology would not impact the performance of the test because the other primers have no homology to P. jirovecii and hence it would not amplify.</p>

<p>Stipulate if the material is required to be extracted that is volume received, volume extracted, volume eluted, elution buffer to be used in the assay.</p>

<p>The swab is soaked for 10 minutes in the SCoV/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Buffer to inactivate the sample and then 300ul transferred to the cartridge for extraction and amplification.</p>

<p>The assay has a shelf life of 6 months at 2-37℃. No inhibitory reagents have been identified. In addition, the company has stated that the performance of the test was assessed in both fresh and frozen clinical samples and showed equivalent performance. Samples may be frozen at -20 ℃ for up to 2 weeks or at -80℃ at least 3 months.</p>

<h2>
<span class="number">3. </span> Equipment and reagents</h2>

<p>Additional equipment required:</p>

<ul>
  <li>sterile, flocked, rayon or Dacron swabs with plastic shaft for sample collection</li>
  <li>19-0006-AM SAMBA II Assay Module</li>
  <li>19-0006-TM SAMBA II Tablet Module</li>
  <li>SAMBA II printer</li>
</ul>

<h2>
<span class="number">4. </span> Performance characteristics</h2>

<h3>Analytical sensitivity and linearity of SARS COV-2 targets</h3>

<h4>Lowest limits of detection (<abbr title="lowest limits of detection">LLoD</abbr>)</h4>

<p>SARS-CoV-2: the <abbr title="limit of detection">LoD</abbr> of the SAMBA II SARS-CoV-2/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Test was determined for SARS-CoV-2 using 2 lots of reagents. Serial dilutions of inactivated SARS-CoV-2 virus (ATCC VR-1986HK) were prepared in pooled negative combined nose and throat swab samples in SAMBA SCoV buffer. The initial <abbr title="limit of detection">LoD</abbr> was determined by testing target concentrations of 1,000, 500, 350, 300 250 copies/ml. Each panel member was initially tested in at least 2 replicates. The final <abbr title="limit of detection">LoD</abbr> was confirmed as 250 copies/ml (100% in a total of 20 replicates).</p>

<p>The results are summarised in Table 1.</p>

<h4>Table 1. Final <abbr title="limit of detection">LoD</abbr> determination</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Target</th>
      <th scope="col">Virus strain</th>
      <th scope="col" style="text-align: right">Confirmed LoD</th>
      <th scope="col" style="text-align: right">% positive</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>SARS-CoV-2</td>
      <td>USA-WA1/2020 (VR-1986HK)</td>
      <td style="text-align: right">250 cp/ml</td>
      <td style="text-align: right">100% (20/20)</td>
    </tr>
    <tr>
      <td>
<abbr title="Respiratory Syncytial Virus">RSV</abbr>-A</td>
      <td>Long (VR-26PQ)</td>
      <td style="text-align: right">0.5 TCID<sub>50</sub>/ml (513 cp/ml)</td>
      <td style="text-align: right">95% (21/22)</td>
    </tr>
    <tr>
      <td>
<abbr title="Respiratory Syncytial Virus">RSV</abbr>-B</td>
      <td>B/WV/1461/85 (VR-1400)</td>
      <td style="text-align: right">1.5 TCID<sub>50</sub>/ml</td>
      <td style="text-align: right">96% (24/25)</td>
    </tr>
  </tbody>
</table>

<h2>
<span class="number">5. </span> Precision and robustness</h2>

<h3>Intra-assay precision</h3>

<p>The reproducibility of the SAMBA SARS-CoV-2/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Test was assessed using low-level positive (1050 cps/ml), medium-level (3500 cps/ml) positive and negative samples, each run-in replicates of 12 by 2 operators.</p>

<p>All SARS-CoV-2 positive samples were SARS-CoV-2 positive/<abbr title="Respiratory Syncytial Virus">RSV</abbr> negative, and all <abbr title="Respiratory Syncytial Virus">RSV</abbr> A samples were SARS-CoV-2 negative/<abbr title="Respiratory Syncytial Virus">RSV</abbr> positive.</p>

<h4>Specimen types</h4>

<p>A matrix equivalency study was performed on nasal, combined nasal/throat and nasopharyngeal swab specimens collected into SAMBA SCoV/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Buffer. Results indicated that all 3 swab specimens spiked with low (1,050 copies/mL for SARS-CoV-2, 1.5 TCID50/ml for <abbr title="Respiratory Syncytial Virus">RSV</abbr> A) and high (2,100 copies/mL for SARS-CoV-2, 3 TCID50/ml for <abbr title="Respiratory Syncytial Virus">RSV</abbr> A) viral load showed equivalent performance when tested with the SAMBA II SARS CoV-2/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Test.</p>

<h2>
<span class="number">6. </span> Analytical specificity (Interferences and cross-reactions)</h2>

<h3>Cross-reactivity to non-target samples/organisms</h3>

<p>A range of samples either direct clinical samples or spiked samples that are known positives for other diseases, both closely related (that is other coronaviruses), syndromic diseases (that is other respiratory viruses and bacteria) and common diseases (that is HIV, HBV, HCV, VZV, EBV, CMV) should be tested.</p>

<p>The analytical specificity (Cross-reactivity) of the SAMBA II SARS-CoV-2/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Test assay targets were evaluated using both wet testing and in silico analysis. Results are shown in manufacturers IFU and reproduced in tables 2 and 3.</p>

<p>The analytical specificity for this assay was confirmed by testing a panel of different microorganisms which represents the most common respiratory pathogens. No cross-reactivity of the SAMBA II SARS-CoV-2/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Test was observed.</p>

<h4>Table 2. Percent homology of the SAMBA II SARS-CoV-2/<abbr title="Respiratory Syncytial Virus">RSV</abbr> test primers, capture probes and detector probes with high priority organisms likely in the circulating area</h4>

<table>
  <thead>
    <tr>
      <th scope="col">High priority organisms likely in the circulating area</th>
      <th scope="col">Primer RSV R</th>
      <th scope="col">Probe 1 RSV</th>
      <th scope="col">Probe 2 RSV</th>
      <th scope="col">Primer RSV F</th>
      <th scope="col">Primer Orf F</th>
      <th scope="col">Primer Orf R</th>
      <th scope="col">Probe Orf 1</th>
      <th scope="col">Probe Orf 2</th>
      <th scope="col">Primer N F 2</th>
      <th scope="col">Primer N R</th>
      <th scope="col">Probe N 1</th>
      <th scope="col">Probe N 2</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Adenovirus (for example C1 Ad. 71)</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Human Metapneumovirus (hMPV)</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Parainfluenza virus 1-4</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Influenza A</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Influenza B</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Enterovirus (for example EV68)</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Respiratory syncytial virus</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Rhinovirus</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Chlamydia pneumoniae</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Haemophilus influenzae</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Legionella pneumophila</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Mycobacterium tuberculosis</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Streptococcus pneumoniae</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Streptococcus pyogenes</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Bordetella pertussis</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Mycoplasma pneumoniae</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Pneumocystis jirovecii (PJP)</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>81%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Candida albicans</td>
      <td>&lt;80%</td>
      <td>80% Up to 16/20</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Pseudomonas aeruginosa</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Staphylococcus epidermis</td>
      <td>&lt;80%</td>
      <td>90% Up to 18/20</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Streptococcus salivarius</td>
      <td>80% Up to 16/20</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>SARS-CoV-1</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>SARS-CoV-2</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>Human coronavirus 229E</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Human coronavirus OC43</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Human coronavirus HKNI63</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>SARS-CoV HKU339849</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>MERS-coronavirus</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Corynebacterium diptheriae</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Legionella non-pneumophila</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Bacillus anthracis (anthrax)</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Moraxella catarrhalis</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Neisseria elongata</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Neisseria meningitidis</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Morbillivirus (measles)</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
    <tr>
      <td>Varicella zoster (chicken pox)</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
      <td>&lt;80%</td>
    </tr>
  </tbody>
</table>

<p>Wet cross reactivity testing was carried out on a subset of organisms as listed in Table 3, which lists representative strains from Table 2. Each organism was spiked individually at ~105TCID50/ml for viruses and ~106 cells/ml for bacteria and fungi into combined nasal and throat / SAMBA SCoV Buffer samples and tested unspiked or spiked with SARS CoV-2 (1,050 GE copies/ml) and <abbr title="Respiratory Syncytial Virus">RSV</abbr> A (1.5 TCID50 /ml). This was intended to further verify the specificity of the assay against this microbial panel and at the same time assess if these microorganisms if present in high titer, would interfere with the assay to detect the target organisms (SARS CoV-2 and <abbr title="Respiratory Syncytial Virus">RSV</abbr>). Results from wet cross reactivity testing indicated that the SAMBA II SARS CoV-2/<abbr title="Respiratory Syncytial Virus">RSV</abbr> Test is not negatively impacted by these microorganisms.</p>

<h4>Table 3. Cross-reactivity wet testing</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Microorganism</th>
      <th scope="col">Source</th>
      <th scope="col">Catalog/Accession No.</th>
      <th scope="col" style="text-align: right">Concentration tested</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Human coronavirus NI63</td>
      <td>BEI</td>
      <td>NR-470</td>
      <td style="text-align: right">1 x 10<sup>5</sup>pfu/ml</td>
    </tr>
    <tr>
      <td>SARS Coronavirus, Gamma-irradiated and sucrose-purified</td>
      <td>BEI</td>
      <td>NR-9323</td>
      <td style="text-align: right">1 x 10<sup>5</sup>pfu/ml</td>
    </tr>
    <tr>
      <td>Adenovirus 21</td>
      <td>BEI</td>
      <td>NR-51436</td>
      <td style="text-align: right">2.5 x 10<sup>5</sup> TCID<sub>50</sub>/ml</td>
    </tr>
    <tr>
      <td>Human Parainfluenza Virus 4a, M-25</td>
      <td>BEI</td>
      <td>NR-3237</td>
      <td style="text-align: right">1.0 x 10<sup>3</sup> TCID<sub>50</sub>/ml</td>
    </tr>
    <tr>
      <td>Enterovirus 71, Tainan/4643/1998</td>
      <td>BEI</td>
      <td>NR-471</td>
      <td style="text-align: right">1.6 x 10<sup>5</sup> TCID<sub>50</sub>/ml</td>
    </tr>
    <tr>
      <td>Human Respiratory Syncytial Virus, A1998/3-2</td>
      <td>BEI</td>
      <td>NR-28529</td>
      <td style="text-align: right">1.6 x 10<sup>5</sup> TCID<sub>50</sub>/ml</td>
    </tr>
    <tr>
      <td>Human Respiratory Syncytial Virus B</td>
      <td>BEI</td>
      <td>NR-790</td>
      <td style="text-align: right">1 x 10<sup>5</sup> TCID<sub>50</sub>/ml</td>
    </tr>
    <tr>
      <td>Rhinovirus 20, 15-CV19</td>
      <td>BEI</td>
      <td>NR-51439</td>
      <td style="text-align: right">5 x 10<sup>4</sup> TCID<sub>50</sub>/ml</td>
    </tr>
    <tr>
      <td>Chlamydophila pneumoniae Strain TW-183</td>
      <td>ATCC</td>
      <td>VR-2282</td>
      <td style="text-align: right">~10<sup>6</sup>cells/ml</td>
    </tr>
    <tr>
      <td>Haemophilus influenzae</td>
      <td>ATCC</td>
      <td>49766</td>
      <td style="text-align: right">~10<sup>6</sup>cells/ml</td>
    </tr>
    <tr>
      <td>Legionella pneumophila</td>
      <td>ATCC</td>
      <td>33152</td>
      <td style="text-align: right">~10<sup>6</sup>cells/ml</td>
    </tr>
    <tr>
      <td>Mycobacterium tuberculosis</td>
      <td>BEI</td>
      <td>NR-49100</td>
      <td style="text-align: right">~10<sup>6</sup>cells/ml</td>
    </tr>
    <tr>
      <td>Bordetella pertussis</td>
      <td>BEI</td>
      <td>NR-42460</td>
      <td style="text-align: right">~10<sup>6</sup>cells/ml</td>
    </tr>
    <tr>
      <td>Mycoplasma pulmonis</td>
      <td>BEI</td>
      <td>NR-3858</td>
      <td style="text-align: right">~10<sup>6</sup>cells/ml</td>
    </tr>
    <tr>
      <td>Pneumocystis jirovecii (formerly P. carinii)</td>
      <td>ATCC</td>
      <td>PRA-159</td>
      <td style="text-align: right">~10<sup>6</sup>cells/ml</td>
    </tr>
    <tr>
      <td>Staphylococcus epidermidis</td>
      <td>ATCC</td>
      <td>14990</td>
      <td style="text-align: right">~10<sup>6</sup>cells/ml</td>
    </tr>
    <tr>
      <td>Streptococcus salivarius</td>
      <td>BEI</td>
      <td>HM-121</td>
      <td style="text-align: right">~10<sup>6</sup>cells/ml</td>
    </tr>
  </tbody>
</table>

<h4>Potential interfering substances</h4>
<p>The effect of potentially interfering substances was evaluated on the following panel of elevated levels of endogenous substances.</p>

<h4>Table 4. Interfering substances</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Substance</th>
      <th scope="col">Active ingredient</th>
      <th scope="col" style="text-align: right">Concentration tested</th>
      <th scope="col">Inhibition (yes/no)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Control</td>
      <td>None</td>
      <td style="text-align: right">N/A</td>
      <td>No</td>
    </tr>
    <tr>
      <td>Blood</td>
      <td>Whole blood 5% v/v</td>
      <td style="text-align: right">5%</td>
      <td>No</td>
    </tr>
    <tr>
      <td>MWE Sigma Virocult</td>
      <td>Transport media</td>
      <td style="text-align: right">1:1</td>
      <td>No</td>
    </tr>
    <tr>
      <td>Copan UTM-RT</td>
      <td>Transport media</td>
      <td style="text-align: right">1:1</td>
      <td>No</td>
    </tr>
    <tr>
      <td>Remel M4RT</td>
      <td>Transport media</td>
      <td style="text-align: right">1:1</td>
      <td>No</td>
    </tr>
    <tr>
      <td>Benacort</td>
      <td>Budenoside 64 µg/spray</td>
      <td style="text-align: right">5%</td>
      <td>Yes</td>
    </tr>
    <tr>
      <td>Benacort</td>
      <td>Budenoside 64 µg/spray</td>
      <td style="text-align: right">1%</td>
      <td>No</td>
    </tr>
    <tr>
      <td>Sudafed nasal spray</td>
      <td>Xylometazoline hydrochloride 0.1%</td>
      <td style="text-align: right">5%</td>
      <td>No</td>
    </tr>
    <tr>
      <td>Sinex nasal spray</td>
      <td>Oxymetazoline hydrochloride 0.5mg/ml</td>
      <td style="text-align: right">5%</td>
      <td>No</td>
    </tr>
    <tr>
      <td>Nurofen oral suspension</td>
      <td>Ibuprofen 100mg/5ml</td>
      <td style="text-align: right">5%</td>
      <td>No</td>
    </tr>
    <tr>
      <td>Tamiflu powder suspension</td>
      <td>Oseltamivir 6mg/ml</td>
      <td style="text-align: right">5%</td>
      <td>No</td>
    </tr>
    <tr>
      <td>Ventolin syrup</td>
      <td>1% w/v Salbutamol sulfate 2mg/5ml</td>
      <td style="text-align: right">5%</td>
      <td>No</td>
    </tr>
    <tr>
      <td>Calpol suspension</td>
      <td>Paracetamol 250mg/5ml</td>
      <td style="text-align: right">5%</td>
      <td>Yes</td>
    </tr>
    <tr>
      <td>Calpol suspension</td>
      <td>Paracetamol 250mg/5ml</td>
      <td style="text-align: right">1%</td>
      <td>No</td>
    </tr>
    <tr>
      <td>Snufflebabe</td>
      <td>Sodium chloride 0.9% w/v</td>
      <td style="text-align: right">5%</td>
      <td>No</td>
    </tr>
    <tr>
      <td>Children’s allergy relief</td>
      <td>Loratadine 5mg/5ml</td>
      <td style="text-align: right">5%</td>
      <td>No</td>
    </tr>
  </tbody>
</table>

<p>Calpol paracetamol suspension (250 mg/5ml) and Benacort Budenoside nasal spray (64 µg/spray) inhibited at 5% v/v. However, neither showed any interference when tested at 1% v/v. None of the other substances showed any interference in spiked or unspiked samples.</p>

<h2>
<span class="number">7. </span> Diagnostic sensitivity and specificity (clinical validation with confirmed positives and negatives)</h2>

<h3>Diagnostic sensitivity</h3>

<p>Overall sensitivity was calculated using 150 positive specimens – 95% confidence limits were calculated. In addition, sensitivity was calculated across the dynamic range of the assay and the results shown in table 5. The comparator assay was the TaqPath COVID-19, FluA/B, <abbr title="Respiratory Syncytial Virus">RSV</abbr> Combo kit.</p>

<p>For specificity the manufacturer provided data for 280 respiratory samples. As shown in Table 5 the specificity of the assay is 100% (CL 98.6-100).</p>

<h4>Table 5. Sensitivity and specificity</h4>

<p>Sensitivity (95% CI): 100% (97.5-100)</p>

<p>Specificity (95% CI):  100% (98.6-100)</p>

<p>Limits of detection: 250cps/ml</p>

<table>
  <thead>
    <tr>
      <th scope="col">CT</th>
      <th scope="col" style="text-align: right">Number of samples</th>
      <th scope="col" style="text-align: right">Sensitivity</th>
      <th scope="col" style="text-align: right">95% CI</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>&lt;25</td>
      <td style="text-align: right">68 (45.3%)</td>
      <td style="text-align: right">100%</td>
      <td style="text-align: right">94.7-100</td>
    </tr>
    <tr>
      <td>25-30</td>
      <td style="text-align: right">44 (29.3%)</td>
      <td style="text-align: right">100%</td>
      <td style="text-align: right">92.0-100</td>
    </tr>
    <tr>
      <td>30-35</td>
      <td style="text-align: right">36 (24.0%)</td>
      <td style="text-align: right">100%</td>
      <td style="text-align: right">90.4-100</td>
    </tr>
    <tr>
      <td>&gt;35</td>
      <td style="text-align: right">2 (1.3%)</td>
      <td style="text-align: right">100%</td>
      <td style="text-align: right">34.2-100</td>
    </tr>
  </tbody>
</table>

<h2>Summary</h2>

<p>The SAMBA II SARS-CoV-2/<abbr title="Respiratory Syncytial Virus">RSV</abbr> test assay meets the diagnostic performance criteria for both sensitivity and specificity.</p>

<p>The sensitivity is achieved with a greater proportion of specimens with a ct&lt;25 included in the application. However, if the number of specimens with a comparator ct&lt;25 is reduced to bring the total proportion of specimen in that banding to 40% the lcl remains above 94%, above the requirement to reach category-1 status under the Coronavirus Test Device Authority (<abbr title="Coronavirus Test Device Authority">CTDA</abbr>) regulations. In addition, the number of specimens with a ct&gt;35 does not reach the requirement of the <abbr title="Coronavirus Test Device Authority">CTDA</abbr> guidance; however, the Technical Virology Group allows approval of assays where the combined numbers of specimens above ct30 is at least 20% of the total number of specimens.</p>

<p>The evaluation was performed using a mixture of upper respiratory sample types including combined nasal and throat swabs, nasal and nasopharyngeal swabs. For <abbr title="Coronavirus Test Device Authority">CTDA</abbr> approval sufficient numbers of specimens of each sample type, across the dynamic range of the assay, would need to be supplied.</p>

</div>


</div>
</div>
  </div>

  <div class="sticky-element" data-sticky-element="">
    <a class="govuk-link app-c-back-to-top dont-print" href="#contents">
    <svg class="app-c-back-to-top__icon" focusable="false" height="17" viewBox="0 0 13 17" width="13" xmlns="http://www.w3.org/2000/svg">
      <path d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z" fill="currentColor"></path>
    </svg>
    Contents
</a>

    <div class="sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>