<main class="html-publication" lang="en" role="main">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          
<div class="gem-c-organisation-logo brand--department-of-health">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color" href="/government/organisations/uk-health-security-agency">
      <span class="gem-c-organisation-logo__name">UK Health<br>Security<br>Agency</span>
</a>
</div>
        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  

<div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
      <span class="govuk-caption-xl gem-c-title__context">
    Guidance
  </span>


  <h1 class="gem-c-title__text govuk-heading-xl">
    UK SMI ID 20 issue 4 (April 2022): identification of Shigella species
  </h1>
</div>
  <p class="publication-header__last-changed">Updated 6 May 2022</p>

  </header>



<div class="govuk-grid-row sidebar-with-body" data-module="sticky-element-container">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav aria-label="Contents" class="gem-c-contents-list" data-module="gem-track-click" role="navigation">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 1" data-track-category="contentsClicked" data-track-label="#acknowledgments" data-track-options="{&quot;dimension29&quot;:&quot;Acknowledgments&quot;}" href="#acknowledgments">Acknowledgments</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 2" data-track-category="contentsClicked" data-track-label="#amendment-table" data-track-options="{&quot;dimension29&quot;:&quot;Amendment table&quot;}" href="#amendment-table">Amendment table</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 3" data-track-category="contentsClicked" data-track-label="#general-information" data-track-options="{&quot;dimension29&quot;:&quot;\n1. \tGeneral information&quot;}" href="#general-information"><span class="gem-c-contents-list__number">1. </span><span class="gem-c-contents-list__numbered-text">General information</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 4" data-track-category="contentsClicked" data-track-label="#scientific-information" data-track-options="{&quot;dimension29&quot;:&quot;\n2. \tScientific information&quot;}" href="#scientific-information"><span class="gem-c-contents-list__number">2. </span><span class="gem-c-contents-list__numbered-text">Scientific information</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 5" data-track-category="contentsClicked" data-track-label="#scope-of-document" data-track-options="{&quot;dimension29&quot;:&quot;\n3. \tScope of document&quot;}" href="#scope-of-document"><span class="gem-c-contents-list__number">3. </span><span class="gem-c-contents-list__numbered-text">Scope of document</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 6" data-track-category="contentsClicked" data-track-label="#introduction" data-track-options="{&quot;dimension29&quot;:&quot;\n4. \tIntroduction&quot;}" href="#introduction"><span class="gem-c-contents-list__number">4. </span><span class="gem-c-contents-list__numbered-text">Introduction</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 7" data-track-category="contentsClicked" data-track-label="#technical-information-and-limitations" data-track-options="{&quot;dimension29&quot;:&quot;\n5. \tTechnical information and limitations&quot;}" href="#technical-information-and-limitations"><span class="gem-c-contents-list__number">5. </span><span class="gem-c-contents-list__numbered-text">Technical information and limitations</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 8" data-track-category="contentsClicked" data-track-label="#safety-considerations" data-track-options="{&quot;dimension29&quot;:&quot;\n6. \tSafety considerations&quot;}" href="#safety-considerations"><span class="gem-c-contents-list__number">6. </span><span class="gem-c-contents-list__numbered-text">Safety considerations</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 9" data-track-category="contentsClicked" data-track-label="#target-organisms" data-track-options="{&quot;dimension29&quot;:&quot;\n7. \tTarget organisms&quot;}" href="#target-organisms"><span class="gem-c-contents-list__number">7. </span><span class="gem-c-contents-list__numbered-text">Target organisms</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 10" data-track-category="contentsClicked" data-track-label="#identification" data-track-options="{&quot;dimension29&quot;:&quot;\n8. \tIdentification&quot;}" href="#identification"><span class="gem-c-contents-list__number">8. </span><span class="gem-c-contents-list__numbered-text">Identification</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 11" data-track-category="contentsClicked" data-track-label="#microscopic-appearance" data-track-options="{&quot;dimension29&quot;:&quot;\n8.1 \tMicroscopic appearance&quot;}" href="#microscopic-appearance"><span class="gem-c-contents-list__number">8.1 </span><span class="gem-c-contents-list__numbered-text">Microscopic appearance</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 12" data-track-category="contentsClicked" data-track-label="#primary-isolation-media" data-track-options="{&quot;dimension29&quot;:&quot;\n8.2 \tPrimary isolation media&quot;}" href="#primary-isolation-media"><span class="gem-c-contents-list__number">8.2 </span><span class="gem-c-contents-list__numbered-text">Primary isolation media</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 13" data-track-category="contentsClicked" data-track-label="#colonial-appearance" data-track-options="{&quot;dimension29&quot;:&quot;\n8.3 \tColonial appearance&quot;}" href="#colonial-appearance"><span class="gem-c-contents-list__number">8.3 </span><span class="gem-c-contents-list__numbered-text">Colonial appearance</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 14" data-track-category="contentsClicked" data-track-label="#test-procedures" data-track-options="{&quot;dimension29&quot;:&quot;\n8.4 \tTest procedures&quot;}" href="#test-procedures"><span class="gem-c-contents-list__number">8.4 </span><span class="gem-c-contents-list__numbered-text">Test procedures</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 15" data-track-category="contentsClicked" data-track-label="#further-identification" data-track-options="{&quot;dimension29&quot;:&quot;\n8.5 \tFurther identification&quot;}" href="#further-identification"><span class="gem-c-contents-list__number">8.5 </span><span class="gem-c-contents-list__numbered-text">Further identification</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 16" data-track-category="contentsClicked" data-track-label="#other-specialised-tests" data-track-options="{&quot;dimension29&quot;:&quot;\n8.6 \tOther specialised tests&quot;}" href="#other-specialised-tests"><span class="gem-c-contents-list__number">8.6 </span><span class="gem-c-contents-list__numbered-text">Other specialised tests</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 17" data-track-category="contentsClicked" data-track-label="#section8" data-track-options="{&quot;dimension29&quot;:&quot;\n8.7 \tStorage and referral&quot;}" href="#section8"><span class="gem-c-contents-list__number">8.7 </span><span class="gem-c-contents-list__numbered-text">Storage and referral</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 18" data-track-category="contentsClicked" data-track-label="#reporting" data-track-options="{&quot;dimension29&quot;:&quot;\n9. \tReporting&quot;}" href="#reporting"><span class="gem-c-contents-list__number">9. </span><span class="gem-c-contents-list__numbered-text">Reporting</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 19" data-track-category="contentsClicked" data-track-label="#infection-specialist" data-track-options="{&quot;dimension29&quot;:&quot;\n9.1 \tInfection specialist&quot;}" href="#infection-specialist"><span class="gem-c-contents-list__number">9.1 </span><span class="gem-c-contents-list__numbered-text">Infection specialist</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 20" data-track-category="contentsClicked" data-track-label="#preliminary-identification" data-track-options="{&quot;dimension29&quot;:&quot;\n9.2 \tPreliminary identification&quot;}" href="#preliminary-identification"><span class="gem-c-contents-list__number">9.2 </span><span class="gem-c-contents-list__numbered-text">Preliminary identification</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 21" data-track-category="contentsClicked" data-track-label="#confirmation-of-identification" data-track-options="{&quot;dimension29&quot;:&quot;\n9.3 \tConfirmation of identification&quot;}" href="#confirmation-of-identification"><span class="gem-c-contents-list__number">9.3 </span><span class="gem-c-contents-list__numbered-text">Confirmation of identification</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 22" data-track-category="contentsClicked" data-track-label="#health-protection-team-hpt" data-track-options="{&quot;dimension29&quot;:&quot;\n9.4 \tHealth Protection Team (HPT)&quot;}" href="#health-protection-team-hpt"><span class="gem-c-contents-list__number">9.4 </span><span class="gem-c-contents-list__numbered-text">Health Protection Team (HPT)</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 23" data-track-category="contentsClicked" data-track-label="#uk-health-security-agency" data-track-options="{&quot;dimension29&quot;:&quot;\n9.5 \tUK Health Security Agency&quot;}" href="#uk-health-security-agency"><span class="gem-c-contents-list__number">9.5 </span><span class="gem-c-contents-list__numbered-text">UK Health Security Agency</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 24" data-track-category="contentsClicked" data-track-label="#infection-prevention-and-control-team" data-track-options="{&quot;dimension29&quot;:&quot;\n9.6 \tInfection prevention and control team&quot;}" href="#infection-prevention-and-control-team"><span class="gem-c-contents-list__number">9.6 </span><span class="gem-c-contents-list__numbered-text">Infection prevention and control team</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 25" data-track-category="contentsClicked" data-track-label="#referral-to-reference-laboratories" data-track-options="{&quot;dimension29&quot;:&quot;\n10. \tReferral to reference laboratories&quot;}" href="#referral-to-reference-laboratories"><span class="gem-c-contents-list__number">10. </span><span class="gem-c-contents-list__numbered-text">Referral to reference laboratories</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 26" data-track-category="contentsClicked" data-track-label="#algorithm-identification-of-shigella-species" data-track-options="{&quot;dimension29&quot;:&quot;Algorithm: identification of Shigella species&quot;}" href="#algorithm-identification-of-shigella-species">Algorithm: identification of Shigella species</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 27" data-track-category="contentsClicked" data-track-label="#references" data-track-options="{&quot;dimension29&quot;:&quot;References&quot;}" href="#references">References</a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img class="print-meta-data-licence" src="/assets/government-frontend/open-government-licence-min-93b6a51b518ff99714a1aa2a7d2162735c155ec3cb073c75fb88b2a332fa83d3.png">
</p>
<p>
  © Crown copyright 2022
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/smi-id-20-identification-of-shigella-species/uk-smi-id-20-issue-4-april-2022-identification-of-shigella-species
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
    
    
      <div class="govspeak">
<h2>Acknowledgments</h2>

<p>UK Standards for Microbiology Investigations (UK <abbr title="UK Standards for Microbiology Investigations">SMIs</abbr>) are developed under the auspices of the UK Health Security Agency (<abbr title="UK Health Security Agency">UKHSA</abbr>) working in partnership with the partner organisations whose logos are displayed below and listed on the <a class="govuk-link" href="https://www.gov.uk/uk-standards-for-microbiology-investigations-smi-quality-and-consistency-in-clinical-laboratories">UK <abbr title="UK Standards for Microbiology Investigations">SMI</abbr> website</a>. UK <abbr title="UK Standards for Microbiology Investigations">SMIs</abbr> are developed, reviewed and revised by various working groups which are overseen by a steering committee (see <a class="govuk-link" href="https://www.gov.uk/government/groups/standards-for-microbiology-investigations-steering-committee">the Steering Committee page on GOV.UK</a>).</p>

<p>The contributions of many individuals in clinical, specialist and reference laboratories who have provided information and comments during the development of this document are acknowledged. We are grateful to the medical editors for editing the medical content.</p>

<p>UK <abbr title="UK Standards for Microbiology Investigations">SMIs</abbr> are produced in association with:</p>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/148927/Image_1.png"></div>
<figcaption><p>Displayed logos correct as of March 2021.</p></figcaption></figure>

<ul>
  <li>The Association for Clinical Biochemistry Microbiology Group</li>
  <li>British Infection Association (BIA)</li>
  <li>British Society for Antimicrobial Chemotherapy (BSAC)</li>
  <li>British Society for Medical Mycology</li>
  <li>British Society For Microbial Technology (BSMT)</li>
  <li>British Society for Parasitology</li>
  <li>Welsh Microbiological Association (WMA)</li>
  <li>Healthcare Infection Society</li>
  <li>Public Health Agency</li>
  <li>Institute of Biomedical Science (IBMS)</li>
  <li>Microbiology Society</li>
  <li>National Services Scotland</li>
  <li>Northern Ireland Microbiology Forum and Audit Group</li>
  <li>Public Health Scotland</li>
  <li>Public Health Wales</li>
  <li>The Royal College of Pathologists</li>
  <li>Society of Anaerobic Microbiology (SAM)</li>
  <li>Society for Applied Microbiology (SfAM)</li>
  <li>Scottish Microbiology and Virology Network</li>
  <li>UK Clinical Mycology Network (UKCMN)</li>
  <li>The UK Clinical Virology Network</li>
</ul>

<h2>Amendment table</h2>

<p>Each UK <abbr title="UK Standards for Microbiology Investigations">SMI</abbr> document has an individual record of amendments. The amendments are listed on this page. The amendment history is available from <a class="govuk-link" href="standards@phe.gov.uk">standards@phe.gov.uk</a></p>

<p>Any alterations to this document should be controlled in accordance with the local document control process.</p>

<table>
  <tbody>
    <tr>
      <th scope="row">    Amendment number/date</th>
      <td>6 / 22 April   2022</td>
    </tr>
    <tr>
      <th scope="row">    Issue number   discarded</th>
      <td>3</td>
    </tr>
    <tr>
      <th scope="row">    Insert issue number</th>
      <td>4</td>
    </tr>
    <tr>
      <th scope="row">    Anticipated next   review date*</th>
      <td>22 April 2025</td>
    </tr>
    <tr>
      <th scope="row">    Section(s) involved</th>
      <td>Amendment</td>
    </tr>
    <tr>
      <th scope="row">    Whole document</th>
      <td>Document   has been transferred to a new template and some text has been   moved to     the relevant sections</td>
    </tr>
    <tr>
      <th scope="row">    Section</th>
      <td>Scope,   introduction, safety considerations updated</td>
    </tr>
    <tr>
      <th scope="row">    8.2</th>
      <td>Primary isolation media   updated</td>
    </tr>
    <tr>
      <th scope="row">    8.5</th>
      <td>Removal of some further identification tests</td>
    </tr>
    <tr>
      <th scope="row">   Appendix</th>
      <td>Flowchart updated   with <abbr title="matrix-assisted laser desorption/ionisation time of flight mass spectrometry">MALDI-TOF</abbr>, Whole genome sequencing and molecular tests</td>
    </tr>
    <tr>
      <th scope="row">    References</th>
      <td>References reviewed and updated</td>
    </tr>
  </tbody>
</table>

<p>*Reviews can be extended up to 5 years where appropriate</p>

<h2>
<span class="number">1. </span>	General information</h2>

<p><a class="govuk-link" href="https://www.gov.uk/guidance/uk-standards-for-microbiology-investigations-smi-quality-and-consistency-in-clinical-laboratories">View general information</a> related to UK <abbr title="UK Standards for Microbiology Investigations">SMIs</abbr>.</p>

<h2>
<span class="number">2. </span>	Scientific information</h2>

<p><a class="govuk-link" href="https://www.gov.uk/guidance/uk-standards-for-microbiology-investigations-uk-smi-scientific-information">View scientific information</a> related to UK <abbr title="UK Standards for Microbiology Investigations">SMIs</abbr>.</p>

<h2>
<span class="number">3. </span>	Scope of document</h2>

<p>This UK <abbr title="UK Standards for Microbiology Investigations">SMI</abbr> describes the identification of Shigella species. Shigella species are amongst the most important enteric pathogens and are transmitted via the faecal oral route, by person-to-person spread or by ingestion of contaminated food and water. Historically, shigellosis in the UK was most commonly associated with travellers’ diarrhoea and outbreaks in institutional settings, mainly schools and nurseries.</p>

<p>More recently, the highest burden of domestic acquisition of Shigella species in the UK has been linked to sexual transmission between men <sup><a class="govuk-link" href="#ref1">(1)</a></sup>. The incubation period is from 1 to 7 days and symptoms include watery diarrhoea and dysentery. Individuals may shed viable bacteria for up to 4 weeks and remain infectious to other individuals even though they may not show symptoms.</p>

<p>This UK <abbr title="UK Standards for Microbiology Investigations">SMI</abbr> includes routine culture and biochemical test methods for the identification of microorganisms. Some laboratories will perform biochemical tests for confirmation. However, many laboratories are implementing rapid techniques such as whole genome sequencing (<abbr title="whole genome sequencing">WGS</abbr>), matrix-assisted laser desorption/ionisation time of flight mass spectrometry (MALDI- TOF MS), nucleic acid amplification tests (<abbr title="nucleic acid amplification tests">NAATs</abbr>) and molecular methods for the identification of Shigella species.</p>

<p>This UK <abbr title="UK Standards for Microbiology Investigations">SMI</abbr> should be used in conjunction with other UK <abbr title="UK Standards for Microbiology Investigations">SMIs</abbr>.</p>

<h2>
<span class="number">4. </span>	Introduction</h2>

<h3>
<span class="number">4.1 </span>	Taxonomy and characteristics</h3>

<p>The genus Shigella belongs to the family Enterobacterales and consists of 4 species: Shigella dysenteriae, Shigella flexneri, Shigella boydii, and Shigella sonnei. Each of these species, with the exception of S. sonnei, is subdivided by serotype.</p>

<p>Shigella species are differentiated into 4 subgroups on the basis of their O (somatic) antigens and further differentiated into serotypes <sup><a class="govuk-link" href="#ref2">(2)</a></sup>:</p>

<ul>
  <li>S. dysenteriae (Group A) contains 15 distinct antigenic serotypes</li>
  <li>S. flexneri (Group B) contains 8 serotypes and 9 subserotypes</li>
  <li>S. boydii (Group C) contains 20 distinct antigenic serotypes <sup><a class="govuk-link" href="#ref3">(3)</a></sup>
</li>
  <li>S. sonnei (Group D) contains only 1 serotype that may occur in 2 forms, form I (smooth) and form II (rough)</li>
</ul>

<p>The type species is Shigella dysenteriae.</p>

<p>These species are human pathogens and have no animal reservoir except for non-human primates. Shigella infection can occur from a low infectious dose of 10 to 100 organisms <sup><a class="govuk-link" href="#ref4">(4)</a></sup>. Shigella species are facultative anaerobes with an optimum growth temperature of 37°C and they are non-spore formers. Unlike Salmonella species, Shigella species do not possess flagella and hence are non-motile. They reduce nitrate to nitrite, are oxidase negative and urease negative. Shigella species are catalase positive with the exception of Shigella dysenteriae.</p>

<p>With a few exceptions, Shigella species ferment sugars without gas production. They are non-lactose fermenters (however, S. sonnei can ferment lactose after prolonged incubation) and do not produce gas from carbohydrates. They all ferment glucose and do not decarboxylate lysine and give variable results for indole (with the exception of S. sonnei that is always indole negative) <sup><a class="govuk-link" href="#ref2">(2)</a>, <a class="govuk-link" href="#ref3">(3)</a>, <a class="govuk-link" href="#ref5">(5)</a></sup>.</p>

<p>Phylogenetically, each Shigella species belongs to different clonal complexes within the genus Escherichia coli. Although the relatedness of DNA from E. coli and Shigella indicate that they constitute a single species, they are maintained as separate entities in the interests of epidemiology and clinical medicine. There is often a clinical or public health requirement to differentiate between E. coli and Shigella and to specify which species of Shigella has been isolated <sup><a class="govuk-link" href="#ref6">(6)</a></sup>.</p>

<p>All 4 Shigella species can cause dysentery; however, infection with S. dysenteriae type 1 may progress to haemolytic uremic syndrome (<abbr title="haemolytic uremic syndrome">HUS</abbr>) due to the production of Shiga toxin. <abbr title="haemolytic uremic syndrome">HUS</abbr> is a systemic condition characterised by kidney failure, cardiac and neurological complications that can be fatal.</p>

<p>In the UK, S. dysenteriae is a cause of travellers’ diarrhoea. In England, S. dysenteriae is the least commonly isolated species (4%) compared to S. sonnei (49%), S. flexneri (39%) and S. boydii (7%) <sup><a class="govuk-link" href="#ref7">(7)</a></sup>.</p>

<p>In the UK, S. flexneri is associated with travellers’ diarrhoea and gastrointestinal symptoms in men who have sex with other men (<abbr title="men who have sex with other men">MSM</abbr>), and may be endemic in defined communities and populations. <abbr title="UK Health Security Agency">UKHSA</abbr> Gastrointestinal Bacteria Reference Unit (GBRU) for England and Wales reported 18,266 Shigella cases between 2004 and 2015 <sup><a class="govuk-link" href="#ref8">(8)</a></sup>.</p>

<p><abbr title="UK Health Security Agency">UKHSA</abbr> has presented laboratory surveillance data in regard to sexual transmission of Shigella species among <abbr title="men who have sex with other men">MSM</abbr> in England. It was reported that between 2004 and 2019, there was an increase in the number of Shigella species diagnosed amongst adults. Due to the coronavirus (COVID-19) pandemic, there were fewer cases reported in 2020 compared to 2019. Before 2010, many cases had been travel associated <sup><a class="govuk-link" href="#ref1">(1)</a></sup>.</p>

<p>Each year <abbr title="UK Health Security Agency">UKHSA</abbr> Second-Generation Surveillance System (<abbr title="Second-Generation Surveillance System">SGSS</abbr>) receives approximately 1,100 laboratory confirmed cases of S. sonnei reported by local hospital laboratories. Of these, approximately 800 isolates are from faecal specimens or community cases with symptoms of gastrointestinal disease which are sent for bacterial identification and typing <sup><a class="govuk-link" href="#ref9">(9)</a></sup>.</p>

<p>It is estimated that globally there are 164.7 million cases of shigellosis each year. The majority of cases occur in low income countries and S. flexneri causes 60% of these cases <sup><a class="govuk-link" href="#ref4">(4)</a></sup>.</p>

<h2>
<span class="number">5. </span>	Technical information and limitations</h2>

<h3>
<span class="number">5.1 </span>	Quality control</h3>

<p>If using commercially manufactured antisera, check suitability of use for all methods. Each new lot or shipment of antisera and commercial identification systems should be tested and validated for positive and negative reactivity using known control strains, ensuring it is fit for purpose. Laboratories must follow manufacturer’s instructions when using these products.</p>

<h3>
<span class="number">5.2 </span>	Agglutination test</h3>

<p>Slide agglutination tests are rapid and will presumptively identify Shigella species. However, agglutination with diagnostic antisera is time consuming and costly, requiring up to 10 separate agglutination tests using antisera for type antigens I, II, III, IV, V, and VI, antisera for group antigens 3 and 4, 7 and 8, and 6, and the monoclonal antibody MASF1c for serotype 1c <sup><a class="govuk-link" href="#ref10">(10)</a></sup>.</p>

<h2>
<span class="number">6. </span>	Safety considerations</h2>

<p>The section covers specific safety considerations <sup><a class="govuk-link" href="#ref11">(11 to 32)</a></sup> related to this UK <abbr title="UK Standards for Microbiology Investigations">SMI</abbr>, and should be read in conjunction with the <a class="govuk-link" href="https://www.gov.uk/guidance/uk-standards-for-microbiology-investigations-uk-smi-scientific-information#safety-considerations">general safety considerations</a> on GOV.UK.</p>

<p>Shigella species are Hazard Group 2 organisms with the exception of Shigella dysenteriae (Type 1) which is a Hazard Group 3 organism.</p>

<p>To minimise exposure to laboratory workers, appropriate personal protective equipment (<abbr title="personal protective equipment">PPE</abbr>), such as gloves, must be worn when handling Shigella species. Techniques designed to minimise exposure should be adhered to at all times. It is also very important to decontaminate the workspace where the samples are handled. Handwashing is important to minimise the spread of infection.</p>

<p>The most effective method for preventing laboratory acquired infections is the adoption of safe working practices including processing faecal samples and presumptive Shigella isolates in a microbiological safety cabinet.</p>

<p>The above guidance should be supplemented with local Control of Substances Hazardous to Health (COSHH) and risk assessments.</p>

<p>Compliance with postal and transport regulations is essential and should be verified. Currently there is no vaccine available to protect against infections caused by Shigella species. Production of a vaccine will contribute to enhanced safety and reduction of infections <sup><a class="govuk-link" href="#ref33">(33)</a></sup>. Refer to <a class="govuk-link" href="https://www.gov.uk/government/publications/immunisation-against-infectious-disease-the-green-book-front-cover-and-contents-page">immunisation against infectious disease: the green book</a> for further guidance.</p>

<h2>
<span class="number">7. </span>	Target organisms</h2>

<p>All species cause human infections.</p>

<table>
  <tbody>
    <tr>
      <td>Shigella dysenteriae</td>
      <td>Group A consisting of 15 serotypes (1 to 15)</td>
    </tr>
    <tr>
      <td>Shigella flexneri</td>
      <td>Group B consisting of 6 serotypes (1a, 1b, 2a, 2b, 3a, 3b,   4a, 4b, 4c, 5a, 5b, and 6)</td>
    </tr>
    <tr>
      <td>Shigella boydii</td>
      <td>Group C consisting of serotypes (1 to 20)</td>
    </tr>
    <tr>
      <td>Shigella sonnei</td>
      <td>Group D with 1 serotype <sup><a class="govuk-link" href="#ref3">(3)</a></sup>
</td>
    </tr>
  </tbody>
</table>

<h2>
<span class="number">8. </span>	Identification</h2>

<h2>
<span class="number">8.1 </span>	Microscopic appearance</h2>

<p>Gram stain refer to <a class="govuk-link" href="https://www.gov.uk/government/collections/standards-for-microbiology-investigations-smi#test-procedures">TP 39 – Staining procedures</a>.</p>

<p>Shigella species are 1 to 3μm in length and 0.7 to 1.0μm in diameter and are identified as Gram negative straight rods <sup><a class="govuk-link" href="#ref2">(2)</a></sup>.</p>

<h2>
<span class="number">8.2 </span>	Primary isolation media</h2>

<p>Isolates from primary culture are identified by colony appearance on selective media, biochemical tests and serology (agglutination with specific antisera). Plesiomonas shigelloides cross reacts with S. sonnei antisera. If confirmation of identification is required, isolates should be sent to a reference laboratory. <a class="govuk-link" href="#section8">Refer to section 8.7</a>.</p>

<p>Isolation of Shigella species from food and water should involve an enrichment stage in Shigella broth, as the infecting dose is 10 to 100 organisms.</p>

<p>Faecal specimens should be sampled at the acute stage and examined as soon after collection as possible.</p>

<p>All identification tests should ideally be performed from non-selective agar. The following media supports the growth of Shigella species:</p>

<ul>
  <li>MacConkey agar (<abbr title="MacConkey agar">MAC</abbr>) is considered a differential between lactose fermenters and non-lactose fermenters, incubated in air at 35 to 37°C for 18 to 24 hour</li>
  <li>Xylose-lysine-desoxycholate agar (<abbr title="Xylose-lysine-desoxycholate agar">XLD</abbr>) incubated in air at 35 to 37°C for 18 to 24hour</li>
  <li>Desoxycholate citrate agar (<abbr title="Desoxycholate citrate agar">DCA</abbr>) incubated in air at 35 to 37°C for 18 to 24 hour</li>
  <li>Hektoen enteric medium (<abbr title="Hektoen enteric medium">HE</abbr>) incubated in air at 35 to 37°C for 18 to 24 hour</li>
  <li>Salmonella-Shigella agar (<abbr title="Salmonella-Shigella agar">SS</abbr>) incubated in air at 35 to 37°C for 18 to 24 hour</li>
  <li>Blood agar incubated at 35°C for 16 to 24 hour</li>
</ul>

<p>Non-inhibitory agar such as <abbr title="MacConkey agar">MAC</abbr> should be used alongside the inhibitory agar such as <abbr title="Xylose-lysine-desoxycholate agar">XLD</abbr>, <abbr title="Desoxycholate citrate agar">DCA</abbr> or <abbr title="Hektoen enteric medium">HE</abbr> agar as some strains of Shigella grow poorly on inhibitory media.</p>

<h2>
<span class="number">8.3 </span>	Colonial appearance</h2>

<p><abbr title="Xylose-lysine-desoxycholate agar">XLD</abbr> – Colonies are red, 1 to 2mm diameter, and with no black centre.</p>

<p><abbr title="Desoxycholate citrate agar">DCA</abbr> – Colonies are colourless (S. sonnei may form pale pink colonies because of late lactose fermentation).</p>

<p><abbr title="MacConkey agar">MAC</abbr> – contains lactose, neutral red, selective inhibitors crystal violet and bile salts. Shigella species are non-lactose fermenters oxidase negative and are transparent or colourless.</p>

<p><abbr title="Hektoen enteric medium">HE</abbr> – contains lactose, sucrose, salicin, bromothymol blue, Andrade’s pH indicator, ferric citrate and sodium desoxycholate. Based on bile salts that reduce growth of Gram positive organisms <sup><a class="govuk-link" href="#ref3">(3)</a></sup>. Colonies appear blue green.</p>

<p><abbr title="Salmonella-Shigella agar">SS</abbr> agar is similar to <abbr title="Hektoen enteric medium">HE</abbr>. Detection is based on fermentation of lactose and non-fermenters. Highly selective agar inhibits growth of some Shigella species. Colonies appear colourless.</p>

<p>Nutrient agar (<abbr title="Nutrient agar">NA</abbr>) for pure cultures for slide agglutination. Colonies appear smooth, circular convex greyish or colorless, translucent often 2 to 3mm diameter.</p>

<p>Blood agar – large grey colonies often with a fringe or feathered edges and do not exhibit pigmentation <sup><a class="govuk-link" href="#ref3">(3)</a></sup>.</p>

<h2>
<span class="number">8.4 </span>	Test procedures</h2>

<h3>
<span class="number">8.4.1 </span>	Biochemical reactions</h3>

<p>D-mannitol is especially important in classifying Shigella. Members of Group A cannot ferment D-mannitol, whilst members of Group B, C and D can ferment this sugar <sup><a class="govuk-link" href="#ref2">(2)</a></sup>.</p>

<h3>
<span class="number">8.4.2 </span>	Serotyping</h3>

<p>Serotyping is a subtyping method based on the immuno-reactivity of various antigens. Shigella species are by definition non-motile, and as such, only the somatic (O) antigens are utilised for the determination of serotype. Flagellar (H) antigens are not expressed.</p>

<p>Serological identification is performed by slide agglutination with polyvalent, somatic (O) antigen grouping sera, followed by testing with monovalent antisera for specific serotype identification <sup><a class="govuk-link" href="#ref3">(3)</a></sup>.</p>

<p>Serotyping involves agglutination tests with diagnostic antisera: Shigella sonnei <sup><a class="govuk-link" href="#ref1">(1)</a></sup>, Shigella flexneri <sup><a class="govuk-link" href="#ref1">(1 to 6)</a></sup> (x,y) and Shigella boydii <sup><a class="govuk-link" href="#ref1">(1 to 6)</a>, <a class="govuk-link" href="#ref7">(7 to 11)</a>, <a class="govuk-link" href="#ref12">(12 to 15)</a></sup>. Not all serotypes are contained in polyvalent antisera. The results require visual assessment of the agglutination reaction <sup><a class="govuk-link" href="#ref10">(10)</a></sup>.</p>

<p>Phenotypic serotyping is useful for the detection and investigation of local outbreaks but can be an expensive and labour intensive method. It is difficult to quality control and novel combinations of antigens can result in errors at the interpretation stage and unresolvable cross reactions <sup><a class="govuk-link" href="#ref4">(4)</a></sup>.</p>

<p>Refer to manufacturer’s instructions.</p>

<h2>
<span class="number">8.5 </span>	Further identification</h2>

<p>Culture based techniques have been used by laboratories as the gold standard method for identification, and involves multiple stages such as appropriate selective agar, slide agglutination and commercial biochemical identification kits. Laboratories are moving towards molecular techniques for rapid results. However, these techniques require specialised laboratories, trained staff and expensive reagents which may not be available to all laboratories <sup><a class="govuk-link" href="#ref34">(34)</a></sup>.</p>

<h3>
<span class="number">8.5.1 </span>	Rapid molecular methods</h3>

<p>Molecular techniques have made identification of many species more rapid and precise than is possible with phenotypic techniques.</p>

<h3>
<span class="number">8.5.2 </span>	Whole genome sequencing (<abbr title="whole genome sequencing">WGS</abbr>)</h3>

<p>This is also known as ‘full genome sequencing, complete genome sequencing, or entire genome sequencing’. It is a laboratory process that determines the complete DNA sequence of an organism’s genome at a single time. There are several high-throughput techniques that are available to sequence an entire genome such as pyrosequencing, nanopore technology, IIIumina sequencing, Ion Torrent sequencing and others.</p>

<p>This sequencing method gives rapid, accurate, and comprehensive identification of bacterial transmission pathways in hospital and community settings, with concomitant reduction in infections, morbidity, and costs.</p>

<p><abbr title="whole genome sequencing">WGS</abbr> is routinely used by <abbr title="UK Health Security Agency">UKHSA</abbr> and has greatly improved surveillance capabilities and monitoring trends in antimicrobial resistance <sup><a class="govuk-link" href="#ref7">(7)</a></sup>. <abbr title="whole genome sequencing">WGS</abbr> has replaced traditional phenotypic and polymerase chain reaction (<abbr title="polymerase chain reaction">PCR</abbr>) methods for routine surveillance.</p>

<p>The <abbr title="whole genome sequencing">WGS</abbr> data for S. flexneri and historical data provides a robust and highly discriminatory typing approach to outbreak detection and investigation. Analysis of <abbr title="whole genome sequencing">WGS</abbr> data also enables monitoring of the emergence of novel serotypes and the opportunity to examine the effects of prophage and plasmids on O-antigen expression and evolution <sup><a class="govuk-link" href="#ref4">(4)</a></sup>.</p>

<h2>
<span class="number">8.6 </span>	Other specialised tests</h2>

<h3>
<span class="number">8.6.1 </span>	<abbr title="matrix-assisted laser desorption/ionisation time of flight mass spectrometry">MALDI-TOF MS</abbr>
</h3>

<p>E. coli and Shigella species have similar gene sequences and protein expression making them difficult to classify and identify. E. coli and Shigella species cannot be reliably differentiated using <abbr title="matrix-assisted laser desorption/ionisation time of flight mass spectrometry">MALDI TOF MS</abbr>, so therefore traditional methods such as biochemical tests and agglutination are required <sup><a class="govuk-link" href="#ref35">(35)</a></sup>.</p>

<h3>
<span class="number">8.6.2 </span>	<abbr title="nucleic acid amplification tests">NAATs</abbr>
</h3>

<p>For reliable identification of Shigella species <abbr title="polymerase chain reaction">PCR</abbr> based methods have been used. Although <abbr title="polymerase chain reaction">PCR</abbr> is a reliable method for the detection of Shigella species, and despite efforts on the specific detection, Shigella species are still regarded as indistinguishable from enteroinvasive E. coli by <abbr title="polymerase chain reaction">PCR</abbr>. Other methods, such as <abbr title="whole genome sequencing">WGS</abbr> can differentiate both species.</p>

<h2>
<span class="number">8.7 </span>	Storage and referral</h2>

<p>The pure isolate is saved on a nutrient agar slope for referral to a reference laboratory.
As Shigella is a notifiable disease, for public health management of cases, all isolates of S. flexneri, S. boydii and S. dysenteriae should be referred to a reference laboratory for typing <sup><a class="govuk-link" href="#ref27">(27)</a></sup>. Following the increase in sexual transmission of MDR S. sonnei it is recommended that all isolates of S. sonnei are sent to a reference laboratory for further testing.</p>

<h2>
<span class="number">9. </span>	Reporting</h2>

<h2>
<span class="number">9.1 </span>	Infection specialist</h2>

<p>According to local protocols, inform the medical microbiologist of presumptive or confirmed S. boydii, S. flexneri and S. dysenteriae O1 isolates.</p>

<p>The medical microbiologist should also be informed of presumptive or confirmed Shigella species (S. sonnei, S. dysenteriae) if the request card bears relevant information such as:</p>

<ul>
  <li>enterocolitis, dysentery (especially if complicated by haemolytic-uraemic syndrome)</li>
  <li>history of laboratory work</li>
  <li>investigations of outbreak situations</li>
</ul>

<p>Follow local protocols for reporting to clinician.</p>

<h2>
<span class="number">9.2 </span>	Preliminary identification</h2>

<p>If appropriate growth characteristics, colonial appearance, urease and serology results are demonstrated.</p>

<h2>
<span class="number">9.3 </span>	Confirmation of identification</h2>

<p>Further biochemical tests, molecular methods and reference laboratory report.</p>

<p>For confirmation and identification please see <a class="govuk-link" href="https://www.gov.uk/guidance/specialist-and-reference-microbiology-laboratory-tests-and-services">Specialist and reference microbiology: laboratory tests and services</a> for reference laboratory user manuals and request forms.</p>

<h2>
<span class="number">9.4 </span>	Health Protection Team (<abbr title="Health Protection Team">HPT</abbr>)</h2>

<p>Refer to local agreements in devolved administrations.</p>

<h2>
<span class="number">9.5 </span>	UK Health Security Agency</h2>

<p>Refer to current guidelines on <abbr title="Second-Generation Surveillance System">SGSS</abbr> reporting <sup><a class="govuk-link" href="#ref27">(27)</a></sup>.</p>

<h2>
<span class="number">9.6 </span>	Infection prevention and control team</h2>

<p>Inform the infection prevention and control team of presumptive or confirmed isolates of
Shigella species.</p>

<h2>
<span class="number">10. </span>	Referral to reference laboratories</h2>

<p>For information on the tests offered, turnaround times, transport procedure and the other requirements of the reference laboratory <a class="govuk-link" href="https://www.gov.uk/guidance/specialist-and-reference-microbiology-laboratory-tests-and-services">see user manuals and request forms</a>.</p>

<p>Contact the appropriate reference laboratory for information on the tests available, turnaround times, transport procedure and any other requirements for sample submission:</p>

<ul>
  <li><a class="govuk-link" href="https://www.gov.uk/guidance/specialist-and-reference-microbiology-laboratory-tests-and-services">England</a></li>
  <li><a class="govuk-link" href="https://phw.nhs.wales/" rel="external">Wales</a></li>
  <li><a class="govuk-link" href="http://www.hps.scot.nhs.uk/reflab/index.aspx" rel="external">Scotland</a></li>
  <li><a class="govuk-link" href="http://www.publichealth.hscni.net/directorate-public-health/health-protection" rel="external">Northern Ireland</a></li>
</ul>

<p>Note: in case of sending away to laboratories for processing, ensure that specimen is placed in appropriate package and transported accordingly.</p>

<h2>Algorithm: identification of Shigella species</h2>

<p><a class="govuk-link" href="https://www.gov.uk/government/publications/smi-id-20-identification-of-shigella-species/text-description-of-algorithm-identification-of-shigella-species">An accessible text description of this flowchart</a> is provided with this document.</p>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/149222/Flowchart_complete.png"></div></figure>

<h2>References</h2>

<p>An explanation of the reference assessment used is available in the <a class="govuk-link" href="https://www.gov.uk/guidance/uk-standards-for-microbiology-investigations-uk-smi-scientific-information#references">scientific information section on the UK <abbr title="UK Standards for Microbiology Investigations">SMI</abbr> website</a>.</p>

<p>1. Public Health England (PHE). ‘Sexually transmitted Shigella spp. in England 2016 to 2020’ PHE 2021 ++</p>

<p>2. Strockbine. A.N MTA. Shigella. ‘Bergey’s Manual of Systematics of Archaea and Bacteria’ 2015 ++</p>

<p>3. Blake W. Buchan MLF, Romney, Humphries M, John Dekker, A. aN and others. ‘Escherichia, Shigella, and Salmonella. Manual of Clinical Microbiology’. 12th edition ed. Vol 12th edition Washington DC: ASM press 2019: pages 688-723 ++</p>

<p>4. Gentle A, Ashton PM, Dallman TJ, Jenkins C. ‘Evaluation of Molecular Methods for Serotyping Shigella flexneri’ Journal of Clinical Microbiology 2016: volume 54, pages 1456-61 3++	10.1128/jcm.03386-15</p>

<p>5. Devanga Ragupathi NK, Muthuirulandi Sethuvel DP, Inbanathan FY, Veeraraghavan B. <a class="govuk-link" href="https://doi.org/10.1016/j.nmni.2017.09.003" rel="external">Accurate differentiation of Escherichia coli and Shigella serogroups: challenges and strategies</a> New Microbes and New Infections 2018: volume 21, pages 58-62 3++</p>

<p>6. Chattaway MA, Schaefer U, Tewolde R, Dallman TJ, Jenkins C. ‘Identification of Escherichia coli and Shigella Species from Whole-Genome Sequences’ Journal of Clinical Microbiology 2017: volume 55, page 616 3++  10.1128/JCM.01790-16</p>

<p>7. Terry LM, Barker CR, Day MR, Greig DR, Dallman TJ, Jenkins C. ‘Antimicrobial resistance profiles of Shigella dysenteriae isolated from travellers returning to the UK, 2004 to 2017’ Journal of Medical Microbiology 2018: issue 67, pages 1022-30 2++ 10.1099/jmm.0.000779</p>

<p>8. Chattaway MA, Greig DR, Gentle A, Hartman HB, Dallman TJ, Jenkins C. ‘Whole- Genome Sequencing for National Surveillance of Shigella flexneri’ Frontiers in Microbiology 2017: volume 8, page 1700 3++	10.3389/fmicb.2017.01700</p>

<p>9. Rew V, Mook P, Trienekens S, Baker KS, Dallman TJ, Jenkins C and others. ‘Whole- genome sequencing revealed concurrent outbreaks of shigellosis in the English Orthodox Jewish Community caused by multiple importations of Shigella sonnei from Israel’ Microbial Genomics, Microbiology Society 2018: volume 4 3++ 10.1099/mgen.0.000170</p>

<p>10. Liu J, Pholwat S, Zhang J, Taniuchi M, Haque R, Alam M and others. ‘Evaluation of Molecular Serotyping Assays for Shigella flexneri Directly on Stool Samples’ Journal of Clinical Microbiology 2021: volume 59 3++	10.1128/jcm.02455-20</p>

<p>11. Advisory Committee on Dangerous Pathogens. ‘The Approved List of Biological Agents’ Health and Safety Executive 2021: pages 1-39 ++</p>

<p>12. British Standards Institution (BSI). ‘BS EN12469 – Biotechnology – performance criteria for microbiological safety cabinets 2000’ ++</p>

<p>13. British Standards Institution (BSI). ‘BS 5726:2005 – Microbiological safety cabinets. Information to be supplied by the purchaser and to the vendor and to the installer, and siting and use of cabinets’ Recommendations and guidance 2005: pages 1-14 ++</p>

<p>14. Centers for Disease Control and Prevention. ‘Guidelines for Safe Work Practices in Human and Animal Medical Diagnostic Laboratories’ Morbidity and Mortality Weekly Report Surveillance Summaries 2012: volume 61, pages 1-102 +</p>

<p>15. Department for Transport, Maritime and Coastguard Agency, HSENI, Civil Aviation Authority. ‘Transport of infectious substances UN2814, UN2900 and UN3373 Guidance note number 17/2012 (revision 7)’ 2013 ++</p>

<p>16. Department of Health. ‘Health Protection Legislation (England) Guidance’ 2010: pages 1-112 ++</p>

<p>17. Gizzie N, Adukwu E. ‘Evaluation of Liquid-Based Swab Transport Systems against the New Approved CLSI M40-A2 Standard’ Journal of Clinical Microbiology 2016: volume 54, pages 1152-6 2+	10.1128/JCM.03337-15</p>

<p>18. Health and Safety Executive. <a class="govuk-link" href="https://www.hse.gov.uk/simple-health-safety/risk/index.htm" rel="external">Managing risks and risk assessment at work</a> (viewed on 28 July 2021) ++</p>

<p>19. Health and Safety Executive. ‘Safe use of pneumatic air tube transport systems for pathology specimens’ 2009 ++</p>

<p>20. Health and Safety Executive (HSE). ‘Control of Substances Hazardous to Health Regulations’ The Control of Substances Hazardous to Health Regulations 2002 (as amended). Approved Code of Practice and guidance L5 (sixth edition). HSE Books 2013 ++</p>

<p>21. HSE. ‘Risk assessment: A brief guide to controlling risks in the workplace’ HSE 2014 ++</p>

<p>22. HSE Advisory Committee on Dangerous Pathogens. ‘Management and operation of microbiological containment laboratories’ HSE 2019 ++</p>

<p>23. Health Services Advisory Committee. ‘Safe Working and the Prevention of Infection in Clinical Laboratories and Similar Facilities’ HSE Books 2003 ++</p>

<p>24. Home Office. ‘Public Health Act (Northern Ireland) 1967 Chapter 36’ 1967 ++</p>

<p>25. Home Office. ‘Anti-terrorism, Crime and Security Act’ 2001 ++</p>

<p>26. Official Journal of the European Communities. ‘Directive 98/79/EC of the European Parliament and of the Council of 27 October 1998 on in vitro diagnostic medical devices 1998’ pages 1-37 ++</p>

<p>27. PHE. ‘Laboratory reporting to Public Health England: a guide for diagnostic laboratories’ PHE 2020: pages 1-31 ++</p>

<p>28. Scottish Government. ‘Public Health (Scotland) Act’ 2008 ++</p>

<p>29. Royal College of Pathologists. ‘The retention and storage of pathological records and specimens (5th edition)’ 2015: pages 1-59 ++</p>

<p>30. Welsh Assembly Government. ‘Health Protection Legislation (Wales) Guidance’ 2010 ++</p>

<p>31. Tyrrell KL, Citron DM, Leoncio ES, Goldstein EJ. ‘Comparison of the Copan eSwab System with an Agar Swab Transport System for Maintenance of Fastidious Anaerobic Bacterium Viability’ Journal of Clinical Microbiology 2016: volume 54, pages 1364-7 2+	10.1128/JCM.03246-15</p>

<p>32. World Health Organization (WHO). ‘Guidance on regulations for the transport of infectious substances 2019 to 2020’ WHO 2019 ++</p>

<p>33. Raqib R, Venkatesan M. ‘Shigella conjugate vaccine efficacy trial in controlled human model and potential immune correlates of protection’ eBioMedicine 2021: volume 66: 103343 ++ 10.1016/j.ebiom.2021.103343</p>

<p>34. Wang Y, Wang Y, Luo L, Liu D, Luo X, Xu Y and others. ‘Rapid and Sensitive Detection of Shigella spp. and Salmonella spp. by Multiple Endonuclease Restriction Real-Time Loop-Mediated Isothermal Amplification Technique’ Frontiers in Microbiology 2015: 6 3+	10.3389/fmicb.2015.01400</p>

<p>35. Feng B, Shi H, Xu F, Hu F, He J, Yang H and others. ‘FTIR-assisted <abbr title="matrix-assisted laser desorption/ionisation time of flight mass spectrometry">MALDI-TOF MS</abbr> for the identification and typing of bacteria’ Analytica Chimica Acta 2020: volume 1111, pages 75-82 3++	10.1016/j.aca.2020.03.037</p>

</div>


</div>
</div>
  </div>

  <div class="sticky-element" data-sticky-element="">
    <a class="govuk-link app-c-back-to-top dont-print" href="#contents">
    <svg class="app-c-back-to-top__icon" focusable="false" height="17" viewBox="0 0 13 17" width="13" xmlns="http://www.w3.org/2000/svg">
      <path d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z" fill="currentColor"></path>
    </svg>
    Contents
</a>

    <div class="sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>