<main lang="en" class="html-publication" role="main">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          <div class="gem-c-organisation-logo brand--department-of-health">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color" href="/government/organisations/public-health-england">
      <span class="gem-c-organisation-logo__name">Public Health <br>England</span>
</a>
</div>

        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  <div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
    <span class="govuk-caption-xl gem-c-title__context">
      Research and analysis
    </span>
  <h1 class="gem-c-title__text govuk-heading-xl">
    Variants: distribution of cases data
  </h1>
</div>
  <p class="publication-header__last-changed">Updated 1 April 2021</p>

  </header>




<div class="govuk-grid-row sidebar-with-body" data-module="sticky-element-container">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav class="gem-c-contents-list  gem-c-contents-list--no-underline " aria-label="Contents" role="navigation" data-module="gem-track-click">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a href="#variants-of-concern-or-under-investigation-data-up-to-31-march-2021" data-track-label="#variants-of-concern-or-under-investigation-data-up-to-31-march-2021" data-track-action="content_item 1" data-track-options="{&quot;dimension29&quot;:&quot;Variants of concern or under investigation: data up to 31 March 2021&quot;}" data-track-category="contentsClicked" class="gem-c-contents-list__link govuk-link ">Variants of concern or under investigation: data up to 31 March 2021</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-options="{&quot;dimension29&quot;:&quot;Differences between a Variant of Concern and a Variant Under Investigation&quot;}" data-track-action="content_item 2" data-track-category="contentsClicked" class="gem-c-contents-list__link govuk-link " data-track-label="#differences-between-a-variant-of-concern-and-a-variant-under-investigation" href="#differences-between-a-variant-of-concern-and-a-variant-under-investigation">Differences between a Variant of Concern and a Variant Under Investigation</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 3" data-track-options="{&quot;dimension29&quot;:&quot;Changes in nomenclature&quot;}" href="#changes-in-nomenclature" class="gem-c-contents-list__link govuk-link " data-track-category="contentsClicked" data-track-label="#changes-in-nomenclature">Changes in nomenclature</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-label="#Variant1" data-track-options="{&quot;dimension29&quot;:&quot;\nVOC-20DEC-01&quot;}" class="gem-c-contents-list__link govuk-link " href="#Variant1" data-track-category="contentsClicked" data-track-action="content_item 4">
VOC-20DEC-01</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 5" data-track-category="contentsClicked" data-track-label="#Variant2" data-track-options="{&quot;dimension29&quot;:&quot;\nVOC-20DEC-02&quot;}" href="#Variant2" class="gem-c-contents-list__link govuk-link ">
VOC-20DEC-02</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 6" data-track-label="#Variant3" class="gem-c-contents-list__link govuk-link " data-track-category="contentsClicked" data-track-options="{&quot;dimension29&quot;:&quot;\nVUI-21JAN-01&quot;}" href="#Variant3">
VUI-21JAN-01</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a href="#Variant4" data-track-category="contentsClicked" class="gem-c-contents-list__link govuk-link " data-track-label="#Variant4" data-track-action="content_item 7" data-track-options="{&quot;dimension29&quot;:&quot;\nVOC-21JAN-02&quot;}">
VOC-21JAN-02</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-options="{&quot;dimension29&quot;:&quot;\nVUI-21FEB-01&quot;}" class="gem-c-contents-list__link govuk-link " data-track-category="contentsClicked" href="#Variant5" data-track-action="content_item 8" data-track-label="#Variant5">
VUI-21FEB-01</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-label="#Variant6" class="gem-c-contents-list__link govuk-link " data-track-category="contentsClicked" data-track-action="content_item 9" data-track-options="{&quot;dimension29&quot;:&quot;\nVOC-21FEB-02&quot;}" href="#Variant6">
VOC-21FEB-02</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-options="{&quot;dimension29&quot;:&quot;\nVUI-21FEB-03&quot;}" href="#Variant7" class="gem-c-contents-list__link govuk-link " data-track-label="#Variant7" data-track-action="content_item 10" data-track-category="contentsClicked">
VUI-21FEB-03</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 11" href="#Variant8" data-track-category="contentsClicked" class="gem-c-contents-list__link govuk-link " data-track-options="{&quot;dimension29&quot;:&quot;\nVUI-21FEB-04&quot;}" data-track-label="#Variant8">
VUI-21FEB-04</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a href="#Variant9" data-track-options="{&quot;dimension29&quot;:&quot;\nVUI-21MAR-01&quot;}" data-track-action="content_item 12" data-track-label="#Variant9" data-track-category="contentsClicked" class="gem-c-contents-list__link govuk-link ">
VUI-21MAR-01</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link " data-track-action="content_item 13" data-track-category="contentsClicked" data-track-label="#Variant10" data-track-options="{&quot;dimension29&quot;:&quot;\nVUI-21MAR-02&quot;}" href="#Variant10">
VUI-21MAR-02</a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img src="/assets/government-frontend/open-government-licence-min-e98134ae65ff5fe99a524d8c383350b3998f91c298dc24b36126e65f1656c578.png" class="print-meta-data-licence">
</p>
<p>
  © Crown copyright 2021
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/covid-19-variants-genomically-confirmed-case-numbers/variants-distribution-of-cases-data
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div data-module="govspeak" class="gem-c-govspeak govuk-govspeak ">
    
    
      <div class="govspeak">
<h2>Variants of concern or under investigation: data up to 31 March 2021</h2>

<h3>Surveillance overview</h3>

<h4>UK total distribution</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Variant</th>
      <th scope="col">Other names by which this variant may be known*</th>
      <th scope="col">Lineage</th>
      <th scope="col">Country in which first detected</th>
      <th scope="col">Genomically confirmed cases**</th>
      <th scope="col">Genomically probable cases**</th>
      <th scope="col">Total genomically confirmed and probable cases^</th>
      <th scope="col">New cases since last update (data up to 24 March)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
<abbr title="Variant of Concern">VOC</abbr>-20DEC-01</td>
      <td>
<abbr title="Variant of Concern">VOC</abbr>-202012/01</td>
      <td>B.1.1.7</td>
      <td>England, UK</td>
      <td>N/A</td>
      <td>N/A</td>
      <td>173,042</td>
      <td>+17,513</td>
    </tr>
    <tr>
      <td>
<abbr title="Variant of Concern">VOC</abbr>-20DEC-02</td>
      <td>
<abbr title="Variant of Concern">VOC</abbr>-202012/02</td>
      <td>501Y.V2<br>B.1.351</td>
      <td>South Africa</td>
      <td>459</td>
      <td>10</td>
      <td>469</td>
      <td>+57</td>
    </tr>
    <tr>
      <td>
<abbr title="Variant Under Investigation">VUI</abbr>-21JAN-01</td>
      <td>
<abbr title="Variant Under Investigation">VUI</abbr>-202101/01</td>
      <td>P2</td>
      <td>Brazil</td>
      <td>59</td>
      <td>0</td>
      <td>59</td>
      <td>+2</td>
    </tr>
    <tr>
      <td>
<abbr title="Variant of Concern">VOC</abbr>-21JAN-02</td>
      <td>
<abbr title="Variant of Concern">VOC</abbr>-202101/02</td>
      <td>P1</td>
      <td>Japan ex Manaus, Brazil</td>
      <td>31</td>
      <td>1</td>
      <td>32</td>
      <td>+5</td>
    </tr>
    <tr>
      <td>
<abbr title="Variant Under Investigation">VUI</abbr>-21FEB-01</td>
      <td>
<abbr title="Variant Under Investigation">VUI</abbr>-202102/01</td>
      <td>A.23.1 with E484K</td>
      <td>England, UK</td>
      <td>78</td>
      <td>0</td>
      <td>78</td>
      <td>0</td>
    </tr>
    <tr>
      <td>
<abbr title="Variant of Concern">VOC</abbr>-21FEB-02</td>
      <td>
<abbr title="Variant of Concern">VOC</abbr>-202102/02</td>
      <td>B.1.1.7 with E484K</td>
      <td>England, UK</td>
      <td>43</td>
      <td>0</td>
      <td>43</td>
      <td>0</td>
    </tr>
    <tr>
      <td>
<abbr title="Variant Under Investigation">VUI</abbr>-21FEB-03</td>
      <td>
<abbr title="Variant Under Investigation">VUI</abbr>-202102/03</td>
      <td>B.1.525 (previously designated UK1188)</td>
      <td>England, UK</td>
      <td>292</td>
      <td>5</td>
      <td>297</td>
      <td>+46</td>
    </tr>
    <tr>
      <td>
<abbr title="Variant Under Investigation">VUI</abbr>-21FEB-04</td>
      <td>
<abbr title="Variant Under Investigation">VUI</abbr>-202102/04</td>
      <td>B1.1.318</td>
      <td>TBC</td>
      <td>74</td>
      <td>5</td>
      <td>79</td>
      <td>+12</td>
    </tr>
    <tr>
      <td>
<abbr title="Variant Under Investigation">VUI</abbr>-21MAR-01</td>
      <td>
<abbr title="Variant Under Investigation">VUI</abbr>-202103/01</td>
      <td>B1.324.1 with E484K</td>
      <td>TBC</td>
      <td>2</td>
      <td>0</td>
      <td>2</td>
      <td>0</td>
    </tr>
    <tr>
      <td>
<abbr title="Variant Under Investigation">VUI</abbr>-21MAR-02</td>
      <td>N/A</td>
      <td>P3</td>
      <td>TBC</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>* Currently, there is no agreed international naming system for variants.</td>
    </tr>
    <tr>
      <td>** Genomically confirmed cases are those where all lineage defining positions can be assessed. Probable cases are those where the sequencing is of a lower quality and not all lineage defining positions are called. <a href="#definition" class="govuk-link">Full case definitions</a>.</td>
    </tr>
    <tr>
      <td>^ As of 31 March 2021, confirmed and probable genomic confidence categories assigned to each case have been unified into a single confirmed category (pertaining to data from England). This is being done as a probable genomic confidence indicates a ~99% confidence in variant assignment and the public health significance is the same.</td>
    </tr>
  </tbody>
</table>

<p>The <a href="#Variant1" class="govuk-link"><abbr title="Variant of Concern">VOC</abbr>-20DEC-01</a> variant was first detected in the UK and was first sequenced in the UK in September 2020.</p>

<p>The <a class="govuk-link" href="#Variant2"><abbr title="Variant of Concern">VOC</abbr>-20DEC-02</a> variant was first detected in South Africa and was first sequenced in the UK in December 2020.</p>

<p>The <a class="govuk-link" href="#Variant3"><abbr title="Variant Under Investigation">VUI</abbr>-21JAN-01</a> variant was first detected in Brazil and was first sequenced in the UK in November 2020.</p>

<p>The <a class="govuk-link" href="#Variant4"><abbr title="Variant of Concern">VOC</abbr>-21JAN-02</a> variant was first detected in Japan in travellers from Brazil in January 2021 and was first detected in the UK in February 2021.</p>

<p>The <a class="govuk-link" href="#Variant5"><abbr title="Variant Under Investigation">VUI</abbr>-21FEB-01</a> variant was first detected in the UK in December 2020.</p>

<p>The <a class="govuk-link" href="#Variant6"><abbr title="Variant of Concern">VOC</abbr>-21FEB-02</a> variant was first detected in the UK in December 2020.</p>

<p>The <a class="govuk-link" href="#Variant7"><abbr title="Variant Under Investigation">VUI</abbr>-21FEB-03</a> variant was first detected in the UK in December 2020.</p>

<p>The <a href="#Variant8" class="govuk-link"><abbr title="Variant Under Investigation">VUI</abbr>-21FEB-04</a> variant’s location of first detection is to be confirmed.</p>

<p>The <a href="#Variant9" class="govuk-link"><abbr title="Variant Under Investigation">VUI</abbr>-21MAR-01</a> variant’s location of first detection is to be confirmed.</p>

<p>The <a href="#Variant10" class="govuk-link"><abbr title="Variant Under Investigation">VUI</abbr>-21MAR-02</a> variant’s location of first detection is to be confirmed.</p>

<h2>Differences between a Variant of Concern and a Variant Under Investigation</h2>

<p>SARS-CoV-2 variants, if considered to have concerning epidemiological, immunological or pathogenic properties, are raised for formal investigation. At this point they are designated Variant Under Investigation (<abbr title="Variant Under Investigation">VUI</abbr>) with a year, month, and number. Following a risk assessment with the relevant expert committee, they may be designated Variant of Concern (<abbr title="Variant of Concern">VOC</abbr>).</p>

<h2>Changes in nomenclature</h2>

<p>The previous variant names frequently include similar numbers with similar or repeating patterns. This makes referring to them verbally challenging. To overcome this challenge, a revised nomenclature has been agreed involving dropping the first 2 numbers of the year and replacing the 2-month numbers with a 3-letter abbreviation. This shortens the name and reduces the degree of repeated sequences of numbers. The table containing surveillance overview ‘UK total distribution’ contains a list of new variant names alongside previous or other names in use for VOCs and VUIs.</p>

<h2>
<abbr title="Variant of Concern">VOC</abbr>-20DEC-01</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant was first detected in the UK and was first sequenced in the UK in September 2020.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>151,344</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>12,208</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>7,230</td>
    </tr>
    <tr>
      <td>Northern Ireland &nbsp;&nbsp;&nbsp;&nbsp;</td>
      <td>2,260</td>
    </tr>
    <tr>
      <td><strong>Total cases</strong></td>
      <td><strong>173,042</strong></td>
    </tr>
  </tbody>
</table>

<p>Cases for England include confirmed and probable <abbr title="Variant of Concern">VOC</abbr>-20DEC-01. Scotland, Wales, and Northern Ireland have not reported probable <abbr title="Variant of Concern">VOC</abbr>-20DEC-01.</p>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant of Concern">VOC</abbr>-20DEC-01 through whole genome <br>sequencing (WGS) where results of all lineage defining positions are called as alternate <br>(variant) base</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>COVID-19* confirmed case with at least 5 <abbr title="Variant of Concern">VOC</abbr>-20DEC-01 lineage defining positions<br> are called as alternate (variant) base and all other defining positions reported as N<br> (unknown) or mixed bases</td>
    </tr>
    <tr>
      <td>Low-quality genome (LQG)</td>
      <td>COVID-19* confirmed case where fewer than 4 lineage defining positions are called <br>as alternate (variant) base and all other defining positions reported as N (unknown) or mixed bases</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

<h2>
<abbr title="Variant of Concern">VOC</abbr>-20DEC-02</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant was first detected in South Africa and was first sequenced in the UK in December 2020.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total genomically confirmed cases</th>
      <th scope="col">Total genomically probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>410</td>
      <td>-</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>22</td>
      <td>5</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>21</td>
      <td>5</td>
    </tr>
    <tr>
      <td>Northern Ireland &nbsp;&nbsp;&nbsp;&nbsp;</td>
      <td>6</td>
      <td>0</td>
    </tr>
    <tr>
      <td><strong>Total cases</strong></td>
      <td><strong>459</strong></td>
      <td><strong>10</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant of Concern">VOC</abbr>-20DEC-02 through whole genome sequencing (WGS) where results of all lineage defining positions are called as alternate (variant) base.</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>COVID-19* confirmed case where:<br> • at least 4 <abbr title="Variant of Concern">VOC</abbr>-20DEC-02 lineage defining positions are called as alternate (variant) base, and <br>• all other defining positions reported as N (unknown) or mixed bases <br> or<br> • at least 5 of the 9 non-synonymous changes.</td>
    </tr>
    <tr>
      <td>High-risk low-quality genome (HR-LQG)</td>
      <td>COVID-19* confirmed case where:<br> • fewer than 4 positions are called but at least one is called as alternate (variant) base, and<br>• all other defining positions reported as N (unknown) or mixed bases<br> or<br> • at least one but fewer than 5 of the 9 non-synonymous changes, and<br> • fewer than 5 wild type calls.</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

<h2>
<abbr title="Variant Under Investigation">VUI</abbr>-21JAN-01</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant was first detected in Brazil and was first sequenced in the UK in November 2020.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total genomically confirmed cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>53</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>5</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>1</td>
    </tr>
    <tr>
      <td>Northern Ireland &nbsp;&nbsp;&nbsp;&nbsp;</td>
      <td>0</td>
    </tr>
    <tr>
      <td><strong>Total cases</strong></td>
      <td><strong>59</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant Under Investigation">VUI</abbr>-21JAN-01 through whole genome sequencing (WGS) where: <br>• results of all lineage defining non-synonymous changes called as alternate base <br> or <br> • 6 of 7 non-synonymous changes called as alternate base, and<br>• remaining position either N or mixed bases</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>N/A</td>
    </tr>
    <tr>
      <td>High-risk low-quality genome (HR-LQG)</td>
      <td>COVID-19* confirmed case where fewer than 6 positions are called but at least one is called as alternate (variant) base and all other defining positions reported as N (unknown) or mixed bases.</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

<h2>
<abbr title="Variant of Concern">VOC</abbr>-21JAN-02</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant was first detected in Japan in travellers from Brazil in January 2021 and was first detected in the UK in February 2021.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total genomically confirmed cases</th>
      <th scope="col">Total genomically probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>27</td>
      <td>-</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>4</td>
      <td>1</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Northern Ireland &nbsp;&nbsp;&nbsp;&nbsp;</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td><strong>Total cases</strong></td>
      <td><strong>31</strong></td>
      <td><strong>1</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant of Concern">VOC</abbr>-21JAN-02 through whole genome sequencing (WGS) where results of all lineage defining non-synonymous changes called as alternate bases.</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>COVID-19* confirmed case with: <br>• at least 5 lineage defining non-synonymous changes called as alternate base, and <br>• all other positions either N or mixed bases</td>
    </tr>
    <tr>
      <td>High-risk low-quality genome (HR-LQG)</td>
      <td>COVID-19* confirmed case where: <br>• fewer than 5 positions are called but at least one is called as alternate (variant) base, and <br>• all other defining positions reported as N (unknown) or mixed bases</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

<h2>
<abbr title="Variant Under Investigation">VUI</abbr>-21FEB-01</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant was first detected in the UK in December 2020.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total genomically confirmed cases</th>
      <th scope="col">Total genomically probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>78</td>
      <td>-</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td><strong>Total cases</strong></td>
      <td><strong>78</strong></td>
      <td><strong>0</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant Under Investigation">VUI</abbr>-21FEB-01 through whole genome sequencing (WGS) where results of all lineage defining non-synonymous changes called as alternate bases.</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>COVID-19* confirmed case with: <br>• at least 5 lineage defining non-synonymous changes called as alternate base, and <br>• all other positions either N or mixed base<br> or<br> • at least 5 of the 9 non-synonymous changes.</td>
    </tr>
    <tr>
      <td>High-risk low-quality genome (HR-LQG)</td>
      <td>COVID-19* confirmed case where: <br>• fewer than 5 positions are called but at least one is called as alternate (variant) base, and <br>• all other defining positions reported as N (unknown) or mixed bases.</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

<h2>
<abbr title="Variant of Concern">VOC</abbr>-21FEB-02</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant was first detected in the UK in December 2020.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total genomically confirmed cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>43</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>0</td>
    </tr>
    <tr>
      <td><strong>Total cases</strong></td>
      <td><strong>43</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant of Concern">VOC</abbr>-21FEB-02 through whole genome sequencing (WGS) where results of all lineage defining non-synonymous changes called as alternate bases.</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>N/A</td>
    </tr>
    <tr>
      <td>High-risk low-quality genome (HR-LQG)</td>
      <td>COVID-19* confirmed case where <br>• probable or confirmed <abbr title="Variant of Concern">VOC</abbr>-20DEC-01, and <br>• fewer than 4 positions are called but at least one is called as alternate (variant) base, and <br>• all other defining positions reported as N (unknown) or mixed bases.</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

<h2>
<abbr title="Variant Under Investigation">VUI</abbr>-21FEB-03</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant was first detected in the UK in December 2020.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total genomically confirmed cases</th>
      <th scope="col">Total genomically probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>277</td>
      <td>-</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>9</td>
      <td>5</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>6</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td><strong>Total cases</strong></td>
      <td><strong>292</strong></td>
      <td><strong>5</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant Under Investigation">VUI</abbr>-21FEB-03 through whole genome sequencing (WGS) where results of all lineage defining non-synonymous changes called as alternate bases.</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>COVID-19* confirmed case with: <br>• at least 5 lineage defining non-synonymous changes called as alternate base, and <br>• all other positions either N or mixed base.</td>
    </tr>
    <tr>
      <td>High-risk low-quality genome (HR-LQG)</td>
      <td>COVID-19* confirmed case where: <br>• fewer than 5 positions are called but at least one is called as alternate (variant) base, and  <br>• all other defining positions reported as N (unknown) or mixed bases.</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

<h2>
<abbr title="Variant Under Investigation">VUI</abbr>-21FEB-04</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant’s location of first detection is to be confirmed.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total genomically confirmed cases</th>
      <th scope="col">Total genomically probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>72</td>
      <td>-</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>2</td>
      <td>5</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td><strong>Total cases</strong></td>
      <td><strong>74</strong></td>
      <td><strong>5</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant Under Investigation">VUI</abbr>-21FEB-04 through whole genome sequencing (WGS) where results of all lineage defining non-synonymous changes called as alternate bases.</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>COVID-19* confirmed case with: <br>• at least 5 lineage defining non-synonymous changes called as alternate base, and <br>• all other positions either N or mixed base<br> or<br> • at least 5 of the 9 non-synonymous changes.</td>
    </tr>
    <tr>
      <td>High-risk low-quality genome (HR-LQG)</td>
      <td>COVID-19* confirmed case where: <br>• fewer than 5 positions are called but at least one is called as alternate (variant) base, and <br>• all other defining positions reported as N (unknown) or mixed bases.</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

<h2>
<abbr title="Variant Under Investigation">VUI</abbr>-21MAR-01</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant’s location of first detection is to be confirmed.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total genomically confirmed cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>2</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>0</td>
    </tr>
    <tr>
      <td><strong>Total cases</strong></td>
      <td><strong>2</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant Under Investigation">VUI</abbr>-21MAR-01 through whole genome sequencing (WGS) where results of all lineage defining non-synonymous changes called as alternate bases.</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>N/A</td>
    </tr>
    <tr>
      <td>High-risk low-quality genome (HR-LQG)</td>
      <td>COVID-19* confirmed case where: <br>• fewer than 8 variant defining changes called as alternate base, and <br>• all other positions either N or mixed base.</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

<h2>
<abbr title="Variant Under Investigation">VUI</abbr>-21MAR-02</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant’s location of first detection is to be confirmed.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total genomically confirmed cases</th>
      <th scope="col">Total genomically probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>5</td>
      <td>-</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td><strong>Total cases</strong></td>
      <td><strong>5</strong></td>
      <td><strong>0</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant Under Investigation">VUI</abbr>-21MAR-02 through whole genome sequencing (WGS) where results of all variant defining changes called as alternate base.</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>COVID-19* confirmed case with: <br>• at least 7 variant defining changes called as alternate base, and <br>• all other positions either N or mixed base</td>
    </tr>
    <tr>
      <td>High-risk low-quality genome (HR-LQG)</td>
      <td>COVID-19* confirmed case where: <br>• fewer than 7 positions are called but at least one is called as alternate base, and <br>• all other positions reported as N (unknown) or mixed bases.</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

</div>


</div>
</div>
  </div>

  <div data-sticky-element="" class="sticky-element">
    <a class="govuk-link app-c-back-to-top dont-print" href="#contents">
    <svg class="app-c-back-to-top__icon" width="13" height="17" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 13 17">
      <path d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z" fill="currentColor"></path>
    </svg>
    Contents
</a>

    <div class="sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>