<main lang="en" class="html-publication" role="main">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          <div class="gem-c-organisation-logo brand--department-of-health">
    <a href="/government/organisations/public-health-england" class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color">
      <span class="gem-c-organisation-logo__name">Public Health <br>England</span>
</a>
</div>

        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  <div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
    <span class="govuk-caption-xl gem-c-title__context">
      Research and analysis
    </span>
  <h1 class="gem-c-title__text ">
    Variants: distribution of cases data
  </h1>
</div>
  <p class="publication-header__last-changed">Updated 9 March 2021</p>

  </header>




<div class="govuk-grid-row sidebar-with-body" data-module="sticky-element-container">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav data-module="gem-track-click" aria-label="Contents" role="navigation" class="gem-c-contents-list  gem-c-contents-list--no-underline ">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 1" data-track-options="{&quot;dimension29&quot;:&quot;Variants of concern or under investigation: data up to 8 March 2021&quot;}" data-track-label="#variants-of-concern-or-under-investigation-data-up-to-8-march-2021" href="#variants-of-concern-or-under-investigation-data-up-to-8-march-2021" data-track-category="contentsClicked" class="gem-c-contents-list__link govuk-link ">Variants of concern or under investigation: data up to 8 March 2021</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 2" href="#Variant1" data-track-category="contentsClicked" data-track-options="{&quot;dimension29&quot;:&quot;Differences between a Variant of Concern and a Variant Under Investigation&quot;}" class="gem-c-contents-list__link govuk-link " data-track-label="#Variant1">Differences between a Variant of Concern and a Variant Under Investigation</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 3" href="#variant-voc-20201201" class="gem-c-contents-list__link govuk-link " data-track-options="{&quot;dimension29&quot;:&quot;Variant VOC-202012/01&quot;}" data-track-category="contentsClicked" data-track-label="#variant-voc-20201201">Variant VOC-202012/01</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-category="contentsClicked" class="gem-c-contents-list__link govuk-link " data-track-action="content_item 4" data-track-label="#Variant2" data-track-options="{&quot;dimension29&quot;:&quot;Variant VOC-202012/02&quot;}" href="#Variant2">Variant VOC-202012/02</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-label="#Variant3" data-track-category="contentsClicked" href="#Variant3" data-track-options="{&quot;dimension29&quot;:&quot;Variant VUI-202101/01&quot;}" class="gem-c-contents-list__link govuk-link " data-track-action="content_item 5">Variant VUI-202101/01</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 6" data-track-category="contentsClicked" data-track-label="#Variant4" data-track-options="{&quot;dimension29&quot;:&quot;Variant VOC-202101/02&quot;}" href="#Variant4" class="gem-c-contents-list__link govuk-link ">Variant VOC-202101/02</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a href="#Variant5" class="gem-c-contents-list__link govuk-link " data-track-action="content_item 7" data-track-options="{&quot;dimension29&quot;:&quot;\nVUI-202102/01&quot;}" data-track-label="#Variant5" data-track-category="contentsClicked">
VUI-202102/01</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-options="{&quot;dimension29&quot;:&quot;\nVOC-202102/02&quot;}" data-track-category="contentsClicked" href="#Variant6" data-track-action="content_item 8" class="gem-c-contents-list__link govuk-link " data-track-label="#Variant6">
VOC-202102/02</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link " data-track-options="{&quot;dimension29&quot;:&quot;\nVUI-202102/03&quot;}" href="#Variant7" data-track-label="#Variant7" data-track-category="contentsClicked" data-track-action="content_item 9">
VUI-202102/03</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-category="contentsClicked" data-track-action="content_item 10" data-track-label="#Variant8" class="gem-c-contents-list__link govuk-link " data-track-options="{&quot;dimension29&quot;:&quot;\nVUI-202102/04&quot;}" href="#Variant8">
VUI-202102/04</a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img class="print-meta-data-licence" src="/assets/government-frontend/open-government-licence-min-e98134ae65ff5fe99a524d8c383350b3998f91c298dc24b36126e65f1656c578.png">
</p>
<p>
  © Crown copyright 2021
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/covid-19-variants-genomically-confirmed-case-numbers/variants-distribution-of-cases-data
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
    
    
      <div class="govspeak">
<h2>Variants of concern or under investigation: data up to 8 March 2021</h2>

<h3>Surveillance overview</h3>

<h4>UK total distribution</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Variant</th>
      <th scope="col">Other names by which this variant may be known*</th>
      <th scope="col">Country in which first detected</th>
      <th scope="col">Genomically confirmed cases**</th>
      <th scope="col">Genomically probable cases**</th>
      <th scope="col">Total genomically confirmed and probable cases</th>
      <th scope="col">New cases since last update</th>
      <td></td>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
<abbr title="Variant of Concern">VOC</abbr>-202012/01</td>
      <td>B.1.1.7</td>
      <td>England, UK</td>
      <td>-</td>
      <td>-</td>
      <td>115,558</td>
      <td>+7,291</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>
<abbr title="Variant of Concern">VOC</abbr>-202012/02</td>
      <td>501Y.V2<br>B.1.351</td>
      <td>South Africa</td>
      <td>227</td>
      <td>83</td>
      <td>3105¥¥</td>
      <td>+15</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>
<abbr title="Variant Under Investigation">VUI</abbr>-202101/01</td>
      <td>P2 (descendent of B.1.1.28)</td>
      <td>Brazil</td>
      <td>47</td>
      <td>0</td>
      <td>47^^</td>
      <td>+4</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>
<abbr title="Variant of Concern">VOC</abbr>-202101/02 ¥¥¥</td>
      <td>P1 (descendent of B.1.1.28)</td>
      <td>Japan ex Manaus, Brazil</td>
      <td>5</td>
      <td>1</td>
      <td>6</td>
      <td>-</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>
<abbr title="Variant Under Investigation">VUI</abbr>-202102/01</td>
      <td>A.23.1 with E484K</td>
      <td>England, UK</td>
      <td>59***</td>
      <td>18</td>
      <td>77***</td>
      <td>-1</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>
<abbr title="Variant of Concern">VOC</abbr>-202102/02</td>
      <td>B.1.1.7 with E484K</td>
      <td>England, UK</td>
      <td>35</td>
      <td>0</td>
      <td>35</td>
      <td>+1</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>
<abbr title="Variant Under Investigation">VUI</abbr>-202102/03</td>
      <td>B.1.525 (previously designated UK1188)</td>
      <td>England, UK</td>
      <td>78</td>
      <td>29</td>
      <td>107¥</td>
      <td>+21</td>
      <td>&nbsp;</td>
    </tr>
    <tr>
      <td>
<abbr title="Variant Under Investigation">VUI</abbr>-202102/04</td>
      <td>B1.1.318</td>
      <td>TBC</td>
      <td>20</td>
      <td>0</td>
      <td>20^^^</td>
      <td>+4</td>
      <td>&nbsp;</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>* Currently, there is no agreed international naming system for variants.</td>
    </tr>
    <tr>
      <td>** Genomically confirmed cases are those where all lineage defining positions can be assessed. Probable cases are those where the sequencing is of a lower quality and not all lineage defining positions are called. <a href="#definition" class="govuk-link">Full case definitions</a>.</td>
    </tr>
    <tr>
      <td>*** Excludes 1 <abbr title="Variant Under Investigation">VUI</abbr>-202102/01 case not yet linked to individuals and under further follow-up (1 probable). Two confirmed cases have been reassigned to the NW region, 1 probable case was identified as a duplicate and has been removed, 1 probable case in London is now excluded as it was missing the E484K mutation.</td>
    </tr>
    <tr>
      <td>^^ Excludes 7 sequences reported from devolved administrations, 1 sequence of low quality, and 5 sequences not yet linked to individuals and under further follow-up (5 confirmed).</td>
    </tr>
    <tr>
      <td>^^^ Excludes 2 <abbr title="Variant Under Investigation">VUI</abbr>-202102/04 cases not yet linked to individuals and under further follow-up (2 confirmed).</td>
    </tr>
    <tr>
      <td>¥ Excludes 10 <abbr title="Variant Under Investigation">VUI</abbr>-202102/03 cases not yet linked to individuals and under further follow-up (1 confirmed, 9 probable).</td>
    </tr>
    <tr>
      <td>¥¥ Excludes 51 <abbr title="Variant of Concern">VOC</abbr>-202012/02 sequences reported from devolved administrations and 19 sequences not yet linked to individuals and under further follow-up (7 confirmed, 12 probable).</td>
    </tr>
    <tr>
      <td>¥¥¥ P.1 data is correct as at 4 March.</td>
    </tr>
  </tbody>
</table>

<p>The <a class="govuk-link" href="#Variant1"><abbr title="Variant of Concern">VOC</abbr>-202012/01</a> variant was first detected in the UK and was first sequenced in the UK in September 2020.</p>

<p>The <a href="#Variant2" class="govuk-link"><abbr title="Variant of Concern">VOC</abbr>-202012/02</a> variant was first detected in South Africa and was first sequenced in the UK in December 2020.</p>

<p>The <a href="#Variant3" class="govuk-link"><abbr title="Variant Under Investigation">VUI</abbr>-202101/01</a> variant was first detected in Brazil and was first sequenced in the UK in November 2020.</p>

<p>The <a class="govuk-link" href="#Variant4"><abbr title="Variant of Concern">VOC</abbr>-202101/02</a> variant was first detected in Japan in travellers from Brazil in January 2021 and was first detected in the UK in February 2021.</p>

<p>The <a href="#Variant5" class="govuk-link"><abbr title="Variant Under Investigation">VUI</abbr>-202102/01</a> variant was first detected in the UK in December 2020.</p>

<p>The <a class="govuk-link" href="#Variant6"><abbr title="Variant of Concern">VOC</abbr>-202102/02</a> variant was first detected in the UK in December 2020.</p>

<p>The <a class="govuk-link" href="#Variant7"><abbr title="Variant Under Investigation">VUI</abbr>-202102/03</a> variant was first detected in the UK in December 2020.</p>

<p>The <a class="govuk-link" href="#Variant8"><abbr title="Variant Under Investigation">VUI</abbr>-202102/04</a> variant’s location of first detection is to be confirmed.</p>

<h2>Differences between a Variant of Concern and a Variant Under Investigation</h2>

<p>SARS-CoV-2 variants, if considered to have concerning epidemiological, immunological or pathogenic properties, are raised for formal investigation. At this point they are designated Variant Under Investigation (<abbr title="Variant Under Investigation">VUI</abbr>) with a year, month, and number. Following a risk assessment with the relevant expert committee, they may be designated Variant of Concern (<abbr title="Variant of Concern">VOC</abbr>).</p>

<h2>Variant <abbr title="Variant of Concern">VOC</abbr>-202012/01</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant was first detected in the UK and was first sequenced in the UK in September 2020.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>104,922</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>4,470</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>5,081</td>
    </tr>
    <tr>
      <td>Northern Ireland &nbsp;&nbsp;&nbsp;&nbsp;</td>
      <td>1,085</td>
    </tr>
    <tr>
      <td><strong>Total cases</strong></td>
      <td><strong>115,558</strong></td>
    </tr>
  </tbody>
</table>

<p>Cases for England include confirmed and probable <abbr title="Variant of Concern">VOC</abbr>-202012/01. Scotland, Wales, and Northern Ireland have not reported probable <abbr title="Variant of Concern">VOC</abbr>-202012/01.</p>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant of Concern">VOC</abbr>-202012/01 through whole genome <br>sequencing (WGS) where results of all lineage defining positions are called as alternate <br>(variant) base</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>COVID-19* confirmed case with at least 5 <abbr title="Variant of Concern">VOC</abbr>-202012/01 lineage defining positions<br> are called as alternate (variant) base and all other defining positions reported as N<br> (unknown) or mixed bases</td>
    </tr>
    <tr>
      <td>Low-quality genome (LQG)</td>
      <td>COVID-19* confirmed case where fewer than 4 lineage defining positions are called <br>as alternate (variant) base and all other defining positions reported as N (unknown) or mixed bases</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

<h2>Variant <abbr title="Variant of Concern">VOC</abbr>-202012/02</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant was first detected in South Africa and was first sequenced in the UK in December 2020.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total genomically confirmed cases</th>
      <th scope="col">Total genomically probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>185</td>
      <td>76</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>17</td>
      <td>2</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>20</td>
      <td>5</td>
    </tr>
    <tr>
      <td>Northern Ireland &nbsp;&nbsp;&nbsp;&nbsp;</td>
      <td>5</td>
      <td>0</td>
    </tr>
    <tr>
      <td><strong>Total cases</strong></td>
      <td><strong>227</strong></td>
      <td><strong>83</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant of Concern">VOC</abbr>-202012/02 through whole genome sequencing (WGS) where results of all lineage defining positions are called as alternate (variant) base.</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>COVID-19* confirmed case where:<br> • at least 4 <abbr title="Variant of Concern">VOC</abbr>-202012/02 lineage defining positions are called as alternate (variant) base, and <br>• all other defining positions reported as N (unknown) or mixed bases <br> or<br> • at least 5 of the 9 non-synonymous changes.</td>
    </tr>
    <tr>
      <td>High-risk low-quality genome (HR-LQG)</td>
      <td>COVID-19* confirmed case where:<br> • fewer than 4 positions are called but at least one is called as alternate (variant) base, and<br>• all other defining positions reported as N (unknown) or mixed bases<br> or<br> • at least one but fewer than 5 of the 9 non-synonymous changes, and<br> • fewer than 5 wild type calls.</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

<h2>Variant <abbr title="Variant Under Investigation">VUI</abbr>-202101/01</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant was first detected in Brazil and was first sequenced in the UK in November 2020.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total genomically confirmed cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>42</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>5</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Northern Ireland &nbsp;&nbsp;&nbsp;&nbsp;</td>
      <td>0</td>
    </tr>
    <tr>
      <td><strong>Total cases</strong></td>
      <td><strong>47</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant Under Investigation">VUI</abbr>-202101/01 through whole genome sequencing (WGS) where: <br>• results of all lineage defining non-synonymous changes called as alternate base <br> or <br> • 6 of 7 non-synonymous changes called as alternate base, and<br>• remaining position either N or mixed bases</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>N/A</td>
    </tr>
    <tr>
      <td>High-risk low-quality genome (HR-LQG)</td>
      <td>COVID-19* confirmed case where fewer than 6 positions are called but at least one is called as alternate (variant) base and all other defining positions reported as N (unknown) or mixed bases.</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

<h2>Variant <abbr title="Variant of Concern">VOC</abbr>-202101/02</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant was first detected in Japan in travellers from Brazil in January 2021 and was first detected in the UK in February 2021.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total genomically confirmed cases</th>
      <th scope="col">Total genomically probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>4</td>
      <td>1</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>2</td>
      <td>1</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Northern Ireland &nbsp;&nbsp;&nbsp;&nbsp;</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td><strong>Total cases</strong></td>
      <td><strong>5</strong></td>
      <td><strong>1</strong></td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>¥¥¥ P.1 data is correct as at 4 March.</td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant of Concern">VOC</abbr>-202101/02 through whole genome sequencing (WGS) where results of all lineage defining non-synonymous changes called as alternate bases.</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>COVID-19* confirmed case with: <br>• at least 5 lineage defining non-synonymous changes called as alternate base, and <br>• all other positions either N or mixed bases</td>
    </tr>
    <tr>
      <td>High-risk low-quality genome (HR-LQG)</td>
      <td>COVID-19* confirmed case where: <br>• fewer than 5 positions are called but at least one is called as alternate (variant) base, and <br>• all other defining positions reported as N (unknown) or mixed bases</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

<h2>
<abbr title="Variant Under Investigation">VUI</abbr>-202102/01</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant was first detected in the UK in December 2020.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total genomically confirmed cases</th>
      <th scope="col">Total genomically probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England (Merseyside and North West)</td>
      <td>58</td>
      <td>18</td>
    </tr>
    <tr>
      <td>England (other)</td>
      <td>1</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td><strong>Total cases</strong></td>
      <td><strong>59</strong></td>
      <td><strong>18</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant Under Investigation">VUI</abbr>-202102/01 through whole genome sequencing (WGS) where results of all lineage defining non-synonymous changes called as alternate bases.</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>COVID-19* confirmed case with: <br>• at least 5 lineage defining non-synonymous changes called as alternate base, and <br>• all other positions either N or mixed base<br> or<br> • at least 5 of the 9 non-synonymous changes.</td>
    </tr>
    <tr>
      <td>High-risk low-quality genome (HR-LQG)</td>
      <td>COVID-19* confirmed case where: <br>• fewer than 5 positions are called but at least one is called as alternate (variant) base, and <br>• all other defining positions reported as N (unknown) or mixed bases.</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

<h2>
<abbr title="Variant of Concern">VOC</abbr>-202102/02</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant was first detected in the UK in December 2020.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total genomically confirmed cases</th>
      <th scope="col">Total genomically probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England (Bristol and South West)</td>
      <td>22</td>
      <td>0</td>
    </tr>
    <tr>
      <td>England (other)</td>
      <td>13</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td><strong>Total cases</strong></td>
      <td><strong>35</strong></td>
      <td><strong>0</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant of Concern">VOC</abbr>-202102/02 through whole genome sequencing (WGS) where results of all lineage defining non-synonymous changes called as alternate bases.</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>N/A</td>
    </tr>
    <tr>
      <td>High-risk low-quality genome (HR-LQG)</td>
      <td>Probable or confirmed <abbr title="Variant of Concern">VOC</abbr>-202012/01 and fewer than 4 positions are called but at least one is called as alternate (variant) base, and all other defining positions reported as N (unknown) or mixed bases.</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

<h2>
<abbr title="Variant Under Investigation">VUI</abbr>-202102/03</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant was first detected in the UK in December 2020.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total genomically confirmed cases</th>
      <th scope="col">Total genomically probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>75</td>
      <td>27</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>0</td>
      <td>2</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>3</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td><strong>Total cases</strong></td>
      <td><strong>78</strong></td>
      <td><strong>29</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant Under Investigation">VUI</abbr>-202102/03 through whole genome sequencing (WGS) where results of all lineage defining non-synonymous changes called as alternate bases.</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>COVID-19* confirmed case with: <br>• at least 5 lineage defining non-synonymous changes called as alternate base, and <br>• all other positions either N or mixed base.</td>
    </tr>
    <tr>
      <td>High-risk low-quality genome (HR-LQG)</td>
      <td>COVID-19* confirmed case where: <br>• fewer than 5 positions are called but at least one is called as alternate (variant) base, and  <br>• all other defining positions reported as N (unknown) or mixed bases.</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

<h2>
<abbr title="Variant Under Investigation">VUI</abbr>-202102/04</h2>

<h3>Surveillance overview</h3>

<h4>UK distribution – summary of 4 nations distribution</h4>

<p>This variant’s location of first detection is to be confirmed.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total genomically confirmed cases</th>
      <th scope="col">Total genomically probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>18</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>2</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>0</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Total cases</td>
      <td><strong>20</strong></td>
      <td><strong>0</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed</td>
      <td>COVID-19* case confirmed as infected with <abbr title="Variant Under Investigation">VUI</abbr>-202102/04 through whole genome sequencing (WGS) where results of all lineage defining non-synonymous changes called as alternate bases.</td>
    </tr>
    <tr>
      <td>Probable</td>
      <td>COVID-19* confirmed case with: <br>• at least 5 lineage defining non-synonymous changes called as alternate base, and <br>• all other positions either N or mixed base<br> or<br> • at least 5 of the 9 non-synonymous changes.</td>
    </tr>
    <tr>
      <td>High-risk low-quality genome (HR-LQG)</td>
      <td>COVID-19* confirmed case where: <br>• fewer than 5 positions are called but at least one is called as alternate (variant) base, and <br>• all other defining positions reported as N (unknown) or mixed bases.</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: an individual who is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (PCR).</td>
    </tr>
  </tbody>
</table>

</div>


</div>
</div>
  </div>

  <div class="sticky-element" data-sticky-element="">
    <a class="govuk-link app-c-back-to-top dont-print" href="#contents">
    <svg width="13" class="app-c-back-to-top__icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 13 17" height="17">
      <path fill="currentColor" d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z"></path>
    </svg>
    Contents
</a>

    <div class="sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button data-module="print-link" class="govuk-link gem-c-print-link__button">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>