<main class="html-publication" lang="en" role="main">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          
<div class="gem-c-organisation-logo brand--department-of-health">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color" href="/government/organisations/uk-health-security-agency">
      <span class="gem-c-organisation-logo__name">UK Health<br>Security<br>Agency</span>
</a>
</div>
        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  

<div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
      <span class="govuk-caption-xl gem-c-title__context">
    Research and analysis
  </span>


  <h1 class="gem-c-title__text govuk-heading-xl">
    Variants: distribution of case data, 6 May 2022
  </h1>
</div>
  <p class="publication-header__last-changed">Updated 20 May 2022</p>

  </header>



<div class="govuk-grid-row sidebar-with-body" data-module="sticky-element-container">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav aria-label="Contents" class="gem-c-contents-list" data-module="gem-track-click" role="navigation">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 1" data-track-category="contentsClicked" data-track-label="#changes-to-variant-classification-system" data-track-options="{&quot;dimension29&quot;:&quot;Changes to variant classification system&quot;}" href="#changes-to-variant-classification-system">Changes to variant classification system</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 2" data-track-category="contentsClicked" data-track-label="#surveillance-overview" data-track-options="{&quot;dimension29&quot;:&quot;Surveillance overview&quot;}" href="#surveillance-overview">Surveillance overview</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 3" data-track-category="contentsClicked" data-track-label="#uk-total-distribution-for-variants-with-new-reports-in-the-last-90-days" data-track-options="{&quot;dimension29&quot;:&quot;UK total distribution for variants with new reports in the last 90 days&quot;}" href="#uk-total-distribution-for-variants-with-new-reports-in-the-last-90-days">UK total distribution for variants with new reports in the last 90 days</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 4" data-track-category="contentsClicked" data-track-label="#uk-total-distribution-for-variants-detected-in-the-uk-with-no-or-lowsupsup-activity-in-the-last-90-days" data-track-options="{&quot;dimension29&quot;:&quot;UK total distribution for variants detected in the UK with no or low$ activity in the last 90 days&quot;}" href="#uk-total-distribution-for-variants-detected-in-the-uk-with-no-or-lowsupsup-activity-in-the-last-90-days">UK total distribution for variants detected in the UK with no or low$ activity in the last 90 days</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 5" data-track-category="contentsClicked" data-track-label="#note-on-omicron-voc-21nov-01" data-track-options="{&quot;dimension29&quot;:&quot;Note on Omicron (VOC-21NOV-01)&quot;}" href="#note-on-omicron-voc-21nov-01">Note on Omicron (VOC-21NOV-01)</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 6" data-track-category="contentsClicked" data-track-label="#alpha" data-track-options="{&quot;dimension29&quot;:&quot;Alpha&quot;}" href="#alpha">Alpha</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 7" data-track-category="contentsClicked" data-track-label="#beta" data-track-options="{&quot;dimension29&quot;:&quot;Beta&quot;}" href="#beta">Beta</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 8" data-track-category="contentsClicked" data-track-label="#gamma" data-track-options="{&quot;dimension29&quot;:&quot;Gamma&quot;}" href="#gamma">Gamma</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 9" data-track-category="contentsClicked" data-track-label="#delta" data-track-options="{&quot;dimension29&quot;:&quot;Delta&quot;}" href="#delta">Delta</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 10" data-track-category="contentsClicked" data-track-label="#v-21oct-01" data-track-options="{&quot;dimension29&quot;:&quot;V-21OCT-01&quot;}" href="#v-21oct-01">V-21OCT-01</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 11" data-track-category="contentsClicked" data-track-label="#omicron" data-track-options="{&quot;dimension29&quot;:&quot;Omicron&quot;}" href="#omicron">Omicron</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 12" data-track-category="contentsClicked" data-track-label="#zeta" data-track-options="{&quot;dimension29&quot;:&quot;Zeta&quot;}" href="#zeta">Zeta</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 13" data-track-category="contentsClicked" data-track-label="#kappa" data-track-options="{&quot;dimension29&quot;:&quot;Kappa&quot;}" href="#kappa">Kappa</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 14" data-track-category="contentsClicked" data-track-label="#v-21may-02" data-track-options="{&quot;dimension29&quot;:&quot;V-21MAY-02&quot;}" href="#v-21may-02">V-21MAY-02</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 15" data-track-category="contentsClicked" data-track-label="#lambda" data-track-options="{&quot;dimension29&quot;:&quot;Lambda&quot;}" href="#lambda">Lambda</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 16" data-track-category="contentsClicked" data-track-label="#omicron-ba2" data-track-options="{&quot;dimension29&quot;:&quot;Omicron BA.2&quot;}" href="#omicron-ba2">Omicron BA.2</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 17" data-track-category="contentsClicked" data-track-label="#v-22apr-02" data-track-options="{&quot;dimension29&quot;:&quot;V-22APR-02&quot;}" href="#v-22apr-02">V-22APR-02</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 18" data-track-category="contentsClicked" data-track-label="#omicron-ba4" data-track-options="{&quot;dimension29&quot;:&quot;Omicron BA.4&quot;}" href="#omicron-ba4">Omicron BA.4</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 19" data-track-category="contentsClicked" data-track-label="#omicron-ba5" data-track-options="{&quot;dimension29&quot;:&quot;Omicron BA.5&quot;}" href="#omicron-ba5">Omicron BA.5</a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img class="print-meta-data-licence" src="/assets/government-frontend/open-government-licence-min-93b6a51b518ff99714a1aa2a7d2162735c155ec3cb073c75fb88b2a332fa83d3.png">
</p>
<p>
  © Crown copyright 2022
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/covid-19-variants-genomically-confirmed-case-numbers/variants-distribution-of-case-data-6-may-2022
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
    
    
      <div class="govspeak">
<h2>Changes to variant classification system</h2>

<p>The UK Health Security Agency (<abbr title="UK Health Security Agency">UKHSA</abbr>) variant classification system assigns the Variant of Concern (<abbr title="variant of concern">VOC</abbr>) label to emerging or circulating variants which have, or can be predicted to have, a:</p>

<ul>
  <li>detrimental change in biological properties (changes in transmissibility, severity or immune evasion) compared to the current dominant variant(s); and</li>
  <li>growth rate potentially compatible with maintaining transmission and/or displacing the current dominant variant</li>
</ul>

<p>Previous variants of concern which no longer meet the criteria above have been redesignated. New variants that do not meet the above criteria but are of note based on genomic features and growth, now receive a variant number (V-date-number).</p>

<p>These changes took effect as of 1 April 2022.</p>

<h2>Surveillance overview</h2>

<p>From 31 January 2022, <abbr title="UK Health Security Agency">UKHSA</abbr> has reported episodes of infection in its coronavirus (COVID-19) surveillance. Previous surveillance of COVID-19 in England was based on positive samples from the same person grouped together, so that the person is only counted once and reported using the date of the first positive result.</p>

<p>The ‘episode’ definition introduced groups positive results up to and including 90 days apart into episodes. Positive test results which are more than 90 days apart will be classed as separate episodes.</p>

<p>The <abbr title="UK Health Security Agency">UKHSA</abbr> methodology for reporting of COVID-19 cases varies between England and the devolved authorities. England and Scotland report 90-day infection episodes. Wales reports 42-day infection episodes. Northern Ireland reports data at individual level (only one infection episode is reported per person) but is working towards reporting 90-day infection episodes.</p>

<p>Information pertaining to V-21MAR-01, B1.324.1 with E484K has been removed as this variant has been reclassified as provisionally extinct. Variants V-21FEB-01 (A.23.1 with E484K), and V-21FEB-02 (Alpha with E484K) are provisionally extinct (not detected for 12 weeks in the UK or in GISAID) therefore are no longer being included in the publication since 12 November 2021. Variant V-21MAY-01, AV.1 has been removed as of 3 December 2021 as it has been classified as provisionally extinct. Variant V-21JUL-01 (Mu) has been removed as of 17 February 2022 as it has been classified as provisionally extinct. 
 </p>

<h2>UK total distribution for variants with new reports in the last 90 days</h2>

<table>
  <thead>
    <tr>
      <th scope="col">Variant</th>
      <th scope="col">Other names by which this variant may be known*</th>
      <th scope="col">Lineage</th>
      <th scope="col">Total confirmed (sequencing) and probable (genotyping) cases** ^</th>
      <th scope="col">New cases since last update (data up to 27 April 2022)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Delta<sup>+</sup>
</td>
      <td>V-21APR-02</td>
      <td>B.1.617.2 AY.1    AY.2</td>
      <td>1,732,401<sup>+</sup>
</td>
      <td>-42</td>
    </tr>
    <tr>
      <td>Omicron</td>
      <td>
<abbr title="variant of concern">VOC</abbr>-21NOV-01</td>
      <td>B.1.1.529</td>
      <td>2,178,620</td>
      <td>+623</td>
    </tr>
    <tr>
      <td>Omicron BA.2</td>
      <td>
<abbr title="variant of concern">VOC</abbr>-22JAN-01</td>
      <td>B.1.1.529.BA.2</td>
      <td>422,208</td>
      <td>+15,253</td>
    </tr>
    <tr>
      <td>V-22APR-02</td>
      <td>V-22APR-02</td>
      <td>B.1.1.529.XE</td>
      <td>1,416</td>
      <td>+45</td>
    </tr>
  </tbody>
</table>

<p>*<abbr title="World Health Organization">WHO</abbr> naming has been included where available.</p>

<p>**Cumulative numbers may be revised up or down as a result of reclassification of results, re-infections and changes to diagnostic tests, new variants or public health management levels.</p>

<p>&nbsp;^Confirmed individuals are confirmed COVID-19 cases with a validated sequencing result meeting the confirmed case definitions outlined in the sections below. Probable individuals are confirmed COVID-19 cases with a genotyping result where sequencing confirmation is awaited or not available. Published figures include genotyping assay data for England, using a combination of highly specific mutations as an indicator of a probable variant. Genotyping assay data is currently only available for Alpha, Beta (for specimen dates up to and including 30 June 2021), Delta and Gamma.</p>

<p>+Of which 793 had the K417N mutation.</p>

<h2>UK total distribution for variants detected in the UK with no or low<sup>$</sup> activity in the last 90 days</h2>

<table>
  <thead>
    <tr>
      <th scope="col">Variant</th>
      <th scope="col">Other names by which this variant may be known*</th>
      <th scope="col">Lineage</th>
      <th scope="col">Total confirmed (sequencing) and probable (genotyping) cases** ^</th>
      <th scope="col">Last reported specimen date$</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Alpha</td>
      <td>V-20DEC-01</td>
      <td>B.1.1.7</td>
      <td>281,330</td>
      <td>5 February 2022</td>
    </tr>
    <tr>
      <td>Beta<sup>¥</sup>
</td>
      <td>V-20DEC-02</td>
      <td>B.1.351</td>
      <td>1,064<sup>¥</sup>
</td>
      <td>12 November 2021</td>
    </tr>
    <tr>
      <td>Gamma</td>
      <td>V-21JAN-02</td>
      <td>P.1</td>
      <td>295</td>
      <td>17 March 2022</td>
    </tr>
    <tr>
      <td>V-21OCT-01***</td>
      <td>V-21OCT-01</td>
      <td>A.Y 4.2</td>
      <td>123,461</td>
      <td>13 March 2022</td>
    </tr>
    <tr>
      <td>Zeta</td>
      <td>V-21JAN-01</td>
      <td>P.2</td>
      <td>50</td>
      <td>14 April 2021</td>
    </tr>
    <tr>
      <td>Kappa</td>
      <td>V-21APR-01</td>
      <td>B.1.617.1</td>
      <td>466</td>
      <td>14 September 2021</td>
    </tr>
    <tr>
      <td>V-21MAY-02</td>
      <td>V-21MAY-02</td>
      <td>C.36.3</td>
      <td>136</td>
      <td>26 June 2021</td>
    </tr>
    <tr>
      <td>Lambda</td>
      <td>V-21JUN-01</td>
      <td>C.37</td>
      <td>8</td>
      <td>18 June 2021</td>
    </tr>
    <tr>
      <td>Omicron BA.4</td>
      <td>V-22APR-03</td>
      <td>B.1.1.529.BA.4</td>
      <td>9</td>
      <td>22 April 2022</td>
    </tr>
    <tr>
      <td>Omicron BA.5</td>
      <td>V-22APR-04</td>
      <td>B.1.1.529.BA.5</td>
      <td>5</td>
      <td>20 April 2022</td>
    </tr>
  </tbody>
</table>

<p>*<abbr title="World Health Organization">WHO</abbr> naming has been included where available.</p>

<p>**Cumulative numbers may be revised up or down as a result of reclassification of results, re-infections and changes to diagnostic tests, new variants or public health management levels.</p>

<p>&nbsp;^Confirmed individuals are confirmed COVID-19 cases with a validated sequencing result meeting the confirmed case definitions outlined in the sections below. Probable individuals are confirmed COVID-19 cases with a genotyping result where sequencing confirmation is awaited or not available. Published figures include genotyping assay data for England, using a combination of highly specific mutations as an indicator of a probable variant. Genotyping assay data is currently only available for Alpha, Beta (for specimen dates up to and including 30 June 2021), Delta and Gamma.</p>

<p><sup>$</sup>Fewer than 10 reports in any week during the last 90 days.</p>

<p><sup>¥</sup>Due to having identical genotyping profiles, since 1 July Beta and Mu cases identified by genotyping have been categorised as ‘Beta or Mu’. In addition to the above numbers, no further ‘Beta or Mu’ provisional genotyping cases have been identified.</p>

<p>***Confirmed V-21APR-02 with at least 2 V-21OCT-01 lineage defining positions called as alternate (variant) base and all other defining positions reported as N (unknown) or mixed bases.</p>

<h2>Note on Omicron (<abbr title="variant of concern">VOC</abbr>-21NOV-01)</h2>

<p>As genotyping assays cannot distinguish between BA.1 and BA.2 genomes, Omicron (<abbr title="variant of concern">VOC</abbr>-21NOV-01) includes cases of BA.1 and BA.2 genomes where those genomes were identified by genotyping only and not confirmed by genome sequencing.</p>

<h2>Alpha</h2>

<h3>Surveillance overview</h3>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total confirmed and probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>230,398</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>9,365</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>29,123</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>12,444</td>
    </tr>
    <tr>
      <th scope="row">Total cases</th>
      <td><strong>281,330</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed <br>(sequencing)</td>
      <td>COVID-19* confirmed case with at least 5 V-20DEC-01 lineage defining positions are called as alternate (variant) base and all other defining positions reported as N (unknown) or mixed bases</td>
    </tr>
    <tr>
      <td>Probable (genotyping)</td>
      <td>COVID-19* confirmed case with a genotyping result where sequencing confirmation is awaited or not available</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: a case where the individual is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (<abbr title="polymerase chain reaction">PCR</abbr>).</td>
    </tr>
  </tbody>
</table>

<h2>Beta</h2>

<h3>Surveillance overview</h3>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total confirmed and probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>950</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>9</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>62</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>43</td>
    </tr>
    <tr>
      <th scope="row">Total cases</th>
      <td><strong>1,064</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed <br>(sequencing)</td>
      <td>COVID-19* confirmed case where:<br> • at least 4 V-20DEC-02 lineage defining positions are called as alternate (variant) base, and <br>• all other defining positions reported as N (unknown) or mixed bases <br> or<br> • at least 5 of the 9 non-synonymous changes</td>
    </tr>
    <tr>
      <td>Probable (genotyping)</td>
      <td>COVID-19* confirmed case with a genotyping result where sequencing confirmation is awaited or not available</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: a case where the individual is confirmed to be infected with SARS-CoV-2 by <abbr title="polymerase chain reaction">PCR</abbr>.</td>
    </tr>
  </tbody>
</table>

<h2>Gamma</h2>

<h3>Surveillance overview</h3>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total confirmed and probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>270</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>23</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>2</td>
    </tr>
    <tr>
      <th scope="row">Total cases</th>
      <td><strong>295</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed <br>(sequencing)</td>
      <td>COVID-19* confirmed case with: <br>• at least 5 lineage defining non-synonymous changes called as alternate base, and <br>• all other positions either N or mixed bases</td>
    </tr>
    <tr>
      <td>Probable (genotyping)</td>
      <td>COVID-19* confirmed case with a genotyping result where sequencing confirmation is awaited or not available</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: a case where the individual is confirmed to be infected with SARS-CoV-2 by <abbr title="polymerase chain reaction">PCR</abbr>.</td>
    </tr>
  </tbody>
</table>

<h2>Delta</h2>

<h3>Surveillance overview</h3>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total confirmed and probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>1,414,183</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>93,794</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>133,282</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>91,142</td>
    </tr>
    <tr>
      <th scope="row">Total cases</th>
      <td><strong>1,732,401</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed<br>(sequencing)</td>
      <td>COVID-19* confirmed case with at least 4 variant defining changes called as alternate base</td>
    </tr>
    <tr>
      <td>Probable (genotyping)</td>
      <td>COVID-19* confirmed case with a genotyping result where sequencing confirmation is awaited or not available</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: a case where the individual is confirmed to be infected with SARS-CoV-2 by <abbr title="polymerase chain reaction">PCR</abbr>.</td>
    </tr>
  </tbody>
</table>

<h2>V-21OCT-01</h2>

<h3>Surveillance overview</h3>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total confirmed and probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>94,018</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>757</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>13,650</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>15,036</td>
    </tr>
    <tr>
      <th scope="row">Total cases</th>
      <td><strong>123,461</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed <br>(sequencing)</td>
      <td>COVID-19* confirmed case that are confirmed <abbr title="variant of concern">VOC</abbr>-21APR-02 with at least 2 V-21OCT-01 lineage defining positions called as alternate (variant) base and all other defining positions reported as N (unknown) or mixed bases.</td>
    </tr>
  </tbody>
</table>

<p>*A confirmed COVID-19 case: a case where the individual is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (<abbr title="polymerase chain reaction">PCR</abbr>).</p>

<h2>Omicron</h2>

<h3>Surveillance overview</h3>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total confirmed and probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>1,724,457</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>109,397</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>248,703</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>96,063</td>
    </tr>
    <tr>
      <th scope="row">    Total cases</th>
      <td><strong>2,178,620</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed <br>(sequencing)</td>
      <td>COVID-19* confirmed case with at least 10 lineage defining non-synonymous changes called as alternate base and no more than 7 defining positions called as wild-type</td>
    </tr>
    <tr>
      <td>Probable (genotyping)</td>
      <td>COVID-19* confirmed case with a genotyping result where sequencing confirmation is awaited or not available</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: a case where the individual is confirmed to be infected with SARS-CoV-2 by <abbr title="polymerase chain reaction">PCR</abbr>.</td>
    </tr>
  </tbody>
</table>

<h2>Zeta</h2>

<h3>Surveillance overview</h3>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total confirmed and probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>44</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>5</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>1</td>
    </tr>
    <tr>
      <th scope="row">    Total cases</th>
      <td><strong>50</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed <br>(sequencing)</td>
      <td>COVID-19* case confirmed as infected with V-21JAN-01 through whole genome sequencing (<abbr title="whole genome sequencing">WGS</abbr>) where: <br>• results of all lineage defining non-synonymous changes called as alternate base <br> or <br> • 6 of 7 non-synonymous changes called as alternate base, and<br>• remaining position either N or mixed bases</td>
    </tr>
    <tr>
      <td>Probable (genotyping)</td>
      <td>COVID-19* confirmed case with a genotyping result where sequencing confirmation is awaited or not available</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: a case where the individual is confirmed to be infected with SARS-CoV-2 by <abbr title="polymerase chain reaction">PCR</abbr>.</td>
    </tr>
  </tbody>
</table>

<h2>Kappa</h2>

<h3>Surveillance overview</h3>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total confirmed and probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>419</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>5</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>27</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>15</td>
    </tr>
    <tr>
      <th scope="row">    Total cases</th>
      <td><strong>466</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed <br>(sequencing)</td>
      <td>COVID-19* confirmed case with: <br>• at least 6 variant defining changes called as alternate base, and <br>• all other positions either N or mixed base</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: a case where the individual is confirmed to be infected with SARS-CoV-2 by <abbr title="polymerase chain reaction">PCR</abbr>.</td>
    </tr>
  </tbody>
</table>

<h2>V-21MAY-02</h2>

<h3>Surveillance overview</h3>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total confirmed and probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>125</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>3</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>4</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>4</td>
    </tr>
    <tr>
      <th scope="row">    Total cases</th>
      <td><strong>136</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed <br>(sequencing)</td>
      <td>COVID-19* confirmed case with at least 5 variant defining changes called as alternate base, and<br> • all other positions either N or mixed base.</td>
    </tr>
  </tbody>
</table>

<p>*A confirmed COVID-19 case: a case where the individual is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (<abbr title="polymerase chain reaction">PCR</abbr>).</p>

<h2>Lambda</h2>

<h3>Surveillance overview</h3>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total confirmed and probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>8</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>0</td>
    </tr>
    <tr>
      <th scope="row">    Total cases</th>
      <td><strong>8</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed <br>(sequencing)</td>
      <td>COVID-19* confirmed case with at least 5 V-21JUN-01 lineage defining positions are called as alternate (variant) base and all other defining positions reported as N (unknown) or mixed bases.</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: a case where the individual is confirmed to be infected with SARS-CoV-2 by <abbr title="polymerase chain reaction">PCR</abbr>.</td>
    </tr>
  </tbody>
</table>

<h2>Omicron BA.2</h2>

<h3>Surveillance overview</h3>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total confirmed and probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>303,078</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>8,291</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>83,726</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>27,113</td>
    </tr>
    <tr>
      <th scope="row">   Total cases</th>
      <td><strong>422,208</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed <br>(sequencing)</td>
      <td>COVID-19* confirmed case with at least 10 lineage defining non-synonymous changes called as alternate base and no more than 7 defining positions called as wild-type.</td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>*A confirmed COVID-19 case: a case where the individual is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (<abbr title="polymerase chain reaction">PCR</abbr>).</td>
    </tr>
  </tbody>
</table>

<h2>V-22APR-02</h2>

<h3>Surveillance overview</h3>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total confirmed and probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>1,399</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>17</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>Not reported</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>Not reported</td>
    </tr>
    <tr>
      <th scope="row">    Total cases</th>
      <td><strong>1,416</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed <br>(sequencing)</td>
      <td>Awaiting final case definitions.</td>
    </tr>
  </tbody>
</table>

<p>*A confirmed COVID-19 case: a case where the individual is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (<abbr title="polymerase chain reaction">PCR</abbr>).</p>

<h2>Omicron BA.4</h2>

<h3>Surveillance overview</h3>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total confirmed and probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>9</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>Not reported</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>Not reported</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>Not reported</td>
    </tr>
    <tr>
      <th scope="row">    Total cases</th>
      <td><strong>9</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed <br>(sequencing)</td>
      <td>Awaiting final case definitions.</td>
    </tr>
  </tbody>
</table>

<h2>Omicron BA.5</h2>

<h3>Surveillance overview</h3>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Total confirmed and probable cases</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>England</td>
      <td>5</td>
    </tr>
    <tr>
      <td>Northern Ireland</td>
      <td>Not reported</td>
    </tr>
    <tr>
      <td>Scotland</td>
      <td>Not reported</td>
    </tr>
    <tr>
      <td>Wales</td>
      <td>Not reported</td>
    </tr>
    <tr>
      <th scope="row">    Total cases</th>
      <td><strong>5</strong></td>
    </tr>
  </tbody>
</table>

<h4>Case definitions</h4>

<table>
  <thead>
    <tr>
      <th scope="col">Genomic match</th>
      <th scope="col">Criteria</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Confirmed <br>(sequencing)</td>
      <td>Awaiting final case definitions.</td>
    </tr>
  </tbody>
</table>

<p>*A confirmed COVID-19 case: a case where the individual is confirmed to be infected with SARS-CoV-2 by polymerase chain reaction (<abbr title="polymerase chain reaction">PCR</abbr>).</p>

</div>


</div>
</div>
  </div>

  <div class="sticky-element" data-sticky-element="">
    <a class="govuk-link app-c-back-to-top dont-print" href="#contents">
    <svg class="app-c-back-to-top__icon" focusable="false" height="17" viewBox="0 0 13 17" width="13" xmlns="http://www.w3.org/2000/svg">
      <path d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z" fill="currentColor"></path>
    </svg>
    Contents
</a>

    <div class="sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>