<main class="html-publication" role="main" lang="en">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          
<div class="gem-c-organisation-logo brand--department-of-health">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color" href="/government/organisations/uk-health-security-agency">
      <span class="gem-c-organisation-logo__name">UK Health<br>Security<br>Agency</span>
</a>
</div>
        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  

<div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
      <span class="govuk-caption-xl gem-c-title__context">
    Transparency data
  </span>

  
  <h1 class="gem-c-title__text govuk-heading-xl">
    EMHP wastewater monitoring of SARS-CoV-2 in England: 1 June to 10 January 2022
  </h1>
</div>
  <p class="publication-header__last-changed">Published 27 January 2022</p>

  </header>



<div class="govuk-grid-row sidebar-with-body" data-module="sticky-element-container">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav role="navigation" class="gem-c-contents-list" data-module="gem-track-click" aria-label="Contents">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-label="#background-on-the-emhp-wastewater-monitoring-programme" data-track-options="{&quot;dimension29&quot;:&quot;Background on the EMHP wastewater monitoring programme&quot;}" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" href="#background-on-the-emhp-wastewater-monitoring-programme" data-track-category="contentsClicked" data-track-action="content_item 1">Background on the EMHP wastewater monitoring programme</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 2" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-label="#emhp-sampling-sites" data-track-options="{&quot;dimension29&quot;:&quot;\nEMHP sampling sites&quot;}" href="#emhp-sampling-sites" data-track-category="contentsClicked">
EMHP sampling sites</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-category="contentsClicked" data-track-options="{&quot;dimension29&quot;:&quot;Concentration of SARS-CoV-2 RNA in wastewater samples&quot;}" href="#concentration-of-sars-cov-2-rna-in-wastewater-samples" data-track-action="content_item 3" data-track-label="#concentration-of-sars-cov-2-rna-in-wastewater-samples" class="gem-c-contents-list__link govuk-link govuk-link--no-underline">Concentration of SARS-CoV-2 RNA in wastewater samples</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-category="contentsClicked" data-track-options="{&quot;dimension29&quot;:&quot;Regional average concentration of SARS-CoV-2 RNA in wastewater&quot;}" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 4" data-track-label="#regional-average-concentration-of-sars-cov-2-rna-in-wastewater" href="#regional-average-concentration-of-sars-cov-2-rna-in-wastewater">Regional average concentration of SARS-CoV-2 RNA in wastewater</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-category="contentsClicked" data-track-action="content_item 5" data-track-options="{&quot;dimension29&quot;:&quot;Wastewater variant detection&quot;}" data-track-label="#wastewater-variant-detection" href="#wastewater-variant-detection">Wastewater variant detection</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a href="#methodology" data-track-options="{&quot;dimension29&quot;:&quot;Methodology&quot;}" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 6" data-track-label="#methodology" data-track-category="contentsClicked">Methodology</a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img src="/assets/government-frontend/open-government-licence-min-e98134ae65ff5fe99a524d8c383350b3998f91c298dc24b36126e65f1656c578.png" class="print-meta-data-licence">
</p>
<p>
  © Crown copyright 2022
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/monitoring-of-sars-cov-2-rna-in-england-wastewater-monthly-statistics-1-june-2021-to-10-january-2022/emhp-wastewater-monitoring-of-sars-cov-2-in-england-1-june-to-10-january-2022
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
    
    
      <div class="govspeak">
<p>Data showing the concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> and variants of concern (<abbr title="variants of concern">VOCs</abbr>) detected in wastewater by the Environmental Monitoring for Health Protection (<abbr title="Environmental Monitoring for Health Protection">EMHP</abbr>) wastewater monitoring programme. New additions to this monthly publication include Omicron detections in wastewater from 22 November 2021 to 2 January 2022.</p>

<p>Please note that, due to a decrease in the number of wastewater samples taken over the Christmas period, concentration data for the period 21 December to 27 December 2021 and Omicron detection data for the period 20 December to 26 December 2021 are not included in this publication.</p>

<h2>Background on the <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> wastewater monitoring programme</h2>

<p>People infected with coronavirus (COVID-19) shed the virus during daily activities such as going to the toilet and blowing their nose. The virus enters the sewer system through sinks, drains and toilets. Fragments of SARS-CoV-2 (the virus that causes COVID-19) can be detected in samples of wastewater (untreated sewage).</p>

<p>The <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> programme is led by the UK Health Security Agency (<abbr title="UK Health Security Agency">UKHSA</abbr>) and run in partnership with the Department for Environment, Food and Rural Affairs (Defra), the Environment Agency (EA), the Centre for Environment, Fisheries and Aquaculture Science (Cefas), academia, and water companies. It provides coverage of approximately 40 million people across England. The <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> team coordinates with programmes in the devolved administrations to provide UK-wide wastewater monitoring. The programme tests sewage for fragments of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr>. Untreated influent samples are taken from approximately 275 sewage treatment works (<abbr title="sewage treatment works">STWs</abbr>) in England, 3 times a week (see <a href="#emhp-sampling-sites" class="govuk-link">figures 1 to 9</a>). <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> also samples from sewer network sites (manholes). Untreated influent samples are taken from approximately 195 sewer network sites in England 4 times a week. Data from sewer network sites is not included in the concentration data in this publication as these sites generally fall within sewage treatment work (<abbr title="sewage treatment work">STW</abbr>) catchments. Data from sewer network sites is included in the Omicron detection data in this publication. Further detail about <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> wastewater coverage can be found in the publication <a class="govuk-link" href="https://www.gov.uk/government/publications/wastewater-testing-coverage-data-for-19-may-2021-emhp-programme/wastewater-testing-coverage-data-for-the-environmental-monitoring-for-health-protection-emhp-programme">Wastewater testing coverage data for the Environmental Monitoring for Health Protection (<abbr title="Environmental Monitoring for Health Protection">EMHP</abbr>) programme</a>.</p>

<p>The programme helps identify where the virus is circulating in England as well as detecting mutations of the virus, including those associated with known <abbr title="variants of concern">VOCs</abbr>, and variants under investigation (<abbr title="variants under investigation">VUIs</abbr>). Findings are reported to national decision makers and local stakeholders on a regular basis, helping to inform national strategy and localised action. Wastewater analysis has the benefit of detecting the virus regardless of whether people have symptoms or whether they are tested. Wastewater monitoring complements other testing programmes and public health actions to help protect against the threat of new variants.</p>

<p>As the threat of variants has emerged, the programme has played a key role in the detection of mutations of the virus, including those associated with known <abbr title="variants of concern">VOCs</abbr> and <abbr title="variants under investigation">VUIs</abbr>. This is achieved through genomic sequencing of wastewater samples, to provide an indication of where <abbr title="variants of concern">VOCs</abbr> and <abbr title="variants under investigation">VUIs</abbr> may be present across England. Findings are reported to national decision makers and local stakeholders on a regular basis, helping to inform national strategy and localised action. Wastewater analysis has the benefit of detecting the virus regardless of whether people have symptoms or whether they are tested. Wastewater monitoring complements other testing programmes and public health actions to help protect against the threat of new variants.</p>

<p>Wastewater variant detection data (Omicron variant only) included in the <a class="govuk-link" href="https://www.gov.uk/government/publications/investigation-of-sars-cov-2-variants-technical-briefings">Investigation of SARS-CoV-2 variants: technical briefings</a> is now available in this month’s publication as experimental statistics. We are exploring the feasibility of adding more variant detection data in future releases. We continue to develop our approach for identifying variants and mutations in wastewater. The data presented in this publication is generated by non-accredited research laboratories and should be considered experimental. Formal validation of wastewater monitoring as a variant surveillance system is underway. <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> advises caution when directly comparing data from <abbr title="sewage treatment works">STWs</abbr> and concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> in wastewater with clinical testing data. Wastewater detections of a variant are reported as either confirmed or possible, based on the number of signature or unique single nucleotide polymorphisms (<abbr title="single nucleotide polymorphisms">SNPs</abbr>), and co-occurrence on the same amplicon. This describes the level of confidence <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> has in the specificity of the detection; ‘confirmed’ status is assigned when there is a high degree of confidence the variant is present, ‘possible’ is assigned where there is some evidence that the variant is present. These definitions are different to the clinical case definitions and the 2 should not be compared. Lack of a positive sequencing result is not necessarily confirmation that no variant cases reside within a given geography, due to factors such as rainfall dilution impacting <abbr title="ribonucleic acid">RNA</abbr> recovery.</p>

<p>Interpretation of all data in this publication should be made with careful consideration of the points in the <a href="#uncertainty" class="govuk-link">Uncertainty, data quality and revisions section</a>.</p>

<h2>
<abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> sampling sites</h2>

<p><abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> samples at sewer network sites (manholes in the street which generally serve a local area) and <abbr title="sewage treatment works">STWs</abbr>  (which generally serve wider areas such as a city or town). Associated with every <abbr title="sewage treatment works">STWs</abbr> and sewer network site is a catchment, the geographical area from which wastewater flows into the sampling location. <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> concentrations data from sewer network sites is not included in this publication as these sites generally fall within <abbr title="sewage treatment work">STW</abbr> catchments. In addition, <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> changes sampling strategy and locations of sewer network sites to meet epidemiological priorities (for example to support local public health responses). Further detail about <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> wastewater coverage can be found in the publication <a class="govuk-link" href="https://www.gov.uk/government/publications/wastewater-testing-coverage-data-for-19-may-2021-emhp-programme/wastewater-testing-coverage-data-for-the-environmental-monitoring-for-health-protection-emhp-programme">Wastewater testing coverage data for the Environmental Monitoring for Health Protection (<abbr title="Environmental Monitoring for Health Protection">EMHP</abbr>) programme</a>.</p>

<p>Figures 1 to 9 show the location of <abbr title="sewage treatment works">STWs</abbr> that <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> samples at in each region.</p>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/140941/1_East_Midlands_STW.png"></div>
<figcaption><p>Figure 1. Map showing the location of STWs sampled in the East Midlands.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in the East Midlands:</p>

<ul>
  <li>Alfreton</li>
  <li>Anwick</li>
  <li>Boston</li>
  <li>Bourne</li>
  <li>Brackley</li>
  <li>Chesterfield</li>
  <li>Corby</li>
  <li>Daventry</li>
  <li>Derby</li>
  <li>Grantham</li>
  <li>Ingoldmells</li>
  <li>Leicester</li>
  <li>Lincoln</li>
  <li>Loughborough</li>
  <li>Louth</li>
  <li>Mablethorpe</li>
  <li>Market Harborough</li>
  <li>Mansfield</li>
  <li>Melton Mowbray</li>
  <li>Northampton</li>
  <li>Nottingham</li>
  <li>Oakham</li>
  <li>Retford</li>
  <li>Spalding</li>
  <li>Stamford</li>
  <li>Wellingborough</li>
  <li>Whaley Bridge</li>
  <li>Wigston</li>
  <li>Worksop</li>
</ul>

<figure class="image embedded"><div class="img"><img src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/140990/2_East_of_England_STW.png" alt=""></div>
<figcaption><p>Figure 2. Map showing the location of STWs sampled in the East of England.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in the East of England:</p>

<ul>
  <li>Basildon</li>
  <li>Beccles</li>
  <li>Bedford</li>
  <li>Braintree</li>
  <li>Breckland</li>
  <li>Bury St Edmunds</li>
  <li>Cambridge</li>
  <li>Chalton</li>
  <li>Chelmsford</li>
  <li>Colchester</li>
  <li>Cromer</li>
  <li>Diss</li>
  <li>Downham Market</li>
  <li>Felixstowe</li>
  <li>Great Yarmouth</li>
  <li>Harlow</li>
  <li>Haverhill</li>
  <li>Hunstanton</li>
  <li>Huntingdon</li>
  <li>Ipswich</li>
  <li>Jaywick</li>
  <li>Kings Lynn</li>
  <li>Letchworth</li>
  <li>Lowestoft</li>
  <li>Luton</li>
  <li>March</li>
  <li>Needham Market</li>
  <li>Newmarket</li>
  <li>Norwich</li>
  <li>Peterborough</li>
  <li>Radlett</li>
  <li>Reepham</li>
  <li>Royston</li>
  <li>Saffron Walden</li>
  <li>Shefford</li>
  <li>Soham</li>
  <li>St Albans</li>
  <li>Stalham</li>
  <li>Stowmarket</li>
  <li>Sudbury</li>
  <li>Southend-on-Sea</li>
  <li>Thetford</li>
  <li>Tilbury</li>
  <li>Wells-next-the-Sea</li>
  <li>Wisbech</li>
  <li>Witham</li>
</ul>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/140991/3_London_STW.png"></div>
<figcaption><p>Figure 3. Map showing the location of STWs sampled in London.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in London:</p>

<ul>
  <li>Beckton</li>
  <li>Beddington</li>
  <li>Crossness</li>
  <li>Deepham</li>
  <li>Hogsmill Valley</li>
  <li>Long Reach</li>
  <li>Mogden</li>
  <li>Riverside</li>
</ul>

<figure class="image embedded"><div class="img"><img src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141085/4-North_East.png" alt=""></div>
<figcaption><p>Figure 4. Map showing the location of STWs sampled in the North East.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in the North East:</p>

<ul>
  <li>Billingham</li>
  <li>Bishop Auckland</li>
  <li>Consett</li>
  <li>Cramlington</li>
  <li>Darlington</li>
  <li>Durham (Barkers Haugh)</li>
  <li>Durham (Belmont)</li>
  <li>Hartlepool</li>
  <li>Horden</li>
  <li>Houghton-le-Spring</li>
  <li>Middlesbrough</li>
  <li>Newcastle</li>
  <li>Newton Aycliffe</li>
  <li>Sunderland</li>
  <li>Washington</li>
</ul>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141086/5_North_West_STW.png"></div>
<figcaption><p>Figure 5. Map showing the location of STWs sampled in the North West.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in the North West:</p>

<ul>
  <li>Barnoldswick</li>
  <li>Barrow-in-Furness</li>
  <li>Blackburn</li>
  <li>Bolton</li>
  <li>Burnley</li>
  <li>Bury</li>
  <li>Carlisle</li>
  <li>Clitheroe</li>
  <li>Congleton</li>
  <li>Crewe</li>
  <li>Ellesmere Port</li>
  <li>Fleetwood</li>
  <li>Hyde</li>
  <li>Hyndburn</li>
  <li>Kendal</li>
  <li>Lancaster</li>
  <li>Leigh</li>
  <li>Liverpool (Sandon)</li>
  <li>Macclesfield</li>
  <li>Maghull</li>
  <li>Northwich</li>
  <li>Penrith</li>
  <li>Preston</li>
  <li>Rochdale</li>
  <li>Rossendale</li>
  <li>Skelmersdale</li>
  <li>Stockport</li>
  <li>Walton-Le-Dale</li>
  <li>Warrington</li>
  <li>Wigan</li>
  <li>Wirral</li>
  <li>Workington</li>
</ul>

<figure class="image embedded"><div class="img"><img src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141087/6_South_East_STW.png" alt=""></div>
<figcaption><p>Figure 6. Map showing the location of STWs sampled in the South East.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in the South East:</p>

<ul>
  <li>Andover</li>
  <li>Alton</li>
  <li>Ashford</li>
  <li>Aylesbury</li>
  <li>Banbury</li>
  <li>Basingstoke</li>
  <li>Bexhill</li>
  <li>Bicester</li>
  <li>Bordon</li>
  <li>Buckingham</li>
  <li>Burgess Hill</li>
  <li>Camberley</li>
  <li>Canterbury</li>
  <li>Chatham</li>
  <li>Chesham</li>
  <li>Chichester</li>
  <li>Crawley</li>
  <li>Didcot</li>
  <li>Dover and Folkestone</li>
  <li>Eastleigh</li>
  <li>Fareham and Gosport</li>
  <li>Guildford</li>
  <li>Hailsham</li>
  <li>Hythe</li>
  <li>Isle of Wight</li>
  <li>Lewes</li>
  <li>Littlehampton and Bognor</li>
  <li>Lymington and New Milton</li>
  <li>Maidstone and Aylesford</li>
  <li>Milton Keynes</li>
  <li>Newbury</li>
  <li>New Forest</li>
  <li>Oxford</li>
  <li>Portsmouth and Havant</li>
  <li>Ramsgate, Sandwich and Deal</li>
  <li>Reading</li>
  <li>Reigate</li>
  <li>Scaynes Hill</li>
  <li>Sittingbourne</li>
  <li>Slough</li>
  <li>Southampton</li>
  <li>Tonbridge</li>
  <li>Tunbridge Wells</li>
  <li>Witney</li>
  <li>Woking</li>
  <li>Worthing</li>
  <li>Wycombe</li>
</ul>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141088/7_South_West_STW.png"></div>
<figcaption><p>Figure 7. Map showing the location of STWs sampled in the South West.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in the South West:</p>

<ul>
  <li>Barnstaple</li>
  <li>Bath</li>
  <li>Bideford</li>
  <li>Blandford Forum</li>
  <li>Bodmin Sc Well</li>
  <li>Bournemouth (Central)</li>
  <li>Bridport</li>
  <li>Bristol</li>
  <li>Camborne</li>
  <li>Chard</li>
  <li>Cheltenham</li>
  <li>Chippenham</li>
  <li>Cirencester</li>
  <li>Clevedon and Nailsea</li>
  <li>Ernesettle and Saltash</li>
  <li>Exmouth</li>
  <li>Falmouth</li>
  <li>Gloucester</li>
  <li>Helston</li>
  <li>Liskeard</li>
  <li>Lydney</li>
  <li>Menagwins</li>
  <li>Minehead</li>
  <li>Newquay</li>
  <li>Newton Abbot</li>
  <li>Par</li>
  <li>Plymouth (Camels Head)</li>
  <li>Plymouth</li>
  <li>Plympton</li>
  <li>Salisbury</li>
  <li>Shaftesbury</li>
  <li>Shepton Mallet</li>
  <li>Sidmouth</li>
  <li>St Ives and Penzance</li>
  <li>Stroud</li>
  <li>Swanage</li>
  <li>Swindon</li>
  <li>Taunton</li>
  <li>Tavistock</li>
  <li>Tiverton</li>
  <li>Torquay</li>
  <li>Trowbridge</li>
  <li>Wellington</li>
  <li>Weston-super-Mare</li>
  <li>Weymouth</li>
  <li>Yeovil</li>
</ul>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141089/8_West_Midlands_STW.png"></div>
<figcaption><p>Figure 8. Map showing the location of STWs sampled in the West Midlands.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in the West Midlands:</p>

<ul>
  <li>Barston</li>
  <li>Brancote</li>
  <li>Checkley</li>
  <li>Birmingham (Coleshill)</li>
  <li>Birmingham (Minworth)</li>
  <li>Burton on Trent</li>
  <li>Coventry</li>
  <li>Evesham</li>
  <li>Kidderminster</li>
  <li>Leek</li>
  <li>Ludlow</li>
  <li>Malvern</li>
  <li>Market Drayton</li>
  <li>Nuneaton</li>
  <li>Oswestry</li>
  <li>Rugby</li>
  <li>Spernal</li>
  <li>Stoke-on-Trent</li>
  <li>Stourbridge and Halesowen</li>
  <li>Telford</li>
  <li>Telford South</li>
  <li>Walsall</li>
  <li>Warwick</li>
  <li>Wolverhampton</li>
  <li>Worcester</li>
</ul>

<figure class="image embedded"><div class="img"><img src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141090/9_Yorkshire_and_The_Humber_STW.png" alt=""></div>
<figcaption><p>Figure 9. Map showing the location of STWs sampled in Yorkshire and the Humber.</p></figcaption></figure>

<p>Location of <abbr title="sewage treatment works">STWs</abbr> sampled in Yorkshire and the Humber:</p>

<ul>
  <li>Barnsley</li>
  <li>Barton-upon-Humber</li>
  <li>Beverley</li>
  <li>Bradford</li>
  <li>Bridlington</li>
  <li>Colburn</li>
  <li>Dewsbury</li>
  <li>Doncaster (Thorne)</li>
  <li>Doncaster (Sandall)</li>
  <li>Driffield</li>
  <li>Grimsby</li>
  <li>Harrogate</li>
  <li>Huddersfield</li>
  <li>Hull</li>
  <li>Keighley</li>
  <li>Leeds</li>
  <li>Malton</li>
  <li>Northallerton</li>
  <li>Pontefract</li>
  <li>Scarborough</li>
  <li>Scunthorpe</li>
  <li>Sheffield (Blackburn Meadows)</li>
  <li>Sheffield (Woodhouse Mill)</li>
  <li>Wakefield</li>
  <li>York</li>
</ul>

<h2>Concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> in wastewater samples</h2>

<p><a href="https://www.gov.uk/government/publications/monitoring-of-sars-cov-2-rna-in-england-wastewater-monthly-statistics-1-june-2021-to-10-january-2022" class="govuk-link"><abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> wastewater concentration data from 1 June 2021 to 10 January 2022</a> is available in the accompanying spreadsheet. The spreadsheet includes data from previous months’ publications, along with the latest data from December and January. The most recent 6 weeks of data is visualised in figures 10 to 12; England concentration map. Note that <abbr title="sewage treatment works">STWs</abbr> are omitted from figures 10 to 12 if no samples were taken during the week.</p>

<p>Since September 2021, the concentrations of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> measured in wastewater appeared to diverge from reported clinical data. This trend is now not as pronounced as previously observed, nor is it as evident when considering the data over a longer timeframe. <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> is continuing to investigate the relationship between COVID-19 cases and wastewater and how this may be fluctuating over time. <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> advises caution when directly comparing data from <abbr title="sewage treatment works">STWs</abbr> and concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> in wastewater with clinical testing data. Interpretation should be made with careful consideration of the points in the <a href="#uncertainty" class="govuk-link">Uncertainty, data quality and revisions section</a>.</p>

<p>In the week beginning 30 November, 117 of 268 <abbr title="sewage treatment works">STWs</abbr> (44%) had a decrease in average concentration levels compared to the previous week. In the week beginning 7 December, 214 of 272 <abbr title="sewage treatment works">STWs</abbr> (79%) had higher average concentration than the preceding week. In the week beginning 14 December, 172 of 273 <abbr title="sewage treatment works">STWs</abbr> (63%) had a decrease in average concentration relative to the previous week. In the week beginning 21 December, there was limited wastewater sampling due to the Christmas holidays, therefore there is not enough data to assess changes in concentration levels over this period. In the week beginning 28 December 2021, 135 out of 232 <abbr title="sewage treatment works">STWs</abbr> (58%) showed an increase in average concentrations compared to week beginning 14 December 2021. In the week beginning 4 January 2022, 122 of 229 <abbr title="sewage treatment works">STWs</abbr> (53%) showed a decrease in average concentrations. Interpretation should be made with careful consideration of the points in the <a class="govuk-link" href="#uncertainty">Uncertainty, data quality and revisions section</a> below.</p>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141091/10_LOD_160_National_STW_Locations.png"></div>
<figcaption><p>Figure 10. England concentration map: map showing the location of STWs sampled. Colour indicates the weekly-average SARS-CoV-2 RNA concentration (gene copies/litre) in the last 2 weeks. See regional maps for the names of each STW.</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141092/11_LOD_160_National_STW_Locations.png"></div>
<figcaption><p>Figure 11. England concentration map: map showing the location of STWs sampled. Colour indicates the weekly-average SARS-CoV-2 RNA concentration (gene copies/litre) in the mid-2 weeks. See regional maps for the names of each STW.</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141093/12_LOD_160_National_STW_Locations.png" alt=""></div>
<figcaption><p>Figure 12. England concentration map: map showing the location of STWs sampled. Colour indicates the weekly-average SARS-CoV-2 RNA concentration (gene copies/litre) in the first 2 weeks. See regional maps for the names of each STW.</p></figcaption></figure>

<h2>Regional average concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> in wastewater</h2>

<p>Over the month of December, concentration levels in London were above the national average, corresponding to the spread of Omicron which was seen first in London. Concentration levels in the East Midlands were also higher than the national average for some of December while the other regions remained on or below the national average. Although there is only one week of data in January 2022, the concentration data shows signs of a downward trend in most regions. There is high variability in concentration between individual <abbr title="sewage treatment works">STWs</abbr>, nationally. Interpretation should be made with careful consideration of the points in the <a href="#uncertainty" class="govuk-link">Uncertainty, data quality and revisions section</a>.</p>

<p>Figures 13 to 21 show the 7-day rolling average SARS-CoV-2 concentration detected in wastewater for each region in England, together with the national average of SARS-CoV-2 concentration detected in wastewater and individual site data. Samples where no SARS-CoV-2 was detected are assigned the value 160 (gc/l) which is the Theoretical Limit of Detection. Data is weighted by catchment population to account for varying catchment sizes. For further information, see the <a class="govuk-link" href="#methodology">Methodology section</a> below.</p>

<p>Note: there was limited wastewater sampling over the Christmas period, therefore trends are not shown for the period 21 December 2021 to 28 December 2021.</p>

<figure class="image embedded"><div class="img"><img src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141094/13_East_Midlands.png" alt=""></div>
<figcaption><p>Figure 13. East Midlands 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141095/14_East_of_England.png" alt=""></div>
<figcaption><p>Figure 14. East of England 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141096/15_London.png"></div>
<figcaption><p>Figure 15. London 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141097/16_North_East.png" alt=""></div>
<figcaption><p>Figure 16. North East 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141098/17_North_West.png" alt=""></div>
<figcaption><p>Figure 17. North West 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141099/18_South_East.png"></div>
<figcaption><p>Figure 18. South East 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141100/19_South_West.png" alt=""></div>
<figcaption><p>Figure 19. South West 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141101/20_West_Midlands.png"></div>
<figcaption><p>Figure 20. West Midlands 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141102/21_Yorkshire_and_The_Humber.png"></div>
<figcaption><p>Figure 21. Yorkshire and the Humber 7-day rolling average SARS-CoV-2 RNA concentration</p></figcaption></figure>

<h2>Wastewater variant detection</h2>

<h3>Omicron detections 22 November 2021 to 2 January 2022</h3>

<p>Applying the Wastewater Variant Genomic Case Definition (see <a class="govuk-link" href="#methodology">Methodology section</a>), wastewater samples collected from sites across England from 22 November 2021 up to 2 January 2022 have been analysed for the presence of Omicron. <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> Omicron detection data is available in the <a class="govuk-link" href="https://www.gov.uk/government/publications/monitoring-of-sars-cov-2-rna-in-england-wastewater-monthly-statistics-1-june-2021-to-10-january-2022">accompanying spreadsheet</a>. The spreadsheet includes a table showing the number of sequenced samples by region and the number of confirmed and possible Omicron detections. Note that data presented here is subject to change as methods are further developed. Due to the additional time required to analyse wastewater detections versus concentrations, the latest detection data presented here is one week prior to the latest concentration data, there is a one-week lag on the data we are able to include in this publication.</p>

<p>From 22 November up to 2 January there were 1,674 confirmed or possible detections of Omicron in wastewater samples across England from approximately 5,900 sequenced for SARS-CoV-2 mutations. The first 5 Omicron detections in wastewater were identified week commencing 22 November 2021. Between 29 November and 5 December, 43 detections of Omicron in wastewater were reported, primarily in London and the South East. In the week commencing 6 December, Omicron was identified in 264 wastewater samples across all regions, a 6-fold increase from the previous week, aligning with the rapid growth of confirmed and probable Omicron cases in England. In the week commencing 13 December, there were 767 Omicron detections (51% of all samples sequenced) with the highest proportion of detections observed in London (78%), the South East (63%), the West Midlands (61%) and the East Midlands (61%). Conversely, Omicron was detected in less than 30% of samples sequenced in Yorkshire and the North East. In the week beginning 20 December, there was limited wastewater sampling due to the Christmas holidays, therefore there is not enough data to assess the spread of Omicron over this period. In the week beginning 27 December, the number of samples sequenced was lower than usual due to the Christmas holidays, but coverage was representative and Omicron was detected in 81% of samples sequenced. Omicron was detected in over 80% of all sequenced samples in all regions except the South West (77%) and the East of England (57%), although for both regions this was still an increase on the week commencing 13 December.</p>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141103/22_Omicron_National_map_Last_Two_Weeks.png"></div>
<figcaption><p>Figure 22. Confirmed and possible detections of Omicron VOC-21NOV-01 (B.1.1.529) in wastewater samples collected in England, data from 20 December 2021 to 29 December 2021.</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141104/23_Omicron_National_map_Mid_Two_Weeks.png"></div>
<figcaption><p>Figure 23. Confirmed and possible detections of Omicron VOC-21NOV-01 (B.1.1.529) in wastewater samples collected in England, data from 6 December to 19 December 2021.</p></figcaption></figure>

<figure class="image embedded"><div class="img"><img src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/141105/24_Omicron_National_map_First_Two_Weeks.png" alt=""></div>
<figcaption><p>Figure 24. Confirmed and possible detections of Omicron VOC-21NOV-01 (B.1.1.529) in wastewater samples collected in England, data from 22 November to 5 December 2021.</p></figcaption></figure>

<h2>Methodology</h2>

<h3>About this data</h3>

<h4>SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> concentration</h4>

<p><abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> samples wastewater from <abbr title="sewage treatment works">STWs</abbr> throughout England. Samples are analysed for the concentration of SARS-CoV-2 by quantifying the number of copies of the nucleocapsid gene (N1). Samples are transported to Environment Agency laboratory for quantification of the N1 gene using a process called Reverse Transcriptase Polymerase Chain Reaction. The data reported here is the number of SARS-CoV-2 N1 gene copies per litre of wastewater (shortened to gene copies/litre throughout this document). Generally, the more people with COVID-19 in the community the more viral <abbr title="ribonucleic acid">RNA</abbr> will be shed into wastewater. Therefore, the concentration in wastewater is indicative of the prevalence of COVID-19 in the community.</p>

<p>Samples are analysed for the concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> and concentration is adjusted for flow (see <a class="govuk-link" href="#uncertainty">Uncertainty, data quality and revisions</a>). Typically, 4 samples per week are analysed from each <abbr title="sewage treatment work">STW</abbr>. However, occasionally fewer samples will be collected and/or analysed. This can be due to several reasons such as <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> altering sampling strategy because of testing capacity constraints, changing epidemiological priorities and current local response activities.</p>

<p>The weekly concentration table and daily concentration table in the <a class="govuk-link" href="https://www.gov.uk/government/publications/monitoring-of-sars-cov-2-rna-in-england-wastewater-monthly-statistics-1-june-2021-to-10-january-2022">accompanying spreadsheet</a> show the concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> in wastewater samples (gene copies/litre). Note that there is sensitivity associated with the analysis of wastewater; the Limit of Detection (<abbr title="Limit of Detection">LOD</abbr>) is the minimum concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> that can be reliably detected. Data tables contain the values ‘<abbr title="Theoretical Limit of Detection">tLOD</abbr>’ where analysis of the sample did not detect presence of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr>. For the purposes of averaging data to produce weekly averages in the Weekly concentration table and figures 10 and 11, <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> assigns a concentration of 160 gene copies/litre to all ‘<abbr title="Theoretical Limit of Detection">tLOD</abbr>’ samples. This concentration is the Theoretical Limit of Detection and is calculated by the Environment Agency. Note that there is uncertainty in this calculation and continuing work to refine it; please see the <a class="govuk-link" href="#uncertainty">Uncertainty, data quality and revisions section</a> for further information.</p>

<p>To calculate region averages, data from each <abbr title="sewage treatment work">STW</abbr> was weighted by catchment population. To estimate the population of <abbr title="sewage treatment works">STWs</abbr>, the catchment areas of <abbr title="sewage treatment works">STWs</abbr> are matched to lower layer super output areas (LSOAs). Catchment areas were provided by the 9 water companies serving England. See the <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> publication <a href="https://www.gov.uk/government/publications/wastewater-testing-coverage-data-for-19-may-2021-emhp-programme/wastewater-testing-coverage-data-for-the-environmental-monitoring-for-health-protection-emhp-programme" class="govuk-link">Wastewater testing coverage data for the Environmental Monitoring for Health Protection (<abbr title="Environmental Monitoring for Health Protection">EMHP</abbr>) programme</a> for a detailed description of this method. Regional and national averages were calculated by taking the 7-day rolling arithmetic mean. Figures are displayed on a logarithmic scale for ease of interpretation.</p>

<p>We are exploring the feasibility of adding more data in future releases, such as concentration data from network sites.</p>

<h4>SARS-CoV-2 variant detection</h4>

<p>The wastewater routine analysis is to look for the presence of pre-defined sets of <abbr title="single nucleotide polymorphisms">SNPs</abbr> that identify known <abbr title="variants of concern">VOCs</abbr>, <abbr title="variants under investigation">VUIs</abbr>, and signals in monitoring. It is possible to look for mutations associated with variants in the wastewater, but detection of variants can be transient and the correlation between population prevalence and wastewater variant detection has not been established for Omicron. Wastewater monitoring remains under development and is currently considered as supplementary data in variant monitoring.</p>

<p>A proportion of samples are sent to Liverpool, Nottingham and Exeter Universities Sequencing facilities each week for genomic sequencing, results are analysed by <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> for the presence of known <abbr title="variants of concern">VOCs</abbr> and <abbr title="variants under investigation">VUIs</abbr> officially recognised by <abbr title="UK Health Security Agency">UKHSA</abbr>. Wastewater detections of a variant are reported as either confirmed or possible, based on the number of signature or unique <abbr title="single nucleotide polymorphisms">SNPs</abbr>, and co-occurrence on the same amplicon. This describes the level of confidence <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> has in the specificity of the detection; ‘confirmed’ status is assigned when there is a high degree of confidence the variant is present, ‘possible’ is assigned where there is some evidence that the variant is present. These definitions are different to the clinical case definitions and the 2 should not be compared.  Lack of a positive sequencing result is not necessarily confirmation that no variant cases reside within a given geography, due to factors such as rainfall dilution impacting <abbr title="ribonucleic acid">RNA</abbr> recovery. The wastewater variant genomic case definition used for detections in this publication is detailed below. Note that this definition is under review and subject to change.</p>

<h4>Wastewater variant genomic case definition</h4>

<h5>B.1.1.529 (VUI-21NOV-01) Omicron</h5>

<p>For the detection of Omicron VOC-21NOV-01 (B.1.1.529) in wastewater, the following definition is based on the detection of a number of <abbr title="single nucleotide polymorphisms">SNPs</abbr> from the list in the official <abbr title="UK Health Security Agency">UKHSA</abbr> definition. No distinction is made between <abbr title="single nucleotide polymorphisms">SNPs</abbr> which can be used to define wastewater detections unless otherwise stated.</p>

<p>Confirmed – ≥ 16 of 22 variant defining <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected, ≥ 9 of 20 unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected and co-occurrence detected on at least 7 OR If ≥ 16 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> and ≥ 9 unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> are present without co-occurrence, OR co-presence of <abbr title="single nucleotide polymorphisms">SNPs</abbr> at the following positions on amplicon 121 with co-occurrence: 22882, 22898, 22992, 23013, 23040, 23048 and 23055.</p>

<p>Possible – ≥ 10 of 22 variant defining <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected and ≥ 6 of 20 unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected OR If &lt; 10 signature <abbr title="single nucleotide polymorphisms">SNPs</abbr> and ≥ 6 unique <abbr title="single nucleotide polymorphisms">SNPs</abbr> are detected, but ≥ 9 <abbr title="single nucleotide polymorphisms">SNPs</abbr> are not covered possible presence can be assigned if those not covered are present across 2 dates from the same site, but in the same sequencing run.</p>

<p>Not detected – &lt; 7 of 22 variant defining <abbr title="single nucleotide polymorphisms">SNPs</abbr> detected unless co-presence of the following <abbr title="single nucleotide polymorphisms">SNPs</abbr> on amplicon 121: 22882, 22898, 22992, 23013, 23040, 23048 and 23055.</p>

<p>The Omicron detections table in the accompanying spreadsheet shows the number of sequenced samples per week in each region and the number of confirmed and possible Omicron detections. Note that this data is subject to change, as methods are improved or if data is reanalysed.</p>

<h3>Data sources</h3>

<p>Wastewater samples are collected by water companies and transported to an Environment Agency Laboratory. Concentration data is from Environment Agency analysis of wastewater samples. Sequencing data is from Liverpool, Nottingham and Exeter Universities Sequencing facilities. Variant detection’s data is from <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> Genomics Analysis.</p>

<p>Figures in this publication show the location of <abbr title="sewage treatment works">STWs</abbr>. These locations were obtained from the <a href="https://uwwtd.eu/United-Kingdom/treatment-plant/ukenthtwutp000014/2016" class="govuk-link" rel="external">European Commission urban wastewater website</a>. Note that names differ between those used in this document and the European Commission urban wastewater website. Sites can be linked using the ‘<abbr title="sewage treatment work">STW</abbr> Site Code’ in the attached data tables.</p>

<p>The figures and analysis presented will evolve over time to ensure the most relevant information is included and the needs of stakeholders are met. These figures will initially be published monthly, from June 2021, and we will continuously evaluate the frequency and date-range based on need and public interest.</p>

<h3>How these figures can be used</h3>

<p>These figures and data can be used to view the concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> in wastewater, which is indicative of prevalence in the community and an indication of where <abbr title="variants of concern">VOCs</abbr> and <abbr title="variants under investigation">VUIs</abbr> may be present across England. Due to factors relating to data uncertainty, <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> recommends careful interpretation of this data alongside other data sources such as Weekly Statistics for NHS Test and Trace (England).</p>

<h3>Uncertainty, data quality and revisions</h3>

<p>The figures here are compiled by professional analysts and have been quality assured. However, the analytical pipeline to produce this data should still be considered experimental; it is subject to an ongoing quality assurance process. Any revisions to past publications will be in line with the Department of Health and Social Care’s revision policy and highlighted in future publications accordingly.</p>

<h4>SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> concentration</h4>

<p>The sites from which <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> samples can change on a regular basis depending on testing capacity, epidemiological priorities, and current local response activities. This can result in changes to specific <abbr title="sewage treatment works">STWs</abbr> covered. This means the <abbr title="sewage treatment work">STW</abbr> included in subsequent publications may change.</p>

<p>This publication shows <abbr title="sewage treatment works">STWs</abbr> as single points. Every <abbr title="sewage treatment work">STW</abbr> has an associated geographical catchment area (area from which wastewater flows into the sampling location). These catchments are not yet presented in these statistics publications. There is uncertainty surrounding catchment geography, as catchments vary in size and population covered, and in some cases an <abbr title="sewage treatment work">STW</abbr> may not be located within the catchment it serves (for example, if wastewater is pumped to the <abbr title="sewage treatment work">STW</abbr>). Therefore, the location of <abbr title="sewage treatment works">STWs</abbr> should be used as a general guide only.</p>

<p>Wastewater is an inherently variable material and can change greatly within and between days (foodstuffs being consumed; when washing machines are switched on and many other factors), the physical infrastructure and the type and levels of industrial wastes. It is generally more useful to consider trends over medium- and long-term periods of time, rather than between individual days.</p>

<p>There are several factors that can impact the quality of the signal recorded from wastewater and introduce uncertainty.</p>

<ol>
  <li>The concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> in wastewater may be impacted by weather, for example heavy rainfall wastewater may cause dilution. <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> mitigates this by adjusting concentration to consider flow.</li>
  <li>Whether samples are ‘composite’ or ‘grab’. Composite samples are where an autosampler (a machine which gathers wastewater at regular intervals over a set time period) gathers wastewater throughout the day. Grab samples are where a single sample is taken at one point in the day and are subject to greater variability; they are influenced by flow characteristics, time of day, and are more susceptible to outliers in the sewage. <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> aims to mitigate this by sampling during times of peak load and ensuring the sample is collected mid-stream rather than close to the edges or bottom of the pipe.</li>
  <li>Calculating the Limit of Detection (<abbr title="Limit of Detection">LOD</abbr>; the minimum concentration of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> that can be reliably detected in a wastewater sample) can be achieved by several methods. In this data we have used the Theoretical <abbr title="Limit of Detection">LOD</abbr> as calculated by the Environment Agency. The Theoretical <abbr title="Limit of Detection">LOD</abbr> uses several assumptions. Laboratory data is required to inform the more realistic Practical <abbr title="Limit of Detection">LOD</abbr>. The Practical <abbr title="Limit of Detection">LOD</abbr> for this analysis is not yet reported here. It is likely that the <abbr title="Limit of Detection">LOD</abbr> used in producing these statistics will change in future publications. <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> will clearly state when this occurs.</li>
  <li>Although the exact value of the <abbr title="Limit of Detection">LOD</abbr> is uncertain, it is unlikely to have a noticeable impact on the interpretation of this data. This is because the difference between Theoretical and Practical <abbr title="Limit of Detection">LOD</abbr> is relatively small, with most impact at low levels of the target being measured.</li>
  <li>As explained in the <a href="#data" class="govuk-link">About this data section</a>, typically 4 wastewater samples are taken at each <abbr title="sewage treatment work">STW</abbr> per week. The average of all samples within the week is presented in the weekly concentration table. Uncertainty occurs where fewer samples are taken. For example, if only one sample is taken during a week, that one sample defines the week average. This means weeks with fewer samples are more susceptible to variability and outliers. The sample count table shows how many samples were taken at an <abbr title="sewage treatment work">STW</abbr> for each week in this release.</li>
</ol>

<p>For the above reasons, <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> advises caution when directly comparing data from <abbr title="sewage treatment works">STWs</abbr> and concentrations of SARS-CoV-2 <abbr title="ribonucleic acid">RNA</abbr> in wastewater with clinical testing data. Methodology and limitations will differ across data sources and this should be taken into account.</p>

<h4>SARS-CoV-2 variant detection</h4>

<p>The lack of a positive sequencing result is not confirmation that no variant cases reside within a given geography (given limitations of wastewater testing).</p>

<p>Environmental factors such as rainfall dilution can influence the detection of variants in wastewater. Therefore, low total detections on some days may not be reflective of low community infection levels.</p>

<p>Due to limitations of wastewater testing, <abbr title="Environmental Monitoring for Health Protection">EMHP</abbr> cannot estimate the number of cases associated with a detection. In small catchments it is possible, but unlikely, that the detection is due to a single individual. In larger catchments a detection is likely caused by a higher number of cases.</p>

<p><a class="govuk-link" href="#omicron">Figures 22 to 24</a> show the location of Omicron detections in <abbr title="sewage treatment works">STWs</abbr> and sewer networks as single points. Sewer networks tend to be relatively small geographical areas. Please note that the underlying characteristics of city sewer networks can be complex. For example, there may be varying flow from other areas into the catchment, and/or the exact catchment geography may be uncertain. With this in mind, maps are presented as a general guide to the location of a detection.</p>

<p>It is possible that a detection in a catchment is contributed to by case(s) who do not reside in the catchment; interpretation with other epidemiological indicators, such as known clinical cases, is encouraged.</p>

<p>We continue to develop our approach for identifying variants and mutations in wastewater. The data presented in this publication is generated by non-accredited research laboratories and should be considered experimental. Formal validation of wastewater monitoring as a variant surveillance system is underway.</p>

<p>If you would like to give user feedback, please email <a class="govuk-link" href="mailto:testandtrace.statistics@dhsc.gov.uk">testandtrace.statistics@dhsc.gov.uk</a></p>

</div>


</div>
</div>
  </div>

  <div data-sticky-element="" class="sticky-element">
    <a href="#contents" class="govuk-link app-c-back-to-top dont-print">
    <svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 13 17" width="13" height="17" class="app-c-back-to-top__icon">
      <path d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z" fill="currentColor"></path>
    </svg>
    Contents
</a>

    <div class="sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button data-module="print-link" class="govuk-link govuk-body-s gem-c-print-link__button">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>