<main lang="en" class="html-publication" role="main">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          
<div class="gem-c-organisation-logo brand--cabinet-office">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link brand__border-color" href="/government/organisations/scientific-advisory-group-for-emergencies">
      <span class="gem-c-organisation-logo__name">Scientific Advisory Group for Emergencies</span>
</a>
</div>
        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  

<div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
      <span class="govuk-caption-xl gem-c-title__context">
    Research and analysis
  </span>

  
  <h1 class="gem-c-title__text govuk-heading-xl">
    NERVTAG: Antiviral drug resistance and the use of directly acting antiviral drugs (DAAs) for COVID-19, 8 December 2021
  </h1>
</div>
  <p class="publication-header__last-changed">Published 8 December 2021</p>

  </header>



<div class="govuk-grid-row sidebar-with-body" data-module="sticky-element-container">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav class="gem-c-contents-list" aria-label="Contents" role="navigation" data-module="gem-track-click">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-options="{&quot;dimension29&quot;:&quot;\n1. Summary&quot;}" href="#summary" data-track-category="contentsClicked" data-track-action="content_item 1" data-track-label="#summary" class="gem-c-contents-list__link govuk-link govuk-link--no-underline"><span class="gem-c-contents-list__number">1. </span><span class="gem-c-contents-list__numbered-text">Summary</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-label="#recommendations" data-track-options="{&quot;dimension29&quot;:&quot;\n2. Recommendations&quot;}" href="#recommendations" data-track-category="contentsClicked" data-track-action="content_item 2"><span class="gem-c-contents-list__number">2. </span><span class="gem-c-contents-list__numbered-text">Recommendations</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a href="#background" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 3" data-track-label="#background" data-track-options="{&quot;dimension29&quot;:&quot;\n3. Background&quot;}" data-track-category="contentsClicked"><span class="gem-c-contents-list__number">3. </span><span class="gem-c-contents-list__numbered-text">Background</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a href="#current-covid-19-situation-and-direct-acting-antivirals" data-track-label="#current-covid-19-situation-and-direct-acting-antivirals" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-options="{&quot;dimension29&quot;:&quot;\n4. Current COVID-19 situation and direct acting antivirals&quot;}" data-track-category="contentsClicked" data-track-action="content_item 4"><span class="gem-c-contents-list__number">4. </span><span class="gem-c-contents-list__numbered-text">Current COVID-19 situation and direct acting antivirals</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-category="contentsClicked" data-track-options="{&quot;dimension29&quot;:&quot;\n5. Recommendations&quot;}" data-track-action="content_item 5" data-track-label="#recommendations-1" href="#recommendations-1"><span class="gem-c-contents-list__number">5. </span><span class="gem-c-contents-list__numbered-text">Recommendations</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-label="#supporting-references" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-options="{&quot;dimension29&quot;:&quot;\n6. Supporting references&quot;}" href="#supporting-references" data-track-category="contentsClicked" data-track-action="content_item 6"><span class="gem-c-contents-list__number">6. </span><span class="gem-c-contents-list__numbered-text">Supporting references</span></a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img src="/assets/government-frontend/open-government-licence-min-e98134ae65ff5fe99a524d8c383350b3998f91c298dc24b36126e65f1656c578.png" class="print-meta-data-licence">
</p>
<p>
  © Crown copyright 2021
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/nervtag-antiviral-drug-resistance-and-the-use-of-directly-acting-antiviral-drugs-daas-for-covid-19-8-december-2021/nervtag-antiviral-drug-resistance-and-the-use-of-directly-acting-antiviral-drugs-daas-for-covid-19-8-december-2021
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
    
    
      <div class="govspeak">
<p><strong>Main authors:</strong> Peter Horby; Wendy Barclay; Julian Hiscox; Meera Chand; Judith Breuer; Emma Sherwood; Andrew Owen.</p>

<p><strong>Contributors:</strong> Calum Semple; Ravindra Gupta; Ben Killingley; Wei Shen Lim; Catherine Moore; Peter Openshaw; Andrew Hayward; James Rubin and Maria Zambon.</p>

<p><strong>Date:</strong> 6 December 2021</p>

<h2>
<span class="number">1. </span>Summary</h2>

<p>1. Directly-acting antivirals (<abbr title="directly-acting antivirals">DAAs</abbr>) selectively target key stages of the viral life cycle to suppress or inhibit viral replication, thereby reducing the likelihood of severe disease, hospitalisation and death. <abbr title="directly-acting antivirals">DAAs</abbr> include complex biologicals such as monoclonal antibodies and small molecule drugs such as remdesivir.</p>

<p>2. Resistance to antiviral drugs arises when spontaneous mutations resulting from imprecise viral genomic replication (or genetic recombination) confer a survival advantage for that virus variant in the presence of an antiviral drug.</p>

<p>3. Drug resistant viral variants may, however, incur a ‘fitness’ cost, meaning that they are not able to efficiently replicate and transmit. In vitro, in vivo, and clinical studies are required to assess the clinical and public health implications of virus variants with a resistant genotype.</p>

<p>4. Resistance risk is increased by (i) increased use of antiviral drugs (increased virus-drug exposure), (ii) monotherapy, (iii) inadequate dosing, (iv) antiviral use in prolonged infections (for example in the immunocompromised).</p>

<p>5. Antiviral drugs will have the greatest effect (and the greatest potential for resistance to emerge) at the time of maximum viral replication, which in <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> occurs early in infection.</p>

<p>6. Two ‘biological’ <abbr title="directly-acting antivirals">DAAs</abbr> against <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> are currently approved for use in the <abbr title="United Kingdom">UK</abbr> – ronapreve and sotrovimab.</p>

<p>7. Three small molecule <abbr title="directly-acting antivirals">DAAs</abbr> against <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> are currently either in use or close to use. Two nucleoside analogues; remdesivir (licensed) and molnupiravir (under conditional <abbr title="Medicines and Healthcare products Regulatory Agency">MHRA</abbr> approval). PF-00835231, Paxlovid (under review) a protease inhibitor.</p>

<p>8. The emergence of new <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> variants such as Omicron (B.1.1.529) with the potential to (partially or fully) evade protection from vaccines and monoclonal antibodies increases the potential clinical and public health importance of small molecule <abbr title="directly-acting antivirals">DAAs</abbr>. Omicron may remove some possible combination therapy options (such as small molecule plus monoclonal). Assuming small molecule <abbr title="directly-acting antivirals">DAAs</abbr> retain activity against Omicron, it is important to preserve this activity, namely avoid resistance.</p>

<p>9. <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> resistance to monoclonal antibodies is a significant problem, but very little evidence has been published on the potential for <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> to develop resistance to small molecule antiviral agents. No studies or clinical trial data investigating resistance to molnupiravir or Paxlovid are available. Resistance has been reported for other coronaviruses to some <abbr title="directly-acting antivirals">DAAs</abbr>, however the resistance phenotype impaired viral fitness.</p>

<p>10. Different antivirals have different susceptibilities to resistance development depending on their target and their pharmacokinetics.</p>

<p>11. In other viral infections, resistance to nucleoside analogue monotherapies emerges extremely readily whereas protease inhibitors are considered to be more durable. However, no data are yet available to establish if this is the case for <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr>.</p>

<h2>
<span class="number">2. </span>Recommendations</h2>

<p>12. Although the propensity of current small molecule <abbr title="directly-acting antivirals">DAAs</abbr> to develop <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> resistance of clinical and public health significance is unknown, efforts to avoid, detect and mitigate resistance to these <abbr title="directly-acting antivirals">DAAs</abbr> should be a high priority.</p>

<p>13. <abbr title="directly-acting antiviral">DAA</abbr> dosing and duration should consider the need to minimise the risk of resistance. These are the same principles used for other antimicrobial therapies. It may be appropriate to provide monitoring and support of adherence.</p>

<p>14. Combination anti-viral therapy is, in theory, preferable to monotherapy in terms of a reduced risk of resistance and, possibly, improved clinical efficacy. The theoretical risk of promoting resistance is higher in patients who are immunocompromised, especially with inadequate therapy. There is, therefore, a pressing need to assess the safety, efficacy, and resistance potential of combinations of <abbr title="directly-acting antivirals">DAAs</abbr> (and other) drugs.</p>

<p>15. From a regulatory and business perspective, consideration should be given around how trials that aim to combine drugs can be conducted and how combinations can be licensed.</p>

<p>16. Viral load and resistance monitoring, particularly in immunocompromised patients, will be needed to rapidly detect, evaluate, and mitigate emerging resistance.</p>

<p>17. Methods and procedures should be established to quickly assess the “fitness” and the clinical and public health relevance of any emerging antiviral resistant viral variants.</p>

<p>18. Prospective work is needed to establish which mutations can confer resistance on all potential <abbr title="directly-acting antivirals">DAAs</abbr>, whether they carry a fitness cost for replication and transmission, and whether they affect disease. There is also a need to determine whether secondary mutations can subsequently restore fitness.</p>

<p>19. Ongoing work is needed to establish whether newly emerging variants, such as Omicron, differ in their propensity to evolve resistance because of other unrelated mutations in their genomes and or their increased replication rate (if confirmed).</p>

<h2>
<span class="number">3. </span>Background</h2>

<p>20. Antiviral drugs can be classified as either host-directed (for example <abbr title="angiotensin-converting enzyme 2">ACE2</abbr> receptor blockade) or directly-acting antivirals (<abbr title="directly-acting antivirals">DAAs</abbr>) depending upon their mechanism of action. <abbr title="directly-acting antivirals">DAAs</abbr> directly target parts of the virus replication cycle and are considered to be more potent as antivirals than host-directed drugs.</p>

<p>21. <abbr title="directly-acting antivirals">DAAs</abbr> include complex biologicals such as monoclonal antibodies and small molecule drugs such as remdesivir. They work by inhibiting or suppressing viral replication, leading to more rapid clearance of virus and decreased likelihood of immunopathological complications that result in severe disease, hospitalisation and or death.</p>

<p>22. For <abbr title="directly-acting antivirals">DAAs</abbr>, it is important to find viral targets which can be selectively inhibited, so as to minimise the impact on human proteins that conduct mechanistically similar processes. The latter carries increased risk of adverse drug reactions. Typically, key stages of the life cycle that have been proven to be successful for other viruses include host cell entry, copying of the viral genome, maturation of the virus or viral budding from the host cell. Key target proteins for <abbr title="ribonucleic acid">RNA</abbr> viruses (not exhaustive), and examples (not exhaustive) for <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> are shown in the table below.</p>

<table>
  <thead>
    <tr>
      <td></td>
      <th scope="col">Replication stage</th>
      <th scope="col">Example targets</th>
      <th scope="col">Medicines (in bold = MHRA approval)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>1</td>
      <td>Viral attachment to host cell</td>
      <td>Spike protein: monoclonal antibodies (<abbr title="monoclonal antibodies">MAbs</abbr>). Transmembrane serine protease 2 (<abbr title="transmembrane serine protease 2">TMPRSS2</abbr>)</td>
      <td>
<strong>Ronapreve, sotrovimab</strong> , bamlanivimab + etesevimab. Nafamostat.</td>
    </tr>
    <tr>
      <td>2</td>
      <td>Viral membrane fusion and endocytosis</td>
      <td>&nbsp;</td>
      <td>-</td>
    </tr>
    <tr>
      <td>3</td>
      <td>Copying of the <abbr title="ribonucleic acid">RNA</abbr> genome</td>
      <td>
<abbr title="ribonucleic acid">RNA</abbr>-dependent <abbr title="ribonucleic acid">RNA</abbr> polymerase (<abbr title="RNA-dependent RNA polymerase">RdRP</abbr>)</td>
      <td>
<strong>Nucleoside analogues</strong>. Chain terminators: <strong>Remdesivir.</strong> Lethal mutagenesis: Favipiravir and <strong>molnupiravir.</strong>
</td>
    </tr>
    <tr>
      <td>4</td>
      <td>Viral maturation</td>
      <td>Protease (3CL or main protease (<abbr title="main protease">Mpro</abbr>)</td>
      <td>PF-00835231, Paxlovid</td>
    </tr>
    <tr>
      <td>5</td>
      <td>Assembly</td>
      <td>&nbsp;</td>
      <td>-</td>
    </tr>
    <tr>
      <td>6</td>
      <td>Virus particle release</td>
      <td>&nbsp;</td>
      <td>-</td>
    </tr>
  </tbody>
</table>

<p>23. Antiviral drugs will have the greatest effect (and the greatest potential for resistance to emerge) at the time of maximum viral replication, which in <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> occurs early in infection.</p>

<p>24. Viruses have a short replication cycle and billions of virus offspring are generated during a single infection. This provides huge opportunities for the random generation of virus variants through replication errors (for example, point mutations, insertions, deletions or duplications). In addition, recombination is a hallmark trait of this genus, meaning that sections of genome are readily exchanged, resulting in insertion or deletion of tracts of <abbr title="ribonucleic acid">RNA</abbr>.</p>

<p>25. Resistance to antiviral drugs arises spontaneously through inherent variation in the genome sequence that occurs because of imprecise replication or recombination. When an antiviral drug is present, virions with chance genetic variation that confers a survival advantage against the drug are placed under a positive selective pressure. The resistance risk increases with increased numbers of infections and increased use of antiviral drugs.</p>

<p>26. The presence of resistance mutations may not, however, necessarily have clinical or public health implications. When a resistant variant is selected there can sometimes be a fitness cost to the virus meaning that the resistant variant may be disabled or in the absence of drug may replicate at a slower rate than its non-resistant counterparts. Therefore, it is important to assess the clinical and public health relevance of resistant viruses through in vitro, in vivo and clinical studies.</p>

<p>27. However, a lower replication rate may subsequently generate a selective pressure for accessory mutations in different sequences that restore fitness to the resistant variant.</p>

<p>28. If resistant virions can replicate efficiently and can be transmitted from one individual to another and if antiviral drugs are widely used, the selective pressure amplifies the number of resistant variants within the population as a whole.</p>

<p>29. Some general principles of antiviral drug resistance are:</p>

<p>a. <em><strong>Variable versus conserved targets.</strong></em> Some antiviral drug targets are inherently more variable than others, either because the targets are themselves under a selection pressure (for example the Spike glycoprotein is under immune selection pressure) or there is limited capacity for change in the target without rendering the virus non-viable. As such, different antivirals can have very different susceptibility to resistance development.</p>

<p>b. <em><strong>Suboptimal dosage or pharmacokinetics</strong></em> results in drug concentrations that are unable to fully inhibit replication and or mean that concentrations fall below an adequate level between doses. At high drug concentrations, all viruses are prevented from replication and there is no opportunity for the random generation (and then selection) of resistant variants. At very low doses the selective pressure is also very low, so resistance is unlikely. However, there may be a middle ground where the virus is able to replicate and the drug concentration is sufficient to exert significant selective pressure. Currently available <abbr title="directly-acting antivirals">DAAs</abbr> have low potency compared to those that are robustly efficacious in other <abbr title="ribonucleic acid">RNA</abbr> infections meaning that higher concentrations need to be sustained, which is much harder to achieve without impractically high doses and or toxicity. [Note that Paxlovid is boosted with ritonavir, presenting potential drug interaction challenges.]</p>

<p>c. <em><strong>Treatment adherence.</strong></em> Suboptimal adherence to the medication (such as failure to complete a course) also increases the time during which the virus can replicate in suboptimal concentrations of drug and therefore the resistance risk.</p>

<p>d. <em><strong>Monotherapy versus combination therapy.</strong></em> If drugs are deployed as monotherapies, as few as one mutation is required in order for a resistant variant to emerge. When drug combinations are deployed, multiple mutations in different regions of the viral genome are required to arise in unison to simultaneously confer resistance to all drugs in the regimen. If resistance emerges to one drug the other drug(s) inhibit its replication and vice versa. The probability of the spontaneous emergence of multiple key mutations within the same viral genome by chance is much lower than the probability of one key mutation arising. Combination therapy therefore provides a much higher barrier to the emergence of resistance than monotherapy.</p>

<p>e. In addition, two drugs may act synergistically and therefore combination therapy can be more clinically effective, and at lower drug doses than required with monotherapy.</p>

<p>f. For chronic viral infections, antiviral resistance is a major problem and combination therapy for <abbr title="ribonucleic acid">RNA</abbr> viruses that cause chronic and persistent infections including <abbr title="human immunodeficiency virus">HIV</abbr> and <abbr title="hepatitis C virus">HCV</abbr> is the evidence-based first line treatment globally to stem emergence of resistance.</p>

<p>g. The evidence is less clear in short-lived (acute) viral infections. There is mixed evidence of the benefit from combination antiviral therapy for influenza in terms of clinical outcome or emergence of resistance. <sup role="doc-noteref"><a href="#fn:1" rel="footnote" class="govuk-link">[footnote 1]</a></sup> Oseltamivir is used widely as monotherapy in influenza and resistance is rare and has not become established, whereas the amantadine antiviral drugs are now no longer viable due to widespread resistance. Although nucleoside analogues are not routinely used to treat influenza, in vitro studies show that resistance to favipiravir can be selected and the mutations confer little or no fitness cost. <sup role="doc-noteref"><a class="govuk-link" href="#fn:2" rel="footnote">[footnote 2]</a></sup></p>

<p>h. <em><strong>Combination therapies can take three different approaches.</strong></em></p>

<p>i. Small molecules targeting different parts of the replication pathway for example molnupiravir (nucleoside analogue) and Paxlovid (protease inhibitor).</p>

<p>ii. Different modalities targeting different parts of the replication pathway for example small molecule drugs (molnupiravir or paxlovid) with monoclonal antibodies (ronapreve, sotrovimab).</p>

<p>iii. Multiple drugs targeting the same part of the replication pathway in different ways for example molnupiravir (<strong>pyrimidine</strong> nucleoside analogue) and remdesivir (<strong>purine</strong> nucleoside analogue).</p>

<p>i. <em><strong>Prolonged infection in immunocompromised people.</strong></em> Immunosuppressed people are prone to prolonged infection because of their reduced ability to control virus replication. In the absence of an effective host antiviral response, it is harder to control viral replication with antiviral medicines alone and short course antivirals may be stopped before virus is fully eradicated. This can promote resistance in several ways. (i). There may be prolonged exposure to antiviral drugs whilst there is ongoing replication, increasing the opportunities for the generation and selection of resistant variants. (ii) Prolonged replication may also provide an opportunity for the development of mutations that compensate for any loss of fitness in resistant viruses. (iii) Whilst the probability of simultaneous acquisition of two mutations conferring resistance to two different targets is very small, the probability of acquiring these sequentially is much higher (particularly for drugs with very different pharmacokinetic half-lives such as monoclonals and small molecules). Therefore, prolonged viral replication also increases the chances of multi-drug resistance.</p>

<p>j. <em><strong>Class-wide resistance.</strong></em> For other viruses, like <abbr title="human immunodeficiency virus">HIV</abbr>, class-wide resistance (<abbr title="class-wide resistance">CWR</abbr>) has been a significant challenge. <abbr title="class-wide resistance">CWR</abbr> arises when resistance to one drug confers resistance to other drugs in the same class (for example resistance to one nucleoside conferring resistance to other nucleosides). The <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> polymerase and protease will undoubtedly remain high value small molecule drug targets in future antiviral development and the avoidance of <abbr title="class-wide resistance">CWR</abbr> therefore warrants careful consideration.</p>

<p>k. Once resistance has emerged and become common within the population any subsequent attempts to combine that drug with another drug may be futile.</p>

<h2>
<span class="number">4. </span>Current <abbr title="coronavirus">COVID-19</abbr> situation and direct acting antivirals</h2>

<p>30. At the time of writing, the emergence of new variants such as Omicron (B.1.1.529) with the potential to partially or fully evade immune protection from monoclonal antibodies may mean an increased importance of the role of small molecule <abbr title="directly-acting antivirals">DAAs</abbr>, assuming they retain activity.</p>

<p>31. One <abbr title="directly-acting antiviral">DAA</abbr> (remdesivir) is licensed, one <abbr title="directly-acting antiviral">DAA</abbr> (molnupiravir) has a conditional approval from <abbr title="Medicines and Healthcare products Regulatory Agency">MHRA</abbr>, and one <abbr title="directly-acting antiviral">DAA</abbr> (Paxlovid) shows promise as therapies to inhibit <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> replication.</p>

<p>32. Remdesivir is a nucleoside analogue that stalls <abbr title="ribonucleic acid">RNA</abbr> assembly by the <abbr title="ribonucleic acid">RNA</abbr>-dependent <abbr title="ribonucleic acid">RNA</abbr> polymerase through chain termination.</p>

<p>33. Molnupiravir is a nucleoside analogue that when incorporated into the replicated chain of viral <abbr title="ribonucleic acid">RNA</abbr> induces errors in subsequent copies which drive the virus towards ‘error catastrophe’ through lethal mutagenesis. The molecular event which is liable to resistance is the same for molnupiravir as it is for remdesvir (nucleoside incorporation by the polymerase). Molnupiravir is incorporated in place of pyrimidine nucleosides while remdesivir is incorporated in place of purine nucleosides.</p>

<p>34. PF-00835231, Paxlovid, targets the viral 3C protease (non-structural protein 5) which is necessary for the virus to mature into infectious virus able to infect new cells.</p>

<p>35. In <abbr title="human immunodeficiency virus">HIV</abbr>, resistance to first generation nucleoside analogues <sup role="doc-noteref"><a rel="footnote" class="govuk-link" href="#fn:3">[footnote 3]</a></sup> <sup role="doc-noteref"><a href="#fn:4" class="govuk-link" rel="footnote">[footnote 4]</a></sup> <sup role="doc-noteref"><a rel="footnote" href="#fn:5" class="govuk-link">[footnote 5]</a></sup> and protease inhibitors  <sup role="doc-noteref"><a class="govuk-link" rel="footnote" href="#fn:6">[footnote 6]</a></sup> <sup role="doc-noteref"><a href="#fn:7" class="govuk-link" rel="footnote">[footnote 7]</a></sup> readily emerged when drugs like zidovudine, saquinavir or ritonavir were given as monotherapies. Subsequent development focused upon</p>

<p>a) ritonavir-boosting of protease inhibitors to improve the pharmacokinetics,</p>

<p>b) optimising activity of next generation protease inhibitors against known protease inhibitor-resistant strains of virus, and</p>

<p>c) combining protease inhibitors with two nucleoside analogue drugs. As a result, protease inhibitor-based triple drug combinations are considered to have a high genetic barrier to drug resistance and became a mainstay of “salvage therapy” in patients with drug resistance. Insufficient data are currently available to take an informed view on whether there is a higher resistance liability in <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> for nucleoside analogues (monupiravir or remdesivir) or the boosted protease inhibitor (paxlovid).</p>

<p>36. Very little evidence has been published on the potential of <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> resistance to antiviral agents. However, <em>in-vitro</em> research with remdesivir demonstrated selection of an escape mutant with a specific amino acid substitution in the inhibitor targeted protein. Furthermore, a very recent preprint <sup role="doc-noteref"><a href="#fn:8" class="govuk-link" rel="footnote">[footnote 8]</a></sup> described the emergence of this same mutation within an immunocompromised patient receiving remdesivir, and there have been two reports <sup role="doc-noteref"><a href="#fn:9" class="govuk-link" rel="footnote">[footnote 9]</a></sup> <sup role="doc-noteref"><a rel="footnote" href="#fn:10" class="govuk-link">[footnote 10]</a></sup> of two remdesivir treatment courses being required to achieve a virological response, viral clearance and clinical resolution in two patients with compromised immune systems.</p>

<p>37. No studies or clinical trial data investigating resistance to molnupiravir or Paxlovid are available. The impact of the remdesivir escape mutation on the activity of molnupiravir also has not been described.</p>

<p>38. Resistance to monotherapies with some monoclonal antibodies has been consistently demonstrated in vitro and in clinical trials. The primary reason that monoclonal antibody therapies have been developed as dual therapy is to reduce the resistance risk associated with monotherapy.</p>

<p>39. Given the paucity of available research regarding antiviral resistance to <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr>, research from similar antivirals and viruses can help inform evidence. Resistance has been reported for other coronaviruses to viral 3C protease inhibitors and <abbr title="ribonucleic acid">RNA</abbr> dependent <abbr title="ribonucleic acid">RNA</abbr> polymerase inhibitors, however the resistance phenotype impaired viral fitness in vitro and attenuated virulence in in vivo models. <sup role="doc-noteref"><a class="govuk-link" rel="footnote" href="#fn:11">[footnote 11]</a></sup></p>

<p>40. Paxlovid has been shown in vitro to have additive or synergistic effects with remdesivir <sup role="doc-noteref"><a href="#fn:12" class="govuk-link" rel="footnote">[footnote 12]</a></sup> so the combination with molnupiravir is also likely to show at least additive effect. A combination of favipiravir and molnupiravir was demonstrated to be more potent in Syrian golden hamsters than either drug alone. <sup role="doc-noteref"><a rel="footnote" class="govuk-link" href="#fn:13">[footnote 13]</a></sup> Early-stage experimental data has indicated several antiviral combinations with therapeutic potential in severe influenza <sup role="doc-noteref"><a class="govuk-link" rel="footnote" href="#fn:14">[footnote 14]</a></sup> , but there remains a need to assess efficacy in clinical trials.</p>

<h2>
<span class="number">5. </span>Recommendations</h2>

<p>41. Although the propensity of current <abbr title="directly-acting antivirals">DAAs</abbr> to develop <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> resistance of clinical and public health significance is unknown, efforts to avoid, detect and mitigate resistance to these <abbr title="directly-acting antivirals">DAAs</abbr> should be a high priority.</p>

<p>42. <abbr title="directly-acting antiviral">DAA</abbr> dosing and duration should consider the need to minimise the risk of resistance. These are the same principles used for other antimicrobial therapies. It may be appropriate to provide monitoring and support of adherence.</p>

<p>43. Combination anti-viral therapy is, in theory, preferable to monotherapy in terms of a reduced risk of resistance and, possibly, improved clinical efficacy. The theoretical risk of promoting resistance is higher in patients who are immunocompromised, especially with inadequate therapy. There is, therefore, a pressing need to assess the safety, efficacy, and resistance potential of combinations of <abbr title="directly-acting antivirals">DAAs</abbr> (and other) drugs.</p>

<p>44. From a regulatory and business perspective, consideration should be given around how trials that aim to combine drugs can be conducted and how combinations can be licensed.</p>

<p>45. Viral load and resistance monitoring, particularly in immunocompromised patients, will be needed to rapidly detect, evaluate, and mitigate emerging resistance.</p>

<p>46. Methods and procedures should be established to quickly assess the “fitness” and the clinical and public health relevance of any emerging antiviral resistant viral variants.</p>

<p>47. Prospective work is needed to establish which mutations can confer resistance on all potential <abbr title="directly-acting antivirals">DAAs</abbr>, whether they carry a fitness cost for replication and transmission, and whether they affect disease. There is also a need to determine whether secondary mutations can subsequently restore fitness.</p>

<p>48. Further work is needed to establish whether newly emerging variants, such as Omicron, differ in their propensity to evolve resistance because of other unrelated mutations in their genomes and or their increased replication rate (if confirmed).</p>

<h2>
<span class="number">6. </span>Supporting references:</h2>

<div role="doc-endnotes" class="footnotes">
  <ol>
    <li role="doc-endnote">
      <p>Lampejo T. Influenza and antiviral resistance: an overview. Eur J Clin Microbiol Infect Dis. 2020;39(7):1201–8.&nbsp;<a aria-label="go to where this is referenced" role="doc-backlink" class="govuk-link" href="#fnref:1">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Goldhill DH, Te Velthuis AJW, Fletcher RA, Zambon M, Lackenby A, Barclay WS, et al. The mechanism of resistance to favipiravir in influenza. Proc Natl Acad Sci U S A. 2018 Nov 6;115(45):11613-11618.&nbsp;<a href="#fnref:2" class="govuk-link" role="doc-backlink" aria-label="go to where this is referenced">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Larder BA, Kemp SD. Multiple mutations in <abbr title="human immunodeficiency virus">HIV</abbr>-1 reverse transcriptase confer high-level resistance to zidovudine (AZT). Science. 1989 Dec 1;246(4934):1155-8.&nbsp;<a role="doc-backlink" aria-label="go to where this is referenced" class="govuk-link" href="#fnref:3">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Kellam P, Boucher CA, Larder BA. Fifth mutation in human immunodeficiency virus type 1 reverse transcriptase contributes to the development of high-level resistance to zidovudine, Proc Natl Acad Sci U S A. 1992 Mar 1;89(5):1934-8.&nbsp;<a aria-label="go to where this is referenced" href="#fnref:4" class="govuk-link" role="doc-backlink">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Boucher CA, O’Sullivan E, Mulder JW, Ramautarsing C, Kellam P, Larder BA, Kempf DJ. et al. Ordered appearance of zidovudine resistance mutations during treatment of 18 human immunodeficiency virus-positive subjects. J Infect Dis. 1992 Jan;165(1):105-10.&nbsp;<a aria-label="go to where this is referenced" role="doc-backlink" class="govuk-link" href="#fnref:5">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Roberts NA. Drug-resistance patterns of saquinavir and other <abbr title="human immunodeficiency virus">HIV</abbr> proteinase inhibitors, AIDS, 1995 Dec;9 Suppl 2:27-S32.&nbsp;<a role="doc-backlink" href="#fnref:6" class="govuk-link" aria-label="go to where this is referenced">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Molla A, Korneyeva M, Gao Q, Vasavanonda S, Schipper PJ, et al. Ordered accumulation of mutations in <abbr title="human immunodeficiency virus">HIV</abbr> protease confers resistance to ritonavir. Nat Med. 1996 Jul;2(7):760-6.&nbsp;<a href="#fnref:7" class="govuk-link" role="doc-backlink" aria-label="go to where this is referenced">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Gandhi S, Klein J, Robertson A, Peña-Hernández MA, Lin MJ, Roychoudhury P, et al. De novo emergence of a remdesivir resistance mutation during treatment of persistent <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> infection in an immunocompromised patient: A case report. medRxiv. 2021 Jan 1;2021.11.08.21266069.&nbsp;<a href="#fnref:8" aria-label="go to where this is referenced" role="doc-backlink" class="govuk-link">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Malsy J, Veletzky L, Heide J, Hennigs A, Gil-Ibanez I, Stein A, et al. Sustained response after remdesivir and convalescent plasma therapy in a B-cell depleted patient with protracted <abbr title="coronavirus">COVID-19</abbr>. Clin Infect Dis an Off Publ Infect Dis Soc Am. 2020 Oct 26; 6:ciaa1637.&nbsp;<a role="doc-backlink" href="#fnref:9" aria-label="go to where this is referenced" class="govuk-link">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Helleberg M, Niemann CU, Moestrup KS, Kirk O, Lebech A-M, Lane C, et al. Persistent <abbr title="coronavirus">COVID-19</abbr> in an Immunocompromised Patient Temporarily Responsive to Two Courses of Remdesivir Therapy. J Infect Dis. 2020 Sep;222(7):1103–7.&nbsp;<a class="govuk-link" href="#fnref:10" role="doc-backlink" aria-label="go to where this is referenced">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Artese A, Svicher V, Costa G, Salpini R, Di Maio VC, Alkhatib M, et al. Current status of antivirals and druggable targets of SARS CoV-2 and other human pathogenic coronaviruses. Drug Resist Updat Rev Comment Antimicrob Anticancer Chemother. 2020 Dec;53:100721.&nbsp;<a aria-label="go to where this is referenced" href="#fnref:11" role="doc-backlink" class="govuk-link">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Boras B, Jones RM, Anson BJ, Arenson D, Aschenbrenner L, Bakowski MA, et al. Preclinical characterization of an intravenous coronavirus 3CL protease inhibitor for the potential treatment of COVID19. Nat Commun. 2021;12(1):6055.&nbsp;<a aria-label="go to where this is referenced" href="#fnref:12" class="govuk-link" role="doc-backlink">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Abdelnabi R, Foo CS, Kaptein SJF, Zhang X, Do TND, Langendries L, et al. The combined treatment of Molnupiravir and Favipiravir results in a potentiation of antiviral efficacy in a <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> hamster infection model. EBioMedicine. 2021 Oct 1;72.&nbsp;<a href="#fnref:13" aria-label="go to where this is referenced" class="govuk-link" role="doc-backlink">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Dunning J, Baillie JK, Cao B, Hayden FG. Antiviral combinations for severe influenza. Lancet Infect Dis. 2014 Dec;14(12):1259–70.&nbsp;<a href="#fnref:14" aria-label="go to where this is referenced" role="doc-backlink" class="govuk-link">↩</a></p>
    </li>
  </ol>
</div>
</div>


</div>
</div>
  </div>

  <div data-sticky-element="" class="sticky-element">
    <a class="govuk-link app-c-back-to-top dont-print" href="#contents">
    <svg focusable="false" width="13" class="app-c-back-to-top__icon" height="17" viewBox="0 0 13 17" xmlns="http://www.w3.org/2000/svg">
      <path fill="currentColor" d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z"></path>
    </svg>
    Contents
</a>

    <div class="sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button data-module="print-link" class="govuk-link govuk-body-s gem-c-print-link__button">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>