<main class="html-publication" role="main" lang="en">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          <div class="gem-c-organisation-logo brand--department-of-health">
    <a href="/government/organisations/department-of-health-and-social-care" class="gem-c-organisation-logo__container gem-c-organisation-logo__link gem-c-organisation-logo__crest gem-c-organisation-logo__crest--single-identity brand__border-color">
      <span class="gem-c-organisation-logo__name">Department<br>of Health &amp;<br>Social Care</span>
</a>
</div>

        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  <div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
    <span class="govuk-caption-xl gem-c-title__context">
      Research and analysis
    </span>
  <h1 class="gem-c-title__text ">
    Evaluation of endpoint PCR (EPCR) as a central laboratory based diagnostic test technology for SARS-CoV-2
  </h1>
</div>
  <p class="publication-header__last-changed">Published 28 January 2021</p>

  </header>




<div class="govuk-grid-row sidebar-with-body" data-module="sticky-element-container">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav data-module="track-click" role="navigation" class="gem-c-contents-list  gem-c-contents-list--no-underline " aria-label="Contents">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 1" data-track-category="contentsClicked" data-track-label="#executive-summary" data-track-options="{&quot;dimension29&quot;:&quot;Executive summary&quot;}" href="#executive-summary" class="gem-c-contents-list__link govuk-link ">Executive summary</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-options="{&quot;dimension29&quot;:&quot;\n1.  Background&quot;}" href="#background" class="gem-c-contents-list__link govuk-link " data-track-category="contentsClicked" data-track-action="content_item 2" data-track-label="#background"><span class="gem-c-contents-list__number">1. </span><span class="gem-c-contents-list__numbered-text">Background</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-category="contentsClicked" data-track-action="content_item 3" class="gem-c-contents-list__link govuk-link " data-track-label="#technical-validation" data-track-options="{&quot;dimension29&quot;:&quot;\n2.  Technical validation&quot;}" href="#technical-validation"><span class="gem-c-contents-list__number">2. </span><span class="gem-c-contents-list__numbered-text">Technical validation</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-label="#diagnostic-sensitivity-and-specificity" data-track-action="content_item 4" data-track-options="{&quot;dimension29&quot;:&quot;\n3.  Diagnostic sensitivity and specificity&quot;}" href="#diagnostic-sensitivity-and-specificity" class="gem-c-contents-list__link govuk-link " data-track-category="contentsClicked"><span class="gem-c-contents-list__number">3. </span><span class="gem-c-contents-list__numbered-text">Diagnostic sensitivity and specificity</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-category="contentsClicked" data-track-options="{&quot;dimension29&quot;:&quot;\n4.  Verification of central lab technical failure rates in process validation studies&quot;}" data-track-action="content_item 5" class="gem-c-contents-list__link govuk-link " href="#verification-of-central-lab-technical-failure-rates-in-process-validation-studies" data-track-label="#verification-of-central-lab-technical-failure-rates-in-process-validation-studies"><span class="gem-c-contents-list__number">4. </span><span class="gem-c-contents-list__numbered-text">Verification of central lab technical failure rates in process validation studies</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a class="gem-c-contents-list__link govuk-link " data-track-options="{&quot;dimension29&quot;:&quot;\n5.  Clinical reporting algorithm&quot;}" href="#clinical-reporting-algorithm" data-track-category="contentsClicked" data-track-label="#clinical-reporting-algorithm" data-track-action="content_item 6"><span class="gem-c-contents-list__number">5. </span><span class="gem-c-contents-list__numbered-text">Clinical reporting algorithm</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a href="#workforce-requirement" class="gem-c-contents-list__link govuk-link " data-track-category="contentsClicked" data-track-action="content_item 7" data-track-label="#workforce-requirement" data-track-options="{&quot;dimension29&quot;:&quot;\n6.  Workforce requirement&quot;}"><span class="gem-c-contents-list__number">6. </span><span class="gem-c-contents-list__numbered-text">Workforce requirement</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-options="{&quot;dimension29&quot;:&quot;\n7.  Conclusions&quot;}" class="gem-c-contents-list__link govuk-link " data-track-action="content_item 8" data-track-label="#conclusions" href="#conclusions" data-track-category="contentsClicked"><span class="gem-c-contents-list__number">7. </span><span class="gem-c-contents-list__numbered-text">Conclusions</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-category="contentsClicked" data-track-options="{&quot;dimension29&quot;:&quot;\n8.  Additional data and references&quot;}" data-track-label="#additional-data-and-references" data-track-action="content_item 9" class="gem-c-contents-list__link govuk-link " href="#additional-data-and-references"><span class="gem-c-contents-list__number">8. </span><span class="gem-c-contents-list__numbered-text">Additional data and references</span></a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-label="#authors" data-track-action="content_item 10" data-track-category="contentsClicked" href="#authors" class="gem-c-contents-list__link govuk-link " data-track-options="{&quot;dimension29&quot;:&quot;\n9.  Authors&quot;}"><span class="gem-c-contents-list__number">9. </span><span class="gem-c-contents-list__numbered-text">Authors</span></a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img class="print-meta-data-licence" src="/assets/government-frontend/open-government-licence-min-e98134ae65ff5fe99a524d8c383350b3998f91c298dc24b36126e65f1656c578.png">
</p>
<p>
  Â© Crown copyright 2021
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/evaluation-of-endpoint-pcr-epcr-as-a-diagnostic-test-technology-for-sars-cov-2/evaluation-of-endpoint-pcr-epcr-as-a-central-laboratory-based-diagnostic-test-technology-for-sars-cov-2
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
    
    
      <div class="govspeak">
<h2>Executive summary</h2>

<p>The UltraDX SARS-CoV-2 N1/N2/RP assay (LGC, Biosearch Technologies) was evaluated according to the published <a href="https://www.gov.uk/government/publications/assessment-and-procurement-of-coronavirus-covid-19-tests" class="govuk-link">Technical Validation Group protocol</a>.</p>

<p>UK BioCentre Ltd (<abbr title="UK BioCentre Ltd">UKBC</abbr>) evaluated <abbr title="endpoint PCR">EPCR</abbr> molecular testing technologies, which are designed as high-sensitivity, ultra-high capacity (more than 150,000 tests per day, per instrument) diagnostic assays to detect SARS-CoV-2 RNA extracted from combined throat and nose swab specimens.</p>

<p>The <abbr title="endpoint PCR">EPCR</abbr> assay was assessed on combined throat and nose swab samples received by NHS Test and Trace at the Milton Keynes Lighthouse Laboratory. Over 65,000 specimens were tested over multiple pilot-stage trials and subsequent deployment readiness studies.</p>

<p>The results of these studies were that <abbr title="endpoint PCR">EPCR</abbr> testing on combined throat and nose swabs, with RNA extraction, had a sensitivity of 98.56% (<abbr title="confidence interval">CI</abbr> 0.976% to 0.992%) and specificity of 99.8% (<abbr title="confidence interval">CI</abbr> 0.997 to .998) across all samples tested (<abbr title="cycle threshold">CT</abbr> less than 40 by real-time <abbr title="Polymerase chain reaction">PCR</abbr> comparator). Concordance testing showed near total concordance (Cohenâs Kappa 0.94) with existing Taqpath real time <abbr title="Polymerase chain reaction">PCR</abbr>. Analytical sensitivity of <abbr title="endpoint PCR">EPCR</abbr> is 50 digital copies / ml which is slightly better than standard of care Taqpath real-time <abbr title="Polymerase chain reaction">PCR</abbr>. This demonstrates that <abbr title="endpoint PCR">EPCR</abbr> technology gives viral detection with excellent sensitivity and specificity, offering the option to use it as an alternative to real-time <abbr title="Polymerase chain reaction">PCR</abbr>.</p>

<p><abbr title="endpoint PCR">EPCR</abbr> offers highly scalable assay technology; this can be used to expand the national test programme capability through development of ultra-high-throughput diagnostic laboratories.  <abbr title="UK BioCentre Ltd">UKBC</abbr> has now established an expanded facility ready for routine ultra-high-throughput testing (greater than 125,000 <abbr title="endpoint PCR">EPCR</abbr> tests per day).</p>

<h2>
<span class="number">1. </span> Background</h2>

<p>1.1 While central lab testing using gold-standard real-time <abbr title="Polymerase chain reaction">PCR</abbr> assays has been essential to help mitigate and manage the SARS-CoV-2 (COVID-19) pandemic, growing overall central lab test capacity to meet our substantial community needs require new, highly scalable test technologies that do not sacrifice high test quality.</p>

<p>1.2 Assay batch size and unit-based test manufacturing limits real-time <abbr title="Polymerase chain reaction">PCR</abbr> capacity: standard laboratory consumables and instruments are anchored around single-unit 96- or 384-well assay plate configurations created for basic research applications. Additionally, quantitative real-time <abbr title="Polymerase chain reaction">PCR</abbr> assay methods use time-intensive and complex assay monitoring approaches to derive simple âpositiveâ or ânegativeâ qualitative outcomes for patients. Viral detection through real-time <abbr title="Polymerase chain reaction">PCR</abbr> testing estimates relative viral RNA quantity by periodically measuring the assay reaction in real time.</p>

<p>1.3. <abbr title="endpoint PCR">EPCR</abbr> offers a simple but efficient means to boost <abbr title="Polymerase chain reaction">PCR</abbr> test capacity. The technology incorporates rapid sample testing into a large batch-size, continuous manufacturing process (greater than or equal to 15,000 samples per run), and takes a single, highly sensitive measurement of the <abbr title="Polymerase chain reaction">PCR</abbr> assay end-products, to speed viral RNA detection to less than 1.5 hours per each assay run time. Through modest re-configurations of the same underlying test technology as real-time <abbr title="Polymerase chain reaction">PCR</abbr> testing, <abbr title="endpoint PCR">EPCR</abbr> can maintain high test performance while greatly expanding test capacity. Two <abbr title="endpoint PCR">EPCR</abbr> instrument systems are designed to have greater capacity than 60 to 70 real-time <abbr title="Polymerase chain reaction">PCR</abbr> instruments running on a continuous basis.</p>

<p>1.4. This report outlines the technical validation undertaken in conjunction with the NHS Test and Trace Technical Validation Group (TVG). The studies were conducted using samples obtained directly through the NHS Test and Trace Lighthouse laboratory network campaign, and therefore provide real-world evidence from ongoing community mass-testing efforts. With an intent to utilise <abbr title="endpoint PCR">EPCR</abbr> technology within the National Testing Programme, UBKC has recently sought diagnostic use approval under a derogation application with the Medicines and Healthcare products Regulatory Agency (<abbr title="Medicines and Healthcare products Regulatory Agency">MHRA</abbr>). <abbr title="UK BioCentre Ltd">UKBC</abbr>'s application to provide an <abbr title="endpoint PCR">EPCR</abbr> mass-test offering was built from a series of pilot readiness and deployment studies run at the facility, and the company intends to offer greater than 125,000 <abbr title="endpoint PCR">EPCR</abbr> tests per day.</p>

<h2>
<span class="number">2. </span> Technical validation</h2>

<p>2.1. The Technical and Validation Group was established under NHS Test and Trace. It includes NHS and PHE experts and working closely with <abbr title="Medicines and Healthcare products Regulatory Agency">MHRA</abbr> and research bodies. The Technical and Validation function considers manufacturers of SARS-CoV-2 (COVID-19) tests for viral detection and registers their interest in the national procurement process if their test meets, or is intended to meet, the requirements of the relevant <abbr title="Medicines and Healthcare products Regulatory Agency">MHRA</abbr> target product profiles. The Technical and Validation function reviews product information; undertakes technical and clinical validation; and establishes and/or works with service evaluation projects and pilots.</p>

<h3>Assay description and intended purpose</h3>

<p>2.2 Endpoint polymerase chain reaction (<abbr title="endpoint PCR">EPCR</abbr>) technology is a high-processivity <abbr title="Polymerase chain reaction">PCR</abbr> assay alternative to diagnostic testing using standard real-time <abbr title="Polymerase chain reaction">PCR</abbr> instruments. <abbr title="endpoint PCR">EPCR</abbr> can be configured for reverse-transcription and <abbr title="Polymerase chain reaction">PCR</abbr> assays that detect viral RNA in human specimens. <abbr title="endpoint PCR">EPCR</abbr> instruments rapidly formulate large numbers of <abbr title="Polymerase chain reaction">PCR</abbr> reactions into reaction vessel âarraysâ using micro-jet dispense technology, which also further miniaturises assay volumes below those typical for 96- or 384-well plate tests. <abbr title="endpoint PCR">EPCR</abbr> thermocycling and assay detection can be scaled up to batch sizes exceeding 15,000 diagnostic tests per 1.5 hour end-to-end process run, while providing sensitive qualitative diagnostic determinations of viral gene presence (similar to real-time <abbr title="Polymerase chain reaction">PCR</abbr>), according the total fluorescence generated upon <abbr title="Polymerase chain reaction">PCR</abbr> assay completion.</p>

<p>2.3 An <abbr title="endpoint PCR">EPCR</abbr> assay specific for SARS-CoV-2, called UltraDx SARS-CoV-2 N1/N2/RP assay (LGC, Biosearch Technologies), contains 2, CDC-characterised amplicon primer sets with 2 separate probes using same fluorescence-based (FAM dye) detection of the viral N-Gene RNA segment in standard RT-<abbr title="Polymerase chain reaction">PCR</abbr> reaction conditions. Input RNA extracted from swab specimens is combined with RT-<abbr title="Polymerase chain reaction">PCR</abbr> reaction mixes (in NEXAR), and then reaction arrays for up 15,000 test assays can be amplified using a water-bath tank thermocycler (Hydrocycler). <abbr title="endpoint PCR">EPCR</abbr> assays are run to completion following 50 cycles of <abbr title="Polymerase chain reaction">PCR</abbr>, and the reactions are then measured by a single scan on a multi-channel fluorescence reader (Araya) âa rolling spool feeds the entire assay set through the array detector during measurement. A multiplex RNase P primer set is utilised as an internal extraction process control, and a passive reference monitoring dye (ROX) is added to each reaction as a dispense and assay interference process control.</p>

<p>2.4 The UltraDx SARS-CoV-2 N1/N2/RP assay is a test for the detection of SARS-CoV-2 in clinical specimens (combined throat and mouth swabs). The assay is intended for use by professionals trained in laboratory settings in the detection of SARS-CoV-2 RNA in association with the LGCâs Biosearch Technologies SARS-CoV-2 <abbr title="Polymerase chain reaction">PCR</abbr>-based testing systems.</p>

<p>2.5 All samples were collected in community surveillance and epidemiologic monitoring settings through pillar 2 of the NHS Test and Trace programme. The <abbr title="endpoint PCR">EPCR</abbr> validation was performed on combined throat and mouth) swabs. Swabs were collected in viral transport medium (various manufacturers, previously evaluated and validated in the pillar 2 programme).</p>

<p>2.6 The specific equipment required to perform testing on the UltraDx SARS-CoV-2 N1/N2/RP assay was supplied by <abbr title="UK BioCentre Ltd">UKBC</abbr> (the <abbr title="Medicines and Healthcare products Regulatory Agency">MHRA</abbr> derogation sponsor, and supply manufacturer for approved use in the UK prior to CE-IVD authorisation). Laboratory consumables, including RNA extraction kits already validated in standard real-time <abbr title="Polymerase chain reaction">PCR</abbr> workstreams (Thermo MagMax platform), were also supplied by <abbr title="UK BioCentre Ltd">UKBC</abbr>.</p>

<h3>Performance characteristics</h3>

<p>2.7 Analytical Sensitivity of SARS COV-2 targets</p>

<p>The analytical sensitivity (ASe) for the UltraDx SARS-CoV-2 N1/N2/RP assay was evaluated using RNA extracted from a panel of inactivated virus inputs ranging from 10<sup>6</sup>/ml to 10<sup>1</sup>/ml (Qnostics Analytical Q Panel, Product Number: SCV2AQP). Samples were extracted using standard RNA extraction procedures with Thermo MagMax reagents and consumables, and RNA inputs were assayed in parallel by real-time <abbr title="Polymerase chain reaction">PCR</abbr> using the TaqPathâ¢ COVIDâ19 CEâIVD RTâ<abbr title="Polymerase chain reaction">PCR</abbr> Kit (Thermo) (called Taqpath real-time <abbr title="Polymerase chain reaction">PCR</abbr> in this document) as a comparator. Similar to the manufacturer's instructions for use, real-time <abbr title="Polymerase chain reaction">PCR</abbr> detected to 10<sup>2</sup> copies/ml (500 copies/ml).</p>

<p>Analytical sensitivity of <abbr title="endpoint PCR">EPCR</abbr> is 10<sup>1</sup> copies / ml (50 digital copies / ml) (see table 1 below).</p>

<h3>Table 1: analytical range performance of <abbr title="endpoint PCR">EPCR</abbr> on inactivated whole virus</h3>

<table>
  <thead>
    <tr>
      <th scope="col">Concentration (digital copies/ml)</th>
      <th scope="col">Positive control specimen replicate 1</th>
      <th scope="col">Positive control specimen  RNA extraction replicate 2</th>
      <th scope="col">Positive control specimen  RNA extraction replicate 3</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><strong>1,000,000</strong></td>
      <td>Positive</td>
      <td>Positive</td>
      <td>Positive</td>
    </tr>
    <tr>
      <td><strong>100,000</strong></td>
      <td>Positive</td>
      <td>Positive</td>
      <td>Positive</td>
    </tr>
    <tr>
      <td><strong>10,000</strong></td>
      <td>Positive</td>
      <td>Positive</td>
      <td>Positive</td>
    </tr>
    <tr>
      <td><strong>5,000</strong></td>
      <td>Positive</td>
      <td>Positive</td>
      <td>Positive</td>
    </tr>
    <tr>
      <td><strong>1,000</strong></td>
      <td>Positive</td>
      <td>Positive</td>
      <td>Positive</td>
    </tr>
    <tr>
      <td><strong>500</strong></td>
      <td>Positive</td>
      <td>Positive</td>
      <td>Positive</td>
    </tr>
    <tr>
      <td><strong>100</strong></td>
      <td>Positive</td>
      <td>Positive</td>
      <td>Positive</td>
    </tr>
    <tr>
      <td><strong>50</strong></td>
      <td>Positive</td>
      <td>Positive</td>
      <td>Positive</td>
    </tr>
    <tr>
      <td><strong>Negative</strong></td>
      <td>Negative</td>
      <td>Negative</td>
      <td>Negative</td>
    </tr>
  </tbody>
</table>

<p>Table 1 criteria for acceptance: in at least 2 of 3 independent extraction control specimens, confirmation of positive detection to at least 50 digital copies per ml. Samples were collected and tests were performed between 26 September and 28 September 2020.</p>

<h3>
<span class="number">2.9 </span> Precision and robustness</h3>

<p>Repeatability was measured by running was measured by testing 188 replicates of an RNA extracted from whole inactivated virus control at 500 copies / ml (Qnostics). Samples were run over 3 separate days, with a minimum of 36 samples a day, assayed on at least 2 separate instrument assay runs. Total <abbr title="Polymerase chain reaction">PCR</abbr> assay signal at endpoint (normalised N gene fluorescence) showed a coefficient of variation of 16.4% across all three days. response signal appeared digital, with a rate of qualitative failure to detect the positive control less than 6% on any day, and 4.10% across all 3 days.</p>

<h3>Table 2: repeatability</h3>

<table>
  <thead>
    <tr>
      <th scope="col">Factor</th>
      <th scope="col">Day 1</th>
      <th scope="col">Day 2</th>
      <th scope="col">Day 3</th>
      <th scope="col">All Days</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><strong>All specimens</strong></td>
      <td>36</td>
      <td>56</td>
      <td>96</td>
      <td>188</td>
    </tr>
    <tr>
      <td><strong>Normalised N1/N2 endpoint fluorescence (arbitrary units)</strong></td>
      <td>10.44</td>
      <td>10.74</td>
      <td>10.63</td>
      <td>10.62</td>
    </tr>
    <tr>
      <td><strong>Standard deviation</strong></td>
      <td>1.69</td>
      <td>0.95</td>
      <td>2.07</td>
      <td>1.73</td>
    </tr>
    <tr>
      <td><strong>Coefficient of variation</strong></td>
      <td>16.2%</td>
      <td>8.9%</td>
      <td>19.5%</td>
      <td>16.3%</td>
    </tr>
    <tr>
      <td><strong>Control specimens below detected threshold (<abbr title="fluorescence unit">FU</abbr> \&lt;8)</strong></td>
      <td>2</td>
      <td>2</td>
      <td>4</td>
      <td>8</td>
    </tr>
    <tr>
      <td><strong>Rate of positive control failure</strong></td>
      <td>5.26%</td>
      <td>3.45%</td>
      <td>4.00%</td>
      <td>4.08%</td>
    </tr>
  </tbody>
</table>

<p>Table 2 criteria for acceptance: less than 5% test failure over all days. Samples were collected and tests were performed on 26 September and 28 September 2020.</p>

<p>Inter-operator precision was assessed by assaying 7,068 RNA specimens obtained under routine swab testing operations and provided to 2 independent operator teams in a blinded fashion. Test agreement between the 2 teams was 96.74%.</p>

<h3>Table 3: inter-operator precision</h3>

<table>
  <thead>
    <tr>
      <th scope="col">Staff team 1</th>
      <th scope="col">Staff team 2</th>
      <th scope="col">Numbers</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Positive</td>
      <td>Positive</td>
      <td>605</td>
    </tr>
    <tr>
      <td>Positive</td>
      <td>Negative</td>
      <td>32</td>
    </tr>
    <tr>
      <td>Negative</td>
      <td>Positive</td>
      <td>77</td>
    </tr>
    <tr>
      <td>Negative</td>
      <td>Negative</td>
      <td>4,973</td>
    </tr>
    <tr>
      <td>Negative</td>
      <td>Void</td>
      <td>32</td>
    </tr>
    <tr>
      <td>Void</td>
      <td>Positive</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Void</td>
      <td>Negative</td>
      <td>0</td>
    </tr>
    <tr>
      <td>Void</td>
      <td>Void</td>
      <td>19</td>
    </tr>
  </tbody>
</table>

<p>Table 3 criteria for acceptance: inter-operator precision. at least 95% test agreement between independent operator teams. Samples were collected and tests were performed on 10 December and 11 December 2020.</p>

<h3>
<span class="number">2.10 </span> Analytical specificity (cross-reactions)</h3>

<p>The analytical specificity of the panel of sequences used in the SARS-CoV-2 endpoint <abbr title="Polymerase chain reaction">PCR</abbr> diagnostic testing (<abbr title="endpoint PCR">EPCR</abbr> test) was assessed for cross-reactivity in an external publication from the CDC.
No cross-reactivity is reported for a wide range of viral respiratory pathogens including other corona and influenza viruses. (See table 4 below).</p>

<h3>Table 4: cross-reactivity of the US CDC  rtPCR panel for detection of SARS-CoV-2 against other respiratory pathogens</h3>
<p>Reproduced from Lu X, Wang L, Sakthivel SK, Whitaker B, Murray J, Kamili S, et al.,
<a class="govuk-link" rel="external" href="https://dx.doi.org/10.3201/eid2608.201246">US CDC Real-Time Reverse Transcription <abbr title="Polymerase chain reaction">PCR</abbr> Panel for Detection of Severe Acute Respiratory Syndrome Coronavirus 2</a>, (Emerg Infect Dis. 2020;26(8):1654 to 1665).</p>

<table>
  <thead>
    <tr>
      <th scope="col">Pathogen (strain)</th>
      <th scope="col">Source</th>
      <th scope="col">Other pathogens  rtPCR (Ct)</th>
      <th scope="col">N1 SARS-CoV-2  rtPCR</th>
      <th scope="col">N2 SARS-CoV-2  rtPCR</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Adenovirus C1 (Ad.71)</td>
      <td>Virus isolate</td>
      <td>Pos (14.0)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Bocavirus</td>
      <td>Clinical specimen</td>
      <td>Pos (14.9)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Coronavirus 229E</td>
      <td>Virus isolate</td>
      <td>Pos (9.6)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Coronavirus OC43</td>
      <td>Virus isolate</td>
      <td>Pos (12.9)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Coronavirus HKU1</td>
      <td>Clinical specimen</td>
      <td>Pos (22.3)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Coronavirus MERS</td>
      <td>Virus isolate</td>
      <td>Pos (14.3)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Coronavirus NL63</td>
      <td>Clinical specimen</td>
      <td>Pos (21.9)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Coronavirus SARS (Urbani)</td>
      <td>Virus isolate</td>
      <td>Pos (27.3)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Enterovirus D68</td>
      <td>Virus isolate</td>
      <td>Pos (21.3)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Human metapneumovirus (CAN 99â81)</td>
      <td>Virus isolate</td>
      <td>Pos (13.8)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Influenza A H1N1 (A/India/2012)</td>
      <td>Virus isolate</td>
      <td>Pos (14.7)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Influenza B (B/Massachusetts/1999)</td>
      <td>Virus isolate</td>
      <td>Pos (8.4)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Influenza B (B/Massachusetts/1999)</td>
      <td>Virus isolate</td>
      <td>Pos (8.4)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Parainfluenza 1 (C35)</td>
      <td>Virus isolate</td>
      <td>Pos (17.2)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Parainfluenza 2 (Greer)</td>
      <td>Virus isolate</td>
      <td>Pos (17.1)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Parainfluenza 3 (C-43)</td>
      <td>Virus isolate</td>
      <td>Pos (20.4)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Parainfluenza 4a (M-25)</td>
      <td>Virus isolate</td>
      <td>Pos (16.7)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Parainfluenza 4b (CH 19503)</td>
      <td>Virus isolate</td>
      <td>Pos (18.2)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Respiratory syncytial virus (Long)</td>
      <td>Virus isolate</td>
      <td>Pos (15.1)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td>Rhinovirus 1A</td>
      <td>Virus isolate</td>
      <td>Pos (15.9)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td><em>Mycoplasma pneumoniae</em></td>
      <td>Cultured bacteria</td>
      <td>Pos (20.7)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
    <tr>
      <td><em>Streptococcus pneumoniae</em></td>
      <td>Cultured bacteria</td>
      <td>Pos (21.1)</td>
      <td>Neg</td>
      <td>Neg</td>
    </tr>
  </tbody>
</table>

<h2>
<span class="number">3. </span> Diagnostic sensitivity and specificity</h2>

<h3>Clinical validation with confirmed positives and negatives</h3>

<p>3.1 Incoming combined throat and nose swab samples (n= 39,986), excluding voids, from the Milton Keynes Lighthouse Laboratory were tested and analysed for test method validation</p>

<p>Validation study specimens were selected randomly during routine daily processing at the Milton Keynes Lighthouse Laboratory and are therefore fully representative of the range of clinical sample diversity.</p>

<p>3.2 Diagnostic sensitivity: confirmed clinical samples from patients (positive real-time <abbr title="Polymerase chain reaction">PCR</abbr> result) were compared across platforms. The qualitative diagnostic determination for both the assessed and comparator assays were included in the submitted validation data. Cycle threshold (real-time <abbr title="Polymerase chain reaction">PCR</abbr>) and endpoint fluorescence <abbr title="endpoint PCR">EPCR</abbr> data were supplied.</p>

<ul>
  <li>906 positive swabs as determined by real-time <abbr title="Polymerase chain reaction">PCR</abbr> were analysed by <abbr title="endpoint PCR">EPCR</abbr> inclusive of voids</li>
</ul>

<p>3.3 Diagnostic assay performance: confirmed clinical samples from patients (negative Real-time <abbr title="Polymerase chain reaction">PCR</abbr> result) were used. The <abbr title="cycle threshold">CT</abbr> values or equivalent for both the assessed and comparator assays were included in the submitted validation data.</p>

<ul>
  <li>39,157 negative swabs as determined by real-time <abbr title="Polymerase chain reaction">PCR</abbr> were analysed by <abbr title="endpoint PCR">EPCR</abbr> inclusive of voids</li>
</ul>

<p>3.4 The diagnostic sensitivity (DSe) and specificity (DSp) derived from section 3.2 and 3.3 are as follows (see table 5 below). DSe and DSp (95% Confidence Intervals (<abbr title="confidence interval">CI</abbr>)) for all samples are summarised below:</p>

<p>â¢	<abbr title="endpoint PCR">EPCR</abbr> on swabs: DSe 98.6% (<abbr title="confidence interval">CI</abbr> 0.976-0.992); DSp 99.8% (<abbr title="confidence interval">CI</abbr> 0.997-0.998).</p>

<p>The calculation of diagnostic sensitivity assumes TaqPath<sup>TM</sup> COVID-19 CE-IVD RT-<abbr title="Polymerase chain reaction">PCR</abbr> Kit is the most-sensitive standard index test.</p>

<h4>Cohenâs Kappa</h4>

<p>3.5 Cohen's kappa was 0.94, which passed acceptance criteria set at minimum of 0.8 (above a level of good to excellent). This is near-total concordance.</p>

<p>Cohenâs kappa assesses predictive performance corrected joint test trial size: measures in excess of 0.8 suggest testing agreement performance should be robust to higher sample capacity and life-time diagnostic run performance. Alongside confidence level boundaries for sensitivity and specificity, Cohen's measure indicates that test trial size and random effects did not materially impact our assessments.</p>

<h3>Table 5: concordance testing summary (post- analytical QC filtering)</h3>

<table>
  <thead>
    <tr>
      <th scope="col">EPCR (UltraDx SARS-CoV-2 N1/N2/RP assay)</th>
      <th scope="col">Taqpath real-time PCR positive</th>
      <th scope="col">Taqpath real-time PCR negative</th>
      <th scope="col">Taqpath real-time PCR void</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><strong>Positive</strong></td>
      <td>889</td>
      <td>92</td>
      <td>0</td>
    </tr>
    <tr>
      <td><strong>Negative</strong></td>
      <td>13</td>
      <td>38,915</td>
      <td>0</td>
    </tr>
    <tr>
      <td><strong>Void</strong></td>
      <td>2</td>
      <td>75</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

<h2>
<span class="number">4. </span> Verification of central lab technical failure rates in process validation studies</h2>

<p>4.1 In process validation studies, the re-testing and total technical failure rates were assessed under business-as-usual operational trials at the newly established Milton Keynes <abbr title="endpoint PCR">EPCR</abbr> test lab unit.</p>

<p>4.2 For 7,068 samples tested over 2 processing days, 619 samples (8.76%) either required a repeat-test assay test run (by <abbr title="endpoint PCR">EPCR</abbr> or real-time <abbr title="Polymerase chain reaction">PCR</abbr>, in order), or yielded an inconclusive outcome where repeated swab testing was indicated. <a href="#table-7" class="govuk-link">See table 7</a>.</p>

<p>4.3 Following assay re-testing, 60 of 7,068 samples yield an inconclusive outcome, yielding a total technical failure rate at our combined <abbr title="endpoint PCR">EPCR</abbr> and real-time <abbr title="Polymerase chain reaction">PCR</abbr> test facility of 0.86% (see table 6).</p>

<h3>Table 6: <abbr title="endpoint PCR">EPCR</abbr> re-test rate during process validation studies</h3>

<table>
  <thead>
    <tr>
      <th scope="col">EPCR outcomes</th>
      <th scope="col">Specimens</th>
      <th scope="col">Percentages</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><strong>Test outcomes available on first run</strong></td>
      <td>6,449</td>
      <td>91.24</td>
    </tr>
    <tr>
      <td><strong>Retest outcome following first run</strong></td>
      <td>619</td>
      <td>8.76</td>
    </tr>
    <tr>
      <td><strong>Void outcome on 1st <abbr title="endpoint PCR">EPCR</abbr> run and no more rerun</strong></td>
      <td>55</td>
      <td>0.09</td>
    </tr>
    <tr>
      <td><strong>1st <abbr title="endpoint PCR">EPCR</abbr> rerun</strong></td>
      <td>372</td>
      <td>5.76</td>
    </tr>
    <tr>
      <td><strong>1st qPCR rerun</strong></td>
      <td>192</td>
      <td>2.98</td>
    </tr>
    <tr>
      <td><strong>Test outcomes available (not void) after re-testing runs</strong></td>
      <td>7,008</td>
      <td>99.14</td>
    </tr>
    <tr>
      <td><strong>Total technical failure rate (after re-testing runs)</strong></td>
      <td>60</td>
      <td>0.86</td>
    </tr>
    <tr>
      <td><strong>Total reactions</strong></td>
      <td>7,068</td>
      <td>100</td>
    </tr>
  </tbody>
</table>

<p>Table 6 criteria for acceptance: re-test rate: no more than 10% re-test rate after first <abbr title="endpoint PCR">EPCR</abbr> assay run. Total technical failure rate: less than 1.0%</p>

<h2>
<span class="number">5. </span> Clinical reporting algorithm</h2>

<p>5.1 While it delivers qualitative outcomes equivalent to Taqpath real-time <abbr title="Polymerase chain reaction">PCR</abbr>, <abbr title="endpoint PCR">EPCR</abbr> employs a different clinical decision tree to evaluate process control and specimen target analyte <abbr title="Polymerase chain reaction">PCR</abbr> products (see figure 1 below).</p>

<ul>
  <li>the <abbr title="endpoint PCR">EPCR</abbr> process utilises a fluorescent dye (ROX) as a passive reference signal to control for proper reaction assay dispense</li>
  <li>a positive control assay amplifying any pan-species RNase P transcripts present in the specimen analyte is allocated to a second fluorescence channel (VIC)</li>
  <li>the SARS-CoV-2 target <em>N</em> gene amplicon is assayed using FAM fluorescent dye as a reporter</li>
</ul>

<p>5.2 <abbr title="endpoint PCR">EPCR</abbr> assay scoring, using either automated software assistance or manual inspection, delivers âpositiveâ infected status outcome determinations when reference dye and RNase P signals are adequate, and when a high N gene signal is observed. A ânegativeâ outcome is determined when low N gene fluorescence is observed along with suitable process control results.</p>

<p>5.3 <abbr title="endpoint PCR">EPCR</abbr> delivers an outcome of positive at limit of detection (<abbr title="positive at limit of detection">PLOD</abbr>) and instruction to repeat swab sampling, when <em>N</em> gene signal is in an inconclusive range between clear positive or negative signal ranges.</p>

<p>5.4 A variety of scenarios will result in a âvoidâ outcome and instruction to repeat swab testing, whenever assay signal or process control results are not suitable to provide a conclusive outcome.</p>

<h3>Figure 1: clinical reporting algorithm to arrive at positive, positive at limit of detection (<abbr title="positive at limit of detection">PLOD</abbr>), negative and void results</h3>

<figure class="image embedded"><div class="img"><img alt="" src="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/image_data/file/112886/Figure_1.PNG"></div></figure>

<p>Description of figire 1, the clinical reporting algorithm:</p>

<ul>
  <li>
    <p>a âpositiveâ infected status outcome is determined when reference dye and RNase P signals are adequate, and when a high N gene signal is observed</p>
  </li>
  <li>
    <p>a ânegativeâ outcome is determined when low N gene fluorescence is observed along with suitable process control results</p>
  </li>
  <li>
    <p>a âpositive at limit of detectionâ outcome is determined when the N gene signal is in an inconclusive range (4000 to 8000 RFU)</p>
  </li>
  <li>
    <p>other scenarios result in a void outcome</p>
  </li>
</ul>

<h2>
<span class="number">6. </span> Workforce requirement</h2>

<p>6.1 Workforce calculations were made to process 225,000 samples per day based on staffing requirements using Taqpath real-time <abbr title="Polymerase chain reaction">PCR</abbr> and <abbr title="endpoint PCR">EPCR</abbr>.</p>

<h3>Table 7: workforce requirement to process 225,000 swabs per day using real-time <abbr title="Polymerase chain reaction">PCR</abbr>, <abbr title="endpoint PCR">EPCR</abbr> and a hybrid of real-time <abbr title="Polymerase chain reaction">PCR</abbr>:<abbr title="endpoint PCR">EPCR</abbr> workflows</h3>

<table>
  <thead>
    <tr>
      <th scope="col">Work station</th>
      <th scope="col">Real-time PCR: 75,000/day</th>
      <th scope="col">EPCR: 150,000/day</th>
      <th scope="col">Real-time PCR 750000/day, plus EPCR 150,000/day</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><strong>Swab container reception</strong></td>
      <td>16</td>
      <td>24</td>
      <td>40</td>
    </tr>
    <tr>
      <td><strong>Sample inactivation and aliquoting</strong></td>
      <td>12</td>
      <td>20</td>
      <td>32</td>
    </tr>
    <tr>
      <td><strong>RNA extraction</strong></td>
      <td>12</td>
      <td>20</td>
      <td>32</td>
    </tr>
    <tr>
      <td><strong><abbr title="Polymerase chain reaction">PCR</abbr> assay dispense</strong></td>
      <td>20</td>
      <td>12</td>
      <td>32</td>
    </tr>
    <tr>
      <td><strong><abbr title="Polymerase chain reaction">PCR</abbr> thermocycling and detection</strong></td>
      <td>12</td>
      <td>0</td>
      <td>12</td>
    </tr>
    <tr>
      <td><strong>Assay analysis and QC</strong></td>
      <td>6</td>
      <td>4</td>
      <td>10</td>
    </tr>
  </tbody>
</table>

<h2>
<span class="number">7. </span> Conclusions</h2>

<p>7.1 In the setting of a centralised diagnostic testing lab, and across a variety samples from the Lighthouse laboratory service, <abbr title="endpoint PCR">EPCR</abbr> demonstrated excellent test sensitivity and specificity. The technical performance of the UltraDx SARS-CoV-2 N1/N2/RP endpoint <abbr title="Polymerase chain reaction">PCR</abbr> (<abbr title="endpoint PCR">EPCR</abbr>) assay demonstrated a sensitivity of 98.6% (<abbr title="confidence interval">CI</abbr> 0.976 to 0.992) and specificity of 99.8% (<abbr title="confidence interval">CI</abbr> 0.997 to 0.998) in comparison to current standard of care Taqpath real-time <abbr title="Polymerase chain reaction">PCR</abbr> testing after RNA extraction. Concordance testing showed near total concordance (Cohen's Kappa 0.94) with an existing Taqpath real-time <abbr title="Polymerase chain reaction">PCR</abbr> in our lab. Analytical sensitivity of <abbr title="endpoint PCR">EPCR</abbr> is 50 digital copies / ml which is slightly better than standard of care Taqpath real-time <abbr title="Polymerase chain reaction">PCR</abbr>. These data support the potential application of this alternative <abbr title="Polymerase chain reaction">PCR</abbr>-based assay detection technology following viral RNA extraction.</p>

<p>7.2 Additionally, test batch-size and continuous manufacturing principles underlying the <abbr title="endpoint PCR">EPCR</abbr> test process make this assay type uniquely suitable for use in ultra-high capacity service lines: the technology is projected to attain run capacity of greater than 150,000 test samples per day, per instrument line. Within the Milton Keynes Lighthouse lab facility, <abbr title="endpoint PCR">EPCR</abbr> has been added as a supplement to ongoing the real-time <abbr title="Polymerase chain reaction">PCR</abbr> service line: with minimal facility or operational changes, introduction of the <abbr title="endpoint PCR">EPCR</abbr> service line should raise total facility test capacity from 75,000 to approximately 225,000 test samples per day.</p>

<p>7.3 The high scalability and excellent performance of <abbr title="endpoint PCR">EPCR</abbr> should enable use of the technology for daily, routine whole-population diagnostic pandemic monitoring, delivered by a limited number of centralised testing labs.</p>

<h2>
<span class="number">8. </span> Additional data and references</h2>

<p>8.1 Data tables are available on request from <a class="govuk-link" href="mailto:queries@ukbiocentre.com">queries@ukbiocentre.com</a></p>

<p>8.2 Cohen's Kappa measure is summarised by Robert Kwiecien, Annette Kopp-Schneider, Maria Blettner, <a rel="external" class="govuk-link" href="https://pubmed.ncbi.nlm.nih.gov/21904584/">Concordance analysis</a>, (Deutsches Ãrzteblatt International. Dtsch Arztebl Int 2011; 108(30): 515 to 521).</p>

<h2>
<span class="number">9. </span> Authors</h2>

<p>Jeff Roix, Malur Sudhanva, Tony Cox, Jonathan Curry, Emma Millican, Derek Craston, Paul Mark Martin, Phil Eeles</p>

</div>


</div>
</div>
  </div>

  <div data-sticky-element="" class="govuk-sticky-element">
    <a class="govuk-link app-c-back-to-top dont-print" href="#contents">
    <svg height="17" xmlns="http://www.w3.org/2000/svg" class="app-c-back-to-top__icon" width="13" viewBox="0 0 13 17">
      <path fill="currentColor" d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z"></path>
    </svg>
    Contents
</a>

    <div class="govuk-sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>