<main lang="en" class="html-publication" role="main">
      

  <div class="publication-external">
    <ul class="organisation-logos">
        <li class="organisation-logos__logo">
          
<div class="gem-c-organisation-logo brand--cabinet-office">
    <a class="gem-c-organisation-logo__container gem-c-organisation-logo__link brand__border-color" href="/government/organisations/scientific-advisory-group-for-emergencies">
      <span class="gem-c-organisation-logo__name">Scientific Advisory Group for Emergencies</span>
</a>
</div>
        </li>
    </ul>
  </div>

  <header class="gem-c-inverse-header  gem-c-inverse-header--padding-top ">
    
  

<div class="gem-c-title gem-c-title--inverse govuk-!-margin-top-8 govuk-!-margin-bottom-0">
      <span class="govuk-caption-xl gem-c-title__context">
    Research and analysis
  </span>

  
  <h1 class="gem-c-title__text govuk-heading-xl">
    NERVTAG: Update note on B.1.1.7 severity - 11 February 2021
  </h1>
</div>
  <p class="publication-header__last-changed">Updated 26 October 2021</p>

  </header>



<div data-module="sticky-element-container" class="govuk-grid-row sidebar-with-body">
    <div class="govuk-grid-column-one-quarter-from-desktop contents-list-container">
        <nav class="gem-c-contents-list" role="navigation" data-module="gem-track-click" aria-label="Contents">
    <h2 class="gem-c-contents-list__title">
      Contents
</h2>
    <ol class="gem-c-contents-list__list">
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-options="{&quot;dimension29&quot;:&quot;Summary&quot;}" data-track-action="content_item 1" href="#summary" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-label="#summary" data-track-category="contentsClicked">Summary</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 2" data-track-options="{&quot;dimension29&quot;:&quot;Full text&quot;}" href="#full-text" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-label="#full-text" data-track-category="contentsClicked">Full text</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-label="#conclusion" data-track-options="{&quot;dimension29&quot;:&quot;Conclusion&quot;}" href="#conclusion" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-action="content_item 3" data-track-category="contentsClicked">Conclusion</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a href="#recommendations" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-category="contentsClicked" data-track-options="{&quot;dimension29&quot;:&quot;Recommendations&quot;}" data-track-action="content_item 4" data-track-label="#recommendations">Recommendations</a>

        </li>
        <li class="gem-c-contents-list__list-item gem-c-contents-list__list-item--numbered">
          <a data-track-action="content_item 5" data-track-label="#annex" data-track-options="{&quot;dimension29&quot;:&quot;Annex&quot;}" class="gem-c-contents-list__link govuk-link govuk-link--no-underline" data-track-category="contentsClicked" href="#annex">Annex</a>

        </li>
    </ol>
</nav>
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button class="govuk-link govuk-body-s gem-c-print-link__button" data-module="print-link">Print this page</button>
</div>
    </div>

  <div class="print-wrapper">
    <div class="print-meta-data">
      <p>
  <img class="print-meta-data-licence" src="/assets/government-frontend/open-government-licence-min-e98134ae65ff5fe99a524d8c383350b3998f91c298dc24b36126e65f1656c578.png">
</p>
<p>
  © Crown copyright 2021
</p>
<p>
  This publication is licensed under the terms of the Open Government Licence v3.0 except where otherwise stated. To view this licence, visit <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3">nationalarchives.gov.uk/doc/open-government-licence/version/3</a> or write to the Information Policy Team, The National Archives, Kew, London TW9 4DU, or email: <a href="mailto:psi@nationalarchives.gov.uk">psi@nationalarchives.gov.uk</a>.
</p>
<p>
  Where we have identified any third party copyright information you will need to obtain permission from the copyright holders concerned.
</p>
<p>
  This publication is available at https://www.gov.uk/government/publications/nervtag-update-note-on-b117-severity-11-february-2021/nervtag-update-note-on-b117-severity-11-february-2021
</p>


    </div>
  </div>

  <div class="main-content-container">
    <div class="gem-c-govspeak-html-publication">
  
<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
    
    
      <div class="govspeak">
<p><strong>Update note on B.1.1.7 severity</strong></p>

<p>Authors: Peter Horby, Iain Bell, Judith Breuer, Muge Cevik, Rob Challen, Nicholas Davies, Gavin Dabrera, John Edmunds, Neil Ferguson, Sebastian Funk, Andrew Hayward, Julia Hippisley-Cox, Ben Humberstone, Catherine Huntley, Jim McMenamin, Paul McKeigue, Graham Medley and Calum Semple.</p>

<p><strong><abbr title="Scientific Advisory Group for Emergencies">SAGE</abbr> meeting date: 11 February 2021</strong></p>

<h2>Summary</h2>

<p>1. On Thursday, 21 January, <abbr title="New and Emerging Respiratory Virus Threats Advisory Group">NERVTAG</abbr> presented evidence to <abbr title="Scientific Advisory Group for Emergencies">SAGE</abbr> of increased disease severity in people infected with variant of concern (<abbr title="Variant of concern">VOC</abbr>) B.1.1.7 compared to people infected with non-<abbr title="Variant of concern">VOC</abbr> virus variants. In that report it was stated that ‘data will accrue in coming weeks, at which time the analyses will become more definitive’.</p>

<p>2. Here we report updated and additional analyses, which together strengthen the earlier finding of increased disease severity in people infected with <abbr title="WHO designated variant Alpha">VOC B.1.1.7</abbr> compared to other virus variants.</p>

<p>3. Independent analyses of the risk of hospitalisation and death for S-gene target failure (<abbr title="s-gene target failure">SGTF</abbr>; a proxy for <abbr title="WHO designated variant Alpha">VOC B.1.1.7</abbr> in the <abbr title="United Kingdom">UK</abbr>) cases and non-<abbr title="s-gene target failure">SGTF</abbr> (non-<abbr title="Variant of concern">VOC</abbr>) cases are (also see data table in annex):</p>

<p>a. <abbr title="London School of Hygiene and Tropical Medicine">LSHTM</abbr>: reported that the relative hazard of death within 28 days of test for <abbr title="Variant of concern">VOC</abbr>-infected individuals compared to non-<abbr title="Variant of concern">VOC</abbr> was 1.58 (95 per cent <abbr title="continuous interval">CI</abbr> 1.40 to 1.79), or 1.71 (95 per cent <abbr title="continuous interval">CI</abbr> 1.48 to 1.97) if adjustment is made for misclassification of <abbr title="s-gene target failure">SGTF</abbr> and missingness of data.</p>

<p>b. Imperial College London: mean ratio of case fatality ratio (<abbr title="case fatality ratio">CFR</abbr>) for <abbr title="Variant of concern">VOC</abbr> infected individuals compared to non-<abbr title="Variant of concern">VOC</abbr> was 1.36 (95 per cent <abbr title="continuous interval">CI</abbr> 1.18 to 1.56) by a case-control weighting method, 1.29 (95 per cent <abbr title="continuous interval">CI</abbr> 1.07 to 1.54) by a standardised <abbr title="case fatality ratio">CFR</abbr> method.</p>

<p>c. University of Exeter: an updated analysis estimated the mortality hazard ratio for <abbr title="Variant of concern">VOC</abbr>-infected individuals compared to non-<abbr title="Variant of concern">VOC</abbr> was 1.7 (95 per cent <abbr title="continuous interval">CI</abbr> 1.3 to 2.2) in a matched cohort study.</p>

<p>d. Public Health England: an updated matched cohort analysis has reported a death risk ratio for <abbr title="Variant of concern">VOC</abbr>-infected individuals compared to non-<abbr title="Variant of concern">VOC</abbr> of 1.65 (95 per cent <abbr title="continuous interval">CI</abbr> 1.21 to 2.25).</p>

<p>e. Public Health Scotland: the <abbr title="REal-time Assessment of Community Transmission">REACT</abbr>-SCOT study found that the hazard ratio was 1.08 (95 per cent <abbr title="continuous interval">CI</abbr> 0.78 to 1.49) for death and 1.40 (95 per cent <abbr title="continuous interval">CI</abbr> 1.28 to 1.53) for death or hospital admission in <abbr title="s-gene target failure">SGTF</abbr> compared to non-<abbr title="s-gene target failure">SGTF</abbr> cases.</p>

<p>f. Public Health Scotland: the <abbr title="Early Pandemic Evaluation and Enhanced Surveillance of COVID-19">EAVE-II</abbr> study found the risk of being admitted to hospital is higher for cases with <abbr title="s-gene target failure">SGTF</abbr> than for those who are S Gene positive - risk Ratio 1.63 (95 per cent <abbr title="continuous interval">CI</abbr> 1.48, 1.80). The relative risk of death within 28 days of a positive test was 1.37 (95 per cent <abbr title="continuous interval">CI</abbr> 1.02, 1.84) for <abbr title="s-gene target failure">SGTF</abbr> compared to S Gene positive.</p>

<p>g. The Hospital Onset Covid Infection (<abbr title="Hospital Onset Covid Infection">HOCI</abbr>) study: found the overall <abbr title="heart rate">HR</abbr> for in hospital mortality of B.1.1.7 was 1.09 (95 per cent <abbr title="continuous interval">CI</abbr> 0.86 to 1.36, P=0.48). Increased mortality was only observed with the <abbr title="Variant of concern">VOC</abbr> in women over 65 years. The overall <abbr title="heart rate">HR</abbr> for <abbr title="Intensive therapy unit">ITU</abbr> admission for B.1.1.7 was 1.15 (95 per cent <abbr title="continuous interval">CI</abbr> 0.86 to 1.53, P=0.35).</p>

<p>h. <abbr title="Intensive Care National Audit and Research Centre">ICNARC</abbr> and <abbr>QRESEARCH</abbr>: found a higher risk of <abbr title="Intensive care unit">ICU</abbr> admission for <abbr title="Variant of concern">VOC</abbr> patients (<abbr title="heart rate">HR</abbr>: 1.44; 95 per cent <abbr title="continuous interval">CI</abbr>: 1.25, 1.67) compared to non-<abbr title="Variant of concern">VOC</abbr> patient no significant difference in the hazard of <abbr title="Intensive care unit">ICU</abbr> mortality between the two groups (<abbr title="heart rate">HR</abbr>: 0.94; 95 per cent <abbr title="continuous interval">CI</abbr>: 0.82, 1.09).</p>

<p>i. <abbr title="Office for National Statistics">ONS</abbr> analysis: found that whilst the hazard ratio suggests that the B.1.1.7 variant is associated with higher risk of all-cause mortality, the number of deaths are too low for reliable inference.</p>

<p>j. <abbr title="COVID-19 Clinical Information Network">CO-CIN</abbr> (hospitalised patients only): found no statistically significant change in in-hospital <abbr title="case fatality ratio">CFR</abbr> comparing proven B.1.1.7 (n=32) with non-<abbr title="Variant of concern">VOC</abbr> (n=184) (OR 0.63, 95 per cent <abbr title="continuous interval">CI</abbr> 0.20 to 1.69).</p>

<p>k. <abbr title="COVID-19 Clinical Information Network">CO-CIN</abbr> (hospitalised patients only): a repeat analysis with an updated dataset did not provide evidence to suggest that the variant of concern is linked to a higher risk of in-hospital case fatality (OR 0.67, 95 per cent <abbr title="continuous interval">CI</abbr> 0.32, 1.40).</p>

<p>l. <abbr title="London School of Hygiene and Tropical Medicine">LSHTM</abbr>: a population-level analysis at the level of upper-tier local authorities resulted in estimates of a 1.4 (1.3 to 1.5) times higher number of hospitalisations per case and 1.4 (1.2 to 1.5) times higher number of fatalities per hospitalisation associated with <abbr title="Variant of concern">VOC</abbr>.</p>

<p>4. There are inevitable limitations to these datasets, including representativeness, power, potential biases in case ascertainment, unmeasured confounders, and secular trends.</p>

<p>5. Whilst studies limited to in-patients did not identify evidence of increased disease severity, this is not incompatible with an overall increase in disease severity.</p>

<p>6. Whilst earlier analyses using linked community testing and mortality data showed comparable increases in case fatality ratios, these were all based on the same datasets, and therefore subject to similar biases reducing the level of certainty in the findings. More recent analyses have added a wider range of data sets and been able to control for additional confounders, increasing confidence in the association of the <abbr title="Variant of concern">VOC</abbr> with increased disease severity.</p>

<p>7. Based on these analyses, it is likely that infection with <abbr title="WHO designated variant Alpha">VOC B.1.1.7</abbr> is associated with an increased risk of hospitalisation and death compared to infection with non-<abbr title="Variant of concern">VOC</abbr> viruses.</p>

<p>8. It should be noted that the absolute risk of death per infection remains low.</p>

<h2>Full text</h2>

<p>9. Previously, preliminary results from a matched-cohort study conducted by <abbr title="Public Health England">PHE</abbr> reported no statistically significant increased risk of hospitalisation or death in <abbr title="Variant of concern">VOC</abbr> infected individuals compared to non-<abbr title="Variant of concern">VOC</abbr>. <sup role="doc-noteref"><a class="govuk-link" href="#fn:1" rel="footnote">[footnote 1]</a></sup></p>

<p>10. On Friday 15 January <abbr title="New and Emerging Respiratory Virus Threats Advisory Group">NERVTAG</abbr> was presented with two papers that reported an increased case fatality rate in subjects with s-gene target failure (<abbr title="s-gene target failure">SGTF</abbr>, a proxy for variant of concern B.1.1.7).</p>

<p>11. Both papers used the same core dataset of <abbr title="s-gene target failure">SGTF</abbr> cases identified through Pillar 2 testing linked to the <abbr title="Public Health England">PHE</abbr> <abbr title="coronavirus">COVID-19</abbr> deaths line list: a paper from <abbr title="London School of Hygiene and Tropical Medicine">LSHTM</abbr> <sup role="doc-noteref"><a href="#fn:2" class="govuk-link" rel="footnote">[footnote 2]</a></sup> and a paper from Imperial College London. <sup role="doc-noteref"><a href="#fn:3" class="govuk-link" rel="footnote">[footnote 3]</a></sup></p>

<p>12. Since then, drawing on analyses of multiple datasets, further evidence of an increase in disease severity in people infected with <abbr title="WHO designated variant Alpha">VOC B.1.1.7</abbr> has emerged. A summary of their findings is presented below.</p>

<p>13. The <abbr title="London School of Hygiene and Tropical Medicine">LSHTM</abbr> paper used a Cox proportional hazards model to estimate change in risk of death within 28 days of test for individuals infected with the <abbr title="Variant of concern">VOC</abbr>. <sup role="doc-noteref"><a href="#fn:2" rel="footnote" class="govuk-link">[footnote 2]</a></sup></p>

<p>a. The study was based on 3,382 deaths among 1 million tested individuals. 1,722 deaths were among <abbr title="s-gene target failure">SGTF</abbr> individuals.</p>

<p>b. Results were controlled for age, sex, index of multiple deprivation, ethnicity, residence type (care home versus residential versus other), specimen date, and Lower Tier Local Authority (<abbr title="Lower Tier Local Authority">LTLA</abbr>).</p>

<p>c. The relative hazard of death within 28 days of test was 1.71 (95 per cent <abbr title="continuous interval">CI</abbr> 1.48 to 1.97) for <abbr title="Variant of concern">VOC</abbr>-infected individuals, compared to non-<abbr title="Variant of concern">VOC</abbr>, with adjustment made for misclassification of <abbr title="s-gene target failure">SGTF</abbr> and missingness of data.</p>

<p>d. Focusing only on individuals with <abbr title="s-gene target failure">SGTF</abbr> after 1 November 2020 (no adjustment for <abbr title="s-gene target failure">SGTF</abbr> misclassification or missingness), the relative hazard of death is 1.58 (95 per cent <abbr title="continuous interval">CI</abbr> 1.40 to 1.79).</p>

<p>e. Relative increases in <abbr title="case fatality ratio">CFR</abbr> appeared to be consistent across age groups, when comparison was done for the age groups 1 to 54, 55 to 69, 70 to 84, 85+ (due to the limited number of deaths below the age of 55), however absolute risk of <abbr title="case fatality ratio">CFR</abbr> is not increased substantially below the age of 55.</p>

<p>f. Sensitivity analyses including further hospital pressure covariates (proportion of beds capable of mechanical ventilation occupied; proportion of beds capable of non-invasive ventilation occupied; number of staff absences per bed among medical staff; and number of staff absences per bed among nursing staff) did not substantially change the measure of effect.</p>

<p>g. There is a statistically significant interaction of <abbr title="s-gene target failure">SGTF</abbr> with residence type suggesting a potential interaction with frailty: the hazard ratio for care home residents was 2.43 (1.72 to 3.35), compared to the hazard ratio for the general population of 1.53 (1.35 to 1.74).</p>

<p>14. The Imperial Paper reported the results of a non-parametric analysis of fatal outcomes associated with B1.1.7. ^[3]</p>

<p>a. Two methods were used to evaluate the differences in mortality between <abbr title="Variant of concern">VOC</abbr> and non-<abbr title="Variant of concern">VOC</abbr> cases: case-control-weighting, and standardised <abbr title="case fatality ratio">CFR</abbr>. In each case, the ratio of s-gene positive to s-gene negative case fatality ratios (CFRs) is calculated.</p>

<p>b. The study considers data from all of England and includes specimen dates in the epidemiological week range 46 to 54 (54 being week 1 of 2021) inclusive. Estimates are adjusted for <abbr title="National Health Service">NHS</abbr> <abbr title="Sustainable Transformation Plan">STP</abbr> area, epidemiological week, ethnicity code, and age band.</p>

<p>c. Across all specimens, the mean ratio of CFRs is 1.36 (95 per cent <abbr title="continuous interval">CI</abbr> 1.18-1.56) by the case-control weighting method, and 1.29 (95 per cent <abbr title="continuous interval">CI</abbr> 1.07 to 1.54) by the standardised <abbr title="case fatality ratio">CFR</abbr> method. This estimate includes a correction for the probability over time that a specimen with <abbr title="s-gene target failure">SGTF</abbr> is the <abbr title="Variant of concern">VOC</abbr>.</p>

<p>d. Relative increases in <abbr title="case fatality ratio">CFR</abbr> appeared to be consistent across age groups when comparison was done for the age groups 1 to 54, 55 to 69, 70 to 84, 85+ (due to the limited number of deaths below the age of 55), however absolute risk of <abbr title="case fatality ratio">CFR</abbr> is not increased substantially below the age of 55.</p>

<p>e. Subsequent correction for possible differences in <abbr title="polymerase chain reaction">PCR</abbr> cycle threshold values (<abbr title="cycle threshold values">ct</abbr>) between <abbr title="Variant of concern">VOC</abbr> and non-<abbr title="Variant of concern">VOC</abbr> cases was included by restricting both groups only to those samples with <abbr title="cycle threshold values">ct</abbr> less than 30. This adjustment made no meaningful difference.</p>

<p>15. A <abbr title="Public Health England">PHE</abbr> retrospective matched cohort study was also reported: <sup role="doc-noteref"><a rel="footnote" class="govuk-link" href="#fn:4">[footnote 4]</a></sup></p>

<p>a. 23 November 2020 to 4 January 2021 study period (period when more than 90 per cent of sequenced <abbr title="s-gene target failure">SGTF</abbr> samples confirmed to be VOC202012/01). Matching based on 10-year age bands, sex, week of test and lower-tier local authority.</p>

<p>b. 92,207 <abbr title="s-gene target failure">SGTF</abbr> cases and corresponding comparators were included in the matched cohort (n = 184,414), although routine hospitalisation data is subject to reporting delays and this should be considered preliminary.</p>

<p>c. The odds of <abbr title="s-gene target failure">SGTF</abbr> cases being admitted was not significantly different to non-<abbr title="s-gene target failure">SGTF</abbr> cases (OR = 1.07, 95 per cent <abbr title="continuous interval">CI</abbr> 0.86 – 1.33).</p>

<p>d. Initial analysis identified 152 deaths following a first positive <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> test, n = 86 (0.09 per cent) <abbr title="s-gene target failure">SGTF</abbr> cases and n = 66 (0.07 per cent) comparator cases. It was noted that 0.07 per cent to 0.09 per cent represents a 28 per cent relative increase in the risk of death, which is compatible with the results from <abbr title="London School of Hygiene and Tropical Medicine">LSHTM</abbr> and Imperial.</p>

<p>e. Initial analysis of 14,939 <abbr title="s-gene target failure">SGTF</abbr> cases and 15,555 comparators who had at least 28 days between specimen date and the study period end date. There were 25 deaths (0.17 per cent) in <abbr title="s-gene target failure">SGTF</abbr> cases and 26 deaths (0.17 per cent) in comparators (<abbr title="Relative Risk">RR</abbr> 1.00, 95 per cent <abbr title="continuous interval">CI</abbr> 0.58 – 1.73).</p>

<p>f. Updated linkage of deaths data to the same matched cohort on 19 January 2021 identified there were 65 deaths among non-<abbr title="s-gene target failure">SGTF</abbr> cases (0.1 per cent) and 104 deaths among <abbr title="s-gene target failure">SGTF</abbr> cases (0.2 per cent), within 28 days of specimen date. With this increased time for follow-up and ascertainment of deaths, the risk ratio increased to 1.65 (95 per cent <abbr title="continuous interval">CI</abbr> 1.21 to 2.25).</p>

<p>16. A subsequent independent case control analysis of Pillar 2 data linked to the death line list by University of Exeter, reported a mortality hazard ratio for <abbr title="Variant of concern">VOC</abbr>-infected individuals compared to non-<abbr title="Variant of concern">VOC</abbr> was 1.7 (95 per cent <abbr title="continuous interval">CI</abbr> 1.3 to 2.2) in a sample size of 54,773 pairs, (n=109,546).</p>

<p>a. Study period: 1 October 2020 to 29 January 2021. Paired S-gene positive “controls” to S-gene negative “cases” by matching on gender, ethnicity, index of multiple deprivation, location (as lower tier local authority region), age (within a tolerance of ±5 years), and date of positive specimen (within a tolerance of ±1 day).</p>

<p>b. Because the data is from Pillar 2, cases were in general younger than typical <abbr title="coronavirus">COVID-19</abbr> cohorts (mean age 46.3) and under-represent elderly (only 0.5 per cent of the cohort were more than 80 years old). The South West, East of England and to a lesser extent the South East are under-represented as the lighthouse labs in those areas do not always report S-gene status.</p>

<p>c. There were 272 deaths following a first positive <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> test, 101 occurring in S-gene positive controls and 171 occurring in S-gene negative cases. A Cox proportional hazards model was used to account for censoring, although this is also controlled for by the close match on specimen date</p>

<p>17. Public Health Scotland presented two studies:</p>

<p>a. A case control study (<abbr title="REal-time Assessment of Community Transmission">REACT</abbr>-SCOT) used a Cox Regression model to estimate hazard of death and hospital admission in <abbr title="s-gene target failure">SGTF</abbr> compared to non <abbr title="s-gene target failure">SGTF</abbr> cases. <sup role="doc-noteref"><a rel="footnote" href="#fn:5" class="govuk-link">[footnote 5]</a></sup></p>

<ul>
  <li>
    <p>i. All Pillar 2 (Lighthouse lab) test-positive cases up to 6 January with <abbr title="s-gene target failure">SGTF</abbr> status scored were linked to deaths and hospital admissions up to 22 January 2021.</p>
  </li>
  <li>
    <p>ii. Hazard ratios were estimated by Cox regression stratifying by calendar time and adjusting for age, sex, recent hospital exposure and mean Ct of the <abbr title="zoonotic viral skin disease that can be spread to humans by handling infected sheep and goats">ORF</abbr> and N gene.</p>
  </li>
  <li>
    <p>iii. The hazard ratio was 1.08 (95 per cent <abbr title="continuous interval">CI</abbr> 0.78 to 1.49) for death (292 events, 74 in <abbr title="s-gene target failure">SGTF</abbr> cases) and 1.40 (95 per cent <abbr title="continuous interval">CI</abbr> 1.28 to 1.53) for death or hospital admission (3765 events, 1220 in <abbr title="s-gene target failure">SGTF</abbr> cases).</p>
  </li>
</ul>

<p>b. A cohort study (<abbr title="Early Pandemic Evaluation and Enhanced Surveillance of COVID-19">EAVE-II</abbr>), which included all of the Scottish population – data linkage of primary care, secondary care, laboratory and death data. <sup role="doc-noteref"><a class="govuk-link" rel="footnote" href="#fn:6">[footnote 6]</a></sup></p>

<ul>
  <li>
    <p>i. A cox survival model was used to estimate the risk of hospitalisation following a positive test result for <abbr title="s-gene target failure">SGTF</abbr> versus <abbr title="Scottish Government">SG</abbr> positive. This model was fitted to all the data using all follow up available. The model included p spline terms for age and calendar time as well as demographic and risk group data.</p>
  </li>
  <li>
    <p>ii. The risk of being admitted to hospital is higher for cases with S Gene deletion than for those who are S Gene positive. Risk Ratio 1.63 (95 per cent <abbr title="continuous interval">CI</abbr> 1.48, 1.80).</p>
  </li>
  <li>
    <p>iii. The relative risk of death within 28 days of a positive test was 1.37 (95 per cent <abbr title="continuous interval">CI</abbr> 1.02, 1.84) for S Gene deletion compared to S Gene positive.</p>
  </li>
</ul>

<p>18. The <abbr title="COVID-19 Genomics UK Consortium">COG-UK</abbr> Hospital Onset <abbr title="coronavirus">COVID-19</abbr> Infection (<abbr title="Hospital Onset Covid Infection">HOCI</abbr>) study reported on mortality and intensive therapy unit (<abbr title="Intensive therapy unit">ITU</abbr>) admissions according to presence of the <abbr title="Variant of concern">VOC</abbr> on viral sequencing. <sup role="doc-noteref"><a href="#fn:7" rel="footnote" class="govuk-link">[footnote 7]</a></sup></p>

<p>a. Study sites are sequencing all positive <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> samples from hospitalised patients and healthcare workers. Data were collected from nine participating hospitals on mortality and <abbr title="Intensive therapy unit">ITU</abbr> admissions for 2386 inpatients with sample date between 16 November 2020 and 10 January 2021.</p>

<p>b. <abbr title="Variant of concern">VOC</abbr> 202012/01 was identified in 1090 (45.5 per cent) of the samples. Mixed effects Cox models were used to estimate the overall association between the <abbr title="Variant of concern">VOC</abbr> and each outcome, with adjustment for sex and age and exploratory analysis of interactions with these variables.</p>

<p>c. 533 (22.3 per cent) patients died during the study period. The overall <abbr title="heart rate">HR</abbr> for mortality of the <abbr title="Variant of concern">VOC</abbr> was 1.09 (95 per cent <abbr title="continuous interval">CI</abbr> 0.86 to 1.36, P=0.48). Evaluation of an interaction between <abbr title="Variant of concern">VOC</abbr> status and sex found an association of the <abbr title="Variant of concern">VOC</abbr> with mortality in females (<abbr title="heart rate">HR</abbr> 1.41, 95 per cent <abbr title="continuous interval">CI</abbr> 1.04 to 1.90) but not in males (0.89, 95 per cent <abbr title="continuous interval">CI</abbr> 0.67 to 1.17), with a further interaction with age suggesting that increased mortality was only observed with the <abbr title="Variant of concern">VOC</abbr> in women over 65 years.</p>

<p>d. 320 (13.4 per cent) patients were admitted to <abbr title="Intensive therapy unit">ITU</abbr>. The overall <abbr title="heart rate">HR</abbr> for <abbr title="Intensive therapy unit">ITU</abbr> admission for the <abbr title="Variant of concern">VOC</abbr> was 1.15 (95 per cent <abbr title="continuous interval">CI</abbr> 0.86 to 1.53, P=0.35). On evaluation of an interaction between <abbr title="Variant of concern">VOC</abbr> status and sex, the <abbr title="heart rate">HR</abbr> for <abbr title="Intensive therapy unit">ITU</abbr> admission for the <abbr title="Variant of concern">VOC</abbr> in males was 0.85 (95 per cent <abbr title="continuous interval">CI</abbr> 0.61 to 1.18) and that for females was 1.94 (1.25 to 2.99). There was no evidence of an interaction between <abbr title="Variant of concern">VOC</abbr> and age.</p>

<p>19. <abbr title="Intensive Care National Audit and Research Centre">ICNARC</abbr> and <abbr>QRESEARCH</abbr> reported two studies which used flexible parametric survival models to estimate the relative risk of <abbr title="Intensive care unit">ICU</abbr> admission and <abbr title="Intensive care unit">ICU</abbr> mortality for <abbr title="Variant of concern">VOC</abbr> compared to non-<abbr title="Variant of concern">VOC</abbr> patients. <sup role="doc-noteref"><a class="govuk-link" rel="footnote" href="#fn:8">[footnote 8]</a></sup></p>

<p>a. Both studies were restricted to Pillar 2 patients tested between the 1 of November and 27 of January 2021.</p>

<p>b. Both studies used flexible parametric survival models (Royston-Parmar model) adjusted for patient demographics (age, sex, deprivation index, geographical region) and medical characteristics. An indicator of the week in which the positive test was taken (<abbr title="Intensive care unit">ICU</abbr> admission analysis) and the week when the patient was admitted to <abbr title="Intensive care unit">ICU</abbr> (<abbr title="Intensive care unit">ICU</abbr> mortality analysis) was included in the models, to account for time dependent biases.</p>

<p>c. The hazard of <abbr title="Intensive care unit">ICU</abbr> admission was estimated using used the QResearch dataset linked with pillar 2 testing data from <abbr title="Public Health England">PHE</abbr> and the <abbr title="Intensive Care National Audit and Research Centre">ICNARC</abbr> Case Mix Programme (<abbr title="Case Mix Programme">CMP</abbr>) dataset. Only patients with at least 12 days of follow up from the date of the positive test were included. The data contained 331,321 patients, of which 99,444 (30.0 per cent) had the <abbr title="Variant of concern">VOC</abbr>. There were 1,171 patients admitted to <abbr title="Intensive care unit">ICU</abbr>, of which 428 (36.5 per cent) had the <abbr title="Variant of concern">VOC</abbr>.</p>

<p>d. The hazard of <abbr title="Intensive care unit">ICU</abbr> mortality was estimated using the <abbr title="Intensive Care National Audit and Research Centre">ICNARC</abbr> <abbr title="Case Mix Programme">CMP</abbr> dataset linked with testing data from <abbr title="Public Health England">PHE</abbr>. There were 6,038 patients admitted to <abbr title="Intensive care unit">ICU</abbr> with a positive test in the study period, of which 2088 had the <abbr title="Variant of concern">VOC</abbr>. The analysis was restricted to patients who had a minimum of 28 days follow-up after <abbr title="Intensive therapy unit">ITU</abbr> admission, as the proportion who had not completed their <abbr title="Intensive care unit">ICU</abbr> care was larger in the <abbr title="Variant of concern">VOC</abbr> group (42 per cent) than in the non-<abbr title="Variant of concern">VOC</abbr> group (27 per cent).</p>

<p>e. There was a higher risk of <abbr title="Intensive care unit">ICU</abbr> admission for <abbr title="Variant of concern">VOC</abbr>-patients (<abbr title="heart rate">HR</abbr>: 1.44; 95 per cent <abbr title="continuous interval">CI</abbr>: 1.25, 1.67) compared to non-<abbr title="Variant of concern">VOC</abbr> patients and a non significant difference in the hazard of <abbr title="Intensive care unit">ICU</abbr> mortality between the two groups (<abbr title="heart rate">HR</abbr>: 0.94; 95 per cent <abbr title="continuous interval">CI</abbr>: 0.82, 1.09).</p>

<p>20. An analysis from the <abbr title="Office for National Statistics">ONS</abbr> Coronavirus Infection Survey. <abbr title="COVID-19 Infection Survey">CIS</abbr> respondents who first tested positive on or after 15 November 2020 (N=8,811) were linked to the mortality registration data for deaths occurring up to 17 January 2021.</p>

<p>a. Out of 4217 positive cases compatible with the new <abbr title="United Kingdom">UK</abbr> variant, 11 deaths were identified. 7 deaths were observed among the 2745 positive cases that were not were compatible with the new <abbr title="United Kingdom">UK</abbr> variant, and 1 among the 1840 cases with unknown variant.</p>

<p>b. Hazard ratios for death were estimated using a Cox proportional hazard regression model, adjusted for age and gender.</p>

<p>c. Whilst the hazard ratio suggests that the new <abbr title="United Kingdom">UK</abbr> variant is associated with higher risk of all-cause mortality, the number of deaths are too low for reliable inference.</p>

<p>d. This analysis will be repeated weekly, as more recent deaths are made available, and the linkage between <abbr title="COVID-19 Infection Survey">CIS</abbr> and <abbr title="Personal Demographics Service">PDS</abbr> (used to retrieve <abbr title="National Health Service">NHS</abbr> numbers) is improved.</p>

<p>21. A rapid analysis of <abbr title="COVID-19 Clinical Information Network">CO-CIN</abbr> data during the period in which <abbr title="Variant of concern">VOC</abbr> B1.1.7 emerged was reported: <sup role="doc-noteref"><a class="govuk-link" href="#fn:9" rel="footnote">[footnote 9]</a></sup></p>

<p>a. Across the whole <abbr title="COVID-19 Clinical Information Network">CO-CIN</abbr> cohort, there is no observed increase in hospital case fatality in the period during which <abbr title="Variant of concern">VOC</abbr> emerged in England, after adjusting for period of admission, age, sex, deprivation, and ethnicity.</p>

<p>b. Compared to March 2020, hospital <abbr title="case fatality ratio">CFR</abbr> continues to be lower and has been stable in September through December.</p>

<p>c. Data return from <abbr title="COVID-19 Clinical Information Network">CO-CIN</abbr> is currently reduced and unevenly distributed, which will impact on the representativeness of findings. Importantly there are a substantial number of outcomes missing from cases admitted in late December, when the impact of <abbr title="Variant of concern">VOC</abbr> emergence would start to be apparent in hospital data.</p>

<p>d. A sub-analysis included linked data from 21,882 cases (21,596 non-<abbr title="Variant of concern">VOC</abbr> and 286 <abbr title="Variant of concern">VOC</abbr>) from across the whole <abbr title="COVID-19 Clinical Information Network">CO-CIN</abbr> cohort. <abbr title="Variant of concern">VOC</abbr> in this sub-study was robustly determined by <abbr title="COVID-19 genome">COG</abbr> sequence lineage, rather than assumed by <abbr title="s-gene target failure">SGTF</abbr>. Outcome data was available for only 143 <abbr title="Variant of concern">VOC</abbr> cases. However; 32 <abbr title="Variant of concern">VOC</abbr> cases were identified at one trust with good data quality returns throughout the period of study.</p>

<p>e. Restricting the analysis to a trust with high proportion of proven <abbr title="Variant of concern">VOC</abbr> which has maintained good quality data returns, and after adjusting for age and sex, found no statistically significant change in hospital <abbr title="case fatality ratio">CFR</abbr> comparing proven <abbr title="Variant of concern">VOC</abbr> (n=32) with non-<abbr title="Variant of concern">VOC</abbr> (n=184) (OR 0.63, 95 per cent <abbr title="continuous interval">CI</abbr> 0.20 to 1.69).</p>

<p>f. An increase in case fatality rates would not necessarily manifest as an increase in case fatality rates amongst those hospitalised. Rather, it may increase the proportion of cases who are ill enough to meet the severity threshold for hospitalisation, but not affect the likelihood of death amongst those who are sick enough to be admitted.</p>

<p>g. An updated analysis of <abbr title="COVID-19 Clinical Information Network">CO-CIN</abbr> data (study period 1 October 2020 to 12 January 2021) did not provide evidence to suggest that the variant of concern is linked to a higher risk of in-hospital case fatality. <sup role="doc-noteref"><a rel="footnote" href="#fn:10" class="govuk-link">[footnote 10]</a></sup></p>

<ul>
  <li>
    <p>i. A mixed-effects multivariable logistic regression model including 202 <abbr title="Variant of concern">VOC</abbr> cases from 91 hospitals (of which 108 had a 28-day mortality outcome ascertained), with 1:1 matching by age, sex, and admission date to non-<abbr title="Variant of concern">VOC</abbr> cases, did not provide evidence of a difference in the 28-day fatality rate in patients admitted to hospital (OR 0.67, 95 per cent <abbr title="continuous interval">CI</abbr> 0.32 to 1.40).</p>
  </li>
  <li>
    <p>ii. A Cox proportional hazards survival regression model where <abbr title="Variant of concern">VOC</abbr> cases were matched to non-<abbr title="Variant of concern">VOC</abbr> cases (1:3, n=103 matchable <abbr title="Variant of concern">VOC</abbr> cases) also did not provide evidence of a difference in the 28-day fatality rate in patients admitted to hospital (<abbr title="heart rate">HR</abbr> 0.81, 95 per cent <abbr title="continuous interval">CI</abbr> 0.50 to 1.32).</p>
  </li>
  <li>
    <p>iii. This analysis is limited by small numbers and the instability of a dynamic dataset.</p>
  </li>
</ul>

<p>22. <abbr title="London School of Hygiene and Tropical Medicine">LSHTM</abbr> undertook a population-level (ecological) analysis of the relationship between cases (Pillar 1 and Pillar 2), hospitalisations, and deaths as a function of local prevalence of <abbr title="s-gene target failure">SGTF</abbr>. <sup role="doc-noteref"><a class="govuk-link" rel="footnote" href="#fn:11">[footnote 11]</a></sup></p>

<p>a. This analysis suggests that <abbr title="Variant of concern">VOC</abbr> has increased the number of hospitalisations per case, and deaths per hospitalisation, which, in turn, is compatible with an increase in the case-hospitalisation rate and the hospitalisation-fatality rate.</p>

<p>b. Allowing baseline ratios to vary at the level of upper-tier local authorities resulted in estimates of a 1.4 (1.3 to 1.5) times higher number of hospitalisations per case and 1.4 (1.2 to 1.5) times higher number of fatalities per hospitalisation associated with <abbr title="Variant of concern">VOC</abbr>.</p>

<p>c. These estimates need to be interpreted with caution as they are likely to be confounded by other factors that varied over time and that could have affected changes in the rate of hospitalisations or deaths.</p>

<p>23. Limitations:</p>

<p>a. The majority of these analyses are limited to community testing data, except the <abbr title="Hospital Onset Covid Infection">HOCI</abbr> and <abbr title="London School of Hygiene and Tropical Medicine">LSHTM</abbr> ecological studies.</p>

<p>b. Less than 10 per cent of all deaths are included in some datasets and the number of deaths are too low for reliable inference.</p>

<p>c. There are several confounding factors which may not be adequately adjusted, for example, in these datasets nursing home status may be poorly identified, there may be confounding by setting of infection acquisition, confounded by comorbidities or goals of care (for example no transfer to acute care hospitals or advanced measures.)</p>

<p>d. While these analyses are adjusted for age, sex, deprivation, <abbr title="Lower Tier Local Authority">LTLA</abbr> etc., given increased transmission potential of <abbr title="Variant of concern">VOC</abbr>, this may not be sufficient adjustment if not a) stratified by <abbr title="Long Term Care Facility">LTCF</abbr> and b) adjusted for comorbidities and c) secular trends.</p>

<h2>Conclusion</h2>

<p>24. There is evidence from analysis of multiple different datasets that infection with <abbr title="Variant of concern">VOC</abbr> B1.1.7 is associated with an increased risk of hospitalisation and death compared to infection with non-<abbr title="Variant of concern">VOC</abbr> viruses.</p>

<p>25. There are potential limitations in all datasets used but together these analyses indicate that it is likely that <abbr title="WHO designated variant Alpha">VOC B.1.1.7</abbr> is associated with an increased risk of hospitalisation and death compared to infection with non-<abbr title="Variant of concern">VOC</abbr> viruses.</p>

<h2>Recommendations</h2>

<p>26. <abbr title="Office for National Statistics">ONS</abbr> are assessing hospitalisation and deaths linked to <abbr title="Office for National Statistics">ONS</abbr> survey participants. This dataset will suffer from less ascertainment bias than other datasets since participants are a random selection and ascertainment of cases is not dependent on symptoms of on seeking a test from the Test and Trace. However, it may have limited power.</p>

<p>27. While the majority of analyses tried to control for confounding by nursing home, poor recording of <abbr title="Long Term Care Facility">LTCF</abbr> residents in hospital data will not allow this to be adequately addressed, therefore residual confounding through unidentified nursing home residence could occur. Detailed documentation of nursing home cases and deaths is needed. Further analysis could be done through specific stratified analyses of pillar 2 among only the care-home residents and adjust for comorbidities.</p>

<p>28. Increased transmission per contact by setting could shift from a null effect to signal (increased <abbr title="case fatality ratio">CFR</abbr> with a variant that leads to more transmission) if confounded by comorbidities in the network, cluster or setting. This could be ameliorated by controlling for comorbidities but this will likely require linkage with <abbr title="COVID-19 Clinical Information Network">CO-CIN</abbr> (which is diminished in coverage now) or primary care analysis.</p>

<h2>Annex</h2>

<p>Data table – preliminary results</p>

<table>
  <thead>
    <tr>
      <th scope="col">Paper</th>
      <th scope="col">Method</th>
      <th scope="col">Sample</th>
      <th scope="col">Outcome</th>
      <th scope="col">Estimate of Effect</th>
      <th scope="col">95 per centCI</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Imperial</td>
      <td>Non- parametric analysis: case- control weighting</td>
      <td>All samples, corrected for probability that S- gene negative samples are the <abbr title="Variant of concern">VOC</abbr>
</td>
      <td>Ratio of S-negative to S-positive case fatality ratios</td>
      <td>1.36</td>
      <td>1.18-1.56</td>
    </tr>
    <tr>
      <td>Imperial</td>
      <td>Non- parametric analysis: standardised <abbr title="case fatality ratio">CFR</abbr>
</td>
      <td>All samples, corrected for probability that S- gene negative samples are the <abbr title="Variant of concern">VOC</abbr>
</td>
      <td>Ratio of S-negative to S-positive case fatality ratios</td>
      <td>1.29</td>
      <td>1.07-1.54</td>
    </tr>
    <tr>
      <td><abbr title="London School of Hygiene and Tropical Medicine">LSHTM</abbr></td>
      <td>Cox proportional hazards model</td>
      <td>All samples, adjusted for misclassification of <abbr title="s-gene target failure">SGTF</abbr> and missingness of data</td>
      <td>Hazard ratio for death for <abbr title="Variant of concern">VOC</abbr>-infected individuals to non-<abbr title="Variant of concern">VOC</abbr> infected individuals</td>
      <td>1.71</td>
      <td>1.48-1.97</td>
    </tr>
    <tr>
      <td><abbr title="London School of Hygiene and Tropical Medicine">LSHTM</abbr></td>
      <td>Cox proportional hazards model</td>
      <td>All samples, after 01.11.20 (not adjusted for misclassification of <abbr title="s-gene target failure">SGTF</abbr> and missingness of data)</td>
      <td>Hazard ratio death for <abbr title="Variant of concern">VOC</abbr>-infected individuals to non-<abbr title="Variant of concern">VOC</abbr> infected individuals</td>
      <td>1.58</td>
      <td>1.40-1.79</td>
    </tr>
    <tr>
      <td>Exeter</td>
      <td>Matched case control study</td>
      <td>Samples since 01 October, various adjustments</td>
      <td>Hazard ratio death for <abbr title="Variant of concern">VOC</abbr>-infected individuals to non-<abbr title="Variant of concern">VOC</abbr> infected individuals</td>
      <td>1.7</td>
      <td>1.3-2.2</td>
    </tr>
    <tr>
      <td><abbr title="COVID-19 Clinical Information Network">CO-CIN</abbr></td>
      <td>Multinomial model</td>
      <td>
<abbr title="COVID-19 Clinical Information Network">CO-CIN</abbr> data from a single trust</td>
      <td>Odds ratio of death in hospitalised <abbr title="Variant of concern">VOC</abbr>-infected individuals to non-<abbr title="Variant of concern">VOC</abbr> infected individuals</td>
      <td>0.63</td>
      <td>0.20-1.69</td>
    </tr>
    <tr>
      <td><abbr title="COVID-19 Clinical Information Network">CO-CIN</abbr></td>
      <td>Mixed-effects multivariable logistic regression model (updated analysis 10 February)</td>
      <td>
<abbr title="COVID-19 Clinical Information Network">CO-CIN</abbr> data from 91 hospitals in England</td>
      <td>Odds ratio of death for <abbr title="Variant of concern">VOC</abbr> compared to non-<abbr title="Variant of concern">VOC</abbr> in hospitalised patients.</td>
      <td>0.67</td>
      <td>0.32-1.40</td>
    </tr>
    <tr>
      <td><abbr title="COVID-19 Clinical Information Network">CO-CIN</abbr></td>
      <td>Cox proportional hazards survival regression model (updated analysis 10 February)</td>
      <td>
<abbr title="COVID-19 Clinical Information Network">CO-CIN</abbr> data from 91 hospitals in England</td>
      <td>Hazard ratio of death for <abbr title="Variant of concern">VOC</abbr> compared to non-<abbr title="Variant of concern">VOC</abbr> in hospitalised patients.</td>
      <td>0.81</td>
      <td>0.50-1.32</td>
    </tr>
    <tr>
      <td><abbr title="Public Health England">PHE</abbr></td>
      <td>Retrospective matched cohort study (initial analysis)</td>
      <td>Cases and comparators with at least 28 days between  specimen date  and study end date</td>
      <td>Odds ratio of hospital admission in <abbr title="s-gene target failure">SGTF</abbr> cases vs non-<abbr title="s-gene target failure">SGTF</abbr> cases</td>
      <td>1.07</td>
      <td>0.86-1.33</td>
    </tr>
    <tr>
      <td><abbr title="Public Health England">PHE</abbr></td>
      <td>Retrospective matched cohort study (initial analysis)</td>
      <td>Whole cohort</td>
      <td>Relative risk of death in <abbr title="s-gene target failure">SGTF</abbr> cases vs non-<abbr title="s-gene target failure">SGTF</abbr> cases within 28 days of a +ve result</td>
      <td>1.3</td>
      <td>0.95-1.79</td>
    </tr>
    <tr>
      <td><abbr title="Public Health England">PHE</abbr></td>
      <td>Retrospective matched cohort study (initial analysis)</td>
      <td>Cases and comparators with at least 28 days between specimen date and study end date</td>
      <td>Relative risk of death in <abbr title="s-gene target failure">SGTF</abbr> cases vs non-<abbr title="s-gene target failure">SGTF</abbr> cases within 28 days of a +ve result</td>
      <td>1.00</td>
      <td>0.58-1.73</td>
    </tr>
    <tr>
      <td><abbr title="Public Health England">PHE</abbr></td>
      <td>Retrospective matched cohort study (updated analysis 19 January)</td>
      <td>Whole cohort</td>
      <td>Relative risk of death in <abbr title="s-gene target failure">SGTF</abbr> cases vs non-<abbr title="s-gene target failure">SGTF</abbr> cases within 28 days of a +ve result</td>
      <td>1.65</td>
      <td>1.21-2.25</td>
    </tr>
    <tr>
      <td>PHS</td>
      <td>Case-only cohort (cases up to 6 January, follow-up to 22 January)</td>
      <td>Whole cohort, survival format allows for censoring</td>
      <td>Rate ratio for death in <abbr title="s-gene target failure">SGTF</abbr> cases vs non-<abbr title="s-gene target failure">SGTF</abbr> cases up to 28 days of a +ve result</td>
      <td>1.08</td>
      <td>0.78-1.49</td>
    </tr>
    <tr>
      <td>PHS</td>
      <td>Case-only cohort (cases up to 6 January, follow-up to 22 January)</td>
      <td>Whole cohort, survival format allows for censoring</td>
      <td>Rate ratio for hospitalisation or death in <abbr title="s-gene target failure">SGTF</abbr> cases vs non-<abbr title="s-gene target failure">SGTF</abbr> cases up to 28 days of a +ve result</td>
      <td>1.40</td>
      <td>1.28-1.53</td>
    </tr>
    <tr>
      <td>PHS</td>
      <td>Cohort study – Cox survival model</td>
      <td>Whole cohort</td>
      <td>Risk ratio of hospitalisation following a positive test result for <abbr title="s-gene target failure">SGTF</abbr> vs <abbr title="Scottish Government">SG</abbr> positive</td>
      <td>1.63</td>
      <td>1.48, 1.80</td>
    </tr>
    <tr>
      <td>PHS</td>
      <td>Cohort study – Cox survival model</td>
      <td>Whole cohort</td>
      <td>Risk ratio of death within 28 days of a positive test following a positive test result for <abbr title="s-gene target failure">SGTF</abbr> vs <abbr title="Scottish Government">SG</abbr> positive</td>
      <td>1.37</td>
      <td>1.02, 1.84</td>
    </tr>
    <tr>
      <td><abbr title="Hospital Onset Covid Infection">HOCI</abbr></td>
      <td>Mixed effects Cox model</td>
      <td>Inpatient cohort from 9 sites</td>
      <td>Hazard ratio for death in <abbr title="Variant of concern">VOC</abbr> vs non-<abbr title="Variant of concern">VOC</abbr>
</td>
      <td>1.09</td>
      <td>0.86-1.36</td>
    </tr>
    <tr>
      <td><abbr title="Hospital Onset Covid Infection">HOCI</abbr></td>
      <td>Mixed effects Cox model</td>
      <td>Inpatient cohort from 9 sites</td>
      <td>Hazard ratio for <abbr title="Intensive therapy unit">ITU</abbr> admission in <abbr title="Variant of concern">VOC</abbr> vs non-<abbr title="Variant of concern">VOC</abbr>
</td>
      <td>1.15</td>
      <td>0.86-1.53</td>
    </tr>
    <tr>
      <td>
<abbr title="Intensive Care National Audit and Research Centre">ICNARC</abbr>/ <abbr>QRESEARCH</abbr>
</td>
      <td>Flexible parametric survival model (Royston- Parmar)</td>
      <td>Patients with at least 12 days follow up from the date of positive test</td>
      <td>Hazard ratio of <abbr title="Intensive care unit">ICU</abbr> admission for <abbr title="Variant of concern">VOC</abbr>- infected individuals compared non-<abbr title="Variant of concern">VOC</abbr> infected individuals</td>
      <td>1.44</td>
      <td>1.25-1.67</td>
    </tr>
    <tr>
      <td><abbr title="Intensive Care National Audit and Research Centre">ICNARC</abbr></td>
      <td>Flexible parametric survival model (Royston- Parmar)</td>
      <td>Patients with a minimum of 28 days follow up after <abbr title="Intensive care unit">ICU</abbr> admission</td>
      <td>Hazard ratio of <abbr title="Intensive care unit">ICU</abbr> mortality for <abbr title="Variant of concern">VOC</abbr>-infected individuals compared non-<abbr title="Variant of concern">VOC</abbr> infected individuals</td>
      <td>0.94</td>
      <td>0.82-1.09</td>
    </tr>
    <tr>
      <td><abbr title="London School of Hygiene and Tropical Medicine">LSHTM</abbr></td>
      <td>Population- level analysis</td>
      <td>Upper-tier Local Authority level</td>
      <td>Multiplicative increase in hospitalisations per case associated with <abbr title="Variant of concern">VOC</abbr>
</td>
      <td>1.4</td>
      <td>1.3-1.5</td>
    </tr>
    <tr>
      <td><abbr title="London School of Hygiene and Tropical Medicine">LSHTM</abbr></td>
      <td>Population- level analysis</td>
      <td>Upper-tier Local Authority level</td>
      <td>Multiplicative increase in fatalities per hospitalisation associated with <abbr title="Variant of concern">VOC</abbr>
</td>
      <td>1.4</td>
      <td>1.2-1.5</td>
    </tr>
  </tbody>
</table>

<div class="footnotes" role="doc-endnotes">
  <ol>
    <li role="doc-endnote">
      <p>Public Health England, “Investigation of novel <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> Variant of Concern 202012/01: Technical briefing 2,” 2020.&nbsp;<a role="doc-backlink" aria-label="go to where this is referenced" href="#fnref:1" class="govuk-link">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>N. Davies, K. Diaz-Ordaz and <abbr title="reproduction number">R</abbr>. Keogh, “Relative fatality hazard in Pillar 2 testing individuals with <abbr title="Variant of concern">VOC</abbr>,” <abbr title="London School of Hygiene and Tropical Medicine">LSHTM</abbr> - Unpublished Analysis, 2021. 11&nbsp;<a href="#fnref:2" role="doc-backlink" class="govuk-link" aria-label="go to where this is referenced">↩</a>&nbsp;<a href="#fnref:2:1" class="govuk-link" aria-label="go to where this is referenced 2" role="doc-backlink">↩<sup>2</sup></a></p>
    </li>
    <li role="doc-endnote">
      <p>N. Ferguson, “Non-parametric analysis of fatal outcomes associated with B1.1.7,” Imperial College London, Unpublished Analysis, 2021.&nbsp;<a href="#fnref:3" class="govuk-link" role="doc-backlink" aria-label="go to where this is referenced">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>Public Health England, Unpublished analysis, 2021.&nbsp;<a class="govuk-link" role="doc-backlink" href="#fnref:4" aria-label="go to where this is referenced">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>P. M. McKeigue, D. McAllister, J. Bishop, S. Hutchinson, N. Lone, J. McMenamin, D. Goldberg, C. Robertson and H. M. Colhoun, “Relation of <abbr title="Severe acute respiratory syndrome coronavirus 2">SARS-CoV-2</abbr> variant 2020-12/01 to transmission-related factors and to severity of <abbr title="coronavirus">COVID-19</abbr> in Scotland: <abbr title="REal-time Assessment of Community Transmission">REACT</abbr>-SCOT case-only linkage study [preliminary results],” Unpublished analysis - Preliminary results, 2021.&nbsp;<a href="#fnref:5" class="govuk-link" role="doc-backlink" aria-label="go to where this is referenced">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>C. Roberston, A. Sheikh, J. McMenamin and EAVE II Study Group, “Clinical Characteristics of S Gene Dropout Cases,” Unpublished analysis, 2021.&nbsp;<a class="govuk-link" aria-label="go to where this is referenced" href="#fnref:6" role="doc-backlink">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p><abbr title="COVID-19 Genomics UK Consortium">COG-UK</abbr> Hospital Onset Covid-19 Infection (<abbr title="Hospital Onset Covid Infection">HOCI</abbr>) Study, Unpublished analysis, 2021.&nbsp;<a aria-label="go to where this is referenced" href="#fnref:7" role="doc-backlink" class="govuk-link">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p><abbr title="Intensive Care National Audit and Research Centre">ICNARC</abbr> and <abbr>QRESEARCH</abbr> Group, Unpublished analysis, 2021.&nbsp;<a role="doc-backlink" aria-label="go to where this is referenced" class="govuk-link" href="#fnref:8">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>A. Docherty, E. Harrison and C. Semple, “Hospital case fatality and emergence of variant of concern B117, rapid report to <abbr title="New and Emerging Respiratory Virus Threats Advisory Group">NERVTAG</abbr> and <abbr title="Scientific Advisory Group for Emergencies">SAGE</abbr>,” Unpublished analysis, 2021.&nbsp;<a role="doc-backlink" aria-label="go to where this is referenced" href="#fnref:9" class="govuk-link">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>C. Cheyne, G. Burnside, D. Hughes, L. Turtle, M. Garcia-Finana and C. Semple, “Analysis on recent hospital case fatality and variant of concern VOC2020-01 using <abbr title="COVID-19 Clinical Information Network">CO-CIN</abbr> data (10th February 2021),” Unpublished analysis, 2021.&nbsp;<a href="#fnref:10" role="doc-backlink" class="govuk-link" aria-label="go to where this is referenced">↩</a></p>
    </li>
    <li role="doc-endnote">
      <p>S. Abbott, S. Funk and <abbr title="Centre for Mathematical Modelling of Infectious Diseases">CMMID</abbr> Covid-19 Working Group, “Population-level association between S-gene target failure and the relationship between cases, hospitalisations, and deaths of Covid-19,” Pre-print, available online, 2021.&nbsp;<a role="doc-backlink" aria-label="go to where this is referenced" href="#fnref:11" class="govuk-link">↩</a></p>
    </li>
  </ol>
</div>
</div>


</div>
</div>
  </div>

  <div data-sticky-element="" class="sticky-element">
    <a href="#contents" class="govuk-link app-c-back-to-top dont-print">
    <svg width="13" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 13 17" height="17" class="app-c-back-to-top__icon" focusable="false">
      <path fill="currentColor" d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z"></path>
    </svg>
    Contents
</a>

    <div class="sticky-element__print-link">
      
<div class="gem-c-print-link govuk-!-display-none-print govuk-!-margin-top-0 govuk-!-margin-bottom-6">
    <button data-module="print-link" class="govuk-link govuk-body-s gem-c-print-link__button">Print this page</button>
</div>
    </div>
  </div>
</div>

    </main>